Index: Makefile
===================================================================
--- Makefile	(revision 329009)
+++ Makefile	(working copy)
@@ -3,6 +3,7 @@
 
 PORTNAME=	pcmanfm
 PORTVERSION=	1.1.2
+PORTREVISION=	1
 CATEGORIES=	x11-fm
 MASTER_SITES=	SF/${PORTNAME}/PCManFM%20%2B%20Libfm%20%28tarball%20release%29/PCManFM/
 
@@ -21,20 +22,23 @@
 CPPFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
 
-MAN1=		pcmanfm.1
-
 OPTIONS_DEFINE=	NLS
 OPTIONS_SUB=	yes
 
 NLS_USES=	gettext
 
-NO_STAGE=	yes
 .include <bsd.port.options.mk>
 
 post-patch:
+	@${REINPLACE_CMD} -e '/pcmanfm.conf/ d' ${WRKSRC}/data/Makefile.in
 .if ! ${PORT_OPTIONS:MNLS}
 	@${REINPLACE_CMD} -e 's|src po|src|' \
 		${WRKSRC}/Makefile.in
 .endif
 
+post-install:
+	@${MKDIR} ${STAGEDIR}${PREFIX}/etc/xdg/pcmanfm/default/
+	${INSTALL_DATA} ${WRKSRC}/data/pcmanfm.conf \
+		${STAGEDIR}${PREFIX}/etc/xdg/pcmanfm/default/pcmanfm.conf.sample
+
 .include <bsd.port.mk>
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 329009)
+++ pkg-plist	(working copy)
@@ -1,5 +1,8 @@
 bin/pcmanfm
-etc/xdg/pcmanfm/default/pcmanfm.conf
+@unexec if cmp -s %D/etc/xdg/pcmanfm/default/pcmanfm.conf %D/etc/xdg/pcmanfm/default/pcmanfm.conf.sample; then rm -f %D/etc/xdg/pcmanfm/default/pcmanfm.conf; fi
+etc/xdg/pcmanfm/default/pcmanfm.conf.sample
+@exec if [ ! -f %D/etc/xdg/pcmanfm/default/pcmanfm.conf ] ; then cp -p %D/%F %B/pcmanfm.conf; fi
+man/man1/pcmanfm.1.gz
 share/applications/pcmanfm-desktop-pref.desktop
 share/applications/pcmanfm.desktop
 %%NLS%%share/locale/ar/LC_MESSAGES/pcmanfm.mo
