===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/net-im/cjc/Makefile ./Makefile
--- /usr/ports/net-im/cjc/Makefile	2013-01-14 17:54:52.000000000 +0100
+++ ./Makefile	2013-04-01 20:26:32.000000000 +0200
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	cjc
-# Date created:		19 April 2006
-# Whom:			Andrew Pantyukhin <infofarmer@FreeBSD.org>
-#
+# Created by: Andrew Pantyukhin <infofarmer@FreeBSD.org>
 # $FreeBSD: head/net-im/cjc/Makefile 300897 2012-07-14 14:29:18Z beat $
-#
 
 PORTNAME=	cjc
 PORTVERSION=	1.2.1
@@ -14,12 +10,22 @@
 COMMENT=	The Console Jabber Client
 
 LICENSE=	GPLv2
-LICENSE_FILE=	${WRKSRC}/COPYING
 
-RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}pyxmpp>=1.1.0:${PORTSDIR}/net-im/py-pyxmpp
+RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}pyxmpp>=0:${PORTSDIR}/net-im/py-pyxmpp
 
 USE_PYTHON=	2.6+
 
+PORTDOCS=	*
+
+DOCSRCDIR1=	${WRKSRC}
+DOC_FILES1=	ChangeLog README TODO
+
+DOCSRCDIR2=	${WRKSRC}/doc
+DOCSDIR2=	${DOCSDIR}/doc
+DOC_FILES2=	manual.html
+
+.include <bsd.port.options.mk>
+
 post-patch:
 	@${GREP} -lR "/usr/bin/python" ${WRKSRC} | ${XARGS} \
 		${REINPLACE_CMD} -e "s|/usr/bin/python|${PYTHON_CMD}|"
@@ -31,12 +37,11 @@
 		 s|INSTALL_DIR=.*|INSTALL_DIR=${MKDIR}|g" ${WRKSRC}/Makefile
 
 post-install:
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/doc/manual.html ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/ChangeLog ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/README ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/TODO ${DOCSDIR}
+	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR2}
+	${INSTALL_DATA} ${DOC_FILES2:S|^|${DOCSRCDIR2}/|} ${DOCSDIR2}
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/net-im/cjc/pkg-plist ./pkg-plist
--- /usr/ports/net-im/cjc/pkg-plist	2013-01-14 17:54:52.000000000 +0100
+++ ./pkg-plist	2013-04-01 20:07:52.000000000 +0200
@@ -95,11 +95,6 @@
 %%DATADIR%%/plugins/version.pyc
 %%DATADIR%%/plugins/xmlconsole.py
 %%DATADIR%%/plugins/xmlconsole.pyc
-%%PORTDOCS%%%%DOCSDIR%%/ChangeLog
-%%PORTDOCS%%%%DOCSDIR%%/README
-%%PORTDOCS%%%%DOCSDIR%%/TODO
-%%PORTDOCS%%%%DOCSDIR%%/manual.html
-%%PORTDOCS%%@dirrm %%DOCSDIR%%
 @dirrm %%DATADIR%%/plugins
 @dirrm %%DATADIR%%/cjc/ui
 @dirrm %%DATADIR%%/cjc
===> Done
