===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/emulators/swine/Makefile ./Makefile
--- /usr/ports/emulators/swine/Makefile	2013-03-13 12:33:24.000000000 +0100
+++ ./Makefile	2013-05-25 14:52:23.000000000 +0200
@@ -1,4 +1,4 @@
-# Created by:   nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/emulators/swine/Makefile 314026 2013-03-13 05:02:11Z beech $
 
 PORTNAME=	swine
@@ -21,40 +21,54 @@
 DATADIRLIB=	${PREFIX}/lib/${PORTNAME}
 
 FETCH_ARGS?=	-Fpr
+USES=		desktop-file-utils
 USE_PYTHON=	yes
 NO_BUILD=	yes
-USE_GMAKE=	yes
-USE_QT4=	linguist_build
 ONLY_FOR_ARCHS=	i386 amd64
 
 PORTDOCS=	README
 
+OPTIONS_DEFINE=	NLS
+
 .include <bsd.port.options.mk>
 
+.if ${PORT_OPTIONS:MNLS}
+USES+=		gettext
+PLIST_SUB+=	NLS=""
+.else
+PLIST_SUB+=	NLS="@comment "
+.endif
+
 .if ${ARCH} == "i386"
-RUN_DEPENDS+=	wine:${PORTSDIR}/emulators/wine-devel
+RUN_DEPENDS+=	wine:${PORTSDIR}/emulators/wine
 .endif
 
 post-patch:
 	@${REINPLACE_CMD} -e 's|/bin/bash|/bin/sh|' ${WRKSRC}/version.sh
-	@${REINPLACE_CMD} -e 's|/usr/share/swine/images/swine32.png|${PREFIX}/share/pixmaps/swine.png|' ${WRKSRC}/resources/swine.desktop
+	@${REINPLACE_CMD} -e 's|^Icon.*|Icon=${PORTNAME}|' \
+		-e 's|^Categories=.*|Categories=System;Emulator;Utility;Game;|' \
+		${WRKSRC}/resources/${PORTNAME}.desktop
 	@${FIND} ${WRKSRC} -name "*.bak" -delete
 
 do-install:
-	${MKDIR} ${DATADIRLIB}
+	@${MKDIR} ${DATADIRLIB}
 	${INSTALL_SCRIPT} ${WRKSRC}/*.py ${DATADIRLIB}
 	${LN} -sf ${DATADIRLIB}/${PORTNAME}.py ${PREFIX}/bin/${PORTNAME}
 	${LN} -sf ${DATADIRLIB}/${PORTNAME}cli.py ${PREFIX}/bin/${PORTNAME}cli
 	${LN} -sf ${DATADIRLIB}/${PORTNAME}run.py ${PREFIX}/bin/${PORTNAME}run
-	${MKDIR} ${DATADIR}
-	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "translations images" ${DATADIR})
+	@${MKDIR} ${DATADIR}
+	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "images" ${DATADIR})
 
 	${LN} -sf ${DATADIR}/images/swine32.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
-	${MKDIR} ${PREFIX}/share/applications
-	${INSTALL_DATA} ${WRKSRC}/resources/*.desktop ${PREFIX}/share/applications/
+	@${MKDIR} ${PREFIX}/share/applications
+	${INSTALL_DATA} ${WRKSRC}/resources/*.desktop ${PREFIX}/share/applications
+
+.if ${PORT_OPTIONS:MNLS}
+	@(cd ${WRKSRC} && ${COPYTREE_SHARE} translations ${DATADIR})
+.endif
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/emulators/swine/pkg-plist ./pkg-plist
--- /usr/ports/emulators/swine/pkg-plist	2013-03-13 12:33:24.000000000 +0100
+++ ./pkg-plist	2013-05-25 14:54:50.000000000 +0200
@@ -71,12 +71,12 @@
 %%DATADIR%%/images/world_go.png
 %%DATADIR%%/images/wrench.png
 %%DATADIR%%/images/wrench_orange.png
-%%DATADIR%%/translations/de.qm
-%%DATADIR%%/translations/en.qm
-%%DATADIR%%/translations/es.qm
-%%DATADIR%%/translations/fr.qm
-%%DATADIR%%/translations/he.qm
-@dirrm %%DATADIR%%/translations
+%%NLS%%%%DATADIR%%/translations/de.qm
+%%NLS%%%%DATADIR%%/translations/en.qm
+%%NLS%%%%DATADIR%%/translations/es.qm
+%%NLS%%%%DATADIR%%/translations/fr.qm
+%%NLS%%%%DATADIR%%/translations/he.qm
+%%NLS%%@dirrm %%DATADIR%%/translations
 @dirrm %%DATADIR%%/images
 @dirrm %%DATADIR%%
 @dirrmtry share/applications
===> Done
