===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/devel/dparser/Makefile ./Makefile
--- /usr/ports/devel/dparser/Makefile	2013-01-24 11:58:03.000000000 +0100
+++ ./Makefile	2013-04-04 10:44:15.000000000 +0200
@@ -2,13 +2,13 @@
 # $FreeBSD: head/devel/dparser/Makefile 310882 2013-01-23 16:49:23Z gahr $
 
 PORTNAME=	dparser
-PORTVERSION=	1.29
+PORTVERSION=	1.30
 CATEGORIES=	devel
 MASTER_SITES=	SF/${PORTNAME}/
 DISTNAME=	d-${PORTVERSION}-src
 
 MAINTAINER=	gahr@FreeBSD.org
-COMMENT=	A simple but powerful tool for parsing
+COMMENT=	Simple but powerful tool for parsing
 
 OPTIONS_DEFINE=	BOEHM DOCS
 BOEHM_DESC=	Boehm garbage collector support
@@ -26,6 +26,9 @@
 
 .include <bsd.port.options.mk>
 
+post-patch:
+	${REINPLACE_CMD} -e 's|%%PREFIX%%|${PREFIX}|' ${WRKSRC}/Makefile
+
 .if ${PORT_OPTIONS:MBOEHM}
 LIB_DEPENDS+=	gc:${PORTSDIR}/devel/boehm-gc
 MAKE_ARGS+=	D_USE_GC=1 \
@@ -41,11 +44,10 @@
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/make_dparser ${PREFIX}/bin
-	${INSTALL_MAN} ${WRKSRC}/make_dparser.1 ${MANPREFIX}/man/man1
-	${INSTALL_DATA} ${WRKSRC}/dparse.h ${PREFIX}/include
-	${INSTALL_DATA} ${WRKSRC}/dparse_tables.h ${PREFIX}/include
-	${INSTALL_DATA} ${WRKSRC}/dparse_tree.h ${PREFIX}/include
-	${INSTALL_DATA} ${WRKSRC}/dsymtab.h ${PREFIX}/include
+	${INSTALL_MAN} ${WRKSRC}/${MAN1} ${MAN1PREFIX}/man/man1
+.for f in dparse.h dparse_tables.h dparse_tree.h dsymtab.h
+	(cd ${WRKSRC} && ${INSTALL_DATA} ${f} ${PREFIX}/include)
+.endfor
 	${INSTALL_DATA} ${WRKSRC}/${LIBDPARSE} ${PREFIX}/lib
 .if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
diff -ruN --exclude=CVS /usr/ports/devel/dparser/distinfo ./distinfo
--- /usr/ports/devel/dparser/distinfo	2013-01-24 11:58:03.000000000 +0100
+++ ./distinfo	2013-04-04 01:31:44.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (d-1.29-src.tar.gz) = 40cf36871324eac96b710cb4db27bd075099a5da2c9dcdcfc0fcf9dcdf2beb14
-SIZE (d-1.29-src.tar.gz) = 239082
+SHA256 (d-1.30-src.tar.gz) = 4c4a61bd5954d75da89cbcfe8f6aac3a113769939b31ce2bf4b20b7b3a31cdd8
+SIZE (d-1.30-src.tar.gz) = 239510
diff -ruN --exclude=CVS /usr/ports/devel/dparser/files/patch-Makefile ./files/patch-Makefile
--- /usr/ports/devel/dparser/files/patch-Makefile	2013-01-24 11:58:03.000000000 +0100
+++ ./files/patch-Makefile	2013-04-04 02:03:12.000000000 +0200
@@ -1,14 +1,18 @@
---- Makefile.orig	2013-01-23 03:56:43.000000000 +0100
-+++ Makefile	2013-01-23 17:40:57.000000000 +0100
-@@ -11,7 +11,7 @@
- MINOR=29
+--- Makefile.orig	2013-03-02 22:22:20.000000000 +0100
++++ Makefile	2013-04-04 01:59:35.000000000 +0200
+@@ -11,10 +11,10 @@
+ MINOR=30
  RELEASE=$(MAJOR).$(MINOR)
  
 -CC = gcc
 +CC ?= gcc
  
  ifndef PREFIX
- PREFIX=/usr/local
+-PREFIX=/usr/local
++PREFIX=%%PREFIX%%
+ endif
+ 
+ .PHONY: all gram test install myexample
 @@ -38,7 +38,7 @@
    ARCH = x86
  endif
@@ -24,30 +28,20 @@
  
 -D_BUILD_VERSION = $(shell git show-ref 2> /dev/null | head -1 | cut -d ' ' -f 1)
 -ifeq ($(D_BUILD_VERSION),)
--  D_BUILD_VERSION = $(shell cat D_BUILD_VERSION)
--endif
 +#D_BUILD_VERSION = $(shell git show-ref 2> /dev/null | head -1 | cut -d ' ' -f 1)
 +#ifeq ($(D_BUILD_VERSION),)
-+  D_BUILD_VERSION = $(shell cat BUILD_VERSION)
+   D_BUILD_VERSION = $(shell cat D_BUILD_VERSION)
+-endif
 +#endif
  CFLAGS += -DD_MAJOR_VERSION=$(MAJOR) -DD_MINOR_VERSION=$(MINOR) -DD_BUILD_VERSION=\"$(D_BUILD_VERSION)\"
  
  CFLAGS += -Wall
-@@ -96,7 +96,7 @@
+@@ -94,7 +94,7 @@
  PYTHON_FILES = d/python/Makefile d/python/*.py d/python/*.c d/python/*.h d/python/*.i d/python/README d/python/*.html d/python/contrib/d* d/python/tests/*.py
  VERILOG_FILES = d/verilog/Makefile d/verilog/verilog.g d/verilog/README d/verilog/ambig.c \
  d/verilog/main.c d/verilog/vparse.c d/verilog/vparse.h d/verilog/verilog_tests
--TAR_FILES = $(AUX_FILES) $(TEST_FILES) $(PYTHON_FILES) $(VERILOG_FILES) d/BUILD_VERSION \
+-TAR_FILES = $(AUX_FILES) $(TEST_FILES) $(PYTHON_FILES) $(VERILOG_FILES) d/D_BUILD_VERSION \
 +TAR_FILES = $(AUX_FILES) $(TEST_FILES) $(PYTHON_FILES) $(VERILOG_FILES) \
  d/grammar.g d/sample.g d/my.g 
  
  LIB_SRCS = arg.c parse.c scan.c symtab.c util.c read_binary.c dparse_tree.c
-@@ -143,7 +143,7 @@
- 
- ALL_SRCS = $(MAKE_PARSER_SRCS) $(BASE_SAMPLE_PARSER_SRCS) $(LIB_SRCS) $(MK_LIB_SRCS)
- 
--all: $(EXECS) $(LIBRARIES) BUILD_VERSION make_dparser.cat
-+all: $(EXECS) $(LIBRARIES) make_dparser.cat
- 
- version:
- 	echo $(OS_TYPE) $(OS_VERSION)
diff -ruN --exclude=CVS /usr/ports/devel/dparser/files/patch-arg.c ./files/patch-arg.c
--- /usr/ports/devel/dparser/files/patch-arg.c	2013-01-14 17:53:59.000000000 +0100
+++ ./files/patch-arg.c	2013-04-04 02:03:41.000000000 +0200
@@ -1,5 +1,5 @@
---- arg.c.orig	Fri Nov  7 03:03:52 2003
-+++ arg.c	Mon Feb  2 18:35:43 2004
+--- arg.c.orig	2013-01-23 03:14:58.000000000 +0100
++++ arg.c	2013-04-04 02:02:01.000000000 +0200
 @@ -40,7 +40,7 @@
            *(double *)desc[i].location = atof(arg);
            break;
===> Done
