===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/devel/pure-readline/Makefile ./Makefile
--- /usr/ports/devel/pure-readline/Makefile	2013-01-14 17:54:04.000000000 +0100
+++ ./Makefile	2013-04-05 22:55:49.000000000 +0200
@@ -1,23 +1,29 @@
-# New ports collection makefile for:	pure-readline
-# Date created:		2011-10-21
-# Whom:			Zhihao Yuan <lichray@gmail.com>
-#
+# Created by: Zhihao Yuan <lichray@gmail.com>
 # $FreeBSD: head/devel/pure-readline/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	pure-readline
-PORTVERSION=	0.1
+PORTVERSION=	0.2
 CATEGORIES=	devel
+#MASTER_SITES=	https://cdn.bitbucket.org/purelang/pure-lang/downloads/
+MASTER_SITES=	https://pure-lang.googlecode.com/files/
+DIST_SUBDIR=	pure
 
 MAINTAINER=	lichray@gmail.com
-COMMENT=	A readline interface for the Pure language
+COMMENT=	Readline interface for the Pure language
 
-USE_PURE=	yes
-NO_PURE_EXAMPLES=	yes
+USES=		pure
 
 PLIST_FILES=	lib/pure/readline.pure \
 		lib/pure/readline.so
 
-.include <bsd.port.pre.mk>
-.include "${PORTSDIR}/lang/pure/bsd.pure.mk"
-.include <bsd.port.post.mk>
+PORTDOCS=	README
+
+.include <bsd.port.options.mk>
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+
+.include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/devel/pure-readline/distinfo ./distinfo
--- /usr/ports/devel/pure-readline/distinfo	2013-01-14 17:54:04.000000000 +0100
+++ ./distinfo	2013-04-05 22:04:08.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (pure/pure-readline-0.1.tar.gz) = a8721d58d2d34c803967923a2ebd1b8f612abfe8d4ad75d6796bee5bbc3c45a2
-SIZE (pure/pure-readline-0.1.tar.gz) = 16358
+SHA256 (pure/pure-readline-0.2.tar.gz) = 543686305921de4a6c295c8320be0c8fb273ae5219dbda9e17a85c27d9cd1baf
+SIZE (pure/pure-readline-0.2.tar.gz) = 19015
diff -ruN --exclude=CVS /usr/ports/devel/pure-readline/pkg-descr ./pkg-descr
--- /usr/ports/devel/pure-readline/pkg-descr	2013-01-14 17:54:04.000000000 +0100
+++ ./pkg-descr	2013-04-05 22:03:53.000000000 +0200
@@ -2,4 +2,4 @@
 Pure expressions, and the C++ Standard Template Library algorithms that act
 on them.
 
-WWW:	http://docs.pure-lang.googlecode.com/hg/pure-readline.html
+WWW: http://docs.pure-lang.googlecode.com/hg/pure-readline.html
===> Done
