===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/devel/pure-stlvec/Makefile ./Makefile
--- /usr/ports/devel/pure-stlvec/Makefile	2013-01-14 17:53:57.000000000 +0100
+++ ./Makefile	2013-04-05 22:56:27.000000000 +0200
@@ -1,25 +1,40 @@
-# New ports collection makefile for:	pure-stlvec
-# Date created:		2011-10-19
-# Whom:			Zhihao Yuan <lichray@gmail.com>
-#
+# Created by: Zhihao Yuan <lichray@gmail.com>
 # $FreeBSD: head/devel/pure-stlvec/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	pure-stlvec
 PORTVERSION=	0.2
+PORTREVISION=	1
 CATEGORIES=	devel
+#MASTER_SITES=	https://cdn.bitbucket.org/purelang/pure-lang/downloads/
+MASTER_SITES=	https://pure-lang.googlecode.com/files/
+DIST_SUBDIR=	pure
 
 MAINTAINER=	lichray@gmail.com
 COMMENT=	Pure interface to C++ STL vector
 
-USE_PURE=	yes
+LICENSE=	BSD
 
-.include <bsd.port.pre.mk>
+USES=		pure
 
 post-patch:
 	@${REINPLACE_CMD} -e \
 		's|^PROD_FLAGS =.*|PROD_FLAGS = $$(CPPFLAGS) $$(CFLAGS) $$(LDFLAGS)|' \
 		${WRKSRC}/Makefile
 
-.include "${PORTSDIR}/lang/pure/bsd.pure.mk"
-.include <bsd.port.post.mk>
+PORTDOCS=	README
+PORTEXAMPLES=	*
+
+.include <bsd.port.options.mk>
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+
+.if ${PORT_OPTIONS:MEXAMPLES}
+	@${MKDIR} ${EXAMPLESDIR}
+	@(cd ${WRKSRC}/examples/ && ${COPYTREE_SHARE} . ${EXAMPLESDIR})
+.endif
+
+.include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/devel/pure-stlvec/pkg-descr ./pkg-descr
--- /usr/ports/devel/pure-stlvec/pkg-descr	2013-01-14 17:53:57.000000000 +0100
+++ ./pkg-descr	2013-04-05 21:58:30.000000000 +0200
@@ -2,4 +2,4 @@
 Pure expressions, and the C++ Standard Template Library algorithms that act
 on them.
 
-WWW:	http://docs.pure-lang.googlecode.com/hg/pure-stlvec.html
+WWW: http://docs.pure-lang.googlecode.com/hg/pure-stlvec.html
===> Done
