===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/devel/synfig/Makefile ./Makefile
--- /usr/ports/devel/synfig/Makefile	2013-01-17 17:32:37.000000000 +0100
+++ ./Makefile	2013-03-16 16:28:28.000000000 +0100
@@ -3,12 +3,14 @@
 
 PORTNAME=	synfig
 DISTVERSION=	0.63.05
-PORTREVISION=	3
+PORTREVISION=	4
 CATEGORIES=	devel graphics multimedia
 MASTER_SITES=	SF
 
 MAINTAINER=	dhn@FreeBSD.org
-COMMENT=	A vector based 2D animation package
+COMMENT=	Vector based 2D animation package
+
+LICENSE=	GPLv3
 
 BUILD_DEPENDS=	${LOCALBASE}/libdata/pkgconfig/ETL.pc:${PORTSDIR}/devel/etl
 LIB_DEPENDS=	glibmm-2.4.1:${PORTSDIR}/devel/glibmm \
@@ -26,9 +28,24 @@
 USE_GETTEXT=	yes
 GNU_CONFIGURE=	yes
 USE_LDCONFIG=	yes
+ALL_TARGET=	-j`${SYSCTL} -n hw.ncpu`
+
 CPPFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
-ALL_TARGET=	-j`${SYSCTL} -n hw.ncpu`
+
+OPTIONS_DEFINE=	NLS
+
+PORTDOCS=	AUTHORS ChangeLog NEWS README TODO
+
+.include <bsd.port.options.mk>
+
+.if ${PORT_OPTIONS:MNLS}
+USE_GETTEXT=	yes
+PLIST_SUB+=	NLS=""
+.else
+CONFIGURE_ARGS+=	--disable-nls
+PLIST_SUB+=	NLS="@comment "
+.endif
 
 post-patch:
 	@${REINPLACE_CMD} -e '/optimization_flags/s|CXXFLAGS=|#CXXFLAGS=| ; \
@@ -47,11 +64,9 @@
 	${MV} ${WRKSRC}/po/el_GR.gmo ${WRKSRC}/po/el.gmo
 
 post-install:
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-.for doc in AUTHORS ChangeLog INSTALL NEWS README TODO
-	${INSTALL_DATA} ${WRKSRC}/${doc} ${DOCSDIR}
-.endfor
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/devel/synfig/pkg-plist ./pkg-plist
--- /usr/ports/devel/synfig/pkg-plist	2013-01-14 17:54:13.000000000 +0100
+++ ./pkg-plist	2013-03-16 16:28:01.000000000 +0100
@@ -1,8 +1,6 @@
 bin/synfig
 bin/synfig-config
 etc/synfig_modules.cfg
-include/synfig-0.0/synfig/valuenode_dilist.h
-include/synfig-0.0/synfig/dashitem.h
 include/synfig-0.0/synfig/activepoint.h
 include/synfig-0.0/synfig/angle.h
 include/synfig-0.0/synfig/blinepoint.h
@@ -13,6 +11,7 @@
 include/synfig-0.0/synfig/context.h
 include/synfig-0.0/synfig/curve_helper.h
 include/synfig-0.0/synfig/curveset.h
+include/synfig-0.0/synfig/dashitem.h
 include/synfig-0.0/synfig/distance.h
 include/synfig-0.0/synfig/exception.h
 include/synfig-0.0/synfig/gamma.h
@@ -71,27 +70,35 @@
 include/synfig-0.0/synfig/value.h
 include/synfig-0.0/synfig/valuenode.h
 include/synfig-0.0/synfig/valuenode_add.h
-include/synfig-0.0/synfig/valuenode_animated.h
+include/synfig-0.0/synfig/valuenode_and.h
 include/synfig-0.0/synfig/valuenode_anglestring.h
+include/synfig-0.0/synfig/valuenode_animated.h
 include/synfig-0.0/synfig/valuenode_atan2.h
 include/synfig-0.0/synfig/valuenode_bline.h
 include/synfig-0.0/synfig/valuenode_blinecalctangent.h
 include/synfig-0.0/synfig/valuenode_blinecalcvertex.h
 include/synfig-0.0/synfig/valuenode_blinecalcwidth.h
 include/synfig-0.0/synfig/valuenode_blinereversetangent.h
+include/synfig-0.0/synfig/valuenode_compare.h
 include/synfig-0.0/synfig/valuenode_composite.h
 include/synfig-0.0/synfig/valuenode_const.h
 include/synfig-0.0/synfig/valuenode_cos.h
-include/synfig-0.0/synfig/valuenode_duplicate.h
+include/synfig-0.0/synfig/valuenode_dilist.h
 include/synfig-0.0/synfig/valuenode_dotproduct.h
+include/synfig-0.0/synfig/valuenode_duplicate.h
 include/synfig-0.0/synfig/valuenode_dynamiclist.h
 include/synfig-0.0/synfig/valuenode_exp.h
-include/synfig-0.0/synfig/valuenode_gradientrotate.h
 include/synfig-0.0/synfig/valuenode_gradientcolor.h
-include/synfig-0.0/synfig/valuenode_intstring.h
+include/synfig-0.0/synfig/valuenode_gradientrotate.h
+include/synfig-0.0/synfig/valuenode_greyed.h
 include/synfig-0.0/synfig/valuenode_integer.h
+include/synfig-0.0/synfig/valuenode_intstring.h
 include/synfig-0.0/synfig/valuenode_join.h
 include/synfig-0.0/synfig/valuenode_linear.h
+include/synfig-0.0/synfig/valuenode_log.h
+include/synfig-0.0/synfig/valuenode_not.h
+include/synfig-0.0/synfig/valuenode_or.h
+include/synfig-0.0/synfig/valuenode_pow.h
 include/synfig-0.0/synfig/valuenode_radialcomposite.h
 include/synfig-0.0/synfig/valuenode_range.h
 include/synfig-0.0/synfig/valuenode_realstring.h
@@ -114,13 +121,6 @@
 include/synfig-0.0/synfig/valuenode_vectorlength.h
 include/synfig-0.0/synfig/valuenode_vectorx.h
 include/synfig-0.0/synfig/valuenode_vectory.h
-include/synfig-0.0/synfig/valuenode_log.h
-include/synfig-0.0/synfig/valuenode_and.h
-include/synfig-0.0/synfig/valuenode_compare.h
-include/synfig-0.0/synfig/valuenode_greyed.h
-include/synfig-0.0/synfig/valuenode_not.h
-include/synfig-0.0/synfig/valuenode_or.h
-include/synfig-0.0/synfig/valuenode_pow.h
 include/synfig-0.0/synfig/valuenode_wplist.h
 include/synfig-0.0/synfig/vector.h
 include/synfig-0.0/synfig/version.h
@@ -137,6 +137,8 @@
 lib/synfig/modules/libmod_bmp.so
 lib/synfig/modules/libmod_dv.la
 lib/synfig/modules/libmod_dv.so
+lib/synfig/modules/libmod_example.la
+lib/synfig/modules/libmod_example.so
 lib/synfig/modules/libmod_ffmpeg.la
 lib/synfig/modules/libmod_ffmpeg.so
 lib/synfig/modules/libmod_filter.la
@@ -163,36 +165,27 @@
 lib/synfig/modules/libmod_png.so
 lib/synfig/modules/libmod_ppm.la
 lib/synfig/modules/libmod_ppm.so
+lib/synfig/modules/libmod_svg.la
+lib/synfig/modules/libmod_svg.so
 lib/synfig/modules/libmod_yuv420p.la
 lib/synfig/modules/libmod_yuv420p.so
-lib/synfig/modules/libmod_svg.so
-lib/synfig/modules/libmod_svg.la
-lib/synfig/modules/libmod_example.so
-lib/synfig/modules/libmod_example.la
 libdata/pkgconfig/synfig.pc
-%%PORTDOCS%%%%DOCSDIR%%/AUTHORS
-%%PORTDOCS%%%%DOCSDIR%%/ChangeLog
-%%PORTDOCS%%%%DOCSDIR%%/INSTALL
-%%PORTDOCS%%%%DOCSDIR%%/NEWS
-%%PORTDOCS%%%%DOCSDIR%%/README
-%%PORTDOCS%%%%DOCSDIR%%/TODO
-share/locale/el/LC_MESSAGES/synfig.mo
-share/locale/fa_IR/LC_MESSAGES/synfig.mo
-share/locale/lt/LC_MESSAGES/synfig.mo
-share/locale/nl/LC_MESSAGES/synfig.mo
-share/locale/sk/LC_MESSAGES/synfig.mo
-share/locale/de/LC_MESSAGES/synfig.mo
-share/locale/en_GB/LC_MESSAGES/synfig.mo
-share/locale/it/LC_MESSAGES/synfig.mo
-share/locale/es/LC_MESSAGES/synfig.mo
-share/locale/fr/LC_MESSAGES/synfig.mo
-share/locale/no/LC_MESSAGES/synfig.mo
-share/locale/pl/LC_MESSAGES/synfig.mo
-share/locale/pt_BR/LC_MESSAGES/synfig.mo
-share/locale/ro/LC_MESSAGES/synfig.mo
-share/locale/ru/LC_MESSAGES/synfig.mo
-share/locale/zh_CN/LC_MESSAGES/synfig.mo
-%%PORTDOCS%%@dirrm %%DOCSDIR%%
+%%NLS%%share/locale/de/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/el/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/en_GB/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/es/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/fa_IR/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/fr/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/it/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/lt/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/nl/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/no/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/pl/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/pt_BR/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/ro/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/ru/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/sk/LC_MESSAGES/synfig.mo
+%%NLS%%share/locale/zh_CN/LC_MESSAGES/synfig.mo
 @dirrm lib/synfig/modules
 @dirrm lib/synfig
 @dirrm include/synfig-0.0/synfig
===> Done
