===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/enblend/Makefile ./Makefile
--- /usr/ports/graphics/enblend/Makefile	2013-03-03 15:55:07.000000000 +0100
+++ ./Makefile	2013-04-09 22:00:07.000000000 +0200
@@ -3,17 +3,18 @@
 
 PORTNAME=	enblend
 PORTVERSION=	4.1.1
+PORTREVISION=	1
 CATEGORIES=	graphics
-MASTER_SITES=	${MASTER_SITE_SOURCEFORGE}
-# Care: the directory is the minor release number
-MASTER_SITE_SUBDIR=	enblend/enblend-enfuse/enblend-enfuse-4.1
+MASTER_SITES=	SF/${PORTNAME}/${PORTNAME}-enfuse/${PORTNAME}-enfuse-${PORTVERSION:R}/
 DISTNAME=	enblend-enfuse-${PORTVERSION}
 
 MAINTAINER=	grog@FreeBSD.org
-COMMENT=	A tool for image blending with multiresolution splines
+COMMENT=	Tool for image blending with multiresolution splines
+
+LICENSE=	GPLv2
 
 BUILD_DEPENDS=	${LOCALBASE}/bin/makeinfo:${PORTSDIR}/print/texinfo \
-		OpenEXR>=1.0:${PORTSDIR}/graphics/OpenEXR
+		OpenEXR>=0:${PORTSDIR}/graphics/OpenEXR
 LIB_DEPENDS=	jpeg:${PORTSDIR}/graphics/jpeg \
 		lcms2:${PORTSDIR}/graphics/lcms2 \
 		xmi:${PORTSDIR}/graphics/plotutils \
@@ -21,10 +22,9 @@
 		boost_filesystem:${PORTSDIR}/devel/boost-libs \
 		vigraimpex:${PORTSDIR}/graphics/vigra \
 		gsl:${PORTSDIR}/math/gsl
-RUN_DEPENDS=	OpenEXR>=1.0:${PORTSDIR}/graphics/OpenEXR
+RUN_DEPENDS=	OpenEXR>=0:${PORTSDIR}/graphics/OpenEXR
 
-INFO=		enfuse enblend
-MAKE_JOBS_SAFE= yes
+MAKE_JOBS_SAFE=	yes
 USE_AUTOTOOLS=	automake:env autoconf:env
 USE_GMAKE=	yes
 # We need a newer version of makinfo for this.
@@ -43,10 +43,19 @@
 LDFLAGS+=	-L${LOCALBASE}/lib
 
 MAN1=		enblend.1 enfuse.1
+INFO=		enfuse enblend
+
 PLIST_FILES=	bin/enblend bin/enfuse
 
+PORTDOCS=	AUTHORS NEWS README
+
+.include <bsd.port.options.mk>
+
 post-install:
-		${MKDIR} ${DOCSDIR}
-		${INSTALL_DATA} ${WRKSRC}/doc/*.info ${PREFIX}/${INFO_PATH}
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+	${INSTALL_DATA} ${WRKSRC}/doc/*.info ${PREFIX}/${INFO_PATH}
 
 .include <bsd.port.mk>
===> Done
