===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/flphoto/Makefile ./Makefile
--- /usr/ports/graphics/flphoto/Makefile	2013-01-14 17:57:03.000000000 +0100
+++ ./Makefile	2013-03-29 02:22:44.000000000 +0100
@@ -1,19 +1,17 @@
-# New ports collection makefile for:   	flPhoto
-# Date created:				24 November, 2003
-# Whom:                			johan
-#
+# Created by: johan
 # $FreeBSD: head/graphics/flphoto/Makefile 309385 2012-12-22 11:51:49Z mandree $
-#
 
 PORTNAME=	flphoto
 PORTVERSION=	1.3.1
-PORTREVISION=	8
+PORTREVISION=	9
 CATEGORIES=	graphics
 MASTER_SITES=	SF/fltk/${PORTNAME}/${PORTVERSION}
 DISTNAME=	${PORTNAME}-${PORTVERSION}-source
 
 MAINTAINER=	ports@FreeBSD.org
-COMMENT=	A basic image management and display program
+COMMENT=	Basic image management and display program
+
+LICENSE=	GPLv2
 
 LIB_DEPENDS=	gphoto2:${PORTSDIR}/graphics/libgphoto2 \
 		fltk:${PORTSDIR}/x11-toolkits/fltk \
@@ -28,26 +26,50 @@
 MAN1=		flphoto.1
 MAN5=		album.5
 
+PORTDOCS=	*
+
+DOCSRCDIR1=	${WRKSRC}
+DOC_FILES1=	ANNOUNCEMENT CHANGES README
+
+DOCSRCDIR2=	${WRKSRC}/doc
+DOC_FILES2=	*.jpg *.html *.png
+
+DESKTOP_ENTRIES="flPhoto" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Graphics;" false
+
+OPTIONS_DEFINE=	NLS
+
+.include <bsd.port.options.mk>
+
+.if ${PORT_OPTIONS:MNLS}
+USE_GETTEXT=	yes
+PLIST_SUB+=	NLS=""
+.else
+PLIST_SUB+=	NLS="@comment "
+.endif
+
 post-patch:
 	@${REINPLACE_CMD} -e 's|@OPTIM@|-DHAVE_CONFIG_H|g' ${WRKSRC}/Makefile.in
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/flphoto ${PREFIX}/bin/flphoto
-	${INSTALL_MAN} ${WRKSRC}/doc/flphoto.man ${MANPREFIX}/man/man1/flphoto.1
+	${INSTALL_MAN} ${WRKSRC}/doc/flphoto.man ${MAN1PREFIX}/man/man1/flphoto.1
 	${INSTALL_MAN} ${WRKSRC}/doc/album.man ${MAN5PREFIX}/man/man5/album.5
-.if !defined(NOPORTDOCS)
+
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/README ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/doc/*.jpg ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/doc/*.html ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/doc/*.png ${DOCSDIR}
+	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${DOCSDIR}
+	${INSTALL_DATA} ${DOC_FILES2:S|^|${DOCSRCDIR2}/|} ${DOCSDIR}
 .endif
-.for lang in de en_CA en_GB es fr it nl pt
+	${LN} -sf ${DOCSDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
+
+.if ${PORT_OPTIONS:MNLS}
+. for lang in de en_CA en_GB es fr it nl pt
 	@${MKDIR} ${PREFIX}/share/locale/${lang}
 	${INSTALL_DATA} ${WRKSRC}/po/${lang}.po \
 		${PREFIX}/share/locale/${lang}/flphoto_${lang}
-.endfor
-
+. endfor
+.endif
 .include <bsd.port.pre.mk>
 
 .if ${ARCH} == "powerpc"
diff -ruN --exclude=CVS /usr/ports/graphics/flphoto/pkg-plist ./pkg-plist
--- /usr/ports/graphics/flphoto/pkg-plist	2013-01-14 17:57:03.000000000 +0100
+++ ./pkg-plist	2013-03-29 02:20:34.000000000 +0100
@@ -1,43 +1,10 @@
 bin/flphoto
-%%PORTDOCS%%%%DOCSDIR%%/README
-%%PORTDOCS%%%%DOCSDIR%%/about.html
-%%PORTDOCS%%%%DOCSDIR%%/albums.html
-%%PORTDOCS%%%%DOCSDIR%%/camera.jpg
-%%PORTDOCS%%%%DOCSDIR%%/contents.html
-%%PORTDOCS%%%%DOCSDIR%%/crop.jpg
-%%PORTDOCS%%%%DOCSDIR%%/download.html
-%%PORTDOCS%%%%DOCSDIR%%/export-dir.jpg
-%%PORTDOCS%%%%DOCSDIR%%/export-general.jpg
-%%PORTDOCS%%%%DOCSDIR%%/export-images.jpg
-%%PORTDOCS%%%%DOCSDIR%%/export-thumbnails.jpg
-%%PORTDOCS%%%%DOCSDIR%%/export-watermark.jpg
-%%PORTDOCS%%%%DOCSDIR%%/export.html
-%%PORTDOCS%%%%DOCSDIR%%/flphoto.jpg
-%%PORTDOCS%%%%DOCSDIR%%/flphoto.png
-%%PORTDOCS%%%%DOCSDIR%%/images.html
-%%PORTDOCS%%%%DOCSDIR%%/import.html
-%%PORTDOCS%%%%DOCSDIR%%/index.html
-%%PORTDOCS%%%%DOCSDIR%%/jpeg.jpg
-%%PORTDOCS%%%%DOCSDIR%%/license.html
-%%PORTDOCS%%%%DOCSDIR%%/options-image.jpg
-%%PORTDOCS%%%%DOCSDIR%%/options-memcard.jpg
-%%PORTDOCS%%%%DOCSDIR%%/options-slideshow.jpg
-%%PORTDOCS%%%%DOCSDIR%%/options.html
-%%PORTDOCS%%%%DOCSDIR%%/print-calendar.jpg
-%%PORTDOCS%%%%DOCSDIR%%/print-general.jpg
-%%PORTDOCS%%%%DOCSDIR%%/print-matted.jpg
-%%PORTDOCS%%%%DOCSDIR%%/print-printer.jpg
-%%PORTDOCS%%%%DOCSDIR%%/print.html
-%%PORTDOCS%%%%DOCSDIR%%/relnotes.html
-%%PORTDOCS%%%%DOCSDIR%%/scale.jpg
-%%PORTDOCS%%%%DOCSDIR%%/slideshow.html
-%%PORTDOCS%%%%DOCSDIR%%/view.html
-share/locale/de/flphoto_de
-share/locale/en_CA/flphoto_en_CA
-share/locale/en_GB/flphoto_en_GB
-share/locale/es/flphoto_es
-share/locale/fr/flphoto_fr
-share/locale/it/flphoto_it
-share/locale/nl/flphoto_nl
-share/locale/pt/flphoto_pt
-%%PORTDOCS%%@dirrm %%DOCSDIR%%
+%%NLS%%share/locale/de/flphoto_de
+%%NLS%%share/locale/en_CA/flphoto_en_CA
+%%NLS%%share/locale/en_GB/flphoto_en_GB
+%%NLS%%share/locale/es/flphoto_es
+%%NLS%%share/locale/fr/flphoto_fr
+%%NLS%%share/locale/it/flphoto_it
+%%NLS%%share/locale/nl/flphoto_nl
+%%NLS%%share/locale/pt/flphoto_pt
+share/pixmaps/flphoto.png
===> Done
