===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/goocanvas/Makefile ./Makefile
--- /usr/ports/graphics/goocanvas/Makefile	2013-01-14 17:57:00.000000000 +0100
+++ ./Makefile	2013-04-06 01:39:11.000000000 +0200
@@ -1,26 +1,58 @@
-# New ports collection makefile for:	goocanvas
-# Date created:		2006-09-20
-# Whom:			Michael Johnson <ahze@FreeBSD.org>
-#
+# Created by: Michael Johnson <ahze@FreeBSD.org>
 # $FreeBSD: head/graphics/goocanvas/Makefile 300896 2012-07-14 13:54:48Z beat $
 #    $MCom: ports/graphics/goocanvas/Makefile,v 1.3 2008/02/27 16:19:11 ahze Exp $
 
 PORTNAME=	goocanvas
 PORTVERSION=	0.15
-PORTREVISION=	4
+PORTREVISION=	5
 CATEGORIES=	graphics
 MASTER_SITES=	GNOME
 
 MAINTAINER=	kwm@FreeBSD.org
 COMMENT=	Canvas widget for GTK+
 
-USE_GNOME=	gtk20 gnomehack ltverhack
+USE_BZIP2=	yes
+USES=		pathfix
+USE_PKGCONFIG=	build
+USE_GNOME=	gtk20 gdkpixbuf2 ltverhack
 USE_AUTOTOOLS=	libtool
 USE_LDCONFIG=	yes
-USE_BZIP2=	yes
+
+CPPFLAGS+=	-I${LOCALBASE}/include
+LDFLAGS+=	-L${LOCALBASE}/lib
+
+PORTDOCS=	*
+
+DOCSRCDIR1=	${WRKSRC}
+DOC_FILES1=	AUTHORS ChangeLog NEWS README TODO
+
+OPTIONS_DEFINE=	NLS
+
+.include <bsd.port.options.mk>
+
+.if ${PORT_OPTIONS:MNLS}
+USE_GETTEXT=	yes
+PLIST_SUB+=	NLS=""
+.else
+PLIST_SUB+=	NLS="@comment "
+.endif
 
 post-patch:
 	@${REINPLACE_CMD} -e 's|$${datadir}/gtk-doc/html|${DOCSDIR}|' \
 		${WRKSRC}/configure
 
+.if ! ${PORT_OPTIONS:MDOCS}
+	@${REINPLACE_CMD} -e '/^SUBDIRS/s|docs||g' ${WRKSRC}/Makefile.in
+.endif
+
+.if ! ${PORT_OPTIONS:MNLS}
+	@${REINPLACE_CMD} -e '/^SUBDIRS/s|po||g' ${WRKSRC}/Makefile.in
+.endif
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${DOCSDIR}
+.endif
+
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/graphics/goocanvas/pkg-plist ./pkg-plist
--- /usr/ports/graphics/goocanvas/pkg-plist	2013-01-14 17:57:00.000000000 +0100
+++ ./pkg-plist	2013-04-06 01:12:05.000000000 +0200
@@ -21,53 +21,7 @@
 lib/libgoocanvas.so
 lib/libgoocanvas.so.3
 libdata/pkgconfig/goocanvas.pc
-%%DOCSDIR%%/goocanvas/GooCanvas.html
-%%DOCSDIR%%/goocanvas/GooCanvasEllipse.html
-%%DOCSDIR%%/goocanvas/GooCanvasEllipseModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasGrid.html
-%%DOCSDIR%%/goocanvas/GooCanvasGridModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasGroup.html
-%%DOCSDIR%%/goocanvas/GooCanvasGroupModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasImage.html
-%%DOCSDIR%%/goocanvas/GooCanvasImageModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasItemModelSimple.html
-%%DOCSDIR%%/goocanvas/GooCanvasItemSimple.html
-%%DOCSDIR%%/goocanvas/GooCanvasPath.html
-%%DOCSDIR%%/goocanvas/GooCanvasPathModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasPolyline.html
-%%DOCSDIR%%/goocanvas/GooCanvasPolylineModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasRect.html
-%%DOCSDIR%%/goocanvas/GooCanvasRectModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasStyle.html
-%%DOCSDIR%%/goocanvas/GooCanvasTable.html
-%%DOCSDIR%%/goocanvas/GooCanvasTableModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasText.html
-%%DOCSDIR%%/goocanvas/GooCanvasTextModel.html
-%%DOCSDIR%%/goocanvas/GooCanvasWidget.html
-%%DOCSDIR%%/goocanvas/ch01.html
-%%DOCSDIR%%/goocanvas/ch02.html
-%%DOCSDIR%%/goocanvas/ch03.html
-%%DOCSDIR%%/goocanvas/ch04.html
-%%DOCSDIR%%/goocanvas/ch05.html
-%%DOCSDIR%%/goocanvas/goocanvas-GooCanvas-Types.html
-%%DOCSDIR%%/goocanvas/goocanvas-architecture.html
-%%DOCSDIR%%/goocanvas/goocanvas-coordinates.html
-%%DOCSDIR%%/goocanvas/goocanvas-creating-items.html
-%%DOCSDIR%%/goocanvas/goocanvas-goocanvasitem.html
-%%DOCSDIR%%/goocanvas/goocanvas-goocanvasitemmodel.html
-%%DOCSDIR%%/goocanvas/goocanvas-model-view-canvas.html
-%%DOCSDIR%%/goocanvas/goocanvas-overview.html
-%%DOCSDIR%%/goocanvas/goocanvas-simple-canvas.html
-%%DOCSDIR%%/goocanvas/goocanvas-wysiwyg.html
-%%DOCSDIR%%/goocanvas/goocanvas.devhelp
-%%DOCSDIR%%/goocanvas/goocanvas.devhelp2
-%%DOCSDIR%%/goocanvas/home.png
-%%DOCSDIR%%/goocanvas/index.html
-%%DOCSDIR%%/goocanvas/index.sgml
-%%DOCSDIR%%/goocanvas/left.png
-%%DOCSDIR%%/goocanvas/right.png
-%%DOCSDIR%%/goocanvas/style.css
-%%DOCSDIR%%/goocanvas/up.png
-@dirrm %%DOCSDIR%%/goocanvas
-@dirrm %%DOCSDIR%%
+%%NLS%%share/locale/en_GB/LC_MESSAGES/goocanvas.mo
+%%NLS%%share/locale/es/LC_MESSAGES/goocanvas.mo
+%%NLS%%share/locale/ja/LC_MESSAGES/goocanvas.mo
 @dirrm include/goocanvas-1.0
===> Done
