===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/mapnik/Makefile ./Makefile
--- /usr/ports/graphics/mapnik/Makefile	2013-02-01 13:15:24.000000000 +0100
+++ ./Makefile	2013-04-09 15:33:28.000000000 +0200
@@ -3,16 +3,15 @@
 
 PORTNAME=	mapnik
 PORTVERSION=	2.1.0
-PORTREVISION=	2
 DISTVERSIONPREFIX=	v
+PORTREVISION=	3
 CATEGORIES=	graphics geography
 MASTER_SITES=	http://cloud.github.com/downloads/${PORTNAME}/${PORTNAME}/
 
 MAINTAINER=	littlesavage@rambler.ru
-COMMENT=	A Free Toolkit For Developing Mapping Applications
+COMMENT=	Free Toolkit For Developing Mapping Applications
 
 LICENSE=	LGPL21
-LICENSE_FILE=	${WRKSRC}/COPYING
 
 LIB_DEPENDS=	proj:${PORTSDIR}/graphics/proj \
 		png15:${PORTSDIR}/graphics/png \
@@ -24,14 +23,15 @@
 BUILD_DEPENDS=	${LOCALBASE}/bin/freetype-config:${PORTSDIR}/print/freetype2 \
 		icu>=4.2:${PORTSDIR}/devel/icu \
 		${PYTHON_PKGNAMEPREFIX}icu>=0.8.1:${PORTSDIR}/devel/py-icu \
-		boost-libs>=1.47:${PORTSDIR}/devel/boost-libs \
+		boost-libs>=1.47:${PORTSDIR}/devel/boost-libs
 
 USE_BZIP2=	yes
 USE_PYTHON=	2.6+
 USE_LDCONFIG=	yes
 USE_SCONS=	yes
 USE_AUTOTOOLS=	libltdl
-USE_GNOME=	pkgconfig libxml2
+USE_PKGCONFIG=	build
+USE_GNOME=	libxml2
 MAKE_JOBS_SAFE=	yes
 WARNING_CXXFLAGS?=	-w
 
@@ -51,6 +51,15 @@
 POSTGIS_DESC=	PostGIS input plugin
 SQLITE3_DESC=	SQLite input plugin
 
+PORTDOCS=	*
+
+DOCSRCDIR1=	${WRKSRC}
+DOC_FILES1=	AUTHORS.md CHANGELOG.md README.md
+
+DOCSRCDIR2=	${WRKSRC}/docs
+DOCSDIR2=	${DOCSDIR}/docs
+DOC_FILES2=	*
+
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MCAIRO}
@@ -134,11 +143,11 @@
 pre-configure:
 .if ${PORT_OPTIONS:MSQLITE3}
 	@if ! ${LOCALBASE}/bin/sqlite3 :memory: 'create virtual table foo using rtree(pkid, xmin, xmax, ymin, ymax)' > /dev/null 2>&1; then \
-	   ${ECHO_MSG} "" ; \
-	   ${ECHO_MSG} "The SQLite plugin requires libsqlite3 built with RTREE support"; \
-	   ${ECHO_MSG} "Please, reinstall 'databases/sqlite3' port with R*Tree module enabled"; \
-	   ${ECHO_MSG} "" ; \
-	   ${FALSE} ; \
+		${ECHO_MSG} "" ; \
+		${ECHO_MSG} "The SQLite plugin requires libsqlite3 built with RTREE support"; \
+		${ECHO_MSG} "Please, reinstall 'databases/sqlite3' port with R*Tree module enabled"; \
+		${ECHO_MSG} "" ; \
+		${FALSE} ; \
 	fi
 .endif
 
@@ -148,6 +157,14 @@
 
 do-install:
 	@cd ${INSTALL_WRKSRC} && ${SETENV} ${SCONS_BUILDENV} ${SCONS_BIN} \
-	    ${SCONS_INSTALL_TARGET}
+		${SCONS_INSTALL_TARGET}
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR2}
+	${INSTALL_DATA} ${DOC_FILES2:S|^|${DOCSRCDIR2}/|} ${DOCSDIR2}
+.endif
 
 .include <bsd.port.post.mk>
diff -ruN --exclude=CVS /usr/ports/graphics/mapnik/pkg-descr ./pkg-descr
--- /usr/ports/graphics/mapnik/pkg-descr	2013-01-14 17:57:03.000000000 +0100
+++ ./pkg-descr	2013-04-09 14:34:50.000000000 +0200
@@ -12,4 +12,4 @@
 the wheel, but utilizes best of breed industry standard libraries from
 boost.org.
 
-WWW:	http://www.mapnik.org/
+WWW: http://www.mapnik.org/
diff -ruN --exclude=CVS /usr/ports/graphics/mapnik/pkg-plist ./pkg-plist
--- /usr/ports/graphics/mapnik/pkg-plist	2013-01-14 17:57:03.000000000 +0100
+++ ./pkg-plist	2013-04-09 22:44:51.000000000 +0200
@@ -194,6 +194,28 @@
 lib/libmapnik.so
 lib/libmapnik.so.2
 lib/libmapnik.so.2.1
+lib/mapnik/fonts/DejaVuSans-Bold.ttf
+lib/mapnik/fonts/DejaVuSans-BoldOblique.ttf
+lib/mapnik/fonts/DejaVuSans-ExtraLight.ttf
+lib/mapnik/fonts/DejaVuSans-Oblique.ttf
+lib/mapnik/fonts/DejaVuSans.ttf
+lib/mapnik/fonts/DejaVuSansCondensed-Bold.ttf
+lib/mapnik/fonts/DejaVuSansCondensed-BoldOblique.ttf
+lib/mapnik/fonts/DejaVuSansCondensed-Oblique.ttf
+lib/mapnik/fonts/DejaVuSansCondensed.ttf
+lib/mapnik/fonts/DejaVuSansMono-Bold.ttf
+lib/mapnik/fonts/DejaVuSansMono-BoldOblique.ttf
+lib/mapnik/fonts/DejaVuSansMono-Oblique.ttf
+lib/mapnik/fonts/DejaVuSansMono.ttf
+lib/mapnik/fonts/DejaVuSerif-Bold.ttf
+lib/mapnik/fonts/DejaVuSerif-BoldItalic.ttf
+lib/mapnik/fonts/DejaVuSerif-Italic.ttf
+lib/mapnik/fonts/DejaVuSerif.ttf
+lib/mapnik/fonts/DejaVuSerifCondensed-Bold.ttf
+lib/mapnik/fonts/DejaVuSerifCondensed-BoldItalic.ttf
+lib/mapnik/fonts/DejaVuSerifCondensed-Italic.ttf
+lib/mapnik/fonts/DejaVuSerifCondensed.ttf
+lib/mapnik/fonts/unifont-5.1.20080907.ttf
 %%CSV%%lib/mapnik/input/csv.input
 %%GDAL%%lib/mapnik/input/gdal.input
 lib/mapnik/input/geojson.input
@@ -205,43 +227,14 @@
 lib/mapnik/input/raster.input
 lib/mapnik/input/shape.input
 %%SQLITE%%lib/mapnik/input/sqlite.input
-lib/mapnik/fonts/unifont-5.1.20080907.ttf
-lib/mapnik/fonts/DejaVuSerif-BoldItalic.ttf
-lib/mapnik/fonts/DejaVuSerifCondensed-BoldItalic.ttf
-lib/mapnik/fonts/DejaVuSerif-Italic.ttf
-lib/mapnik/fonts/DejaVuSerifCondensed-Italic.ttf
-lib/mapnik/fonts/DejaVuSansMono-Oblique.ttf
-lib/mapnik/fonts/DejaVuSerifCondensed-Bold.ttf
-lib/mapnik/fonts/DejaVuSansCondensed-Oblique.ttf
-lib/mapnik/fonts/DejaVuSansCondensed-Bold.ttf
-lib/mapnik/fonts/DejaVuSans-Oblique.ttf
-lib/mapnik/fonts/DejaVuSansCondensed.ttf
-lib/mapnik/fonts/DejaVuSansMono-Bold.ttf
-lib/mapnik/fonts/DejaVuSans.ttf
-lib/mapnik/fonts/DejaVuSans-Bold.ttf
-lib/mapnik/fonts/DejaVuSans-ExtraLight.ttf
-lib/mapnik/fonts/DejaVuSerif.ttf
-lib/mapnik/fonts/DejaVuSerifCondensed.ttf
-lib/mapnik/fonts/DejaVuSansCondensed-BoldOblique.ttf
-lib/mapnik/fonts/DejaVuSansMono-BoldOblique.ttf
-lib/mapnik/fonts/DejaVuSans-BoldOblique.ttf
-lib/mapnik/fonts/DejaVuSerif-Bold.ttf
-lib/mapnik/fonts/DejaVuSansMono.ttf
-%%PYTHON_SITELIBDIR%%/mapnik/__init__.py
-%%PYTHON_SITELIBDIR%%/mapnik/printing.py
-%%PYTHON_SITELIBDIR%%/mapnik/paths.py
-%%PYTHON_SITELIBDIR%%/mapnik/_mapnik.so
-%%PYTHON_SITELIBDIR%%/mapnik2/__init__.py
-@dirrm %%PYTHON_SITELIBDIR%%/mapnik
-@dirrm %%PYTHON_SITELIBDIR%%/mapnik2
 @dirrm lib/mapnik/input
 @dirrm lib/mapnik/fonts
 @dirrm lib/mapnik
-@dirrm include/mapnik/formatting
-@dirrm include/mapnik/grid
-@dirrm include/mapnik/json
-@dirrm include/mapnik/svg
-@dirrm include/mapnik/text_placements
-@dirrm include/mapnik/util
 @dirrm include/mapnik/wkt
+@dirrm include/mapnik/util
+@dirrm include/mapnik/text_placements
+@dirrm include/mapnik/svg
+@dirrm include/mapnik/json
+@dirrm include/mapnik/grid
+@dirrm include/mapnik/formatting
 @dirrm include/mapnik
===> Done
