===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/pngcrush/Makefile ./Makefile
--- /usr/ports/graphics/pngcrush/Makefile	2013-03-06 02:18:28.000000000 +0100
+++ ./Makefile	2013-04-09 23:18:58.000000000 +0200
@@ -1,15 +1,14 @@
 # Created by: Chris D. Faulhaber <jedgar@FreeBSD.org>
 # $FreeBSD: head/graphics/pngcrush/Makefile 313369 2013-03-03 17:35:51Z jsa $
-#
 
 PORTNAME=	pngcrush
-PORTVERSION=	1.7.51
+PORTVERSION=	1.7.57
 CATEGORIES=	graphics
-MASTER_SITES=	SF/pmt/${PORTNAME}/${PORTVERSION}
+MASTER_SITES=	SF/pmt/${PORTNAME}/${PORTVERSION}/
 DISTNAME=	${PORTNAME}-${PORTVERSION}-nolib
 
 MAINTAINER=	jsa@FreeBSD.org
-COMMENT=	An optimizer for PNG files
+COMMENT=	Optimizer for PNG files
 
 LIB_DEPENDS=	png15:${PORTSDIR}/graphics/png
 
@@ -20,9 +19,9 @@
 MAKE_ARGS+=	CC="${CC}" CFLAGS="${CFLAGS}" LD="${CC}" LDFLAGS="${LDFLAGS}"
 MAKE_JOBS_SAFE=	yes
 
-PLIST_FILES=	bin/pngcrush
+PLIST_FILES=	bin/${PORTNAME}
 
 do-install:
-	${INSTALL_PROGRAM} ${WRKSRC}/pngcrush ${PREFIX}/bin/pngcrush
+	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/graphics/pngcrush/distinfo ./distinfo
--- /usr/ports/graphics/pngcrush/distinfo	2013-03-06 02:18:28.000000000 +0100
+++ ./distinfo	2013-04-09 23:15:59.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (pngcrush-1.7.51-nolib.tar.xz) = 78df26dd2c6fdb09e55078d173350bef62abb2ba51ccfe352f4be6f6b8cf7762
-SIZE (pngcrush-1.7.51-nolib.tar.xz) = 53912
+SHA256 (pngcrush-1.7.57-nolib.tar.xz) = 71c4dbdf0ccfc4d26c391c85b341b7567383a7fbd7f077dedfdcf076847b2f89
+SIZE (pngcrush-1.7.57-nolib.tar.xz) = 56280
===> Done
