===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/net/xmlrpc-epi/Makefile ./Makefile
--- /usr/ports/net/xmlrpc-epi/Makefile	2013-01-14 17:56:35.000000000 +0100
+++ ./Makefile	2013-03-31 18:15:12.000000000 +0200
@@ -1,20 +1,19 @@
-# New ports collection makefile for:	xmlrpc-epi
-# Date created:			19 February 2007
-# Whom:				Thierry Thomas <thierry@pompo.net>
-#
+# Created by: Thierry Thomas <thierry@pompo.net>
 # $FreeBSD: head/net/xmlrpc-epi/Makefile 300897 2012-07-14 14:29:18Z beat $
-#
 
 PORTNAME=	xmlrpc-epi
-PORTVERSION=	0.54.1
+PORTVERSION=	0.54.2
 CATEGORIES=	net
-MASTER_SITES=	SF/${PORTNAME}/${PORTNAME}-base/${PORTVERSION}
+MASTER_SITES=	SF/${PORTNAME}/${PORTNAME}-base/${PORTVERSION}/
 
 MAINTAINER=	ports@FreeBSD.org
-COMMENT=	A general purpose implementation of the xmlrpc specification in C
+COMMENT=	General purpose implementation of the xmlrpc specification in C
 
-LIB_DEPENDS=	expat.6:${PORTSDIR}/textproc/expat2
+LICENSE=	MIT
 
+LIB_DEPENDS=	expat:${PORTSDIR}/textproc/expat2
+
+USE_BZIP2=	yes
 USE_GMAKE=	yes
 USE_ICONV=	yes
 USE_LDCONFIG=	yes
@@ -25,29 +24,22 @@
 
 PORTDOCS=	AUTHORS ChangeLog NEWS README
 
-METHODS=	TestNormal TestFault TestStruct TestArray TestBoolean	\
+METHODS=	TestNormal TestFault TestStruct TestArray TestBoolean \
 		TestInt TestString TestDouble TestBase64 TestDateTime
 
 CONFLICTS=	xmlrpc-c-0.* xmlrpc-c-devel-[0-9]*
 
-WRKSRC=	${WRKDIR}/${PORTNAME:C/-.*//}
-
-pre-configure:
-	@${FIND} ${WRKSRC} -name Makefile.in | ${XARGS}	\
-	${REINPLACE_CMD} -e '/^LTCOMPILE =/s/--mode=/--tag=CC --mode=/'
-	@${FIND} ${WRKSRC} -name Makefile.in | ${XARGS}	\
-	${REINPLACE_CMD} -e '/^LINK =/s/--mode=/--tag=CC --mode=/'
+.include <bsd.port.options.mk>
 
 post-install:
-.if !defined(NOPORTDOCS)
-	${MKDIR} ${DOCSDIR}
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 regression-test:
 .for tm in ${METHODS}
-	cd ${WRKSRC}/sample &&	\
-	./sample -method method_${tm}
+	cd ${WRKSRC}/sample && ./sample -method method_${tm}
 .endfor
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/net/xmlrpc-epi/distinfo ./distinfo
--- /usr/ports/net/xmlrpc-epi/distinfo	2013-01-14 17:56:35.000000000 +0100
+++ ./distinfo	2013-03-31 14:41:30.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (xmlrpc-epi-0.54.1.tar.gz) = fa3d0637338f227dc01d9ea9bce1225b925618bd2f36bee7846b690101de038a
-SIZE (xmlrpc-epi-0.54.1.tar.gz) = 913402
+SHA256 (xmlrpc-epi-0.54.2.tar.bz2) = c74ef8fb680b140890138a82f37619714b67f69025a775b9ba2009d62cded0b8
+SIZE (xmlrpc-epi-0.54.2.tar.bz2) = 348478
===> Done
