===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/security/secpanel/Makefile ./Makefile
--- /usr/ports/security/secpanel/Makefile	2013-01-24 11:57:55.000000000 +0100
+++ ./Makefile	2013-05-27 15:58:50.000000000 +0200
@@ -25,8 +25,8 @@
 
 USE_TK_WRAPPER=	yes
 
-DESKTOP_ENTRIES="SecPanel" "${COMMENT}" "${PREFIX}/share/pixmaps/${PORTNAME}.png" \
-		"${PORTNAME}" "" "false"
+DESKTOP_ENTRIES="SecPanel" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Network;Security;" false
 
 post-patch:
 	@${REINPLACE_CMD} \
@@ -35,9 +35,9 @@
 
 do-install:
 	${INSTALL_SCRIPT} ${WRKSRC}/bin/secpanel ${PREFIX}/bin
-	@(cd ${WRKSRC}/lib/secpanel && ${COPYTREE_SHARE} \* ${DATADIR})
+	@(cd ${WRKSRC}/lib/secpanel && ${COPYTREE_SHARE} . ${DATADIR})
 	@${CHMOD} u+rw ${DATADIR}/default.config
 
-	${INSTALL_DATA} ${DISTDIR}/${DIST_SUBDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .include <bsd.port.mk>
===> Done
