===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/gno3dtet/Makefile ./Makefile
--- /usr/ports/games/gno3dtet/Makefile	2013-03-29 00:26:25.000000000 +0100
+++ ./Makefile	2013-04-06 17:15:04.000000000 +0200
@@ -3,9 +3,9 @@
 
 PORTNAME=	gno3dtet
 PORTVERSION=	1.96.1
-PORTREVISION=	10
+PORTREVISION=	11
 CATEGORIES=	games gnome
-MASTER_SITES=	http://eseb.net/ftp/gno3dtet/
+MASTER_SITES=	http://eseb.net/ftp/${PORTNAME}/
 EXTRACT_SUFX=	.tgz
 
 MAINTAINER=	ports@FreeBSD.org
@@ -13,15 +13,28 @@
 
 EXTRACT_DEPENDS=intltoolize:${PORTSDIR}/textproc/intltool
 
-USES=        pathfix
-USE_GNOME=	gnomeprefix intlhack libgnomeui librsvg2
+USES=		pathfix
+USE_PKGCONFIG=	build
+USE_GNOME=	desktopfileutils gnomeprefix intlhack libgnomeui librsvg2
+INSTALLS_OMF=	yes
 USE_GMAKE=	yes
 USE_AUTOTOOLS=	autoheader automake aclocal autoconf
 ACLOCAL_ARGS=	-I m4 -I ${ACLOCAL_DIR} -I ${LOCALBASE}/share/aclocal
 AUTOMAKE_ARGS=	-a -c -f
 CPPFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
-INSTALLS_OMF=	yes
+
+OPTIONS_DEFINE=	NLS
+
+.include <bsd.port.options.mk>
+
+.if ${PORT_OPTIONS:MNLS}
+USE_GETTEXT=	yes
+PLIST_SUB+=	NLS=""
+.else
+CONFIGURE_ARGS+=	--disable-nls
+PLIST_SUB+=	NLS="@comment "
+.endif
 
 run-autotools:: run-autotools-aclocal run-autotools-autoheader \
 		run-autotools-automake run-autotools-autoconf
@@ -31,7 +44,11 @@
 
 post-patch:
 	@${REINPLACE_CMD} -e '/LIBTOOL/d ; \
-		 s|gnome-games|gno3dtet|g' ${WRKSRC}/configure.in
+		s|gnome-games|gno3dtet|g' ${WRKSRC}/configure.in
+
+.if ! ${PORT_OPTIONS:MNLS}
+	@${REINPLACE_CMD} -e '/^always_built_SUBDIRS/s|po||g' ${WRKSRC}/Makefile.am
+.endif
 
 pre-configure:
 	@${INSTALL_SCRIPT} ${AUTOMAKE_DIR}/mkinstalldirs ${WRKSRC}
@@ -42,4 +59,6 @@
 	${CHOWN} root:games ${PREFIX}/share/games/gno3dtet.hof
 	${CHMOD} 664 ${PREFIX}/share/games/gno3dtet.hof
 
+	@-update-desktop-database -q
+
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/gno3dtet/pkg-plist ./pkg-plist
--- /usr/ports/games/gno3dtet/pkg-plist	2013-01-14 17:55:28.000000000 +0100
+++ ./pkg-plist	2013-04-06 17:15:17.000000000 +0200
@@ -3,6 +3,7 @@
 share/games/gno3dtet.hof
 share/gnome/help/gno3dtet/C/gno3dtet.xml
 share/gnome/help/gno3dtet/C/legal.xml
+%%NLS%%share/locale/fr/LC_MESSAGES/gno3dtet.mo
 share/omf/gno3dtet/gno3dtet-C.omf
 share/pixmaps/gno3dtet.png
 share/sounds/gno3dtet/danger.wav
@@ -15,9 +16,11 @@
 share/sounds/gno3dtet/pause.wav
 share/sounds/gno3dtet/score.wav
 share/sounds/gno3dtet/screwd.wav
-share/locale/fr/LC_MESSAGES/gno3dtet.mo
 @dirrm share/sounds/gno3dtet
 @dirrm share/omf/gno3dtet
 @dirrm share/gnome/help/gno3dtet/C
 @dirrm share/gnome/help/gno3dtet
+@dirrmtry share/games
 @dirrmtry share/applications
+@exec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
+@unexec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
===> Done
