===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/x11-toolkits/pure-tk/Makefile ./Makefile
--- /usr/ports/x11-toolkits/pure-tk/Makefile	2013-01-14 17:54:37.000000000 +0100
+++ ./Makefile	2013-04-05 23:59:16.000000000 +0200
@@ -1,19 +1,21 @@
-# New ports collection makefile for:	pure-tk
-# Date created:		2011-11-09
-# Whom:			Zhihao Yuan <lichray@gmail.com>
-#
+# Created by: Zhihao Yuan <lichray@gmail.com>
 # $FreeBSD: head/x11-toolkits/pure-tk/Makefile 308232 2012-12-04 10:43:09Z gahr $
-#
 
 PORTNAME=	pure-tk
 PORTVERSION=	0.3
+PORTREVISION=	1
 CATEGORIES=	x11-toolkits
+#MASTER_SITES=	https://cdn.bitbucket.org/purelang/pure-lang/downloads/
+MASTER_SITES=	https://pure-lang.googlecode.com/files/
+DIST_SUBDIR=	pure
 
 MAINTAINER=	lichray@gmail.com
-COMMENT=	A basic interface between Pure and Tcl/Tk
+COMMENT=	Basic interface between Pure and Tcl/Tk
 
-USE_PURE=	yes
-USE_TK=	83+
+LICENSE=	BSD
+
+USES=		pure
+USE_TK=		83+
 SHORT_TCL_VER=	${TCL_VER:S/8./8/}
 
 MAKE_ARGS+=	tclvers=${TCL_VER}
@@ -24,11 +26,25 @@
 
 post-patch:
 	${REINPLACE_CMD} \
- 		-e "s|/usr/include|${LOCALBASE}/include|g" \
- 		-e "s|-ltcl\$$(tclvers)|-ltcl${SHORT_TCL_VER}|" \
- 		-e "s|-ltk\$$(tclvers)|-ltk${SHORT_TCL_VER}|" \
+		-e "s|/usr/include|${LOCALBASE}/include|g" \
+		-e "s|-ltcl\$$(tclvers)|-ltcl${SHORT_TCL_VER}|" \
+		-e "s|-ltk\$$(tclvers)|-ltk${SHORT_TCL_VER}|" \
 		${WRKSRC}/Makefile
 
-.include <bsd.port.pre.mk>
-.include "${PORTSDIR}/lang/pure/bsd.pure.mk"
-.include <bsd.port.post.mk>
+PORTDOCS=	README
+PORTEXAMPLES=	*
+
+.include <bsd.port.options.mk>
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+
+.if ${PORT_OPTIONS:MEXAMPLES}
+	@${MKDIR} ${EXAMPLESDIR}
+	@(cd ${WRKSRC}/examples/ && ${COPYTREE_SHARE} . ${EXAMPLESDIR})
+.endif
+
+.include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/x11-toolkits/pure-tk/pkg-descr ./pkg-descr
--- /usr/ports/x11-toolkits/pure-tk/pkg-descr	2013-01-14 17:54:37.000000000 +0100
+++ ./pkg-descr	2013-04-05 23:59:22.000000000 +0200
@@ -3,4 +3,4 @@
 in the Tcl interpreter, set and retrieve variable values in the
 interpreter, and invoke Pure callbacks from Tcl/Tk.
 
-WWW:	http://docs.pure-lang.googlecode.com/hg/pure-tk.html
+WWW: http://docs.pure-lang.googlecode.com/hg/pure-tk.html
===> Done
