===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/net-mgmt/cowpatty/Makefile ./Makefile
--- /usr/ports/net-mgmt/cowpatty/Makefile	2013-01-14 17:54:50.000000000 +0100
+++ ./Makefile	2013-05-20 00:55:02.000000000 +0200
@@ -1,15 +1,10 @@
-# New ports collection makefile for:	cowpatty
-# Date created:		March 20, 2008
-# Whom:		Dennis Herrmann <adox@mcx2.org>
-#
+# Created by: Dennis Herrmann <adox@mcx2.org>
 # $FreeBSD: head/net-mgmt/cowpatty/Makefile 300897 2012-07-14 14:29:18Z beat $
-#
 
 PORTNAME=	cowpatty
 PORTVERSION=	4.6
 CATEGORIES=	net-mgmt security
-MASTER_SITES=	http://www.willhackforsushi.com/code/cowpatty/${PORTVERSION}/ \
-		http://mcx2.org/files/
+MASTER_SITES=	http://www.willhackforsushi.com/code/${PORTNAME}/${PORTVERSION}/
 EXTRACT_SUFX=	.tgz
 
 MAINTAINER=	dhn@FreeBSD.org
@@ -22,15 +17,16 @@
 
 PLIST_FILES=	bin/cowpatty bin/genpmk
 
+.include <bsd.port.options.mk>
+
 post-install:
-.if !defined(NOPORTDOCS)
-	@${ECHO_MSG} "Installing additional documentation to ${DOCSDIR}"
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-	@cd ${WRKSRC} && ${INSTALL_DATA} ${PORTDOCS} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
-	@${ECHO_MSG} ""
+	@${ECHO_CMD}
 	@${CAT} ${PKGMESSAGE}
-	@${ECHO_MSG} ""
+	@${ECHO_CMD}
 
 .include <bsd.port.mk>
===> Done
