===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/lang/pure/Makefile ./Makefile
--- /usr/ports/lang/pure/Makefile	2013-01-22 10:16:19.000000000 +0100
+++ ./Makefile	2013-04-06 04:12:51.000000000 +0200
@@ -1,41 +1,41 @@
-# New ports collection makefile for:	pure
-# Date created:		2011-03-17
-# Whom:			Zhihao Yuan <lichray@gmail.com>
-#
-# $FreeBSD: head/lang/pure/Makefile 310792 2013-01-22 02:05:58Z brooks $
-#
+# Created by: Zhihao Yuan <lichray@gmail.com>
+# $FreeBSD: nemysis/lang/pure/Makefile 10924 2013-04-05 16:23:34Z nemysis $
 
 PORTNAME=	pure
-PORTVERSION=	0.55
+PORTVERSION=	0.57
 CATEGORIES=	lang
+MASTER_SITES=	https://cdn.bitbucket.org/purelang/pure-lang/downloads/
 DISTFILES=	${PORTNAME}-${PORTVERSION}.tar.gz ${PORTNAME}-docs-${PORTVERSION}.tar.gz
+DIST_SUBDIR=	pure
 
 MAINTAINER=	lichray@gmail.com
-COMMENT=	A modern-style functional programming language
+COMMENT=	Modern-style functional programming language
 
 LICENSE=	GPLv3 LGPL3
 LICENSE_COMB=	dual
 
 LIB_DEPENDS=	gmp:${PORTSDIR}/math/gmp \
 		mpfr:${PORTSDIR}/math/mpfr
-BUILD_DEPENDS=	llvm31>=3.1:${PORTSDIR}/devel/llvm31
-RUN_DEPENDS=	llvm31>=3.1:${PORTSDIR}/devel/llvm31
+BUILD_DEPENDS=	llvm>=0:${PORTSDIR}/devel/llvm
+RUN_DEPENDS=	llvm>=0:${PORTSDIR}/devel/llvm
 
 MAN1=		pure.1
 SUB_FILES=	pkg-message
 
 USE_AUTOTOOLS=	libltdl
 GNU_CONFIGURE=	yes
-USE_GNOME=	gnomehack pkgconfig
-USE_PURE=	yes
+CONFIGURE_ARGS=	--with-libgmp-prefix=${LOCALBASE} --enable-release
+USES=		pure pathfix
+USE_PKGCONFIG=	build
 USE_LDCONFIG=	yes
 MAKE_JOBS_SAFE=	yes
 
-CONFIGURE_ARGS=	--with-libgmp-prefix=${LOCALBASE} --enable-release
-
-OPTIONS_DEFINE=	EMACS
+OPTIONS_DEFINE=	EMACS ETC
 
 EMACS_DESC=	Compile pure-mode.el with Emacs
+ETC_DESC=	Copy Pure syntax highlighting to ${DATADIR}/etc
+
+OPTIONS_DEFAULT=	ETC
 
 .include <bsd.port.options.mk>
 
@@ -48,26 +48,13 @@
 PLIST_SUB+=	ELC="@comment "
 .endif
 
-# automatically disable readline support if editline support is available
-.if exists(/usr/include/edit/readline/readline.h)
-CONFIGURE_ARGS+=	--without-readline
-.endif
-
-.if ${ARCH} == "amd64"
-CONFIGURE_TARGET=	x86_64-portbld-freebsd
-.endif
-
-PORTDATA=	*
-PORTEXAMPLES=	*
-PORTDOCS=	*
-
-.if !defined(NOPORTDATA)
-SUB_LIST+=	ETC=""
+.if ${PORT_OPTIONS:METC}
+PLIST_SUB+=	ETC=""
 .else
-SUB_LIST+=	ETC="@comment "
+PLIST_SUB+=	ETC="@comment "
 .endif
 
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MDOCS}
 PLIST_SUB+=	DOCS=""
 .else
 PLIST_SUB+=	DOCS="@comment "
@@ -79,26 +66,51 @@
 SUB_LIST+=	NOCLANG="@comment "
 .endif
 
+# automatically disable readline support if editline support is available
+.if exists(/usr/include/edit/readline/readline.h)
+CONFIGURE_ARGS+=	--without-readline
+.endif
+
+.if ${ARCH} == "amd64"
+CONFIGURE_TARGET=	x86_64-portbld-freebsd
+.endif
+
+PORTDOCS=	*
+PORTEXAMPLES=	*
+
 .include <bsd.port.pre.mk>
 
 post-install:
-.if !defined(NOPORTDATA)
-	@${MKDIR} ${DATADIR}/etc/
-	(cd ${WRKSRC} && ${RM} -f etc/*.in && ${COPYTREE_SHARE} etc/ ${DATADIR}/)
+.if ${PORT_OPTIONS:METC}
+	@${MKDIR} ${DATADIR}/etc
+	@(cd ${WRKSRC} && ${RM} -f etc/*.in && ${COPYTREE_SHARE} etc/ ${DATADIR})
 .endif
 
-.if !defined(NOPORTEXAMPLES)
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	@(cd ${WRKDIR}/${PORTNAME}-docs-${PORTVERSION} && ${RM} -f Makefile && ${COPYTREE_SHARE} . ${DOCSDIR})
+	${LN} -sf ${DOCSDIR} ${PREFIX}/lib/${PORTNAME}/docs
+.endif
+
+.if ${PORT_OPTIONS:MEXAMPLES}
 	@${MKDIR} ${EXAMPLESDIR}
-	(cd ${WRKSRC}/examples/ && ${COPYTREE_SHARE} \* ${EXAMPLESDIR})
+	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} . ${EXAMPLESDIR})
 .endif
 
-.if !defined(NOPORTDOCS)
-	@${MKDIR} ${DOCSDIR}
-	(cd ${WRKDIR}/${PORTNAME}-docs-${PORTVERSION} && ${RM} -f Makefile && ${COPYTREE_SHARE} \* ${DOCSDIR})
-	${LN} -fs ${DOCSDIR} ${PREFIX}/lib/${PORTNAME}/docs
+.if ${PORT_OPTIONS:METC}
+	@${ECHO_CMD}
+	@${ECHO_CMD}
+	@${ECHO_CMD} "==============================================================================="
+	@${ECHO_CMD}
+	@${ECHO_CMD} "	Note that we installed a few Pure syntax highlighting"
+	@${ECHO_CMD} "	files for the code editors in ${DATADIR}/etc/"
+	@${ECHO_CMD}
+	@${ECHO_CMD} "==============================================================================="
+	@${ECHO_CMD}
 .endif
 
+	@${ECHO_CMD}
 	@${CAT} ${PKGMESSAGE}
+	@${ECHO_CMD}
 
-.include "bsd.pure.mk"
 .include <bsd.port.post.mk>
diff -ruN --exclude=CVS /usr/ports/lang/pure/bsd.pure.mk ./bsd.pure.mk
--- /usr/ports/lang/pure/bsd.pure.mk	2013-01-14 17:56:12.000000000 +0100
+++ ./bsd.pure.mk	1970-01-01 01:00:00.000000000 +0100
@@ -1,96 +0,0 @@
-# ex:ts=8
-#
-# $FreeBSD: head/lang/pure/bsd.pure.mk 300896 2012-07-14 13:54:48Z beat $
-#
-# bsd.pure.mk - Support for the Pure language packages
-#
-# Author: Zhihao Yuan <lichray@gmail.com>
-#
-# USE_PURE		-	Set it to 'yes' to use libpure.so.
-# 					Others are regarded as a list of *RUN_DEPENDS*
-# 					to pure-* addons. For example,
-# USE_PURE=	ffi 	makes the port depends on pure-ffi at runtime.
-# USE_PURE_BUILD	-	Set it to 'yes' to change the addons in USE_PURE
-# 						list to *BUILD_DEPENDS*.
-# NO_PURE_EXAMPLES	-	Don't automatically install examples/.
-
-.if !defined(PURE_include)
-
-PURE_MAINTAINER=lichray@gmail.com
-PURE_include=	bsd.pure.mk
-
-MASTER_SITES?=	http://pure-lang.googlecode.com/files/
-DIST_SUBDIR=	pure
-
-USE_GMAKE=	yes
-
-_PURE_audio_cat=	audio
-_PURE_csv_cat=		textproc
-_PURE_ffi_cat=		devel
-_PURE_gen_cat=		devel
-_PURE_gen_util=		pure-gen
-_PURE_gl_cat=		graphics
-_PURE_gl_fn=		GL
-_PURE_gtk_cat=		x11-toolkits
-_PURE_mpfr_cat=		math
-_PURE_readline_cat=	devel
-_PURE_sockets_cat=	net
-_PURE_sql3_cat=		databases
-_PURE_tk_cat=		x11-toolkits
-_PURE_xml_cat=		textproc
-
-.ifdef _PURE_${PORTNAME}_cat
-CATEGORIES=		_PURE_${PORTNAME}_cat
-.endif
-
-.if ${PORTNAME} != pure
-
-.if ${USE_PURE} == yes
-LIB_DEPENDS+=	pure.8:${PORTSDIR}/lang/pure
-.else
-. for dep in ${USE_PURE}
-_PURE_LIBDIR=	${LOCALBASE}/lib/pure
-_PURE_PORTDIR=	${PORTSDIR}/${_PURE_${dep}_cat}/pure-${dep}
-.  ifdef _PURE_${dep}_cat
-.   ifdef _PURE_${dep}_util
-BUILD_DEPENDS+=	${_PURE_${dep}_util}:${_PURE_PORTDIR}
-.   else
-.    ifdef USE_PURE_BUILD
-.     ifdef _PURE_${dep}_fn
-BUILD_DEPENDS+=	${_PURE_LIBDIR}/${_PURE_${dep}_fn}.pure:${_PURE_PORTDIR}
-.     else
-BUILD_DEPENDS+=	${_PURE_LIBDIR}/${dep}.pure:${_PURE_PORTDIR}
-.     endif
-.    endif
-.     ifdef _PURE_${dep}_fn
-RUN_DEPENDS+=	${_PURE_LIBDIR}/${_PURE_${dep}_fn}.pure:${_PURE_PORTDIR}
-.     else
-RUN_DEPENDS+=	${_PURE_LIBDIR}/${dep}.pure:${_PURE_PORTDIR}
-.     endif
-.   endif
-.  else
-.   error pure-${dep} is not supported by "bsd.pure.mk"
-.  endif
-. endfor
-.endif
-
-.endif # PORTNAME != pure
-
-MAKE_ARGS+=	prefix=${PREFIX} mandir=${PREFIX}/man \
-		CC=${CC} CFLAGS="${CFLAGS}" \
-		CXX=${CXX} CXXFLAGS="${CXXFLAGS}" \
-		CPPFLAGS+=-I${LOCALBASE}/include \
-		LDFLAGS+=-L${LOCALBASE}/lib
-
-.if ${PORTNAME:Mpure-*} !="" && !defined(NO_PURE_EXAMPLES)
-PORTEXAMPLES=	*
-
-post-install:
-.if !defined(NOPORTEXAMPLES)
-	@${MKDIR} ${EXAMPLESDIR}
-	(cd ${WRKSRC}/examples/ && ${COPYTREE_SHARE} \* ${EXAMPLESDIR})
-.endif
-
-.endif
-
-.endif #!defined(PURE_include)
diff -ruN --exclude=CVS /usr/ports/lang/pure/distinfo ./distinfo
--- /usr/ports/lang/pure/distinfo	2013-01-14 17:56:12.000000000 +0100
+++ ./distinfo	2013-04-05 20:04:33.000000000 +0200
@@ -1,4 +1,4 @@
-SHA256 (pure/pure-0.55.tar.gz) = 36dbf1b77719a8031766b30812ac89102188d01ae9da80c2b53889f501cb58a7
-SIZE (pure/pure-0.55.tar.gz) = 1147787
-SHA256 (pure/pure-docs-0.55.tar.gz) = 2cd0045c8fd126cf41cb847f2842e11b7c0480c8cb8d5ea33470cf7edcfd1db9
-SIZE (pure/pure-docs-0.55.tar.gz) = 3540249
+SHA256 (pure/pure-0.57.tar.gz) = 31c16894f132046dbb8a5826cc8ebd62c76fe2741eca2fd1b6e8799eddca5fe8
+SIZE (pure/pure-0.57.tar.gz) = 1455384
+SHA256 (pure/pure-docs-0.57.tar.gz) = d40705a2d24bea1e0e139bb65bafc5d91da43c07d529b909c49d90bd7565eafa
+SIZE (pure/pure-docs-0.57.tar.gz) = 4785296
diff -ruN --exclude=CVS /usr/ports/lang/pure/files/patch-pure.cc ./files/patch-pure.cc
--- /usr/ports/lang/pure/files/patch-pure.cc	1970-01-01 01:00:00.000000000 +0100
+++ ./files/patch-pure.cc	2013-04-05 04:56:25.000000000 +0200
@@ -0,0 +1,19 @@
+--- pure.cc.orig	2013-02-14 15:03:12.000000000 +0100
++++ pure.cc	2013-04-05 04:54:33.000000000 +0200
+@@ -362,14 +362,14 @@
+ 
+ static char *my_command_input2(const char *prompt)
+ {
+-#ifdef HAVE_READLINE_HISTORY
++#if 0
+   HISTORY_STATE *save_hist = history_get_history_state();
+   int histmax = unstifle_history();
+   history_set_history_state(my_hist);
+   stifle_history(600);
+ #endif
+   char *s = readline(prompt);
+-#ifdef HAVE_READLINE_HISTORY
++#if 0
+   if (s && *s) add_history(s);
+   free(my_hist);
+   my_hist = history_get_history_state();
diff -ruN --exclude=CVS /usr/ports/lang/pure/files/pkg-message.in ./files/pkg-message.in
--- /usr/ports/lang/pure/files/pkg-message.in	2013-01-14 17:56:12.000000000 +0100
+++ ./files/pkg-message.in	2013-04-05 17:07:45.000000000 +0200
@@ -1,11 +1,9 @@
-====
-%%ETC%%Note that we installed a few Pure syntax highlighting
-%%ETC%%files for the code editors in %%DATADIR%%/etc/
-%%ETC%%
+===============================================================================
+
 A text-based browser, such as one of following
 
 	lynx:	www/lynx
-	w3a:	mwww/w3m
+	w3a:	www/w3m
 
 is recommended to use the `help' command inside the
 interpreter. Check pure(1) for details.
@@ -15,4 +13,5 @@
 %%NOCLANG%%	clang:	lang/clang
 %%NOCLANG%%
 %%NOCLANG%%is required.
-====
+
+===============================================================================
diff -ruN --exclude=CVS /usr/ports/lang/pure/pkg-descr ./pkg-descr
--- /usr/ports/lang/pure/pkg-descr	2013-01-14 17:56:12.000000000 +0100
+++ ./pkg-descr	2013-04-05 03:15:35.000000000 +0200
@@ -5,4 +5,4 @@
 interface. The interpreter uses LLVM as a backend to JIT-compile Pure
 programs to fast native code.
 
-WWW:	https://code.google.com/p/pure-lang/
+WWW: http://purelang.bitbucket.org/
diff -ruN --exclude=CVS /usr/ports/lang/pure/pkg-plist ./pkg-plist
--- /usr/ports/lang/pure/pkg-plist	2013-01-14 17:56:12.000000000 +0100
+++ ./pkg-plist	2013-04-05 17:35:24.000000000 +0200
@@ -6,6 +6,7 @@
 lib/pure/array.pure
 lib/pure/avltrees.pure
 lib/pure/dict.pure
+lib/pure/enum.pure
 lib/pure/faustui.pure
 lib/pure/getopt.pure
 lib/pure/heap.pure
@@ -18,7 +19,6 @@
 lib/pure/pure_main.c
 lib/pure/pure_main.o
 lib/pure/quasiquote.pure
-lib/pure/quasiquote1.pure
 lib/pure/records.pure
 lib/pure/regex.pure
 lib/pure/set.pure
@@ -27,6 +27,18 @@
 libdata/pkgconfig/pure.pc
 %%ELC%%%%EMACS_SITE_LISPDIR%%/pure-mode.el
 %%ELC%%%%EMACS_SITE_LISPDIR%%/pure-mode.elc
+%%ETC%%%%DATADIR%%/etc/pure-highlight.lang
+%%ETC%%%%DATADIR%%/etc/pure-mode.el
+%%ELC%%%%ETC%%%%DATADIR%%/etc/pure-mode.elc
+%%ETC%%%%DATADIR%%/etc/pure.lang
+%%ETC%%%%DATADIR%%/etc/pure.nanorc
+%%ETC%%%%DATADIR%%/etc/pure.plist
+%%ETC%%%%DATADIR%%/etc/pure.py
+%%ETC%%%%DATADIR%%/etc/pure.ssh
+%%ETC%%%%DATADIR%%/etc/pure.vim
+%%ETC%%%%DATADIR%%/etc/pure.xml
+%%ETC%%@dirrm %%DATADIR%%/etc
+%%ETC%%@dirrm %%DATADIR%%
 %%DOCS%%lib/pure/docs
 @dirrm lib/pure
 @dirrm include/pure
===> Done
