===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/mail/rspamd/Makefile ./Makefile
--- /usr/ports/mail/rspamd/Makefile	2013-03-23 02:42:33.000000000 +0100
+++ ./Makefile	2013-04-12 20:52:21.000000000 +0200
@@ -11,12 +11,14 @@
 LICENSE=	BSD
 
 LIB_DEPENDS=	pcre.3:${PORTSDIR}/devel/pcre \
-		gmime-2.0:${PORTSDIR}/mail/gmime2 \
+		gmime-2.4:${PORTSDIR}/mail/gmime24 \
 		event-1.4:${PORTSDIR}/devel/libevent
 
-OPTIONS=	RSPAMD_GPERF "Enable google perf-tools profiling" off \
-		RSPAMD_JUDY "Enable Judy arrays usage for performance" off \
-		RSPAMD_REDIRECTOR "Install HTTP redirector" off
+OPTIONS_DEFINE=	RSPAMD_GPERF RSPAMD_JUDY RSPAMD_REDIRECTOR
+
+RSPAMD_GPERF_DESC=	Enable google perf-tools profiling
+RSPAMD_JUDY_DESC=	Enable Judy arrays usage for performance
+RSPAMD_REDIRECTOR_DESC=	Install HTTP redirector
 
 USE_RC_SUBR=	rspamd
 
@@ -38,23 +40,23 @@
 MAN8=	rspamd.8
 MAN1=	rspamc.1
 
-.include <bsd.port.pre.mk>
+.include <bsd.port.options.mk>
 
-.if defined(WITH_RSPAMD_GPERF)
- CMAKE_ARGS+=	-DENABLE_GPERF_TOOLS=ON
- LIB_DEPENDS+=	profiler.1:${PORTSDIR}/devel/google-perftools
+.if ${PORT_OPTIONS:MRSPAMD_GPERF}
+CMAKE_ARGS+=	-DENABLE_GPERF_TOOLS=ON
+LIB_DEPENDS+=	profiler.1:${PORTSDIR}/devel/google-perftools
 .endif
 
-.if defined(WITH_RSPAMD_JUDY)
- LIB_DEPENDS+=	Judy:${PORTSDIR}/devel/judy
+.if ${PORT_OPTIONS:MRSPAMD_JUDY}
+LIB_DEPENDS+=	Judy:${PORTSDIR}/devel/judy
 .endif
 
-.if defined(WITH_RSPAMD_REDIRECTOR)
- USE_PERL5=	yes
- CMAKE_ARGS+=	-DENABLE_REDIRECTOR=ON
- PLIST_SUB+=	REDIRECTOR=""
- USE_RC_SUBR+=	rspamd_redirector
- RUN_DEPENDS+=	\
+.if ${PORT_OPTIONS:MRSPAMD_REDIRECTOR}
+USE_PERL5=	yes
+CMAKE_ARGS+=	-DENABLE_REDIRECTOR=ON
+PLIST_SUB+=	REDIRECTOR=""
+USE_RC_SUBR+=	rspamd_redirector
+RUN_DEPENDS+=	\
 	p5-IO-Socket-SSL>=0:${PORTSDIR}/security/p5-IO-Socket-SSL \
 	p5-IO-String>=0:${PORTSDIR}/devel/p5-IO-String \
 	p5-XML-Parser>=0:${PORTSDIR}/textproc/p5-XML-Parser \
@@ -67,7 +69,7 @@
 	p5-Proc-Daemon>=0:${PORTSDIR}/devel/p5-Proc-Daemon \
 	p5-SWF-File>=0:${PORTSDIR}/graphics/p5-SWF-File
 .else
- PLIST_SUB+=		REDIRECTOR="@comment "
+PLIST_SUB+=	REDIRECTOR="@comment "
 .endif
 
-.include <bsd.port.post.mk>
+.include <bsd.port.mk>
===> Done
