Index: Makefile
===================================================================
--- Makefile	(revision 345227)
+++ Makefile	(working copy)
@@ -3,61 +3,77 @@
 
 PORTNAME=	gajim
 PORTVERSION=	0.15.4
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	net-im
 MASTER_SITES=	http://gajim.org/downloads/0.15/
 
-MAINTAINER=	rm@FreeBSD.org
+MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Jabber client based on a plugin system
 
 LICENSE=	GPLv3
 
+BUILD_DEPENDS=	p5-XML-Parser>=0:${PORTSDIR}/textproc/p5-XML-Parser
 RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}openssl>=0:${PORTSDIR}/security/py-openssl \
 		${PYTHON_PKGNAMEPREFIX}sqlite3>=0:${PORTSDIR}/databases/py-sqlite3 \
 		${PYTHON_PKGNAMEPREFIX}asn1>=0:${PORTSDIR}/devel/py-asn1
 
-GNU_CONFIGURE=	yes
-INSTALLS_ICONS=	yes
+USE_BZIP2=	yes
 USES=		dos2unix gmake pathfix
-USE_BZIP2=	yes
 DOS2UNIX_FILES=	src/plugins/plugins_i18n.py
 USE_GNOME=	pygtk2
+USE_XORG=	x11 xext xscrnsaver
 USE_PYTHON=	2
-USE_XORG=	x11 xext xscrnsaver
+GNU_CONFIGURE=	yes
+CONFIGURE_ENV+= PYTHON=${PYTHON_CMD}
+INSTALLS_ICONS=	yes
 
-CONFIGURE_ENV+=	PYTHON=${PYTHON_CMD}
 LDFLAGS+=	-L${LOCALBASE}/lib
 CFLAGS+=	-I${LOCALBASE}/include
 
-OPTIONS_DEFINE=	AVAHI CRYPTO DBUS DOCS GUPNP KEYRING NLS SPELL
-OPTIONS_DEFAULT=DBUS NLS SPELL
+OPTIONS_DEFINE=	AVAHI  DBUS DOCS E2EE GUPNP KEYRING NLS SPELL
+OPTIONS_DEFAULT=DBUS SPELL
 OPTIONS_SUB=	yes
-CRYPTO_DESC=	End to end encryption support
+
+E2EE_DESC=	End to End message Encryption suport
 KEYRING_DESC=	Gnome Keyring support
+GUPNP_DESC=	UPnP IGD protocol support
 SPELL_DESC=	Spell checking support
 
 AVAHI_RUN_DEPENDS=	avahi-discover:${PORTSDIR}/net/py-avahi
-CRYPTO_RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}pycrypto>0:${PORTSDIR}/security/py-pycrypto
+E2EE_RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}pycrypto>0:${PORTSDIR}/security/py-pycrypto
 DBUS_RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}dbus>=0.82.2:${PORTSDIR}/devel/py-dbus
 GUPNP_RUN_DEPENDS=	${PYTHON_SITELIBDIR}/gupnp/__init__.py:${PORTSDIR}/net/gupnp-igd
 KEYRING_USE=		GNOME=pygnomedesktop
+NLS_CONFIGURE_ENABLE=	nls
 NLS_USES=		gettext
 SPELL_LIB_DEPENDS=	libgtkspell.so:${PORTSDIR}/textproc/gtkspell
 
 PORTDOCS=	AUTHORS ChangeLog README.html
 
+SUB_FILES=	pkg-message
+
 .include <bsd.port.options.mk>
 
+post-patch:
+	@${REINPLACE_CMD} -e 's|$$(pkgdatadir)/icons|${PREFIX}/share/icons|; \
+		s|$$(datadir)/icons|${PREFIX}/share/icons|' \
+		${WRKSRC}/icons/Makefile.in
+
+.if ! ${PORT_OPTIONS:ME2EE}
+	@${REINPLACE_CMD} -e 's|PYCRYPTO = True|PYCRYPTO = False|' \
+		${WRKSRC}/src/common/gajim.py
+.endif
+
 .if ${OPSYS} == FreeBSD && ${OSVERSION} >= 1000000
-EXTRA_PATCHES+=	${FILESDIR}/extra-patch-src-common_resolver.py
+EXTRA_PATCHES+=	${FILESDIR}/extra-patch-src__common__resolver.py
 .endif
 
+do-build:
+	@${PYTHON_CMD} -m compileall ${WRKSRC}/src
+	@${PYTHON_CMD} -O -m compileall ${WRKSRC}/src
+
 post-install:
 	@${MKDIR} ${STAGEDIR}${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${STAGEDIR}${DOCSDIR}
 
-pre-build:
-	${PYTHON_CMD} ${PYTHON_LIBDIR}/compileall.py ${WRKSRC}/src
-	${PYTHON_CMD} -O ${PYTHON_LIBDIR}/compileall.py ${WRKSRC}/src
-
 .include <bsd.port.mk>
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 345227)
+++ pkg-plist	(working copy)
@@ -1,6 +1,9 @@
 bin/gajim
 bin/gajim-history-manager
 bin/gajim-remote
+man/man1/gajim-history-manager.1.gz
+man/man1/gajim-remote.1.gz
+man/man1/gajim.1.gz
 share/applications/gajim.desktop
 %%DATADIR%%/data/activities/default/doing_chores/buying_groceries.png
 %%DATADIR%%/data/activities/default/doing_chores/category.png
@@ -821,61 +824,6 @@
 %%DATADIR%%/data/sounds/message1.wav
 %%DATADIR%%/data/sounds/message2.wav
 %%DATADIR%%/data/sounds/sent.wav
-%%DATADIR%%/icons/hicolor/128x128/apps/gajim-about.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-cam_active.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-cam_inactive.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-earth.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-kbd_input.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-mic_active.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-mic_inactive.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-muc_separator.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-receipt_missing.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-security_high.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-security_low.png
-%%DATADIR%%/icons/hicolor/16x16/actions/gajim-upload.png
-%%DATADIR%%/icons/hicolor/16x16/status/audio-mic-volume-high.png
-%%DATADIR%%/icons/hicolor/16x16/status/audio-mic-volume-low.png
-%%DATADIR%%/icons/hicolor/16x16/status/audio-mic-volume-medium.png
-%%DATADIR%%/icons/hicolor/16x16/status/audio-mic-volume-muted.png
-%%DATADIR%%/icons/hicolor/32x32/actions/gajim-security_high.png
-%%DATADIR%%/icons/hicolor/32x32/actions/gajim-security_low.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-aim.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-bytestreams.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-conference.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-disc.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-error.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-facebook.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-gadu-gadu.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-http-ws.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-icq.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-irc.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-jabber.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-jud.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-mail.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-mrim.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-msn.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-pubsub.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-rss.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-sip.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-sms.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-tv.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-weather.png
-%%DATADIR%%/icons/hicolor/32x32/categories/gajim-agent-yahoo.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-chat_msg_recv.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-connection_lost.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-ft_done.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-ft_error.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-ft_request.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-ft_stopped.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-gc_invitation.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-new_email_recv.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-priv_msg_recv.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-single_msg_recv.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-subscription_request.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-unsubscribed.png
-%%DATADIR%%/icons/hicolor/48x48/actions/gajim-warning.png
-%%DATADIR%%/icons/hicolor/64x64/apps/gajim.png
-%%DATADIR%%/icons/hicolor/scalable/apps/gajim.svg
 %%DATADIR%%/plugins/plugin_installer/__init__.py
 %%DATADIR%%/plugins/plugin_installer/config_dialog.ui
 %%DATADIR%%/plugins/plugin_installer/manifest.ini
@@ -1304,11 +1252,61 @@
 %%DATADIR%%/src/vcard.py
 %%DATADIR%%/src/vcard.pyc
 %%DATADIR%%/src/vcard.pyo
+share/icons/hicolor/128x128/apps/gajim-about.png
+share/icons/hicolor/16x16/actions/gajim-cam_active.png
+share/icons/hicolor/16x16/actions/gajim-cam_inactive.png
+share/icons/hicolor/16x16/actions/gajim-earth.png
+share/icons/hicolor/16x16/actions/gajim-kbd_input.png
+share/icons/hicolor/16x16/actions/gajim-mic_active.png
+share/icons/hicolor/16x16/actions/gajim-mic_inactive.png
+share/icons/hicolor/16x16/actions/gajim-muc_separator.png
+share/icons/hicolor/16x16/actions/gajim-receipt_missing.png
+share/icons/hicolor/16x16/actions/gajim-security_high.png
+share/icons/hicolor/16x16/actions/gajim-security_low.png
+share/icons/hicolor/16x16/actions/gajim-upload.png
+share/icons/hicolor/16x16/status/audio-mic-volume-high.png
+share/icons/hicolor/16x16/status/audio-mic-volume-low.png
+share/icons/hicolor/16x16/status/audio-mic-volume-medium.png
+share/icons/hicolor/16x16/status/audio-mic-volume-muted.png
+share/icons/hicolor/32x32/actions/gajim-security_high.png
+share/icons/hicolor/32x32/actions/gajim-security_low.png
+share/icons/hicolor/32x32/categories/gajim-agent-aim.png
+share/icons/hicolor/32x32/categories/gajim-agent-bytestreams.png
+share/icons/hicolor/32x32/categories/gajim-agent-conference.png
+share/icons/hicolor/32x32/categories/gajim-agent-disc.png
+share/icons/hicolor/32x32/categories/gajim-agent-error.png
+share/icons/hicolor/32x32/categories/gajim-agent-facebook.png
+share/icons/hicolor/32x32/categories/gajim-agent-gadu-gadu.png
+share/icons/hicolor/32x32/categories/gajim-agent-http-ws.png
+share/icons/hicolor/32x32/categories/gajim-agent-icq.png
+share/icons/hicolor/32x32/categories/gajim-agent-irc.png
+share/icons/hicolor/32x32/categories/gajim-agent-jabber.png
+share/icons/hicolor/32x32/categories/gajim-agent-jud.png
+share/icons/hicolor/32x32/categories/gajim-agent-mail.png
+share/icons/hicolor/32x32/categories/gajim-agent-mrim.png
+share/icons/hicolor/32x32/categories/gajim-agent-msn.png
+share/icons/hicolor/32x32/categories/gajim-agent-pubsub.png
+share/icons/hicolor/32x32/categories/gajim-agent-rss.png
+share/icons/hicolor/32x32/categories/gajim-agent-sip.png
+share/icons/hicolor/32x32/categories/gajim-agent-sms.png
+share/icons/hicolor/32x32/categories/gajim-agent-tv.png
+share/icons/hicolor/32x32/categories/gajim-agent-weather.png
+share/icons/hicolor/32x32/categories/gajim-agent-yahoo.png
+share/icons/hicolor/48x48/actions/gajim-chat_msg_recv.png
+share/icons/hicolor/48x48/actions/gajim-connection_lost.png
+share/icons/hicolor/48x48/actions/gajim-ft_done.png
+share/icons/hicolor/48x48/actions/gajim-ft_error.png
+share/icons/hicolor/48x48/actions/gajim-ft_request.png
+share/icons/hicolor/48x48/actions/gajim-ft_stopped.png
+share/icons/hicolor/48x48/actions/gajim-gc_invitation.png
+share/icons/hicolor/48x48/actions/gajim-new_email_recv.png
+share/icons/hicolor/48x48/actions/gajim-priv_msg_recv.png
+share/icons/hicolor/48x48/actions/gajim-single_msg_recv.png
+share/icons/hicolor/48x48/actions/gajim-subscription_request.png
+share/icons/hicolor/48x48/actions/gajim-unsubscribed.png
+share/icons/hicolor/48x48/actions/gajim-warning.png
 share/icons/hicolor/64x64/apps/gajim.png
 share/icons/hicolor/scalable/apps/gajim.svg
-man/man1/gajim.1.gz
-man/man1/gajim-remote.1.gz
-man/man1/gajim-history-manager.1.gz
 %%NLS%%share/locale/be/LC_MESSAGES/gajim.mo
 %%NLS%%share/locale/be@latin/LC_MESSAGES/gajim.mo
 %%NLS%%share/locale/bg/LC_MESSAGES/gajim.mo
@@ -1359,22 +1357,6 @@
 @dirrm %%DATADIR%%/src
 @dirrm %%DATADIR%%/plugins/plugin_installer
 @dirrm %%DATADIR%%/plugins
-@dirrm %%DATADIR%%/icons/hicolor/scalable/apps
-@dirrm %%DATADIR%%/icons/hicolor/scalable
-@dirrm %%DATADIR%%/icons/hicolor/64x64/apps
-@dirrm %%DATADIR%%/icons/hicolor/64x64
-@dirrm %%DATADIR%%/icons/hicolor/48x48/actions
-@dirrm %%DATADIR%%/icons/hicolor/48x48
-@dirrm %%DATADIR%%/icons/hicolor/32x32/categories
-@dirrm %%DATADIR%%/icons/hicolor/32x32/actions
-@dirrm %%DATADIR%%/icons/hicolor/32x32
-@dirrm %%DATADIR%%/icons/hicolor/16x16/status
-@dirrm %%DATADIR%%/icons/hicolor/16x16/actions
-@dirrm %%DATADIR%%/icons/hicolor/16x16
-@dirrm %%DATADIR%%/icons/hicolor/128x128/apps
-@dirrm %%DATADIR%%/icons/hicolor/128x128
-@dirrm %%DATADIR%%/icons/hicolor
-@dirrm %%DATADIR%%/icons
 @dirrm %%DATADIR%%/data/sounds
 @dirrm %%DATADIR%%/data/pixmaps
 @dirrm %%DATADIR%%/data/other
