Index: Makefile
===================================================================
--- Makefile	(revision 329029)
+++ Makefile	(working copy)
@@ -6,7 +6,7 @@
 CATEGORIES=	sysutils
 MASTER_SITES=	SF/lxde/LXInput%20%28Kbd%20and%20amp_%20mouse%20config%29/LXInput%200.3.2/
 
-MAINTAINER=	nemysis@gmx.ch
+MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Keyboard and mouse config tool from the LXDE project
 
 USES=		iconv gmake pkgconfig
@@ -15,19 +15,26 @@
 USE_LDCONFIG=	yes
 CONFIGURE_ARGS=	--prefix=${PREFIX}
 
-MAN1=		lxinput.1
+CPPFLAGS+=	-I${LOCALBASE}/include
+LDFLAGS+=	-L${LOCALBASE}/lib
 
-NO_STAGE=	yes
+PORTDOCS=	AUTHORS README
+
+OPTIONS_DEFINE=	DOCS NLS
+OPTIONS_SUB=  yes
+
+NLS_USES=		gettext
+
 .include <bsd.port.options.mk>
 
-.if ${PORT_OPTIONS:MNLS}
-USES+=		gettext
-PLIST_SUB+=	NLS=""
-CPPFLAGS+=	-I${LOCALBASE}/include
-LDFLAGS+=	-L${LOCALBASE}/lib
-.else
-CONFIGURE_ARGS+=--disable-nls
-PLIST_SUB+=	NLS="@comment "
+post-patch:
+.if ! ${PORT_OPTIONS:MNLS}
+	@${REINPLACE_CMD} -e 's|po man|man|' \
+		${WRKSRC}/Makefile.in
 .endif
 
+post-install:
+	@${MKDIR} ${STAGEDIR}${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${STAGEDIR}${DOCSDIR}
+
 .include <bsd.port.mk>
Index: pkg-descr
===================================================================
--- pkg-descr	(revision 329029)
+++ pkg-descr	(working copy)
@@ -1,3 +1,3 @@
 LXInput is a small program used to configure keyboard and mouse for LXDE.
 
-WWW: http://lxde.sourceforge.net/
+WWW: http://lxde.org/
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 329029)
+++ pkg-plist	(working copy)
@@ -1,4 +1,5 @@
 bin/lxinput
+man/man1/lxinput.1.gz
 share/applications/lxinput.desktop
 %%NLS%%share/locale/af/LC_MESSAGES/lxinput.mo
 %%NLS%%share/locale/ar/LC_MESSAGES/lxinput.mo
@@ -68,32 +69,12 @@
 @dirrm %%DATADIR%%
 %%NLS%%@dirrmtry share/locale/ur_PK/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/ur_PK
-%%NLS%%@dirrmtry share/locale/ur/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/ur
-%%NLS%%@dirrmtry share/locale/ug/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/ug
 %%NLS%%@dirrmtry share/locale/tt_RU/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/tt_RU
-%%NLS%%@dirrmtry share/locale/te/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/te
-%%NLS%%@dirrmtry share/locale/sr@latin/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/sr@latin
-%%NLS%%@dirrmtry share/locale/si/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/si
-%%NLS%%@dirrmtry share/locale/ps/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/ps
 %%NLS%%@dirrmtry share/locale/lg/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/lg
-%%NLS%%@dirrmtry share/locale/kk/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/kk
 %%NLS%%@dirrmtry share/locale/frp/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/frp
-%%NLS%%@dirrmtry share/locale/fo/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/fo
 %%NLS%%@dirrmtry share/locale/es_VE/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/es_VE
-%%NLS%%@dirrmtry share/locale/bn_IN/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/bn_IN
-%%NLS%%@dirrmtry share/locale/ast/LC_MESSAGES
-%%NLS%%@dirrmtry share/locale/ast
 @dirrmtry share/applications
