===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/batik/Makefile ./Makefile
--- /usr/ports/graphics/batik/Makefile	2011-07-21 07:02:37.000000000 +0200
+++ ./Makefile	2012-11-04 11:01:05.000000000 +0100
@@ -7,14 +7,15 @@
 
 PORTNAME=	batik
 PORTVERSION=	1.7
-PORTREVISION=	2
+PORTREVISION=	3
 CATEGORIES=	graphics java
-MASTER_SITES=	${MASTER_SITE_APACHE}
-MASTER_SITE_SUBDIR=	xmlgraphics/${PORTNAME}
+MASTER_SITES=	APACHE/xmlgraphics/${PORTNAME}
 DISTNAME=	${PORTNAME}-src-${PORTVERSION}
 
-MAINTAINER=	ports@FreeBSD.org
-COMMENT=	SVG Toolkit
+MAINTAINER=	nemysis@gmx.ch
+COMMENT=	Java based toolkit SVG Toolkit
+
+LICENSE=	AL2
 
 USE_ZIP=	yes
 USE_JAVA=	1.5+
@@ -27,7 +28,17 @@
 INSTALL_WRKSRC=	${WRKSRC}/${PORTNAME}-${PORTVERSION}
 DATADIR=	${JAVASHAREDIR}/${PORTNAME}
 
+PORTDOCS=	CHANGES NOTICE README
+
+.include <bsd.port.options.mk>
+
 do-install:
-	cd ${INSTALL_WRKSRC} && ${COPYTREE_SHARE} . ${DATADIR}
+	cd ${INSTALL_WRKSRC} && ${COPYTREE_SHARE} \* ${DATADIR}
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/graphics/batik/pkg-descr ./pkg-descr
--- /usr/ports/graphics/batik/pkg-descr	2010-11-30 00:10:50.000000000 +0100
+++ ./pkg-descr	2012-11-04 10:54:26.000000000 +0100
@@ -1,5 +1,5 @@
-Batik is a Java technology based toolkit for applications or applets
-that want to use images in the Scalable Vector Graphics (SVG) format
-for various purposes, such as viewing, generation or manipulation.
+Batik is a Java based toolkit for applications which handle images in the
+Scalable Vector Graphics (SVG) format for various purposes, such as viewing,
+generation or manipulation.
 
 WWW: http://xml.apache.org/batik/
===> Done
