===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/textproc/py-docutils/Makefile ./Makefile
--- /usr/ports/textproc/py-docutils/Makefile	2013-01-14 17:55:54.000000000 +0100
+++ ./Makefile	2013-04-04 23:02:13.000000000 +0200
@@ -21,7 +21,7 @@
 
 post-install:
 .for script in ${BIN_SCRIPTS}
-	@cd ${PREFIX}/bin && ${LN} -s ${script} ${script:C/\.py//}
+	@cd ${PREFIX}/bin && ${LN} -sf ${script} ${script:C/\.py//}
 .endfor
 
 .include <bsd.port.mk>
===> Done
