Index: Makefile
===================================================================
--- Makefile	(revision 329704)
+++ Makefile	(working copy)
@@ -5,7 +5,6 @@
 PORTVERSION=	3.0.3
 CATEGORIES=	graphics
 MASTER_SITES=	SF
-PKGNAMESUFFIX=	-devel
 
 MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Portable framework for OpenGL development
@@ -17,6 +16,7 @@
 USE_GL=		glu
 USE_LDCONFIG=	yes
 USE_GCC=	4.6+
+CMAKE_ARGS+=	-DBUILD_SHARED_LIBS:BOOL=ON
 
 CPPFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
@@ -23,8 +23,6 @@
 
 PORTDOCS=	*
 
-DOCSDIR=	${PREFIX}/share/doc/${PORTNAME}${PKGNAMESUFFIX}
-
 DOCSRCDIR1=	${WRKSRC}
 DOC_FILES1=	README.md
 
@@ -38,8 +36,6 @@
 
 PORTEXAMPLES=	*
 
-EXAMPLESDIR=	${PREFIX}/share/examples/${PORTNAME}${PKGNAMESUFFIX}
-
 EXAMPLESSRCDIR1=	${WRKSRC}/examples
 EXAMPLES_FILES1=	boing gears heightmap simple splitview wave
 
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 329704)
+++ pkg-plist	(working copy)
@@ -4,7 +4,8 @@
 lib/cmake/glfw/glfwConfigVersion.cmake
 lib/cmake/glfw/glfwTargets-release.cmake
 lib/cmake/glfw/glfwTargets.cmake
-lib/libglfw3.a
+lib/libglfw.so.3
+lib/libglfw.so.3.0
 libdata/pkgconfig/glfw3.pc
 @dirrm lib/cmake/glfw
 @dirrmtry lib/cmake
