===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/multimedia/ggrab/Makefile ./Makefile
--- /usr/ports/multimedia/ggrab/Makefile	2013-01-14 17:54:54.000000000 +0100
+++ ./Makefile	2013-03-30 21:35:22.000000000 +0100
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	ggrab
-# Date created:        			Fri Jan  6 20:53:15 CET 2006
-# Whom:    				Markus Brueffer <markus@FreeBSD.org>
-#
+# Created by: Markus Brueffer <markus@FreeBSD.org>
 # $FreeBSD: head/multimedia/ggrab/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	ggrab
 PORTVERSION=	0.22a
@@ -20,6 +16,8 @@
 PLIST_FILES=	bin/ggrab bin/sserver
 PORTDOCS=	README
 
+.include <bsd.port.options.mk>
+
 post-patch:
 	@${REINPLACE_CMD} -e 's/-lpthread/${PTHREAD_LIBS}/g' \
 		${WRKSRC}/Makefile
@@ -27,9 +25,10 @@
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/ggrab ${PREFIX}/bin
 	${INSTALL_PROGRAM} ${WRKSRC}/sserver ${PREFIX}/bin
-.if !defined(NOPORTDOCS)
-	${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/README ${DOCSDIR}
+
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
===> Done
