===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/multimedia/mkxvcd/Makefile ./Makefile
--- /usr/ports/multimedia/mkxvcd/Makefile	2013-01-14 17:54:56.000000000 +0100
+++ ./Makefile	2013-03-30 22:01:12.000000000 +0100
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	mkxvcd
-# Date created:				Fri Mar  5 04:25:02 EST 2004
-# Whom:					michael johnson <ahze@ahze.net>
-#
+# Created by: michael johnson <ahze@ahze.net>
 # $FreeBSD: head/multimedia/mkxvcd/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	mkxvcd
 PORTVERSION=	2.0.2
@@ -14,6 +10,8 @@
 MAINTAINER=	ports@FreeBSD.org
 COMMENT=	Makes high quality X/K/S/VCD movies from many formats
 
+LICENSE=	GPLv2
+
 RUN_DEPENDS=	mpeg2enc:${PORTSDIR}/multimedia/mjpegtools \
 		bash:${PORTSDIR}/shells/bash \
 		mplayer:${PORTSDIR}/multimedia/mplayer \
@@ -22,21 +20,20 @@
 NO_BUILD=	yes
 PLIST_FILES=	bin/mkxvcd
 
-PORTDOCS=	AUTHORS Changelog INSTALL README TODO
+PORTDOCS=	AUTHORS Changelog README TODO
 
-.include <bsd.port.pre.mk>
+.include <bsd.port.options.mk>
 
 post-patch:
-	@${REINPLACE_CMD} -e 's|#!/bin/bash|#!${LOCALBASE}/bin/bash|' \
+	@${REINPLACE_CMD} -e 's|#!/bin/bash|#!${PREFIX}/bin/bash|' \
 		${WRKSRC}/mkxvcd.sh
 
 do-install:
-	@${INSTALL_SCRIPT} ${WRKSRC}/mkxvcd.sh ${PREFIX}/bin/mkxvcd
-.if !defined(NOPORTDOCS)
+	${INSTALL_SCRIPT} ${WRKSRC}/mkxvcd.sh ${PREFIX}/bin/mkxvcd
+
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-.for docfile in ${PORTDOCS}
-	@${INSTALL_DATA} ${WRKSRC}/${docfile} ${DOCSDIR}
-.endfor
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
-.include <bsd.port.post.mk>
+.include <bsd.port.mk>
===> Done
