===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/multimedia/mpeg2play/Makefile ./Makefile
--- /usr/ports/multimedia/mpeg2play/Makefile	2013-01-14 17:54:53.000000000 +0100
+++ ./Makefile	2013-03-29 20:57:27.000000000 +0100
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	mpeg2play
-# Date created:		June 21, 1996
-# Whom:			"Philippe Charnier" <charnier@lirmm.fr>
-#
+# Created by: Philippe Charnier <charnier@lirmm.fr>
 # $FreeBSD: head/multimedia/mpeg2play/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	mpeg2play
 PORTVERSION=	1.1b
@@ -12,19 +8,33 @@
 MASTER_SITES=	ftp://ftp.funet.fi/pub/graphics/packages/mpeg/mpeg2play/
 
 MAINTAINER=	ports@FreeBSD.org
-COMMENT=	A program to play mpeg-2 movies on X displays
+COMMENT=	Program to play mpeg-2 movies on X displays
 
 USE_IMAKE=	yes
 USE_XORG=	x11 xext xp xt
 USE_MOTIF=	yes
 ALL_TARGET=	mpeg2play
 
+PLIST_FILES=	bin/${PORTNAME} \
+		lib/X11/app-defaults/Mpeg2play
+
+PORTDOCS=	README
+
+.include <bsd.port.options.mk>
+
+post-patch:
+	@${REINPLACE_CMD} -e 's|-lXm -lXt -lX11 -lm|$${MOTIFLIB} -lXt -lX11 -lm|' \
+		${WRKSRC}/Imakefile
+	@${REINPLACE_CMD} -e 's|tell(base.infile);|lseek(base.infile, 0L, SEEK_CUR);|' \
+		${WRKSRC}/mpeg2play.c
+
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/mpeg2play ${PREFIX}/bin
 	${INSTALL_DATA} ${WRKSRC}/Mpeg2play ${PREFIX}/lib/X11/app-defaults
-.if !defined(NOPORTDOCS)
-	${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/README ${DOCSDIR}/README
+
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/multimedia/mpeg2play/files/patch-aa ./files/patch-aa
--- /usr/ports/multimedia/mpeg2play/files/patch-aa	2013-01-14 17:54:53.000000000 +0100
+++ ./files/patch-aa	1970-01-01 01:00:00.000000000 +0100
@@ -1,11 +0,0 @@
---- mpeg2play.c	1997/08/11 18:58:37	1.1
-+++ mpeg2play.c	1997/08/11 19:07:22
-@@ -335,7 +335,7 @@
-   XmStringGetLtoR(((XmFileSelectionBoxCallbackStruct *)call_data)->value,
- 		  XmSTRING_DEFAULT_CHARSET, &filename);
- 
--  position = tell(base.infile);
-+  position = lseek(base.infile, 0L, SEEK_CUR);
-   fprintf(stderr, "pos: %d\n", position);
-   fprintf(stderr, "output: %s\n", filename);
- 
diff -ruN --exclude=CVS /usr/ports/multimedia/mpeg2play/files/patch-ab ./files/patch-ab
--- /usr/ports/multimedia/mpeg2play/files/patch-ab	2013-01-14 17:54:53.000000000 +0100
+++ ./files/patch-ab	1970-01-01 01:00:00.000000000 +0100
@@ -1,11 +0,0 @@
---- Imakefile.orig	Fri Oct 28 12:39:48 1994
-+++ Imakefile	Sat Nov 14 16:29:42 1998
-@@ -15,7 +15,7 @@
- 
- EXTRA_DEFINES = $(USE_SHMEM) $(USE_TIME)
- EXTRA_INCLUDES = -I. -Ibitmaps
--LOCAL_LIBRARIES = $(SHM_LIB) -lXm -lXt -lX11 -lm 
-+LOCAL_LIBRARIES = $(SHM_LIB) ${MOTIFLIB} -lXt -lX11 -lm 
- 
- PROGRAM = mpeg2play
- SRCS = mpeg2play.c getpic.c motion.c getvlc.c gethdr.c getblk.c \
diff -ruN --exclude=CVS /usr/ports/multimedia/mpeg2play/pkg-plist ./pkg-plist
--- /usr/ports/multimedia/mpeg2play/pkg-plist	2013-01-14 17:54:53.000000000 +0100
+++ ./pkg-plist	1970-01-01 01:00:00.000000000 +0100
@@ -1,4 +0,0 @@
-bin/mpeg2play
-lib/X11/app-defaults/Mpeg2play
-%%PORTDOCS%%%%DOCSDIR%%/README
-%%PORTDOCS%%@dirrm %%DOCSDIR%%
===> Done
