===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/multimedia/quark/Makefile ./Makefile
--- /usr/ports/multimedia/quark/Makefile	2013-03-29 00:26:21.000000000 +0100
+++ ./Makefile	2013-03-30 20:42:43.000000000 +0100
@@ -8,44 +8,52 @@
 MASTER_SITES=	http://hsgg.github.com/quark/
 
 MAINTAINER=	ports@FreeBSD.org
-COMMENT=	An audio player, for geeks, by geeks
-
-LIB_DEPENDS=	xine.2:${PORTSDIR}/multimedia/libxine
+COMMENT=	Audio player, for geeks, by geeks
 
 LICENSE=	GPLv2
-LICENSE_FILE=	${WRKSRC}/COPYING
 
-USES=        pathfix
+LIB_DEPENDS=	xine:${PORTSDIR}/multimedia/libxine
+
+USES=		pathfix
 USE_GNOME=	gnomeprefix gnomevfs2
-USE_GETTEXT=	yes
-USE_GMAKE=	yes
 GNU_CONFIGURE=	yes
+USE_GMAKE=	yes
 CPPFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
 
 MAKE_JOBS_SAFE=	yes
 
-PORTDOCS=	AUTHORS COPYING INSTALL README
+PORTDOCS=	AUTHORS README
 GCONF_SCHEMAS=	quark.schemas strange-quark.schemas
 
-.if defined(WITHOUT_NLS)
-CONFIGURE_ARGS+=--disable-nls
-PLIST_SUB+=	NLS="@comment "
-.else
+OPTIONS_DEFINE=	NLS
+
+.include <bsd.port.options.mk>
+
+.if ${PORT_OPTIONS:MNLS}
+USE_GETTEXT=	yes
 PLIST_SUB+=	NLS=""
+.else
+CONFIGURE_ARGS+=	--disable-nls
+PLIST_SUB+=	NLS="@comment "
 .endif
 
 post-patch:
-	@${REINPLACE_CMD} -e \
-		's|CFLAGS="-g |CFLAGS="$$CFLAGS |g' ${WRKSRC}/configure
-	@${FIND} ${WRKSRC} -name Makefile.in | ${XARGS} ${REINPLACE_CMD} -e \
-		's|-D.*_DISABLE_DEPRECATED||g'
+	@${REINPLACE_CMD} -e 's|/bin/bash|/bin/sh|g' ${WRKSRC}/charm-quark/charm-quark
+	@${REINPLACE_CMD} -e 's|CFLAGS="-g |CFLAGS="$$CFLAGS |g' \
+		${WRKSRC}/configure
+	@${REINPLACE_CMD} \
+		-e 's|-D.*_DISABLE_DEPRECATED||g' \
+		${WRKSRC}/strange-quark/Makefile.in ${WRKSRC}/quark/Makefile.in
 
 post-install:
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-	(cd ${WRKSRC} && ${INSTALL_DATA} ${PORTDOCS} ${DOCSDIR})
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
+
+	@${ECHO_CMD}
 	@${CAT} ${PKGMESSAGE}
+	@${ECHO_CMD}
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/multimedia/quark/files/patch-charm-quark::charm-quark ./files/patch-charm-quark::charm-quark
--- /usr/ports/multimedia/quark/files/patch-charm-quark::charm-quark	2013-01-14 17:54:56.000000000 +0100
+++ ./files/patch-charm-quark::charm-quark	1970-01-01 01:00:00.000000000 +0100
@@ -1,8 +0,0 @@
---- charm-quark/charm-quark.orig	Mon Jul 21 02:39:44 2003
-+++ charm-quark/charm-quark	Mon Jul 21 02:41:04 2003
-@@ -1,4 +1,4 @@
--#! /bin/bash
-+#! /bin/sh
- 
- NAME="charm-quark"
- VERSION="1.0"
diff -ruN --exclude=CVS /usr/ports/multimedia/quark/pkg-descr ./pkg-descr
--- /usr/ports/multimedia/quark/pkg-descr	2013-01-14 17:54:56.000000000 +0100
+++ ./pkg-descr	2013-03-30 20:06:00.000000000 +0100
@@ -1,9 +1,11 @@
 Quark is an audio player, for geeks, by geeks. It runs in the background
 with access provided via a FIFO in the filesystem. It uses Xine-lib
-for playing music, and can therefore play any file format supported 
+for playing music, and can therefore play any file format supported
 by Xine. 
 
 Quark comes with a couple front-ends to make it more useful. 
 
-	- charm-quark - a CLI interface 
-	- strange-quark - a GTK System Tray interface 
+ - charm-quark - a CLI interface 
+ - strange-quark - a GTK System Tray interface 
+
+WWW: http://hsgg.github.com/quark/
diff -ruN --exclude=CVS /usr/ports/multimedia/quark/pkg-plist ./pkg-plist
--- /usr/ports/multimedia/quark/pkg-plist	2013-01-14 17:54:56.000000000 +0100
+++ ./pkg-plist	2013-03-30 20:38:19.000000000 +0100
@@ -2,7 +2,7 @@
 bin/quark
 bin/strange-quark
 share/applications/quark.desktop
+%%NLS%%share/locale/ko/LC_MESSAGES/quark.mo
 share/pixmaps/quark.png
 share/pixmaps/quark48.png
-%%NLS%%share/locale/ko/LC_MESSAGES/quark.mo
 @dirrmtry share/applications
===> Done
