===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/multimedia/tcmplex-panteltje/Makefile ./Makefile
--- /usr/ports/multimedia/tcmplex-panteltje/Makefile	2013-01-14 17:54:54.000000000 +0100
+++ ./Makefile	2013-03-29 16:43:51.000000000 +0100
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	tcmplex-panteltje
-# Date created:		2004-10-21
-# Whom:			Herve Quiroz <hq@FreeBSD.org>
-#
+# Created by: Herve Quiroz <hq@FreeBSD.org>
 # $FreeBSD: head/multimedia/tcmplex-panteltje/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	tcmplex-panteltje
 PORTVERSION=	0.4.7
@@ -21,16 +17,16 @@
 MAKE_ARGS=	CC="${CC}" CFLAGS="${CFLAGS}"
 ALL_TARGET=	a.out
 
-PORTDOCS=	CHANGES COPYRIGHT README
+PORTDOCS=	CHANGES README
 PLIST_FILES=	bin/${PORTNAME}
 
+.include <bsd.port.options.mk>
+
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-.for file in ${PORTDOCS}
-	${INSTALL_DATA} ${WRKSRC}/${file} ${DOCSDIR}
-.endfor
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
===> Done
