===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/multimedia/vcdgear/Makefile ./Makefile
--- /usr/ports/multimedia/vcdgear/Makefile	2013-01-14 17:54:56.000000000 +0100
+++ ./Makefile	2013-03-29 16:23:31.000000000 +0100
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	vcdgear
-# Date created:	 	06 January 1999
-# Whom:			Chris Piazza <cpiazza@FreeBSD.org>
-#
+# Created by: Chris Piazza <cpiazza@FreeBSD.org>
 # $FreeBSD: head/multimedia/vcdgear/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	vcdgear
 PORTVERSION=	1.6d
@@ -11,11 +7,16 @@
 MASTER_SITES=	http://www.vcdgear.com/files/
 
 MAINTAINER=	ports@FreeBSD.org
-COMMENT=	A tool to convert VCDs from cue/bin format to mpeg
+COMMENT=	Tool to convert VCDs from cue/bin format to mpeg
 
 ONLY_FOR_ARCHS=	alpha i386 amd64
 
+PLIST_FILES=	bin/${PORTNAME}
+
+PORTDOCS=	faq.txt manual.txt whatsnew.txt
+
 .include <bsd.port.pre.mk>
+.include <bsd.port.options.mk>
 
 .if ${ARCH} == i386 || ${ARCH} == amd64
 DISTNAME=	${PORTNAME}16d_i386_freebsd42
@@ -28,11 +29,9 @@
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/vcdgear16_static ${PREFIX}/bin/vcdgear
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-.for file in faq.txt manual.txt whatsnew.txt
-	${INSTALL_DATA} ${WRKSRC}/${file} ${DOCSDIR}
-.endfor
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.post.mk>
diff -ruN --exclude=CVS /usr/ports/multimedia/vcdgear/distinfo ./distinfo
--- /usr/ports/multimedia/vcdgear/distinfo	2013-01-14 17:54:56.000000000 +0100
+++ ./distinfo	2013-03-29 16:29:12.000000000 +0100
@@ -1,4 +1,2 @@
-SHA256 (vcdgear16d_alpha_freebsd41.tar.gz) = cedce8180bd7e4959aaa3b26d4e83d0804c156c94168a1ef7f0f7259f91159e8
-SIZE (vcdgear16d_alpha_freebsd41.tar.gz) = 121796
 SHA256 (vcdgear16d_i386_freebsd42.tar.gz) = c90cc63ff402ab6bcfc545d9b534bd3ce86954d8a5b2069554174073b73bf432
 SIZE (vcdgear16d_i386_freebsd42.tar.gz) = 98035
diff -ruN --exclude=CVS /usr/ports/multimedia/vcdgear/pkg-plist ./pkg-plist
--- /usr/ports/multimedia/vcdgear/pkg-plist	2013-01-14 17:54:56.000000000 +0100
+++ ./pkg-plist	1970-01-01 01:00:00.000000000 +0100
@@ -1,5 +0,0 @@
-bin/vcdgear
-%%PORTDOCS%%share/doc/vcdgear/faq.txt
-%%PORTDOCS%%share/doc/vcdgear/manual.txt
-%%PORTDOCS%%share/doc/vcdgear/whatsnew.txt
-%%PORTDOCS%%@dirrm share/doc/vcdgear
===> Done
