===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/boxer/Makefile ./Makefile
--- /usr/ports/graphics/boxer/Makefile	2013-01-24 11:58:04.000000000 +0100
+++ ./Makefile	2013-05-26 21:33:57.000000000 +0200
@@ -3,39 +3,38 @@
 
 PORTNAME=	boxer
 PORTVERSION=	0.3.8
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	graphics
 MASTER_SITES=	SF/boxc/${PORTNAME:C/^b/B/}%20-%20the%20Box%20GUI/${DISTNAME}/ \
-		http://boxc.sourceforge.net/_images/:icons
-PKGNAMEPREFIX=	${PYTHON_PKGNAMEPREFIX}
+		SF/nemysisfreebsdp/:icons
 DISTFILES=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX} \
-		cycloid.png:icons
+		${PORTNAME}.png:icons
 DIST_SUBDIR=	box
 EXTRACT_ONLY=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX}
 
 MAINTAINER=	nemysis@gmx.ch
 COMMENT=	GUI for Box the figure description language
 
-LICENSE=	GPLv3 # (or later)
+LICENSE=	GPLv3
 
 RUN_DEPENDS=	box:${PORTSDIR}/graphics/box
-EXTRACT_DEPENDS=pngtopnm:${PORTSDIR}/graphics/netpbm
 
 USE_GNOME=	pygtk2
-
-USE_PYTHON=	-2.7
+USE_PYTHON=	yes
 USE_PYDISTUTILS=yes
 
-DESKTOP_ENTRIES="Boxer" "${COMMENT}" \
-		"${PREFIX}/share/pixmaps/${PORTNAME}.png" \
-		"${PORTNAME}" "Graphics;VectorGraphics;" false
-
-post-extract:
-	@(cd ${WRKSRC} && \
-		${LOCALBASE}/bin/pngtopnm ${DISTDIR}/${DIST_SUBDIR}/cycloid.png | \
-		${LOCALBASE}/bin/pamscale .218 | ${LOCALBASE}/bin/pnmtopng > ${PORTNAME}.png)
+PORTDOCS=	ChangeLog
+
+DESKTOP_ENTRIES="Boxer" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Application;Graphics;" false
+
+.include <bsd.port.options.mk>
 
 post-install:
-	${INSTALL_DATA} ${WRKSRC}/${PORTNAME}.png ${PREFIX}/share/pixmaps
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/graphics/boxer/distinfo ./distinfo
--- /usr/ports/graphics/boxer/distinfo	2013-01-14 17:57:09.000000000 +0100
+++ ./distinfo	2013-03-25 00:30:31.000000000 +0100
@@ -1,4 +1,4 @@
 SHA256 (box/boxer-0.3.8.tar.gz) = 626a81f1533b31a861c60a2fbfadb86f58d9d8f273abc9ae3fb8372e9edb292c
 SIZE (box/boxer-0.3.8.tar.gz) = 229126
-SHA256 (box/cycloid.png) = f5ee2a66954ac106dbb1c9611a87cb2f645c03d6f82d2aab38bf62649a27fa2c
-SIZE (box/cycloid.png) = 8727
+SHA256 (box/boxer.png) = c0aa0ba1ff663d7de6f1a1228579e4697fec54f48751a7d1e854f9c6d1391e5d
+SIZE (box/boxer.png) = 1555
===> Done
