===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/cbrpager/Makefile ./Makefile
--- /usr/ports/graphics/cbrpager/Makefile	2013-04-26 16:36:25.000000000 +0200
+++ ./Makefile	2013-05-22 01:43:16.000000000 +0200
@@ -3,12 +3,12 @@
 
 PORTNAME=	cbrpager
 PORTVERSION=	0.9.22
-PORTREVISION=	4
+PORTREVISION=	5
 CATEGORIES=	graphics
 MASTER_SITES=	SF/${PORTNAME}/${PORTNAME}/${PORTNAME}-${PORTVERSION} \
-		http://jcoppens.com/soft/cbrpager/img/:icons
+		SF/nemysisfreebsdp/:icons
 DISTFILES=	${PORTNAME}-${PORTVERSION}${EXTRACT_SUFX} \
-		snap.jpeg:icons
+		${PORTNAME}.png:icons
 DIST_SUBDIR=	${PORTNAME}
 EXTRACT_ONLY=	${PORTNAME}-${PORTVERSION}${EXTRACT_SUFX}
 
@@ -17,8 +17,6 @@
 
 LICENSE=	GPLv3
 
-BUILD_DEPENDS=	jpegtopnm:${PORTSDIR}/graphics/netpbm
-
 GNU_CONFIGURE=	yes
 USES=		pkgconfig
 USE_GNOME=	libgnomeui
@@ -30,9 +28,12 @@
 PLIST_FILES=	bin/${PORTNAME} \
 		share/pixmaps/${PORTNAME}.png
 
-DESKTOP_ENTRIES="cbrPager" "Comic book viewer" \
-		"${PREFIX}/share/pixmaps/${PORTNAME}.png" \
-		"${PORTNAME}" "Graphics;Viewer;GNOME;" true
+PORTDOCS=	CONTRIBUTORS ChangeLog NEWS README TODO
+
+OPTIONS_DEFINE=	NLS
+
+DESKTOP_ENTRIES="cbrPager" "Comic book viewer" "${PORTNAME}" \
+		"${PORTNAME}" "Graphics;Viewer;GNOME;" false
 
 .include <bsd.port.options.mk>
 
@@ -45,17 +46,17 @@
 PLIST_SUB+=	NLS="@comment "
 .endif
 
-post-configure:
-.if !${PORT_OPTIONS:MNLS}
-	@${REINPLACE_CMD} 's|src po|src|' ${WRKSRC}/Makefile
+post-patch:
+.if ! ${PORT_OPTIONS:MNLS}
+	@${REINPLACE_CMD} 's|src po|src|' ${WRKSRC}/Makefile.in
 .endif
 
-post-build:
-	@(cd ${WRKSRC} && \
-		${LOCALBASE}/bin/jpegtopnm ${DISTDIR}/${DIST_SUBDIR}/snap.jpeg | \
-		${LOCALBASE}/bin/pamscale .118 | ${LOCALBASE}/bin/pnmtopng > ${PORTNAME}.png)
-
 post-install:
-	${INSTALL_DATA} ${WRKSRC}/${PORTNAME}.png ${PREFIX}/share/pixmaps
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/graphics/cbrpager/distinfo ./distinfo
--- /usr/ports/graphics/cbrpager/distinfo	2013-01-14 17:56:59.000000000 +0100
+++ ./distinfo	2013-03-25 01:07:32.000000000 +0100
@@ -1,4 +1,4 @@
 SHA256 (cbrpager/cbrpager-0.9.22.tar.gz) = 1852303374d7777f2a5f395fda97d78f970fa87c854ee048495a901922b7dd84
 SIZE (cbrpager/cbrpager-0.9.22.tar.gz) = 146216
-SHA256 (cbrpager/snap.jpeg) = 3cc4264676c69d84620d6c136cfa87270ed214468553a94a64ea8ea1191b5877
-SIZE (cbrpager/snap.jpeg) = 60792
+SHA256 (cbrpager/cbrpager.png) = 9c0660c34361f3afb5e2ceb3361bf52ba1abf6740c3598c3cbfdbe185c00754b
+SIZE (cbrpager/cbrpager.png) = 3809
===> Done
