===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/eos-movrec/Makefile ./Makefile
--- /usr/ports/graphics/eos-movrec/Makefile	2013-03-23 02:42:31.000000000 +0100
+++ ./Makefile	2013-05-21 20:26:48.000000000 +0200
@@ -26,8 +26,7 @@
 PLIST_FILES=	bin/eos_movrec \
 		share/pixmaps/${PORTNAME}.png
 
-DESKTOP_ENTRIES=	"eos_movrec" "EOS Camera Movie Recorder" \
-			"${PREFIX}/share/pixmaps/${PORTNAME}.png" \
+DESKTOP_ENTRIES=	"eos_movrec" "EOS Camera Movie Recorder" ${PORTNAME} \
 			"eos_movrec" "Graphics;Photography;Qt;" false
 
 post-patch:
===> Done
