===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/openexr_ctl/Makefile ./Makefile
--- /usr/ports/graphics/openexr_ctl/Makefile	2013-04-26 16:36:25.000000000 +0200
+++ ./Makefile	2013-05-26 23:46:45.000000000 +0200
@@ -14,7 +14,7 @@
 LIB_DEPENDS=	IlmImf:${PORTSDIR}/graphics/OpenEXR \
 		IlmCtl:${PORTSDIR}/graphics/ampasCTL
 
-USES=		pkgconfig
+USES=		pathfix pkgconfig
 USE_LDCONFIG=	yes
 GNU_CONFIGURE=	yes
 USE_AUTOTOOLS=	libtool
@@ -33,7 +33,7 @@
 
 post-install:
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/graphics/openexr_ctl/files/patch-Makefile.in ./files/patch-Makefile.in
--- /usr/ports/graphics/openexr_ctl/files/patch-Makefile.in	2013-01-14 17:57:07.000000000 +0100
+++ ./files/patch-Makefile.in	1970-01-01 01:00:00.000000000 +0100
@@ -1,11 +0,0 @@
---- Makefile.in.orig	2007-08-04 09:13:15.000000000 +0900
-+++ Makefile.in	2007-10-09 01:15:31.000000000 +0900
-@@ -209,7 +209,7 @@
- 	AUTHORS COPYING ChangeLog INSTALL LICENSE NEWS \
-         README README.CVS README.OSX README.win32 bootstrap
- 
--pkgconfigdir = $(libdir)/pkgconfig
-+pkgconfigdir = $(prefix)/libdata/pkgconfig
- pkgconfig_DATA = OpenEXR_CTL.pc
- all: all-recursive
- 
diff -ruN --exclude=CVS /usr/ports/graphics/openexr_ctl/pkg-descr ./pkg-descr
--- /usr/ports/graphics/openexr_ctl/pkg-descr	2013-01-17 13:26:12.000000000 +0100
+++ ./pkg-descr	2013-03-25 15:17:47.000000000 +0100
@@ -16,6 +16,6 @@
 header fields and most OpenEXR attributes are lost during file
 conversion.
 
-WWW:	http://www.openexr.com/
-WWW:	http://www.oscars.org/science-technology/council/projects/ctl.html
-WWW:	http://ampasctl.sourceforge.net/
+WWW: http://www.openexr.com/
+WWW: http://www.oscars.org/science-technology/council/projects/ctl.html
+WWW: http://ampasctl.sourceforge.net/
===> Done
