===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/graphics/viewnior/Makefile ./Makefile
--- /usr/ports/graphics/viewnior/Makefile	2013-04-26 16:36:27.000000000 +0200
+++ ./Makefile	2013-05-27 11:33:15.000000000 +0200
@@ -1,9 +1,9 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/graphics/viewnior/Makefile 316464 2013-04-24 18:10:30Z ak $
 
 PORTNAME=	viewnior
 PORTVERSION=	1.3
-PORTREVISION=	2
+PORTREVISION=	3
 CATEGORIES=	graphics
 MASTER_SITES=	http://cloud.github.com/downloads/xsisqox/Viewnior/
 
@@ -12,21 +12,19 @@
 
 LICENSE=	GPLv3
 
-BUILD_DEPENDS=	update-mime-database:${PORTSDIR}/misc/shared-mime-info
-RUN_DEPENDS=	update-mime-database:${PORTSDIR}/misc/shared-mime-info
+BUILD_DEPENDS=	p5-XML-Parser>=0:${PORTSDIR}/textproc/p5-XML-Parser
 
+USES=		pkgconfig
+USE_GNOME=	glib20 gtk20
 GNU_CONFIGURE=	yes
 USE_AUTOTOOLS=	autoconf
 USE_GMAKE=	yes
-USE_GNOME=	glib20 gtk20
 INSTALLS_ICONS=	yes
-
-MAN1=		viewnior.1
+MAN1=		${PORTNAME}.1
 
 PORTDOCS=	AUTHORS ChangeLog NEWS README TODO
 
 OPTIONS_DEFINE=	NLS
-OPTIONS_DEFAULT=	NLS
 
 .include <bsd.port.options.mk>
 
@@ -38,10 +36,19 @@
 PLIST_SUB+=	NLS="@comment "
 .endif
 
+post-patch:
+	@${REINPLACE_CMD} -e 's|$$(prefix)/$$(DATADIRNAME)/locale|$$(datarootdir)/locale|' \
+		${WRKSRC}/po/Makefile.in.in
+
+.if ! ${PORT_OPTIONS:MNLS}
+	@${REINPLACE_CMD} -e 's|install: install-data|install: |' ${WRKSRC}/po/Makefile.in.in
+.endif
+
 post-install:
 	${INSTALL_MAN} ${WRKSRC}/man/${MAN1} ${MAN1PREFIX}/man/man1
+
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/graphics/viewnior/files/patch-po_Makefile.in.in ./files/patch-po_Makefile.in.in
--- /usr/ports/graphics/viewnior/files/patch-po_Makefile.in.in	2013-01-14 17:57:07.000000000 +0100
+++ ./files/patch-po_Makefile.in.in	1970-01-01 01:00:00.000000000 +0100
@@ -1,11 +0,0 @@
---- po/Makefile.in.in.orig	2012-04-27 11:51:14.851154779 +0200
-+++ po/Makefile.in.in	2012-04-27 11:51:28.306154800 +0200
-@@ -34,7 +34,7 @@
- datarootdir = @datarootdir@
- libdir = @libdir@
- DATADIRNAME = @DATADIRNAME@
--itlocaledir = $(prefix)/$(DATADIRNAME)/locale
-+itlocaledir = $(datarootdir)/locale
- subdir = po
- install_sh = @install_sh@
- # Automake >= 1.8 provides @mkdir_p@.
diff -ruN --exclude=CVS /usr/ports/graphics/viewnior/pkg-plist ./pkg-plist
--- /usr/ports/graphics/viewnior/pkg-plist	2013-01-14 17:57:07.000000000 +0100
+++ ./pkg-plist	2013-05-27 10:56:55.000000000 +0200
@@ -1,4 +1,11 @@
 bin/viewnior
+share/applications/viewnior.desktop
+share/icons/hicolor/16x16/apps/viewnior.png
+share/icons/hicolor/22x22/apps/viewnior.png
+share/icons/hicolor/24x24/apps/viewnior.png
+share/icons/hicolor/32x32/apps/viewnior.png
+share/icons/hicolor/48x48/apps/viewnior.png
+share/icons/hicolor/scalable/apps/viewnior.svg
 %%NLS%%share/locale/bg/LC_MESSAGES/viewnior.mo
 %%NLS%%share/locale/cs/LC_MESSAGES/viewnior.mo
 %%NLS%%share/locale/da/LC_MESSAGES/viewnior.mo
@@ -25,13 +32,6 @@
 %%NLS%%share/locale/tr/LC_MESSAGES/viewnior.mo
 %%NLS%%share/locale/uk/LC_MESSAGES/viewnior.mo
 %%NLS%%share/locale/zh_CN/LC_MESSAGES/viewnior.mo
-share/applications/viewnior.desktop
-share/icons/hicolor/16x16/apps/viewnior.png
-share/icons/hicolor/22x22/apps/viewnior.png
-share/icons/hicolor/24x24/apps/viewnior.png
-share/icons/hicolor/32x32/apps/viewnior.png
-share/icons/hicolor/48x48/apps/viewnior.png
-share/icons/hicolor/scalable/apps/viewnior.svg
 %%DATADIR%%/pixmaps/object-flip-horizontal.png
 %%DATADIR%%/pixmaps/object-flip-vertical.png
 %%DATADIR%%/pixmaps/object-rotate-left.png
@@ -40,20 +40,6 @@
 %%DATADIR%%/vnr-preferences-dialog.ui
 @dirrm %%DATADIR%%/pixmaps
 @dirrm %%DATADIR%%
-@dirrmtry share/icons/hicolor/scalable/apps
-@dirrmtry share/icons/hicolor/scalable
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor/24x24/apps
-@dirrmtry share/icons/hicolor/24x24
-@dirrmtry share/icons/hicolor/22x22/apps
-@dirrmtry share/icons/hicolor/22x22
-@dirrmtry share/icons/hicolor/16x16/apps
-@dirrmtry share/icons/hicolor/16x16
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
-@dirrmtry share/applications
 %%NLS%%@dirrmtry share/locale/oc/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/oc
+@dirrmtry share/applications
===> Done
