===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/x11-wm/obmenu/Makefile ./Makefile
--- /usr/ports/x11-wm/obmenu/Makefile	2013-01-14 17:55:01.000000000 +0100
+++ ./Makefile	2013-02-08 00:56:00.000000000 +0100
@@ -2,13 +2,15 @@
 
 PORTNAME=	obmenu
 PORTVERSION=	1.0
-PORTREVISION=	5
+PORTREVISION=	6
 CATEGORIES=	x11-wm
 MASTER_SITES=	SF
 
-MAINTAINER=	ports@FreeBSD.org
+MAINTAINER=	nemysis@gmx.ch
 COMMENT=	Menu manager for Openbox supporting both static and pipe menus
 
+LICENSE=	GPLv2
+
 USE_GNOME=		pygtk2
 USE_PYTHON=		yes
 USE_PYDISTUTILS=	yes
@@ -16,12 +18,25 @@
 
 PORTDOCS=	README
 
+SUB_FILES=	pkg-install \
+		pkg-message
+
+DESKTOP_ENTRIES="Obmenu" "Menu manager for Openbox" \
+		"${PREFIX}/share/pixmaps/${PORTNAME}.png" \
+		"${PORTNAME}" "Settings;" false
+
+.include <bsd.port.options.mk>
+
 post-install:
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-.for file in ${PORTDOCS}
-	${INSTALL_DATA} ${WRKSRC}/${file} ${DOCSDIR}
-.endfor
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
+	${LN} -sf ${DATADIR}/mnu48.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
+
+	@${SH} ${PKGINSTALL} ${PKGNAME} POST-INSTALL
+	@${ECHO_CMD}
+	@${CAT} ${PKGMESSAGE}
+	@${ECHO_CMD}
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/x11-wm/obmenu/files/pkg-install.in ./files/pkg-install.in
--- /usr/ports/x11-wm/obmenu/files/pkg-install.in	1970-01-01 01:00:00.000000000 +0100
+++ ./files/pkg-install.in	2013-02-08 00:55:05.000000000 +0100
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+
+echo ""
+echo "Have your user menu.xml for Openbox?"
+echo ""
+
+ls -lh /home/*/.config/openbox | grep menu.xml
+echo ""
diff -ruN --exclude=CVS /usr/ports/x11-wm/obmenu/files/pkg-message.in ./files/pkg-message.in
--- /usr/ports/x11-wm/obmenu/files/pkg-message.in	1970-01-01 01:00:00.000000000 +0100
+++ ./files/pkg-message.in	2013-02-08 01:35:49.000000000 +0100
@@ -0,0 +1,15 @@
+===============================================================================
+
+Obmenu has been installed.
+
+If yours users don't have  ~/.config/openbox/menu.xml
+
+To can use Obmenu, please type as user:
+
+    cp -a %%LOCALBASE%%/etc/xdg/openbox/menu.xml ~/.config/openbox/menu.xml
+
+    chmod chmod 644 ~/.config/openbox/menu.xml
+
+Have fun!
+
+===============================================================================
diff -ruN --exclude=CVS /usr/ports/x11-wm/obmenu/pkg-plist ./pkg-plist
--- /usr/ports/x11-wm/obmenu/pkg-plist	2013-01-14 17:55:01.000000000 +0100
+++ ./pkg-plist	2013-02-08 00:17:26.000000000 +0100
@@ -9,6 +9,7 @@
 %%DATADIR%%/mnu16.png
 %%DATADIR%%/mnu48.png
 %%DATADIR%%/obmenu.glade
+share/pixmaps/obmenu.png
 @dirrm %%DATADIR%%
 @dirrmtry %%PYTHON_SITELIBDIR%%
 @dirrmtry %%PYTHON_LIBDIR%%
===> Done
