===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/audio/alure/Makefile ./Makefile
--- /usr/ports/audio/alure/Makefile	2013-03-23 02:42:31.000000000 +0100
+++ ./Makefile	2013-05-25 13:25:08.000000000 +0200
@@ -1,4 +1,4 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/audio/alure/Makefile 314960 2013-03-22 20:06:14Z makc $
 
 PORTNAME=	alure
@@ -13,13 +13,17 @@
 
 LIB_DEPENDS+=	sndfile:${PORTSDIR}/audio/libsndfile
 
+USES=		cmake
 USE_OPENAL=	soft
+CMAKE_VERBOSE=	yes
 USE_LDCONFIG=	yes
-USES=		cmake
 
 CFLAGS+=	-I${LOCALBASE}/include -L${LOCALBASE}/lib
 
-OPTIONS_DEFINE=		VORBIS FLAC MPG123 DUMB MODPLUG FLUIDSYNTH DOCS EXAMPLES
+PORTDOCS=	*
+PORTEXAMPLES=	*
+
+OPTIONS_DEFINE=		VORBIS FLAC MPG123 DUMB MODPLUG FLUIDSYNTH
 OPTIONS_DEFAULT=	VORBIS FLAC DUMB
 
 DUMB_DESC=		DUMB audio library decoding support
@@ -109,8 +113,8 @@
 
 post-install:
 .if ${PORT_OPTIONS:MEXAMPLES}
-	${MKDIR} ${EXAMPLESDIR}
-	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} \* ${EXAMPLESDIR})
+	@${MKDIR} ${EXAMPLESDIR}
+	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} . ${EXAMPLESDIR})
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/audio/alure/pkg-plist ./pkg-plist
--- /usr/ports/audio/alure/pkg-plist	2013-01-18 22:29:05.000000000 +0100
+++ ./pkg-plist	2013-03-22 05:45:55.000000000 +0100
@@ -8,39 +8,4 @@
 lib/libalure.so.1.2.0
 libdata/pkgconfig/alure-static.pc
 libdata/pkgconfig/alure.pc
-%%PORTDOCS%%%%DOCSDIR%%/html/files/alure-cpp.html
-%%PORTDOCS%%%%DOCSDIR%%/html/files/buffer-cpp.html
-%%PORTDOCS%%%%DOCSDIR%%/html/files/istream-cpp.html
-%%PORTDOCS%%%%DOCSDIR%%/html/files/stream-cpp.html
-%%PORTDOCS%%%%DOCSDIR%%/html/files/streamplay-cpp.html
-%%PORTDOCS%%%%DOCSDIR%%/html/index.html
-%%PORTDOCS%%%%DOCSDIR%%/html/index/Functions.html
-%%PORTDOCS%%%%DOCSDIR%%/html/index/General.html
-%%PORTDOCS%%%%DOCSDIR%%/html/javascript/main.js
-%%PORTDOCS%%%%DOCSDIR%%/html/javascript/searchdata.js
-%%PORTDOCS%%%%DOCSDIR%%/html/search/FunctionsA.html
-%%PORTDOCS%%%%DOCSDIR%%/html/search/GeneralA.html
-%%PORTDOCS%%%%DOCSDIR%%/html/search/GeneralF.html
-%%PORTDOCS%%%%DOCSDIR%%/html/search/GeneralM.html
-%%PORTDOCS%%%%DOCSDIR%%/html/search/GeneralS.html
-%%PORTDOCS%%%%DOCSDIR%%/html/search/NoResults.html
-%%PORTDOCS%%%%DOCSDIR%%/html/styles/1.css
-%%PORTDOCS%%%%DOCSDIR%%/html/styles/2.css
-%%PORTDOCS%%%%DOCSDIR%%/html/styles/main.css
-%%PORTDOCS%%%%DOCSDIR%%/naturaldocs/CustomStyle.css
-%%PORTDOCS%%%%DOCSDIR%%/naturaldocs/Languages.txt
-%%PORTDOCS%%%%DOCSDIR%%/naturaldocs/Menu.txt
-%%PORTDOCS%%%%DOCSDIR%%/naturaldocs/Topics.txt
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/alurecdplay.c
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/alureplay.c
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/alurestream.c
-%%PORTEXAMPLES%%@dirrm %%EXAMPLESDIR%%
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/naturaldocs
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/html/styles
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/html/search
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/html/javascript
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/html/index
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/html/files
-%%PORTDOCS%%@dirrm %%DOCSDIR%%/html
-%%PORTDOCS%%@dirrm %%DOCSDIR%%
 @dirrmtry include/AL
===> Done
