===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/audio/hexter/Makefile ./Makefile
--- /usr/ports/audio/hexter/Makefile	2013-01-14 17:54:21.000000000 +0100
+++ ./Makefile	2013-05-25 13:28:53.000000000 +0200
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	hexter
-# Date created:		2007-01-20
-# Whom:			trasz <trasz@pin.if.uz.zgora.pl>
-#
+# Created by: trasz <trasz@pin.if.uz.zgora.pl>
 # $FreeBSD: head/audio/hexter/Makefile 308445 2012-12-07 20:03:33Z beech $
-#
 
 PORTNAME=	hexter
 PORTVERSION=	1.0.2
@@ -20,9 +16,9 @@
 		jack:${PORTSDIR}/audio/jack
 BUILD_DEPENDS=	${LOCALBASE}/include/dssi.h:${PORTSDIR}/audio/dssi
 
+USE_GNOME=	gtk20
 GNU_CONFIGURE=	yes
 USE_GMAKE=	yes
-USE_GNOME=	gtk20
 
 PORTDOCS=	AUTHORS ChangeLog README TODO
 
@@ -30,7 +26,7 @@
 
 post-install:
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
