===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/astro/viking/Makefile ./Makefile
--- /usr/ports/astro/viking/Makefile	2013-04-29 12:53:36.000000000 +0200
+++ ./Makefile	2013-05-21 14:50:53.000000000 +0200
@@ -2,8 +2,7 @@
 # $FreeBSD: head/astro/viking/Makefile 316684 2013-04-27 18:31:47Z mva $
 
 PORTNAME=	viking
-PORTVERSION=	1.4
-PORTREVISION=	1
+PORTVERSION=	1.4.1
 CATEGORIES=	astro
 MASTER_SITES=	SF/${PORTNAME}/${PORTNAME}/${PORTVERSION}/
 
@@ -23,12 +22,12 @@
 		xcb:${PORTSDIR}/x11/libxcb
 RUN_DEPENDS=	gpsbabel:${PORTSDIR}/astro/gpsbabel
 
-USES=		iconv pkgconfig
-USE_GMAKE=	yes
 GNU_CONFIGURE=	yes
+USE_GMAKE=	yes
+USES=		iconv desktop-file-utils pkgconfig
 USE_XORG=	pixman x11 xau xcomposite xcursor xdamage xdmcp xext \
 		xfixes xi xinerama xrandr xrender
-USE_GNOME=	desktopfileutils gnomedocutils gtk20
+USE_GNOME=	gnomedocutils gtk20
 USE_PERL5_RUN=	yes
 INSTALLS_ICONS=	yes
 INSTALLS_OMF=	yes
@@ -39,11 +38,13 @@
 
 PORTDOCS=	AUTHORS ChangeLog NEWS README TODO
 
-OPTIONS_DEFINE=	GEOTAG GPSD
-OPTIONS_DEFAULT=	GEOTAG
+OPTIONS_DEFINE=	NLS GEOTAG GPSD
+
 GEOTAG_DESC=	Geotag support
 GPSD_DESC=	Realtime GPS tracking
 
+OPTIONS_DEFAULT=	GEOTAG
+
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MNLS}
@@ -68,6 +69,7 @@
 
 post-patch:
 	@${REINPLACE_CMD} -e '/^AM_CFLAGS/s| -g | |' ${WRKSRC}/src/Makefile.in
+	@${REINPLACE_CMD} -e 's|Science;|Education;Science;|' ${WRKSRC}/src/viking.desktop.in
 
 post-install:
 # Port not install in figures jpg and png, this is cleaner as patch
@@ -76,9 +78,8 @@
 	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps/
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
-	@-update-desktop-database -q
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/astro/viking/distinfo ./distinfo
--- /usr/ports/astro/viking/distinfo	2013-02-19 23:46:51.000000000 +0100
+++ ./distinfo	2013-05-21 14:30:49.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (viking-1.4.tar.gz) = 80aa4c7bf3be75441a435f6681831b8aeb08f7eb6f7a5a86872eec16cda67799
-SIZE (viking-1.4.tar.gz) = 1487470
+SHA256 (viking-1.4.1.tar.gz) = 20d91b048d2bfbb33498af4da5dc120a3a9232ed05a4a87a3ce6e341445ef640
+SIZE (viking-1.4.1.tar.gz) = 1490617
diff -ruN --exclude=CVS /usr/ports/astro/viking/pkg-plist ./pkg-plist
--- /usr/ports/astro/viking/pkg-plist	2013-02-19 23:46:51.000000000 +0100
+++ ./pkg-plist	2013-05-21 14:11:34.000000000 +0200
@@ -57,18 +57,9 @@
 %%DATADIR%%/maps.xml
 @dirrm %%DATADIR%%
 @dirrmtry share/omf/viking
-@dirrmtry share/omf
 %%NLS%%@dirrmtry share/locale/jv/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/jv
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrmtry share/gnome/help/viking/C/figures
 @dirrmtry share/gnome/help/viking/C
 @dirrmtry share/gnome/help/viking
-@dirrmtry share/gnome/help
-@dirrmtry share/gnome
 @dirrmtry share/applications
-@exec %%LOCALBASE%%/bin/update-desktop-database  -q || /usr/bin/true
-@unexec %%LOCALBASE%%/bin/update-desktop-database  -q || /usr/bin/true
===> Done
