===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/databases/pure-sql3/Makefile ./Makefile
--- /usr/ports/databases/pure-sql3/Makefile	2013-01-14 17:56:43.000000000 +0100
+++ ./Makefile	2013-04-06 02:53:01.000000000 +0200
@@ -1,25 +1,40 @@
-# New ports collection makefile for:	pure-sql3
-# Date created:		2011-03-18
-# Whom:			Zhihao Yuan <lichray@gmail.com>
-#
+# Created by: Zhihao Yuan <lichray@gmail.com>
 # $FreeBSD: head/databases/pure-sql3/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	pure-sql3
 PORTVERSION=	0.4
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	databases
+#MASTER_SITES=	https://cdn.bitbucket.org/purelang/pure-lang/downloads/
+MASTER_SITES=	https://pure-lang.googlecode.com/files/
+DIST_SUBDIR=	pure
 
 MAINTAINER=	lichray@gmail.com
 COMMENT=	Pure language binding to the SQLite3 library
 
+LICENSE=	BSD
+
 LIB_DEPENDS=	sqlite3:${PORTSDIR}/databases/sqlite3
 
-USE_PURE=	yes
+USES=		pure
 
 PLIST_FILES=	lib/pure/sql3.pure \
 		lib/pure/sql3util.so
 
-.include <bsd.port.pre.mk>
-.include "${PORTSDIR}/lang/pure/bsd.pure.mk"
-.include <bsd.port.post.mk>
+PORTDOCS=	README
+PORTEXAMPLES=	*
+
+.include <bsd.port.options.mk>
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+
+.if ${PORT_OPTIONS:MEXAMPLES}
+	@${MKDIR} ${EXAMPLESDIR}
+	@(cd ${WRKSRC}/examples/ && ${COPYTREE_SHARE} . ${EXAMPLESDIR})
+.endif
+
+.include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/databases/pure-sql3/pkg-descr ./pkg-descr
--- /usr/ports/databases/pure-sql3/pkg-descr	2013-01-14 17:56:43.000000000 +0100
+++ ./pkg-descr	2013-04-06 02:53:35.000000000 +0200
@@ -3,4 +3,4 @@
 developer access to all of Sqlite3's features in a way that is convenient
 for Pure programmers.
 
-WWW:	http://docs.pure-lang.googlecode.com/hg/pure-sql3.html
+WWW: http://docs.pure-lang.googlecode.com/hg/pure-sql3.html
===> Done
