===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/sysutils/dd_rescue/Makefile ./Makefile
--- /usr/ports/sysutils/dd_rescue/Makefile	2013-01-30 14:55:20.000000000 +0100
+++ ./Makefile	2013-02-19 12:52:00.000000000 +0100
@@ -6,7 +6,7 @@
 #
 
 PORTNAME=	dd_rescue
-PORTVERSION=	1.30
+PORTVERSION=	1.32
 CATEGORIES=	sysutils
 MASTER_SITES=	http://www.garloff.de/kurt/linux/ddrescue/
 
@@ -14,18 +14,30 @@
 COMMENT=	A dd tool suitable for rescuing data from a medium with errors
 
 LICENSE=	GPLv2
-LICENSE_FILE=	${WRKSRC}/COPYING
 
 USE_GMAKE=	yes
+ALL_TARGET=	default
 
 WRKSRC=		${WRKDIR}/${PORTNAME}
-ALL_TARGET=	default
+
+PORTDOCS=	README.dd_rescue
+
+OPTIONS_DEFINE=	DOCS
+
+.include <bsd.port.options.mk>
+
+.if ${PORT_OPTIONS:MDOCS}
+PLIST_SUB+=	PORTDOCS=""
+.else
+PLIST_SUB+=	PORTDOCS="@comment "
+.endif
 
 do-install:
-	${INSTALL_PROGRAM} ${WRKSRC}/dd_rescue ${PREFIX}/bin
-.if !defined(NOPORTDOCS)
+	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin/
+
+.if ${PORT_OPTIONS:MDOCS}
 	${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA}	${WRKSRC}/README.dd_rescue ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/sysutils/dd_rescue/distinfo ./distinfo
--- /usr/ports/sysutils/dd_rescue/distinfo	2013-01-30 14:55:20.000000000 +0100
+++ ./distinfo	2013-02-19 12:38:00.000000000 +0100
@@ -1,2 +1,2 @@
-SHA256 (dd_rescue-1.30.tar.gz) = f4e59c5e669b758c7063bd69dc86f18d380ee3d3551f3da23a2b2a24da4bfa3b
-SIZE (dd_rescue-1.30.tar.gz) = 25253
+SHA256 (dd_rescue-1.32.tar.gz) = 9e57d9b1a37d417ea5e2e0ae4de3b742d2370496034bc3a1133147dd9b400dbd
+SIZE (dd_rescue-1.32.tar.gz) = 28968
diff -ruN --exclude=CVS /usr/ports/sysutils/dd_rescue/pkg-descr ./pkg-descr
--- /usr/ports/sysutils/dd_rescue/pkg-descr	2013-01-14 17:55:42.000000000 +0100
+++ ./pkg-descr	2013-02-19 12:26:37.000000000 +0100
@@ -1,10 +1,16 @@
-dd_rescue is a dd tool with following features suitable for rescuing data from
-a medium with errors.
+Like dd, dd_rescue does copy data from one file or block device to another.
+You can specify file positions (called seek and Skip in dd).
+There are several differences:
 
-- Don't abort on errors on the input file unless the user specify the maximum
-  error number.
-- Don't truncate the output file.
-- Can use two different block sizes for normal operation and the case of errors.
-- Can start from the end of the file and move backwards.
+- dd_rescue does not provide character conversions.
+- The command syntax is different. Call dd_rescue -h.
+- dd_rescue does not abort on errors on the input file, unless you specify
+  a maximum error number. Then dd_rescue will abort when this number is reached.
+- dd_rescue does not truncate the output file, unless asked to.
+- You can tell dd_rescue to start from the end of a file and move bcakwards.
+- It uses two block sizes, a large (soft) block size and a small (hard)
+  block size. In case of errors, the size falls back to the small one and is
+  promoted again after a while without errors.
+- It does not (yet) support non-seekable in- or output.
 
 WWW: http://www.garloff.de/kurt/linux/ddrescue/
===> Done
