===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/math/pure-rational/Makefile ./Makefile
--- /usr/ports/math/pure-rational/Makefile	2013-01-14 17:53:37.000000000 +0100
+++ ./Makefile	2013-04-05 23:04:05.000000000 +0200
@@ -1,26 +1,32 @@
-# New ports collection makefile for:	pure-rational
-# Date created:		2011-10-09
-# Whom:			Zhihao Yuan <lichray@gmail.com>
-#
+# Created by: Zhihao Yuan <lichray@gmail.com>
 # $FreeBSD: head/math/pure-rational/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	pure-rational
 PORTVERSION=	0.1
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	math
+#MASTER_SITES=	https://cdn.bitbucket.org/purelang/pure-lang/downloads/
+MASTER_SITES=	https://pure-lang.googlecode.com/files/
+DIST_SUBDIR=	pure
 
 MAINTAINER=	lichray@gmail.com
 COMMENT=	Rational number library for the Pure language
 
 LICENSE=	GPLv3
 
-USE_PURE=	yes
-NO_PURE_EXAMPLES=	yes
+USES=		pure
 
 PLIST_FILES=	lib/pure/rational.pure \
 		lib/pure/rat_interval.pure
 
-.include <bsd.port.pre.mk>
-.include "${PORTSDIR}/lang/pure/bsd.pure.mk"
-.include <bsd.port.post.mk>
+PORTDOCS=	README
+
+.include <bsd.port.options.mk>
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+
+.include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/math/pure-rational/pkg-descr ./pkg-descr
--- /usr/ports/math/pure-rational/pkg-descr	2013-01-14 17:53:37.000000000 +0100
+++ ./pkg-descr	2013-04-05 23:05:21.000000000 +0200
@@ -1,4 +1,4 @@
 pure-rational provides additional operations on the rational number type
 provided by the math.pure module in the standard library.
 
-WWW:	http://docs.pure-lang.googlecode.com/hg/pure-rational.html
+WWW: http://docs.pure-lang.googlecode.com/hg/pure-rational.html
===> Done
