===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/math/pure-mpfr/Makefile ./Makefile
--- /usr/ports/math/pure-mpfr/Makefile	2013-01-14 17:53:39.000000000 +0100
+++ ./Makefile	2013-04-05 23:11:44.000000000 +0200
@@ -1,24 +1,41 @@
-# New ports collection makefile for:	pure-mpfr
-# Date created:		2011-10-19
-# Whom:			Zhihao Yuan <lichray@gmail.com>
-#
+# Created by: Zhihao Yuan <lichray@gmail.com>
 # $FreeBSD: head/math/pure-mpfr/Makefile 300896 2012-07-14 13:54:48Z beat $
-#
 
 PORTNAME=	pure-mpfr
 PORTVERSION=	0.4
+PORTREVISION=	1
 CATEGORIES=	math
+#MASTER_SITES=	https://cdn.bitbucket.org/purelang/pure-lang/downloads/
+MASTER_SITES=	https://pure-lang.googlecode.com/files/
+DIST_SUBDIR=	pure
 
 MAINTAINER=	lichray@gmail.com
 COMMENT=	Multiprecision floats for Pure
 
-USE_PURE=	yes
+LICENSE=	GPLv3 LGPL3
+LICENSE_COMB=	dual
+
+MAKE_ARGS+=	libdir="${PREFIX}/lib"
+
+USES=		pure
 
 PLIST_FILES=	lib/pure/mpfr.pure \
 		lib/pure/mpfr.so
 
-MAKE_ARGS+=	libdir="${PREFIX}/lib"
+PORTDOCS=	README
+PORTEXAMPLES=	*
+
+.include <bsd.port.options.mk>
+
+post-install:
+.if ${PORT_OPTIONS:MDOCS}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
+.endif
+
+.if ${PORT_OPTIONS:MEXAMPLES}
+	@${MKDIR} ${EXAMPLESDIR}
+	@(cd ${WRKSRC}/examples/ && ${COPYTREE_SHARE} . ${EXAMPLESDIR})
+.endif
 
-.include <bsd.port.pre.mk>
-.include "${PORTSDIR}/lang/pure/bsd.pure.mk"
-.include <bsd.port.post.mk>
+.include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/math/pure-mpfr/pkg-descr ./pkg-descr
--- /usr/ports/math/pure-mpfr/pkg-descr	2013-01-14 17:53:39.000000000 +0100
+++ ./pkg-descr	2013-04-05 23:10:43.000000000 +0200
@@ -2,4 +2,4 @@
 mpfr numbers or values) available in Pure, so that they work with the other
 types of Pure numbers in an almost seamless fashion.
 
-WWW:	http://docs.pure-lang.googlecode.com/hg/pure-mpfr.html
+WWW: http://docs.pure-lang.googlecode.com/hg/pure-mpfr.html
===> Done
