Index: Makefile
===================================================================
--- Makefile	(revision 328493)
+++ Makefile	(working copy)
@@ -2,54 +2,107 @@
 
 PORTNAME=	allegro
 DISTVERSION=	4.4.2
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	devel
 MASTER_SITES=	SF/alleg/allegro/${PORTVERSION}
+DISTFILES=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX}
+DIST_SUBDIR=	${PORTNAME}
+EXTRACT_ONLY=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX}
 
-MAINTAINER=	nemysis@gmx.ch
+MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Cross-platform library for games and multimedia programming
 
-USE_GMAKE=	yes
-USES=		cmake pkgconfig
+LIB_DEPENDS+=	libpng15.so:${PORTSDIR}/graphics/png \
+		libvorbis.so:${PORTSDIR}/audio/libvorbis
+
+USES=		gmake cmake pkgconfig
 USE_XORG=	x11 xpm xext xcursor xxf86vm xxf86dga
+USE_GL=		gl glu
 USE_LDCONFIG=	yes
-WANT_GNOME=	yes
-USE_GL=		gl glu
 
-MAKE_JOBS_UNSAFE=	yes
-
 PORTDOCS=	*
+PORTEXAMPLES=	*
 
-CONFLICTS=	allegro-devel-[0-9]*
-PLIST_SUB=	SHLIB_VER="${SHLIB_VER}"
+PLIST_SUB+=	SHLIB_VER="${SHLIB_VER}"
 CMAKE_ARGS+=	-DDOCDIR="${DOCSDIR}"
 
-SHLIB_VER=	4.4.2
+SHLIB_VER=	${PORTVERSION}
 
-PORTSCOUT=	limit:^4\.3\.[0-9]*
+# Wrong versioning upstream
+PORTSCOUT=	ignore:1
 
-CFLAGS+=	-L${LOCALBASE}/lib
-LIB_DEPENDS+=	asound.2:${PORTSDIR}/audio/alsa-lib \
-		jack.0:${PORTSDIR}/audio/jack \
-		png15:${PORTSDIR}/graphics/png
+OPTIONS_DEFINE=	ALSA DOCS JACK OGG
+OPTIONS_GROUP=	DEMOS_EXAMPLES
+OPTIONS_GROUP_DEMOS_EXAMPLES=	DEMOS EXAMPLES
+OPTIONS_DEFAULT=	ALSA DEMOS OGG
+DEMOS_DESC=	Install Demos programs, requires EXAMPLES
 
-NO_STAGE=	yes
+DOCSRCDIR1=	${WRKSRC}
+DOC_FILES1=	AUTHORS CHANGES THANKS readme.txt
+
+DOCSRCDIR2=	${WRKSRC}/docs
+DOCSDIR2=	${DOCSDIR}/docs
+
+OPTIONS_SUB=	yes
+
+EXAMPLES_CMAKE_ON=	-DWANT_EXAMPLES=on
+EXAMPLES_CMAKE_OFF=	-DWANT_EXAMPLES=off
+ALSA_LIB_DEPENDS=	libasound.so:${PORTSDIR}/audio/alsa-lib
+ALSA_CMAKE_ON=		-DWANT_ALSA=on
+ALSA_CMAKE_OFF=		-DWANT_ALSA=off
+JACK_LIB_DEPENDS=	libjack.so:${PORTSDIR}/audio/jack
+JACK_CMAKE_ON=		-DWANT_JACK=on
+JACK_CMAKE_OFF=		-DWANT_JACK=off
+OGG_LIB_DEPENDS=	libogg.so:${PORTSDIR}/audio/libogg
+OGG_CMAKE_ON=		-DWANT_LOGG=on
+OGG_CMAKE_OFF=		-DWANT_LOGG=of
+
 .include <bsd.port.options.mk>
 
-.if ${PORT_OPTIONS:MDOCS}
-.include "Makefile.man"
-INFO=		allegro
+.if ${PORT_OPTIONS:MALSA}
+CFLAGS+=	-L${LOCALBASE}/lib
 .endif
 
+.if ${PORT_OPTIONS:MDEMOS} && ${PORT_OPTIONS:MEXAMPLES}
+SUB_FILES+=	shooter skater skater_agl \
+		pkg-message
+MASTER_SITES+=	SF/nemysisfreebsdp/allegro/:icons
+DISTFILES+=	shooter.png:icons \
+		skater.png:icons
+DESKTOP_ENTRIES+="Shooter" "In a distant corner of the galaxy" "shooter" \
+		"shooter" "Game;ArcadeGame;" false
+DESKTOP_ENTRIES+="Skater" "Allegro's next demo game" "skater" \
+		"skater" "Game;ArcadeGame;" false
+DESKTOP_ENTRIES+="Skater AGL" "Allegro's next demo game" "skater" \
+		"skater_agl" "Game;ArcadeGame;" false
+.endif
+
 post-patch:
+	@${REINPLACE_CMD} -e 's|/allegro-$${ALLEGRO_VERSION}||' \
+		${WRKSRC}/docs/CMakeLists.txt
+	@${REINPLACE_CMD} -e 's|<X11/extensions/xf86dga.h>|<X11/extensions/Xxf86dga.h>|' \
+		${WRKSRC}/src/x/xdga2.c
+.if ${PORT_OPTIONS:MJACK}
+	@${REINPLACE_CMD} -e 's|jack_client = jack_client_new(jack_client_name);|jack_client = jack_client_open(jack_client_name, (jack_options_t)0, NULL);|' \
+		${WRKSRC}/src/unix/jack.c
+.endif
 .if ! ${PORT_OPTIONS:MDOCS}
-	 @${REINPLACE_CMD} -e 's|add_subdirectory(docs)|#add_subdirectory(docs)|' ${WRKSRC}/CMakeLists.txt
+	@${REINPLACE_CMD} -e 's|add_subdirectory(docs)|#add_subdirectory(docs)|' \
+		${WRKSRC}/CMakeLists.txt
 .endif
 
 post-install:
-.if ${PORT_OPTIONS:MDOCS}
-	@cd ${WRKSRC}/docs/man/ && \
-		${FIND} . -type f -and -name "*.3" -exec ${INSTALL_MAN} {} ${PREFIX}/man/man3/{} \;
+	@${INSTALL_MAN} ${WRKSRC}/docs/man/*.3 ${STAGEDIR}${MAN3PREFIX}/man/man3/
+	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} . ${STAGEDIR}${EXAMPLESDIR})
+
+.if ${PORT_OPTIONS:MDEMOS} && ${PORT_OPTIONS:MEXAMPLES}
+	@(cd ${WRKSRC} && ${COPYTREE_SHARE} demos ${STAGEDIR}${EXAMPLESDIR})
+	${INSTALL_SCRIPT} ${WRKDIR}/shooter ${STAGEDIR}${PREFIX}/bin/
+	${INSTALL_DATA} ${_DISTDIR}/shooter.png ${STAGEDIR}${PREFIX}/share/pixmaps/
+	@(cd ${STAGEDIR}${EXAMPLESDIR}/demos/shooter && ${CHMOD} 755 shooter)
+	${INSTALL_SCRIPT} ${WRKDIR}/skater* ${STAGEDIR}${PREFIX}/bin
+	${INSTALL_DATA} ${_DISTDIR}/skater.png ${STAGEDIR}${PREFIX}/share/pixmaps/
+	@(cd ${STAGEDIR}${EXAMPLESDIR}/demos/skater && ${CHMOD} 755 skater*)
 .endif
 
 .include <bsd.port.mk>
Index: Makefile.man
===================================================================
--- Makefile.man	(revision 328493)
+++ Makefile.man	(working copy)
@@ -1,923 +0,0 @@
-MAN3=	ALLEGRO_DATE.3 \
-ALLEGRO_DATE_STR.3 \
-ALLEGRO_SUB_VERSION.3 \
-ALLEGRO_VERSION.3 \
-ALLEGRO_VERSION_STR.3 \
-ALLEGRO_WIP_VERSION.3 \
-AL_ID.3 \
-ASSERT.3 \
-AUDIOSTREAM.3 \
-BITMAP.3 \
-COLOR_MAP.3 \
-COMPILED_SPRITE.3 \
-DATAFILE.3 \
-DAT_ID.3 \
-DIALOG.3 \
-DIALOG_PLAYER.3 \
-END_OF_FUNCTION.3 \
-END_OF_MAIN.3 \
-FONT.3 \
-GFX_MODE.3 \
-GFX_MODE_LIST.3 \
-JOYSTICK_AXIS_INFO.3 \
-JOYSTICK_BUTTON_INFO.3 \
-JOYSTICK_INFO.3 \
-JOYSTICK_STICK_INFO.3 \
-LOCK_FUNCTION.3 \
-LOCK_VARIABLE.3 \
-LZSS_PACK_DATA.3 \
-LZSS_UNPACK_DATA.3 \
-MAKE_VERSION.3 \
-MASK_COLOR_15.3 \
-MASK_COLOR_16.3 \
-MASK_COLOR_24.3 \
-MASK_COLOR_32.3 \
-MASK_COLOR_8.3 \
-MATRIX.3 \
-MATRIX_f.3 \
-MENU.3 \
-MENU_PLAYER.3 \
-MIDI.3 \
-PACKFILE.3 \
-PACKFILE_VTABLE.3 \
-PALETTE.3 \
-PAL_SIZE.3 \
-POLYTYPE_ATEX.3 \
-POLYTYPE_ATEX_LIT.3 \
-POLYTYPE_ATEX_MASK.3 \
-POLYTYPE_ATEX_MASK_LIT.3 \
-POLYTYPE_ATEX_MASK_TRANS.3 \
-POLYTYPE_ATEX_TRANS.3 \
-POLYTYPE_FLAT.3 \
-POLYTYPE_GCOL.3 \
-POLYTYPE_GRGB.3 \
-POLYTYPE_PTEX.3 \
-POLYTYPE_PTEX_LIT.3 \
-POLYTYPE_PTEX_MASK.3 \
-POLYTYPE_PTEX_MASK_LIT.3 \
-POLYTYPE_PTEX_MASK_TRANS.3 \
-POLYTYPE_PTEX_TRANS.3 \
-QUAT.3 \
-RGB.3 \
-RGB_MAP.3 \
-RLE_SPRITE.3 \
-SAMPLE.3 \
-SCREEN_H.3 \
-SCREEN_W.3 \
-TRACE.3 \
-V3D.3 \
-V3D_f.3 \
-VIRTUAL_H.3 \
-VIRTUAL_W.3 \
-ZBUFFER.3 \
-_getpixel.3 \
-_getpixel15.3 \
-_getpixel16.3 \
-_getpixel24.3 \
-_getpixel32.3 \
-_putpixel.3 \
-_putpixel15.3 \
-_putpixel16.3 \
-_putpixel24.3 \
-_putpixel32.3 \
-_set_color.3 \
-_ustrdup.3 \
-acquire_bitmap.3 \
-acquire_screen.3 \
-active_dialog.3 \
-active_menu.3 \
-add_clip_rect.3 \
-adjust_sample.3 \
-al_assert.3 \
-al_ffblk.3 \
-al_ffblk_get_size.3 \
-al_findclose.3 \
-al_findfirst.3 \
-al_findnext.3 \
-al_trace.3 \
-alert.3 \
-alert3.3 \
-allegro_404_char.3 \
-allegro_error.3 \
-allegro_exit.3 \
-allegro_icon.3 \
-allegro_id.3 \
-allegro_init.3 \
-allegro_message.3 \
-allocate_voice.3 \
-append_filename.3 \
-apply_matrix.3 \
-apply_matrix_f.3 \
-apply_quat.3 \
-arc.3 \
-bestfit_color.3 \
-bitmap_color_depth.3 \
-bitmap_mask_color.3 \
-black_palette.3 \
-blit.3 \
-bmp_read_line.3 \
-bmp_unwrite_line.3 \
-bmp_write_line.3 \
-broadcast_dialog_message.3 \
-calc_spline.3 \
-calibrate_joystick.3 \
-calibrate_joystick_name.3 \
-canonicalize_filename.3 \
-centre_dialog.3 \
-check_cpu.3 \
-circle.3 \
-circlefill.3 \
-clear_bitmap.3 \
-clear_keybuf.3 \
-clear_scene.3 \
-clear_to_color.3 \
-clear_zbuffer.3 \
-clip3d.3 \
-clip3d_f.3 \
-close_fli.3 \
-color_map.3 \
-config_is_hooked.3 \
-cpu_capabilities.3 \
-cpu_family.3 \
-cpu_model.3 \
-cpu_vendor.3 \
-create_bitmap.3 \
-create_bitmap_ex.3 \
-create_blender_table.3 \
-create_color_table.3 \
-create_datafile_index.3 \
-create_light_table.3 \
-create_lzss_pack_data.3 \
-create_lzss_unpack_data.3 \
-create_rgb_table.3 \
-create_sample.3 \
-create_scene.3 \
-create_sub_bitmap.3 \
-create_sub_zbuffer.3 \
-create_system_bitmap.3 \
-create_trans_table.3 \
-create_video_bitmap.3 \
-create_zbuffer.3 \
-cross_product.3 \
-cross_product_f.3 \
-d_bitmap_proc.3 \
-d_box_proc.3 \
-d_button_proc.3 \
-d_check_proc.3 \
-d_clear_proc.3 \
-d_ctext_proc.3 \
-d_edit_proc.3 \
-d_icon_proc.3 \
-d_keyboard_proc.3 \
-d_list_proc.3 \
-d_menu_proc.3 \
-d_radio_proc.3 \
-d_rtext_proc.3 \
-d_shadow_box_proc.3 \
-d_slider_proc.3 \
-d_text_list_proc.3 \
-d_text_proc.3 \
-d_textbox_proc.3 \
-d_yield_proc.3 \
-deallocate_voice.3 \
-default_palette.3 \
-delete_file.3 \
-desktop_color_depth.3 \
-desktop_palette.3 \
-destroy_bitmap.3 \
-destroy_compiled_sprite.3 \
-destroy_datafile_index.3 \
-destroy_font.3 \
-destroy_gfx_mode_list.3 \
-destroy_midi.3 \
-destroy_rle_sprite.3 \
-destroy_sample.3 \
-destroy_scene.3 \
-destroy_zbuffer.3 \
-detect_digi_driver.3 \
-detect_midi_driver.3 \
-dialog_message.3 \
-digi_recorder.3 \
-disable_hardware_cursor.3 \
-do_arc.3 \
-do_circle.3 \
-do_dialog.3 \
-do_ellipse.3 \
-do_line.3 \
-do_menu.3 \
-do_uconvert.3 \
-dot_product.3 \
-dot_product_f.3 \
-draw_character_ex.3 \
-draw_compiled_sprite.3 \
-draw_gouraud_sprite.3 \
-draw_lit_rle_sprite.3 \
-draw_lit_sprite.3 \
-draw_rle_sprite.3 \
-draw_sprite.3 \
-draw_sprite_ex.3 \
-draw_sprite_h_flip.3 \
-draw_sprite_v_flip.3 \
-draw_sprite_vh_flip.3 \
-draw_trans_rle_sprite.3 \
-draw_trans_sprite.3 \
-drawing_mode.3 \
-ellipse.3 \
-ellipsefill.3 \
-empty_string.3 \
-enable_hardware_cursor.3 \
-enable_triple_buffer.3 \
-ex12bit.3 \
-ex3buf.3 \
-ex3d.3 \
-exaccel.3 \
-exalpha.3 \
-exbitmap.3 \
-exblend.3 \
-excamera.3 \
-excolmap.3 \
-exconfig.3 \
-excustom.3 \
-exdata.3 \
-exdbuf.3 \
-exexedat.3 \
-exfixed.3 \
-exflame.3 \
-exflip.3 \
-exfont.3 \
-exgui.3 \
-exhello.3 \
-exists.3 \
-exjoy.3 \
-exkeys.3 \
-exlights.3 \
-exmem.3 \
-exmidi.3 \
-exmouse.3 \
-expackf.3 \
-expal.3 \
-expat.3 \
-exquat.3 \
-exrgbhsv.3 \
-exrotscl.3 \
-exsample.3 \
-exscale.3 \
-exscn3d.3 \
-exscroll.3 \
-exshade.3 \
-exspline.3 \
-exsprite.3 \
-exstars.3 \
-exstream.3 \
-exswitch.3 \
-exsyscur.3 \
-extimer.3 \
-extract_font_range.3 \
-extrans.3 \
-extrans2.3 \
-extruec.3 \
-exunicod.3 \
-exupdate.3 \
-exxfade.3 \
-exzbuf.3 \
-fade_from.3 \
-fade_from_range.3 \
-fade_in.3 \
-fade_in_range.3 \
-fade_interpolate.3 \
-fade_out.3 \
-fade_out_range.3 \
-fastline.3 \
-file_exists.3 \
-file_select_ex.3 \
-file_size_ex.3 \
-file_time.3 \
-find_allegro_resource.3 \
-find_datafile_object.3 \
-find_dialog_focus.3 \
-fix_filename_case.3 \
-fix_filename_slashes.3 \
-fixacos.3 \
-fixadd.3 \
-fixasin.3 \
-fixatan.3 \
-fixatan2.3 \
-fixceil.3 \
-fixcos.3 \
-fixdiv.3 \
-fixed.3 \
-fixfloor.3 \
-fixhypot.3 \
-fixmul.3 \
-fixsin.3 \
-fixsqrt.3 \
-fixsub.3 \
-fixtan.3 \
-fixtof.3 \
-fixtoi.3 \
-fixtorad_r.3 \
-fixup_datafile.3 \
-fli_bitmap.3 \
-fli_bmp_dirty_from.3 \
-fli_bmp_dirty_to.3 \
-fli_frame.3 \
-fli_pal_dirty_from.3 \
-fli_pal_dirty_to.3 \
-fli_palette.3 \
-fli_timer.3 \
-floodfill.3 \
-flush_config_file.3 \
-font.3 \
-font_has_alpha.3 \
-for_each_file_ex.3 \
-free_audio_stream_buffer.3 \
-free_config_entries.3 \
-free_lzss_pack_data.3 \
-free_lzss_unpack_data.3 \
-freeze_mouse_flag.3 \
-ftofix.3 \
-generate_332_palette.3 \
-generate_optimized_palette.3 \
-get_align_matrix.3 \
-get_align_matrix_f.3 \
-get_audio_stream_buffer.3 \
-get_camera_matrix.3 \
-get_camera_matrix_f.3 \
-get_clip_rect.3 \
-get_clip_state.3 \
-get_color.3 \
-get_color_conversion.3 \
-get_color_depth.3 \
-get_compiled_sprite.3 \
-get_config_argv.3 \
-get_config_float.3 \
-get_config_hex.3 \
-get_config_id.3 \
-get_config_int.3 \
-get_config_string.3 \
-get_config_text.3 \
-get_datafile_property.3 \
-get_desktop_resolution.3 \
-get_display_switch_mode.3 \
-get_executable_name.3 \
-get_extension.3 \
-get_filename.3 \
-get_filename_encoding.3 \
-get_font_range_begin.3 \
-get_font_range_end.3 \
-get_font_ranges.3 \
-get_gfx_mode.3 \
-get_gfx_mode_list.3 \
-get_gfx_mode_type.3 \
-get_hardware_volume.3 \
-get_midi_length.3 \
-get_mixer_bits.3 \
-get_mixer_buffer_length.3 \
-get_mixer_channels.3 \
-get_mixer_frequency.3 \
-get_mixer_quality.3 \
-get_mixer_voices.3 \
-get_mouse_mickeys.3 \
-get_palette.3 \
-get_palette_range.3 \
-get_refresh_rate.3 \
-get_rle_sprite.3 \
-get_rotation_matrix.3 \
-get_rotation_matrix_f.3 \
-get_rotation_quat.3 \
-get_scaling_matrix.3 \
-get_scaling_matrix_f.3 \
-get_sound_input_cap_bits.3 \
-get_sound_input_cap_parm.3 \
-get_sound_input_cap_rate.3 \
-get_sound_input_cap_stereo.3 \
-get_transformation_matrix.3 \
-get_transformation_matrix_f.3 \
-get_translation_matrix.3 \
-get_translation_matrix_f.3 \
-get_uformat.3 \
-get_vector_rotation_matrix.3 \
-get_vector_rotation_matrix_f.3 \
-get_vector_rotation_quat.3 \
-get_volume.3 \
-get_x_rotate_matrix.3 \
-get_x_rotate_matrix_f.3 \
-get_x_rotate_quat.3 \
-get_y_rotate_matrix.3 \
-get_y_rotate_matrix_f.3 \
-get_y_rotate_quat.3 \
-get_z_rotate_matrix.3 \
-get_z_rotate_matrix_f.3 \
-get_z_rotate_quat.3 \
-geta.3 \
-geta32.3 \
-geta_depth.3 \
-getb.3 \
-getb15.3 \
-getb16.3 \
-getb24.3 \
-getb32.3 \
-getb8.3 \
-getb_depth.3 \
-getg.3 \
-getg15.3 \
-getg16.3 \
-getg24.3 \
-getg32.3 \
-getg8.3 \
-getg_depth.3 \
-getpixel.3 \
-getr.3 \
-getr15.3 \
-getr16.3 \
-getr24.3 \
-getr32.3 \
-getr8.3 \
-getr_depth.3 \
-gfx_capabilities.3 \
-gfx_mode_select.3 \
-gfx_mode_select_ex.3 \
-gfx_mode_select_filter.3 \
-grab_font_from_bitmap.3 \
-gui_bg_color.3 \
-gui_button_proc.3 \
-gui_ctext_proc.3 \
-gui_edit_proc.3 \
-gui_fg_color.3 \
-gui_font_baseline.3 \
-gui_get_screen.3 \
-gui_list_proc.3 \
-gui_menu_draw_menu.3 \
-gui_menu_draw_menu_item.3 \
-gui_mg_color.3 \
-gui_mouse_b.3 \
-gui_mouse_focus.3 \
-gui_mouse_x.3 \
-gui_mouse_y.3 \
-gui_mouse_z.3 \
-gui_set_screen.3 \
-gui_shadow_box_proc.3 \
-gui_strlen.3 \
-gui_text_list_proc.3 \
-gui_textout_ex.3 \
-hline.3 \
-hook_config_section.3 \
-hsv_to_rgb.3 \
-identity_matrix.3 \
-identity_matrix_f.3 \
-identity_quat.3 \
-init_dialog.3 \
-init_menu.3 \
-initialise_joystick.3 \
-install_allegro.3 \
-install_int.3 \
-install_int_ex.3 \
-install_joystick.3 \
-install_keyboard.3 \
-install_keyboard_hooks.3 \
-install_mouse.3 \
-install_param_int.3 \
-install_param_int_ex.3 \
-install_sound.3 \
-install_sound_input.3 \
-install_timer.3 \
-is_color_font.3 \
-is_compatible_font.3 \
-is_inside_bitmap.3 \
-is_linear_bitmap.3 \
-is_memory_bitmap.3 \
-is_mono_font.3 \
-is_planar_bitmap.3 \
-is_relative_filename.3 \
-is_same_bitmap.3 \
-is_screen_bitmap.3 \
-is_sub_bitmap.3 \
-is_system_bitmap.3 \
-is_trans_font.3 \
-is_video_bitmap.3 \
-is_windowed_mode.3 \
-itofix.3 \
-joy.3 \
-key.3 \
-key_led_flag.3 \
-key_shifts.3 \
-keyboard_callback.3 \
-keyboard_lowlevel_callback.3 \
-keyboard_needs_poll.3 \
-keyboard_ucallback.3 \
-keypressed.3 \
-line.3 \
-list_config_entries.3 \
-list_config_sections.3 \
-load_bios_font.3 \
-load_bitmap.3 \
-load_bitmap_font.3 \
-load_bmp.3 \
-load_bmp_pf.3 \
-load_dat_font.3 \
-load_datafile.3 \
-load_datafile_callback.3 \
-load_datafile_object.3 \
-load_datafile_object_indexed.3 \
-load_font.3 \
-load_grx_font.3 \
-load_grx_or_bios_font.3 \
-load_ibk.3 \
-load_joystick_data.3 \
-load_lbm.3 \
-load_midi.3 \
-load_midi_patches.3 \
-load_pcx.3 \
-load_pcx_pf.3 \
-load_sample.3 \
-load_tga.3 \
-load_tga_pf.3 \
-load_txt_font.3 \
-load_voc.3 \
-load_voc_pf.3 \
-load_wav.3 \
-load_wav_pf.3 \
-lock_bitmap.3 \
-lock_midi.3 \
-lock_sample.3 \
-lzss_read.3 \
-lzss_write.3 \
-make_absolute_filename.3 \
-make_relative_filename.3 \
-make_trans_font.3 \
-makeacol.3 \
-makeacol32.3 \
-makeacol_depth.3 \
-makecol.3 \
-makecol15.3 \
-makecol15_dither.3 \
-makecol16.3 \
-makecol16_dither.3 \
-makecol24.3 \
-makecol32.3 \
-makecol8.3 \
-makecol_depth.3 \
-masked_blit.3 \
-masked_stretch_blit.3 \
-matrix_mul.3 \
-matrix_mul_f.3 \
-matrix_to_quat.3 \
-merge_fonts.3 \
-midi_loop_end.3 \
-midi_loop_start.3 \
-midi_meta_callback.3 \
-midi_msg_callback.3 \
-midi_out.3 \
-midi_pause.3 \
-midi_pos.3 \
-midi_recorder.3 \
-midi_resume.3 \
-midi_seek.3 \
-midi_sysex_callback.3 \
-midi_time.3 \
-mouse_b.3 \
-mouse_callback.3 \
-mouse_needs_poll.3 \
-mouse_on_screen.3 \
-mouse_pos.3 \
-mouse_sprite.3 \
-mouse_w.3 \
-mouse_x.3 \
-mouse_x_focus.3 \
-mouse_y.3 \
-mouse_y_focus.3 \
-mouse_z.3 \
-need_uconvert.3 \
-next_fli_frame.3 \
-normalize_vector.3 \
-normalize_vector_f.3 \
-num_joysticks.3 \
-object_message.3 \
-offer_focus.3 \
-open_fli.3 \
-open_memory_fli.3 \
-os_multitasking.3 \
-os_revision.3 \
-os_type.3 \
-os_version.3 \
-override_config_data.3 \
-override_config_file.3 \
-pack_fclose.3 \
-pack_fclose_chunk.3 \
-pack_feof.3 \
-pack_ferror.3 \
-pack_fgets.3 \
-pack_fopen.3 \
-pack_fopen_chunk.3 \
-pack_fopen_vtable.3 \
-pack_fputs.3 \
-pack_fread.3 \
-pack_fseek.3 \
-pack_fwrite.3 \
-pack_getc.3 \
-pack_igetl.3 \
-pack_igetw.3 \
-pack_iputl.3 \
-pack_iputw.3 \
-pack_mgetl.3 \
-pack_mgetw.3 \
-pack_mputl.3 \
-pack_mputw.3 \
-pack_putc.3 \
-pack_ungetc.3 \
-packfile_password.3 \
-palette_color.3 \
-persp_project.3 \
-persp_project_f.3 \
-pivot_scaled_sprite.3 \
-pivot_scaled_sprite_lit.3 \
-pivot_scaled_sprite_trans.3 \
-pivot_scaled_sprite_v_flip.3 \
-pivot_scaled_sprite_v_flip_lit.3 \
-pivot_scaled_sprite_v_flip_trans.3 \
-pivot_sprite.3 \
-pivot_sprite_lit.3 \
-pivot_sprite_trans.3 \
-pivot_sprite_v_flip.3 \
-pivot_sprite_v_flip_lit.3 \
-pivot_sprite_v_flip_trans.3 \
-play_audio_stream.3 \
-play_fli.3 \
-play_looped_midi.3 \
-play_memory_fli.3 \
-play_midi.3 \
-play_sample.3 \
-poll_joystick.3 \
-poll_keyboard.3 \
-poll_mouse.3 \
-poll_scroll.3 \
-polygon.3 \
-polygon3d.3 \
-polygon3d_f.3 \
-polygon_z_normal.3 \
-polygon_z_normal_f.3 \
-pop_config_state.3 \
-popup_dialog.3 \
-position_dialog.3 \
-position_mouse.3 \
-position_mouse_w.3 \
-position_mouse_z.3 \
-push_config_state.3 \
-put_backslash.3 \
-putpixel.3 \
-qnx_get_window.3 \
-qscale_matrix.3 \
-qscale_matrix_f.3 \
-qtranslate_matrix.3 \
-qtranslate_matrix_f.3 \
-quad3d.3 \
-quad3d_f.3 \
-quat_interpolate.3 \
-quat_mul.3 \
-quat_slerp.3 \
-quat_to_matrix.3 \
-radtofix_r.3 \
-read_sound_input.3 \
-readkey.3 \
-reallocate_voice.3 \
-rect.3 \
-rectfill.3 \
-register_assert_handler.3 \
-register_bitmap_file_type.3 \
-register_datafile_object.3 \
-register_font_file_type.3 \
-register_sample_file_type.3 \
-register_trace_handler.3 \
-register_uformat.3 \
-release_bitmap.3 \
-release_screen.3 \
-release_voice.3 \
-reload_config_texts.3 \
-remove_display_switch_callback.3 \
-remove_int.3 \
-remove_joystick.3 \
-remove_keyboard.3 \
-remove_mouse.3 \
-remove_param_int.3 \
-remove_sound.3 \
-remove_sound_input.3 \
-remove_timer.3 \
-render_scene.3 \
-replace_extension.3 \
-replace_filename.3 \
-request_refresh_rate.3 \
-request_scroll.3 \
-request_video_bitmap.3 \
-reserve_voices.3 \
-reset_fli_variables.3 \
-rest.3 \
-rest_callback.3 \
-retrace_count.3 \
-rgb_map.3 \
-rgb_to_hsv.3 \
-rotate_scaled_sprite.3 \
-rotate_scaled_sprite_lit.3 \
-rotate_scaled_sprite_trans.3 \
-rotate_scaled_sprite_v_flip.3 \
-rotate_scaled_sprite_v_flip_lit.3 \
-rotate_scaled_sprite_v_flip_trans.3 \
-rotate_sprite.3 \
-rotate_sprite_lit.3 \
-rotate_sprite_trans.3 \
-rotate_sprite_v_flip.3 \
-rotate_sprite_v_flip_lit.3 \
-rotate_sprite_v_flip_trans.3 \
-save_bitmap.3 \
-save_bmp.3 \
-save_bmp_pf.3 \
-save_joystick_data.3 \
-save_pcx.3 \
-save_pcx_pf.3 \
-save_sample.3 \
-save_tga.3 \
-save_tga_pf.3 \
-scancode_to_ascii.3 \
-scancode_to_name.3 \
-scare_mouse.3 \
-scare_mouse_area.3 \
-scene_gap.3 \
-scene_polygon3d.3 \
-scene_polygon3d_f.3 \
-screen.3 \
-scroll_screen.3 \
-select_mouse_cursor.3 \
-select_palette.3 \
-set_add_blender.3 \
-set_allegro_resource_path.3 \
-set_alpha_blender.3 \
-set_blender_mode.3 \
-set_blender_mode_ex.3 \
-set_burn_blender.3 \
-set_clip_rect.3 \
-set_clip_state.3 \
-set_close_button_callback.3 \
-set_color.3 \
-set_color_blender.3 \
-set_color_conversion.3 \
-set_color_depth.3 \
-set_config_data.3 \
-set_config_file.3 \
-set_config_float.3 \
-set_config_hex.3 \
-set_config_id.3 \
-set_config_int.3 \
-set_config_string.3 \
-set_dialog_color.3 \
-set_difference_blender.3 \
-set_display_switch_callback.3 \
-set_display_switch_mode.3 \
-set_dissolve_blender.3 \
-set_dodge_blender.3 \
-set_filename_encoding.3 \
-set_gfx_mode.3 \
-set_hardware_volume.3 \
-set_hue_blender.3 \
-set_invert_blender.3 \
-set_keyboard_rate.3 \
-set_leds.3 \
-set_luminance_blender.3 \
-set_mixer_quality.3 \
-set_mouse_cursor_bitmap.3 \
-set_mouse_range.3 \
-set_mouse_speed.3 \
-set_mouse_sprite.3 \
-set_mouse_sprite_focus.3 \
-set_multiply_blender.3 \
-set_palette.3 \
-set_palette_range.3 \
-set_projection_viewport.3 \
-set_saturation_blender.3 \
-set_screen_blender.3 \
-set_sound_input_source.3 \
-set_trans_blender.3 \
-set_ucodepage.3 \
-set_uformat.3 \
-set_volume.3 \
-set_volume_per_voice.3 \
-set_window_title.3 \
-set_write_alpha_blender.3 \
-set_zbuffer.3 \
-show_mouse.3 \
-show_os_cursor.3 \
-show_video_bitmap.3 \
-shutdown_dialog.3 \
-shutdown_menu.3 \
-simulate_keypress.3 \
-simulate_ukeypress.3 \
-solid_mode.3 \
-spline.3 \
-start_sound_input.3 \
-stop_audio_stream.3 \
-stop_midi.3 \
-stop_sample.3 \
-stop_sound_input.3 \
-stretch_blit.3 \
-stretch_sprite.3 \
-text_height.3 \
-text_length.3 \
-textout_centre_ex.3 \
-textout_ex.3 \
-textout_justify_ex.3 \
-textout_right_ex.3 \
-textprintf_centre_ex.3 \
-textprintf_ex.3 \
-textprintf_justify_ex.3 \
-textprintf_right_ex.3 \
-three_finger_flag.3 \
-transpose_font.3 \
-triangle.3 \
-triangle3d.3 \
-triangle3d_f.3 \
-uatof.3 \
-uconvert.3 \
-uconvert_ascii.3 \
-uconvert_size.3 \
-uconvert_toascii.3 \
-ucwidth.3 \
-ugetat.3 \
-ugetc.3 \
-ugetx.3 \
-ugetxc.3 \
-uinsert.3 \
-uisdigit.3 \
-uisok.3 \
-uisspace.3 \
-unload_datafile.3 \
-unload_datafile_object.3 \
-unscare_mouse.3 \
-unselect_palette.3 \
-uoffset.3 \
-update_dialog.3 \
-update_menu.3 \
-ureadkey.3 \
-uremove.3 \
-usetat.3 \
-usetc.3 \
-usprintf.3 \
-ustrcat.3 \
-ustrchr.3 \
-ustrcmp.3 \
-ustrcpy.3 \
-ustrdup.3 \
-ustrerror.3 \
-ustricmp.3 \
-ustrlen.3 \
-ustrlwr.3 \
-ustrncat.3 \
-ustrncmp.3 \
-ustrncpy.3 \
-ustrnicmp.3 \
-ustrpbrk.3 \
-ustrrchr.3 \
-ustrsize.3 \
-ustrsizez.3 \
-ustrstr.3 \
-ustrtod.3 \
-ustrtok.3 \
-ustrtok_r.3 \
-ustrtol.3 \
-ustrupr.3 \
-ustrzcat.3 \
-ustrzcpy.3 \
-ustrzncat.3 \
-ustrzncpy.3 \
-uszprintf.3 \
-utolower.3 \
-utoupper.3 \
-uvsprintf.3 \
-uvszprintf.3 \
-uwidth.3 \
-uwidth_max.3 \
-vector_length.3 \
-vector_length_f.3 \
-vline.3 \
-voice_check.3 \
-voice_get_frequency.3 \
-voice_get_pan.3 \
-voice_get_position.3 \
-voice_get_volume.3 \
-voice_ramp_volume.3 \
-voice_set_echo.3 \
-voice_set_frequency.3 \
-voice_set_pan.3 \
-voice_set_playmode.3 \
-voice_set_position.3 \
-voice_set_priority.3 \
-voice_set_tremolo.3 \
-voice_set_vibrato.3 \
-voice_set_volume.3 \
-voice_start.3 \
-voice_stop.3 \
-voice_stop_frequency_sweep.3 \
-voice_stop_pan_sweep.3 \
-voice_stop_volumeramp.3 \
-voice_sweep_frequency.3 \
-voice_sweep_pan.3 \
-vsync.3 \
-xor_mode.3 \
-xwin_set_window_name.3
Index: distinfo
===================================================================
--- distinfo	(revision 328493)
+++ distinfo	(working copy)
@@ -1,2 +1,6 @@
-SHA256 (allegro-4.4.2.tar.gz) = 1b21e7577dbfada02d85ca4510bd22fedaa6ce76fde7f4838c7c1276eb840fdc
-SIZE (allegro-4.4.2.tar.gz) = 4674902
+SHA256 (allegro/allegro-4.4.2.tar.gz) = 1b21e7577dbfada02d85ca4510bd22fedaa6ce76fde7f4838c7c1276eb840fdc
+SIZE (allegro/allegro-4.4.2.tar.gz) = 4674902
+SHA256 (allegro/shooter.png) = cf73f63f0224a9ee77c287b6211beb4a072ba4aa969a57993e60a2e97996e468
+SIZE (allegro/shooter.png) = 1741
+SHA256 (allegro/skater.png) = f0274627c6eae2ecad880c3e2f25bf5f8944f08829ac32ab0a00f77001e8b2f3
+SIZE (allegro/skater.png) = 3395
Index: files/patch-docs_CMakeLists.txt
===================================================================
--- files/patch-docs_CMakeLists.txt	(revision 328493)
+++ files/patch-docs_CMakeLists.txt	(working copy)
@@ -1,11 +0,0 @@
---- docs/CMakeLists.txt.orig	2010-12-15 13:29:42.000000000 +0600
-+++ docs/CMakeLists.txt	2010-12-15 13:29:49.000000000 +0600
-@@ -9,7 +9,7 @@
- endif()
- 
- set(DOCDIR "doc" CACHE STRING "Install docs into $DOCDIR/allegro-$VERSION")
--set(ACTUAL_DOC_DIR "${DOCDIR}/allegro-${ALLEGRO_VERSION}")
-+set(ACTUAL_DOC_DIR "${DOCDIR}")
- 
- add_executable(makedoc
-     src/makedoc/makedoc.c
Index: files/patch-src-x-xkeyboard.c
===================================================================
--- files/patch-src-x-xkeyboard.c	(revision 0)
+++ files/patch-src-x-xkeyboard.c	(working copy)
@@ -0,0 +1,19 @@
+--- src/x/xkeyboard.c.orig	2010-12-08 09:13:50.000000000 +0100
++++ src/x/xkeyboard.c	2013-07-28 00:17:52.000000000 +0200
+@@ -23,6 +23,7 @@
+ #include <X11/Xos.h>
+ #include <X11/Xlib.h>
+ #include <X11/Xutil.h>
++#include <X11/XKBlib.h>
+ #include <X11/Xproto.h>
+ 
+ #include "allegro.h"
+@@ -617,7 +618,7 @@
+ 
+       TRACE (PREFIX_I "Modifier %d:", i + 1);
+       for (j = 0; j < xmodmap->max_keypermod; j++) {
+-	 KeySym sym = XKeycodeToKeysym(_xwin.display,
++	 KeySym sym = XkbKeycodeToKeysym(_xwin.display, 0,
+ 	    xmodmap->modifiermap[i * xmodmap->max_keypermod + j], 0);
+          char *sym_str = XKeysymToString(sym);
+          TRACE(" %s", sym_str ? sym_str : "NULL");

Property changes on: files/patch-src-x-xkeyboard.c
___________________________________________________________________
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Index: files/pkg-message.in
===================================================================
--- files/pkg-message.in	(revision 0)
+++ files/pkg-message.in	(working copy)
@@ -0,0 +1,25 @@
+===============================================================================
+
+Allegro has been installed.
+
+Allegro have two Demos (Games)
+
+Shooter and Skater
+
+
+You can use executable
+
+    %%LOCALBASE%%/bin/shooter
+
+    %%LOCALBASE%%/bin/skater
+
+
+Or can use desktop files
+
+    %%LOCALBASE%%/share/applications/shooter.desktop
+
+    %%LOCALBASE%%/share/applications/skater.desktop
+
+Have fun!
+
+===============================================================================

Property changes on: files/pkg-message.in
___________________________________________________________________
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Index: files/shooter.in
===================================================================
--- files/shooter.in	(revision 0)
+++ files/shooter.in	(working copy)
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# # $FreeBSD$
+#
+
+cd "%%EXAMPLESDIR%%/demos/shooter"
+exec ./shooter "${@}"
Index: files/skater.in
===================================================================
--- files/skater.in	(revision 0)
+++ files/skater.in	(working copy)
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# # $FreeBSD$
+#
+
+cd "%%EXAMPLESDIR%%/demos/skater"
+exec ./skater "${@}"
Index: files/skater_agl.in
===================================================================
--- files/skater_agl.in	(revision 0)
+++ files/skater_agl.in	(working copy)
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# # $FreeBSD$
+#
+
+cd "%%EXAMPLESDIR%%/demos/skater"
+exec ./skater_agl "${@}"
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 328493)
+++ pkg-plist	(working copy)
@@ -8,6 +8,9 @@
 bin/pack
 bin/pat2dat
 bin/rgbmap
+%%DEMOS%%bin/shooter
+%%DEMOS%%bin/skater
+%%DEMOS%%bin/skater_agl
 bin/textconv
 include/alleggl.h
 include/allegro.h
@@ -115,10 +118,11 @@
 include/loadpng.h
 include/logg.h
 include/xalleg.h
+info/allegro.info
 lib/allegro/%%SHLIB_VER%%/alleg-alsadigi.so
 lib/allegro/%%SHLIB_VER%%/alleg-alsamidi.so
 lib/allegro/%%SHLIB_VER%%/alleg-dga2.so
-lib/allegro/%%SHLIB_VER%%/alleg-jack.so
+%%JACK%%lib/allegro/%%SHLIB_VER%%/alleg-jack.so
 lib/allegro/%%SHLIB_VER%%/modules.lst
 lib/liballeg.so
 lib/liballeg.so.4
@@ -132,6 +136,931 @@
 libdata/pkgconfig/jpgalleg.pc
 libdata/pkgconfig/loadpng.pc
 libdata/pkgconfig/logg.pc
+man/man3/ALLEGRO_DATE.3.gz
+man/man3/ALLEGRO_DATE_STR.3.gz
+man/man3/ALLEGRO_SUB_VERSION.3.gz
+man/man3/ALLEGRO_VERSION.3.gz
+man/man3/ALLEGRO_VERSION_STR.3.gz
+man/man3/ALLEGRO_WIP_VERSION.3.gz
+man/man3/AL_ID.3.gz
+man/man3/ASSERT.3.gz
+man/man3/AUDIOSTREAM.3.gz
+man/man3/BITMAP.3.gz
+man/man3/COLOR_MAP.3.gz
+man/man3/COMPILED_SPRITE.3.gz
+man/man3/DATAFILE.3.gz
+man/man3/DAT_ID.3.gz
+man/man3/DIALOG.3.gz
+man/man3/DIALOG_PLAYER.3.gz
+man/man3/END_OF_FUNCTION.3.gz
+man/man3/END_OF_MAIN.3.gz
+man/man3/FONT.3.gz
+man/man3/GFX_MODE.3.gz
+man/man3/GFX_MODE_LIST.3.gz
+man/man3/JOYSTICK_AXIS_INFO.3.gz
+man/man3/JOYSTICK_BUTTON_INFO.3.gz
+man/man3/JOYSTICK_INFO.3.gz
+man/man3/JOYSTICK_STICK_INFO.3.gz
+man/man3/LOCK_FUNCTION.3.gz
+man/man3/LOCK_VARIABLE.3.gz
+man/man3/LZSS_PACK_DATA.3.gz
+man/man3/LZSS_UNPACK_DATA.3.gz
+man/man3/MAKE_VERSION.3.gz
+man/man3/MASK_COLOR_15.3.gz
+man/man3/MASK_COLOR_16.3.gz
+man/man3/MASK_COLOR_24.3.gz
+man/man3/MASK_COLOR_32.3.gz
+man/man3/MASK_COLOR_8.3.gz
+man/man3/MATRIX.3.gz
+man/man3/MATRIX_f.3.gz
+man/man3/MENU.3.gz
+man/man3/MENU_PLAYER.3.gz
+man/man3/MIDI.3.gz
+man/man3/PACKFILE.3.gz
+man/man3/PACKFILE_VTABLE.3.gz
+man/man3/PALETTE.3.gz
+man/man3/PAL_SIZE.3.gz
+man/man3/POLYTYPE_ATEX.3.gz
+man/man3/POLYTYPE_ATEX_LIT.3.gz
+man/man3/POLYTYPE_ATEX_MASK.3.gz
+man/man3/POLYTYPE_ATEX_MASK_LIT.3.gz
+man/man3/POLYTYPE_ATEX_MASK_TRANS.3.gz
+man/man3/POLYTYPE_ATEX_TRANS.3.gz
+man/man3/POLYTYPE_FLAT.3.gz
+man/man3/POLYTYPE_GCOL.3.gz
+man/man3/POLYTYPE_GRGB.3.gz
+man/man3/POLYTYPE_PTEX.3.gz
+man/man3/POLYTYPE_PTEX_LIT.3.gz
+man/man3/POLYTYPE_PTEX_MASK.3.gz
+man/man3/POLYTYPE_PTEX_MASK_LIT.3.gz
+man/man3/POLYTYPE_PTEX_MASK_TRANS.3.gz
+man/man3/POLYTYPE_PTEX_TRANS.3.gz
+man/man3/QUAT.3.gz
+man/man3/RGB.3.gz
+man/man3/RGB_MAP.3.gz
+man/man3/RLE_SPRITE.3.gz
+man/man3/SAMPLE.3.gz
+man/man3/SCREEN_H.3.gz
+man/man3/SCREEN_W.3.gz
+man/man3/TRACE.3.gz
+man/man3/V3D.3.gz
+man/man3/V3D_f.3.gz
+man/man3/VIRTUAL_H.3.gz
+man/man3/VIRTUAL_W.3.gz
+man/man3/ZBUFFER.3.gz
+man/man3/_getpixel.3.gz
+man/man3/_getpixel15.3.gz
+man/man3/_getpixel16.3.gz
+man/man3/_getpixel24.3.gz
+man/man3/_getpixel32.3.gz
+man/man3/_putpixel.3.gz
+man/man3/_putpixel15.3.gz
+man/man3/_putpixel16.3.gz
+man/man3/_putpixel24.3.gz
+man/man3/_putpixel32.3.gz
+man/man3/_set_color.3.gz
+man/man3/_ustrdup.3.gz
+man/man3/acquire_bitmap.3.gz
+man/man3/acquire_screen.3.gz
+man/man3/active_dialog.3.gz
+man/man3/active_menu.3.gz
+man/man3/add_clip_rect.3.gz
+man/man3/adjust_sample.3.gz
+man/man3/al_assert.3.gz
+man/man3/al_ffblk.3.gz
+man/man3/al_ffblk_get_size.3.gz
+man/man3/al_findclose.3.gz
+man/man3/al_findfirst.3.gz
+man/man3/al_findnext.3.gz
+man/man3/al_trace.3.gz
+man/man3/alert.3.gz
+man/man3/alert3.3.gz
+man/man3/allegro_404_char.3.gz
+man/man3/allegro_error.3.gz
+man/man3/allegro_exit.3.gz
+man/man3/allegro_icon.3.gz
+man/man3/allegro_id.3.gz
+man/man3/allegro_init.3.gz
+man/man3/allegro_message.3.gz
+man/man3/allocate_voice.3.gz
+man/man3/append_filename.3.gz
+man/man3/apply_matrix.3.gz
+man/man3/apply_matrix_f.3.gz
+man/man3/apply_quat.3.gz
+man/man3/arc.3.gz
+man/man3/bestfit_color.3.gz
+man/man3/bitmap_color_depth.3.gz
+man/man3/bitmap_mask_color.3.gz
+man/man3/black_palette.3.gz
+man/man3/blit.3.gz
+man/man3/bmp_read_line.3.gz
+man/man3/bmp_unwrite_line.3.gz
+man/man3/bmp_write_line.3.gz
+man/man3/broadcast_dialog_message.3.gz
+man/man3/calc_spline.3.gz
+man/man3/calibrate_joystick.3.gz
+man/man3/calibrate_joystick_name.3.gz
+man/man3/canonicalize_filename.3.gz
+man/man3/centre_dialog.3.gz
+man/man3/check_cpu.3.gz
+man/man3/circle.3.gz
+man/man3/circlefill.3.gz
+man/man3/clear_bitmap.3.gz
+man/man3/clear_keybuf.3.gz
+man/man3/clear_scene.3.gz
+man/man3/clear_to_color.3.gz
+man/man3/clear_zbuffer.3.gz
+man/man3/clip3d.3.gz
+man/man3/clip3d_f.3.gz
+man/man3/close_fli.3.gz
+man/man3/color_map.3.gz
+man/man3/config_is_hooked.3.gz
+man/man3/cpu_capabilities.3.gz
+man/man3/cpu_family.3.gz
+man/man3/cpu_model.3.gz
+man/man3/cpu_vendor.3.gz
+man/man3/create_bitmap.3.gz
+man/man3/create_bitmap_ex.3.gz
+man/man3/create_blender_table.3.gz
+man/man3/create_color_table.3.gz
+man/man3/create_datafile_index.3.gz
+man/man3/create_light_table.3.gz
+man/man3/create_lzss_pack_data.3.gz
+man/man3/create_lzss_unpack_data.3.gz
+man/man3/create_rgb_table.3.gz
+man/man3/create_sample.3.gz
+man/man3/create_scene.3.gz
+man/man3/create_sub_bitmap.3.gz
+man/man3/create_sub_zbuffer.3.gz
+man/man3/create_system_bitmap.3.gz
+man/man3/create_trans_table.3.gz
+man/man3/create_video_bitmap.3.gz
+man/man3/create_zbuffer.3.gz
+man/man3/cross_product.3.gz
+man/man3/cross_product_f.3.gz
+man/man3/d_bitmap_proc.3.gz
+man/man3/d_box_proc.3.gz
+man/man3/d_button_proc.3.gz
+man/man3/d_check_proc.3.gz
+man/man3/d_clear_proc.3.gz
+man/man3/d_ctext_proc.3.gz
+man/man3/d_edit_proc.3.gz
+man/man3/d_icon_proc.3.gz
+man/man3/d_keyboard_proc.3.gz
+man/man3/d_list_proc.3.gz
+man/man3/d_menu_proc.3.gz
+man/man3/d_radio_proc.3.gz
+man/man3/d_rtext_proc.3.gz
+man/man3/d_shadow_box_proc.3.gz
+man/man3/d_slider_proc.3.gz
+man/man3/d_text_list_proc.3.gz
+man/man3/d_text_proc.3.gz
+man/man3/d_textbox_proc.3.gz
+man/man3/d_yield_proc.3.gz
+man/man3/deallocate_voice.3.gz
+man/man3/default_palette.3.gz
+man/man3/delete_file.3.gz
+man/man3/desktop_color_depth.3.gz
+man/man3/desktop_palette.3.gz
+man/man3/destroy_bitmap.3.gz
+man/man3/destroy_compiled_sprite.3.gz
+man/man3/destroy_datafile_index.3.gz
+man/man3/destroy_font.3.gz
+man/man3/destroy_gfx_mode_list.3.gz
+man/man3/destroy_midi.3.gz
+man/man3/destroy_rle_sprite.3.gz
+man/man3/destroy_sample.3.gz
+man/man3/destroy_scene.3.gz
+man/man3/destroy_zbuffer.3.gz
+man/man3/detect_digi_driver.3.gz
+man/man3/detect_midi_driver.3.gz
+man/man3/dialog_message.3.gz
+man/man3/digi_recorder.3.gz
+man/man3/disable_hardware_cursor.3.gz
+man/man3/do_arc.3.gz
+man/man3/do_circle.3.gz
+man/man3/do_dialog.3.gz
+man/man3/do_ellipse.3.gz
+man/man3/do_line.3.gz
+man/man3/do_menu.3.gz
+man/man3/do_uconvert.3.gz
+man/man3/dot_product.3.gz
+man/man3/dot_product_f.3.gz
+man/man3/draw_character_ex.3.gz
+man/man3/draw_compiled_sprite.3.gz
+man/man3/draw_gouraud_sprite.3.gz
+man/man3/draw_lit_rle_sprite.3.gz
+man/man3/draw_lit_sprite.3.gz
+man/man3/draw_rle_sprite.3.gz
+man/man3/draw_sprite.3.gz
+man/man3/draw_sprite_ex.3.gz
+man/man3/draw_sprite_h_flip.3.gz
+man/man3/draw_sprite_v_flip.3.gz
+man/man3/draw_sprite_vh_flip.3.gz
+man/man3/draw_trans_rle_sprite.3.gz
+man/man3/draw_trans_sprite.3.gz
+man/man3/drawing_mode.3.gz
+man/man3/ellipse.3.gz
+man/man3/ellipsefill.3.gz
+man/man3/empty_string.3.gz
+man/man3/enable_hardware_cursor.3.gz
+man/man3/enable_triple_buffer.3.gz
+man/man3/ex12bit.3.gz
+man/man3/ex3buf.3.gz
+man/man3/ex3d.3.gz
+man/man3/exaccel.3.gz
+man/man3/exalpha.3.gz
+man/man3/exbitmap.3.gz
+man/man3/exblend.3.gz
+man/man3/excamera.3.gz
+man/man3/excolmap.3.gz
+man/man3/exconfig.3.gz
+man/man3/excustom.3.gz
+man/man3/exdata.3.gz
+man/man3/exdbuf.3.gz
+man/man3/exexedat.3.gz
+man/man3/exfixed.3.gz
+man/man3/exflame.3.gz
+man/man3/exflip.3.gz
+man/man3/exfont.3.gz
+man/man3/exgui.3.gz
+man/man3/exhello.3.gz
+man/man3/exists.3.gz
+man/man3/exjoy.3.gz
+man/man3/exkeys.3.gz
+man/man3/exlights.3.gz
+man/man3/exmem.3.gz
+man/man3/exmidi.3.gz
+man/man3/exmouse.3.gz
+man/man3/expackf.3.gz
+man/man3/expal.3.gz
+man/man3/expat.3.gz
+man/man3/exquat.3.gz
+man/man3/exrgbhsv.3.gz
+man/man3/exrotscl.3.gz
+man/man3/exsample.3.gz
+man/man3/exscale.3.gz
+man/man3/exscn3d.3.gz
+man/man3/exscroll.3.gz
+man/man3/exshade.3.gz
+man/man3/exspline.3.gz
+man/man3/exsprite.3.gz
+man/man3/exstars.3.gz
+man/man3/exstream.3.gz
+man/man3/exswitch.3.gz
+man/man3/exsyscur.3.gz
+man/man3/extimer.3.gz
+man/man3/extract_font_range.3.gz
+man/man3/extrans.3.gz
+man/man3/extrans2.3.gz
+man/man3/extruec.3.gz
+man/man3/exunicod.3.gz
+man/man3/exupdate.3.gz
+man/man3/exxfade.3.gz
+man/man3/exzbuf.3.gz
+man/man3/fade_from.3.gz
+man/man3/fade_from_range.3.gz
+man/man3/fade_in.3.gz
+man/man3/fade_in_range.3.gz
+man/man3/fade_interpolate.3.gz
+man/man3/fade_out.3.gz
+man/man3/fade_out_range.3.gz
+man/man3/fastline.3.gz
+man/man3/file_exists.3.gz
+man/man3/file_select_ex.3.gz
+man/man3/file_size_ex.3.gz
+man/man3/file_time.3.gz
+man/man3/find_allegro_resource.3.gz
+man/man3/find_datafile_object.3.gz
+man/man3/find_dialog_focus.3.gz
+man/man3/fix_filename_case.3.gz
+man/man3/fix_filename_slashes.3.gz
+man/man3/fixacos.3.gz
+man/man3/fixadd.3.gz
+man/man3/fixasin.3.gz
+man/man3/fixatan.3.gz
+man/man3/fixatan2.3.gz
+man/man3/fixceil.3.gz
+man/man3/fixcos.3.gz
+man/man3/fixdiv.3.gz
+man/man3/fixed.3.gz
+man/man3/fixfloor.3.gz
+man/man3/fixhypot.3.gz
+man/man3/fixmul.3.gz
+man/man3/fixsin.3.gz
+man/man3/fixsqrt.3.gz
+man/man3/fixsub.3.gz
+man/man3/fixtan.3.gz
+man/man3/fixtof.3.gz
+man/man3/fixtoi.3.gz
+man/man3/fixtorad_r.3.gz
+man/man3/fixup_datafile.3.gz
+man/man3/fli_bitmap.3.gz
+man/man3/fli_bmp_dirty_from.3.gz
+man/man3/fli_bmp_dirty_to.3.gz
+man/man3/fli_frame.3.gz
+man/man3/fli_pal_dirty_from.3.gz
+man/man3/fli_pal_dirty_to.3.gz
+man/man3/fli_palette.3.gz
+man/man3/fli_timer.3.gz
+man/man3/floodfill.3.gz
+man/man3/flush_config_file.3.gz
+man/man3/font.3.gz
+man/man3/font_has_alpha.3.gz
+man/man3/for_each_file_ex.3.gz
+man/man3/free_audio_stream_buffer.3.gz
+man/man3/free_config_entries.3.gz
+man/man3/free_lzss_pack_data.3.gz
+man/man3/free_lzss_unpack_data.3.gz
+man/man3/freeze_mouse_flag.3.gz
+man/man3/ftofix.3.gz
+man/man3/generate_332_palette.3.gz
+man/man3/generate_optimized_palette.3.gz
+man/man3/get_align_matrix.3.gz
+man/man3/get_align_matrix_f.3.gz
+man/man3/get_audio_stream_buffer.3.gz
+man/man3/get_camera_matrix.3.gz
+man/man3/get_camera_matrix_f.3.gz
+man/man3/get_clip_rect.3.gz
+man/man3/get_clip_state.3.gz
+man/man3/get_color.3.gz
+man/man3/get_color_conversion.3.gz
+man/man3/get_color_depth.3.gz
+man/man3/get_compiled_sprite.3.gz
+man/man3/get_config_argv.3.gz
+man/man3/get_config_float.3.gz
+man/man3/get_config_hex.3.gz
+man/man3/get_config_id.3.gz
+man/man3/get_config_int.3.gz
+man/man3/get_config_string.3.gz
+man/man3/get_config_text.3.gz
+man/man3/get_datafile_property.3.gz
+man/man3/get_desktop_resolution.3.gz
+man/man3/get_display_switch_mode.3.gz
+man/man3/get_executable_name.3.gz
+man/man3/get_extension.3.gz
+man/man3/get_filename.3.gz
+man/man3/get_filename_encoding.3.gz
+man/man3/get_font_range_begin.3.gz
+man/man3/get_font_range_end.3.gz
+man/man3/get_font_ranges.3.gz
+man/man3/get_gfx_mode.3.gz
+man/man3/get_gfx_mode_list.3.gz
+man/man3/get_gfx_mode_type.3.gz
+man/man3/get_hardware_volume.3.gz
+man/man3/get_midi_length.3.gz
+man/man3/get_mixer_bits.3.gz
+man/man3/get_mixer_buffer_length.3.gz
+man/man3/get_mixer_channels.3.gz
+man/man3/get_mixer_frequency.3.gz
+man/man3/get_mixer_quality.3.gz
+man/man3/get_mixer_voices.3.gz
+man/man3/get_mouse_mickeys.3.gz
+man/man3/get_palette.3.gz
+man/man3/get_palette_range.3.gz
+man/man3/get_refresh_rate.3.gz
+man/man3/get_rle_sprite.3.gz
+man/man3/get_rotation_matrix.3.gz
+man/man3/get_rotation_matrix_f.3.gz
+man/man3/get_rotation_quat.3.gz
+man/man3/get_scaling_matrix.3.gz
+man/man3/get_scaling_matrix_f.3.gz
+man/man3/get_sound_input_cap_bits.3.gz
+man/man3/get_sound_input_cap_parm.3.gz
+man/man3/get_sound_input_cap_rate.3.gz
+man/man3/get_sound_input_cap_stereo.3.gz
+man/man3/get_transformation_matrix.3.gz
+man/man3/get_transformation_matrix_f.3.gz
+man/man3/get_translation_matrix.3.gz
+man/man3/get_translation_matrix_f.3.gz
+man/man3/get_uformat.3.gz
+man/man3/get_vector_rotation_matrix.3.gz
+man/man3/get_vector_rotation_matrix_f.3.gz
+man/man3/get_vector_rotation_quat.3.gz
+man/man3/get_volume.3.gz
+man/man3/get_x_rotate_matrix.3.gz
+man/man3/get_x_rotate_matrix_f.3.gz
+man/man3/get_x_rotate_quat.3.gz
+man/man3/get_y_rotate_matrix.3.gz
+man/man3/get_y_rotate_matrix_f.3.gz
+man/man3/get_y_rotate_quat.3.gz
+man/man3/get_z_rotate_matrix.3.gz
+man/man3/get_z_rotate_matrix_f.3.gz
+man/man3/get_z_rotate_quat.3.gz
+man/man3/geta.3.gz
+man/man3/geta32.3.gz
+man/man3/geta_depth.3.gz
+man/man3/getb.3.gz
+man/man3/getb15.3.gz
+man/man3/getb16.3.gz
+man/man3/getb24.3.gz
+man/man3/getb32.3.gz
+man/man3/getb8.3.gz
+man/man3/getb_depth.3.gz
+man/man3/getg.3.gz
+man/man3/getg15.3.gz
+man/man3/getg16.3.gz
+man/man3/getg24.3.gz
+man/man3/getg32.3.gz
+man/man3/getg8.3.gz
+man/man3/getg_depth.3.gz
+man/man3/getpixel.3.gz
+man/man3/getr.3.gz
+man/man3/getr15.3.gz
+man/man3/getr16.3.gz
+man/man3/getr24.3.gz
+man/man3/getr32.3.gz
+man/man3/getr8.3.gz
+man/man3/getr_depth.3.gz
+man/man3/gfx_capabilities.3.gz
+man/man3/gfx_mode_select.3.gz
+man/man3/gfx_mode_select_ex.3.gz
+man/man3/gfx_mode_select_filter.3.gz
+man/man3/grab_font_from_bitmap.3.gz
+man/man3/gui_bg_color.3.gz
+man/man3/gui_button_proc.3.gz
+man/man3/gui_ctext_proc.3.gz
+man/man3/gui_edit_proc.3.gz
+man/man3/gui_fg_color.3.gz
+man/man3/gui_font_baseline.3.gz
+man/man3/gui_get_screen.3.gz
+man/man3/gui_list_proc.3.gz
+man/man3/gui_menu_draw_menu.3.gz
+man/man3/gui_menu_draw_menu_item.3.gz
+man/man3/gui_mg_color.3.gz
+man/man3/gui_mouse_b.3.gz
+man/man3/gui_mouse_focus.3.gz
+man/man3/gui_mouse_x.3.gz
+man/man3/gui_mouse_y.3.gz
+man/man3/gui_mouse_z.3.gz
+man/man3/gui_set_screen.3.gz
+man/man3/gui_shadow_box_proc.3.gz
+man/man3/gui_strlen.3.gz
+man/man3/gui_text_list_proc.3.gz
+man/man3/gui_textout_ex.3.gz
+man/man3/hline.3.gz
+man/man3/hook_config_section.3.gz
+man/man3/hsv_to_rgb.3.gz
+man/man3/identity_matrix.3.gz
+man/man3/identity_matrix_f.3.gz
+man/man3/identity_quat.3.gz
+man/man3/init_dialog.3.gz
+man/man3/init_menu.3.gz
+man/man3/initialise_joystick.3.gz
+man/man3/install_allegro.3.gz
+man/man3/install_int.3.gz
+man/man3/install_int_ex.3.gz
+man/man3/install_joystick.3.gz
+man/man3/install_keyboard.3.gz
+man/man3/install_keyboard_hooks.3.gz
+man/man3/install_mouse.3.gz
+man/man3/install_param_int.3.gz
+man/man3/install_param_int_ex.3.gz
+man/man3/install_sound.3.gz
+man/man3/install_sound_input.3.gz
+man/man3/install_timer.3.gz
+man/man3/is_color_font.3.gz
+man/man3/is_compatible_font.3.gz
+man/man3/is_inside_bitmap.3.gz
+man/man3/is_linear_bitmap.3.gz
+man/man3/is_memory_bitmap.3.gz
+man/man3/is_mono_font.3.gz
+man/man3/is_planar_bitmap.3.gz
+man/man3/is_relative_filename.3.gz
+man/man3/is_same_bitmap.3.gz
+man/man3/is_screen_bitmap.3.gz
+man/man3/is_sub_bitmap.3.gz
+man/man3/is_system_bitmap.3.gz
+man/man3/is_trans_font.3.gz
+man/man3/is_video_bitmap.3.gz
+man/man3/is_windowed_mode.3.gz
+man/man3/itofix.3.gz
+man/man3/joy.3.gz
+man/man3/key.3.gz
+man/man3/key_led_flag.3.gz
+man/man3/key_shifts.3.gz
+man/man3/keyboard_callback.3.gz
+man/man3/keyboard_lowlevel_callback.3.gz
+man/man3/keyboard_needs_poll.3.gz
+man/man3/keyboard_ucallback.3.gz
+man/man3/keypressed.3.gz
+man/man3/line.3.gz
+man/man3/list_config_entries.3.gz
+man/man3/list_config_sections.3.gz
+man/man3/load_bios_font.3.gz
+man/man3/load_bitmap.3.gz
+man/man3/load_bitmap_font.3.gz
+man/man3/load_bmp.3.gz
+man/man3/load_bmp_pf.3.gz
+man/man3/load_dat_font.3.gz
+man/man3/load_datafile.3.gz
+man/man3/load_datafile_callback.3.gz
+man/man3/load_datafile_object.3.gz
+man/man3/load_datafile_object_indexed.3.gz
+man/man3/load_font.3.gz
+man/man3/load_grx_font.3.gz
+man/man3/load_grx_or_bios_font.3.gz
+man/man3/load_ibk.3.gz
+man/man3/load_joystick_data.3.gz
+man/man3/load_lbm.3.gz
+man/man3/load_midi.3.gz
+man/man3/load_midi_patches.3.gz
+man/man3/load_pcx.3.gz
+man/man3/load_pcx_pf.3.gz
+man/man3/load_sample.3.gz
+man/man3/load_tga.3.gz
+man/man3/load_tga_pf.3.gz
+man/man3/load_txt_font.3.gz
+man/man3/load_voc.3.gz
+man/man3/load_voc_pf.3.gz
+man/man3/load_wav.3.gz
+man/man3/load_wav_pf.3.gz
+man/man3/lock_bitmap.3.gz
+man/man3/lock_midi.3.gz
+man/man3/lock_sample.3.gz
+man/man3/lzss_read.3.gz
+man/man3/lzss_write.3.gz
+man/man3/make_absolute_filename.3.gz
+man/man3/make_relative_filename.3.gz
+man/man3/make_trans_font.3.gz
+man/man3/makeacol.3.gz
+man/man3/makeacol32.3.gz
+man/man3/makeacol_depth.3.gz
+man/man3/makecol.3.gz
+man/man3/makecol15.3.gz
+man/man3/makecol15_dither.3.gz
+man/man3/makecol16.3.gz
+man/man3/makecol16_dither.3.gz
+man/man3/makecol24.3.gz
+man/man3/makecol32.3.gz
+man/man3/makecol8.3.gz
+man/man3/makecol_depth.3.gz
+man/man3/masked_blit.3.gz
+man/man3/masked_stretch_blit.3.gz
+man/man3/matrix_mul.3.gz
+man/man3/matrix_mul_f.3.gz
+man/man3/matrix_to_quat.3.gz
+man/man3/merge_fonts.3.gz
+man/man3/midi_loop_end.3.gz
+man/man3/midi_loop_start.3.gz
+man/man3/midi_meta_callback.3.gz
+man/man3/midi_msg_callback.3.gz
+man/man3/midi_out.3.gz
+man/man3/midi_pause.3.gz
+man/man3/midi_pos.3.gz
+man/man3/midi_recorder.3.gz
+man/man3/midi_resume.3.gz
+man/man3/midi_seek.3.gz
+man/man3/midi_sysex_callback.3.gz
+man/man3/midi_time.3.gz
+man/man3/mouse_b.3.gz
+man/man3/mouse_callback.3.gz
+man/man3/mouse_needs_poll.3.gz
+man/man3/mouse_on_screen.3.gz
+man/man3/mouse_pos.3.gz
+man/man3/mouse_sprite.3.gz
+man/man3/mouse_w.3.gz
+man/man3/mouse_x.3.gz
+man/man3/mouse_x_focus.3.gz
+man/man3/mouse_y.3.gz
+man/man3/mouse_y_focus.3.gz
+man/man3/mouse_z.3.gz
+man/man3/need_uconvert.3.gz
+man/man3/next_fli_frame.3.gz
+man/man3/normalize_vector.3.gz
+man/man3/normalize_vector_f.3.gz
+man/man3/num_joysticks.3.gz
+man/man3/object_message.3.gz
+man/man3/offer_focus.3.gz
+man/man3/open_fli.3.gz
+man/man3/open_memory_fli.3.gz
+man/man3/os_multitasking.3.gz
+man/man3/os_revision.3.gz
+man/man3/os_type.3.gz
+man/man3/os_version.3.gz
+man/man3/override_config_data.3.gz
+man/man3/override_config_file.3.gz
+man/man3/pack_fclose.3.gz
+man/man3/pack_fclose_chunk.3.gz
+man/man3/pack_feof.3.gz
+man/man3/pack_ferror.3.gz
+man/man3/pack_fgets.3.gz
+man/man3/pack_fopen.3.gz
+man/man3/pack_fopen_chunk.3.gz
+man/man3/pack_fopen_vtable.3.gz
+man/man3/pack_fputs.3.gz
+man/man3/pack_fread.3.gz
+man/man3/pack_fseek.3.gz
+man/man3/pack_fwrite.3.gz
+man/man3/pack_getc.3.gz
+man/man3/pack_igetl.3.gz
+man/man3/pack_igetw.3.gz
+man/man3/pack_iputl.3.gz
+man/man3/pack_iputw.3.gz
+man/man3/pack_mgetl.3.gz
+man/man3/pack_mgetw.3.gz
+man/man3/pack_mputl.3.gz
+man/man3/pack_mputw.3.gz
+man/man3/pack_putc.3.gz
+man/man3/pack_ungetc.3.gz
+man/man3/packfile_password.3.gz
+man/man3/palette_color.3.gz
+man/man3/persp_project.3.gz
+man/man3/persp_project_f.3.gz
+man/man3/pivot_scaled_sprite.3.gz
+man/man3/pivot_scaled_sprite_lit.3.gz
+man/man3/pivot_scaled_sprite_trans.3.gz
+man/man3/pivot_scaled_sprite_v_flip.3.gz
+man/man3/pivot_scaled_sprite_v_flip_lit.3.gz
+man/man3/pivot_scaled_sprite_v_flip_trans.3.gz
+man/man3/pivot_sprite.3.gz
+man/man3/pivot_sprite_lit.3.gz
+man/man3/pivot_sprite_trans.3.gz
+man/man3/pivot_sprite_v_flip.3.gz
+man/man3/pivot_sprite_v_flip_lit.3.gz
+man/man3/pivot_sprite_v_flip_trans.3.gz
+man/man3/play_audio_stream.3.gz
+man/man3/play_fli.3.gz
+man/man3/play_looped_midi.3.gz
+man/man3/play_memory_fli.3.gz
+man/man3/play_midi.3.gz
+man/man3/play_sample.3.gz
+man/man3/poll_joystick.3.gz
+man/man3/poll_keyboard.3.gz
+man/man3/poll_mouse.3.gz
+man/man3/poll_scroll.3.gz
+man/man3/polygon.3.gz
+man/man3/polygon3d.3.gz
+man/man3/polygon3d_f.3.gz
+man/man3/polygon_z_normal.3.gz
+man/man3/polygon_z_normal_f.3.gz
+man/man3/pop_config_state.3.gz
+man/man3/popup_dialog.3.gz
+man/man3/position_dialog.3.gz
+man/man3/position_mouse.3.gz
+man/man3/position_mouse_w.3.gz
+man/man3/position_mouse_z.3.gz
+man/man3/push_config_state.3.gz
+man/man3/put_backslash.3.gz
+man/man3/putpixel.3.gz
+man/man3/qnx_get_window.3.gz
+man/man3/qscale_matrix.3.gz
+man/man3/qscale_matrix_f.3.gz
+man/man3/qtranslate_matrix.3.gz
+man/man3/qtranslate_matrix_f.3.gz
+man/man3/quad3d.3.gz
+man/man3/quad3d_f.3.gz
+man/man3/quat_interpolate.3.gz
+man/man3/quat_mul.3.gz
+man/man3/quat_slerp.3.gz
+man/man3/quat_to_matrix.3.gz
+man/man3/radtofix_r.3.gz
+man/man3/read_sound_input.3.gz
+man/man3/readkey.3.gz
+man/man3/reallocate_voice.3.gz
+man/man3/rect.3.gz
+man/man3/rectfill.3.gz
+man/man3/register_assert_handler.3.gz
+man/man3/register_bitmap_file_type.3.gz
+man/man3/register_datafile_object.3.gz
+man/man3/register_font_file_type.3.gz
+man/man3/register_sample_file_type.3.gz
+man/man3/register_trace_handler.3.gz
+man/man3/register_uformat.3.gz
+man/man3/release_bitmap.3.gz
+man/man3/release_screen.3.gz
+man/man3/release_voice.3.gz
+man/man3/reload_config_texts.3.gz
+man/man3/remove_display_switch_callback.3.gz
+man/man3/remove_int.3.gz
+man/man3/remove_joystick.3.gz
+man/man3/remove_keyboard.3.gz
+man/man3/remove_mouse.3.gz
+man/man3/remove_param_int.3.gz
+man/man3/remove_sound.3.gz
+man/man3/remove_sound_input.3.gz
+man/man3/remove_timer.3.gz
+man/man3/render_scene.3.gz
+man/man3/replace_extension.3.gz
+man/man3/replace_filename.3.gz
+man/man3/request_refresh_rate.3.gz
+man/man3/request_scroll.3.gz
+man/man3/request_video_bitmap.3.gz
+man/man3/reserve_voices.3.gz
+man/man3/reset_fli_variables.3.gz
+man/man3/rest.3.gz
+man/man3/rest_callback.3.gz
+man/man3/retrace_count.3.gz
+man/man3/rgb_map.3.gz
+man/man3/rgb_to_hsv.3.gz
+man/man3/rotate_scaled_sprite.3.gz
+man/man3/rotate_scaled_sprite_lit.3.gz
+man/man3/rotate_scaled_sprite_trans.3.gz
+man/man3/rotate_scaled_sprite_v_flip.3.gz
+man/man3/rotate_scaled_sprite_v_flip_lit.3.gz
+man/man3/rotate_scaled_sprite_v_flip_trans.3.gz
+man/man3/rotate_sprite.3.gz
+man/man3/rotate_sprite_lit.3.gz
+man/man3/rotate_sprite_trans.3.gz
+man/man3/rotate_sprite_v_flip.3.gz
+man/man3/rotate_sprite_v_flip_lit.3.gz
+man/man3/rotate_sprite_v_flip_trans.3.gz
+man/man3/save_bitmap.3.gz
+man/man3/save_bmp.3.gz
+man/man3/save_bmp_pf.3.gz
+man/man3/save_joystick_data.3.gz
+man/man3/save_pcx.3.gz
+man/man3/save_pcx_pf.3.gz
+man/man3/save_sample.3.gz
+man/man3/save_tga.3.gz
+man/man3/save_tga_pf.3.gz
+man/man3/scancode_to_ascii.3.gz
+man/man3/scancode_to_name.3.gz
+man/man3/scare_mouse.3.gz
+man/man3/scare_mouse_area.3.gz
+man/man3/scene_gap.3.gz
+man/man3/scene_polygon3d.3.gz
+man/man3/scene_polygon3d_f.3.gz
+man/man3/screen.3.gz
+man/man3/scroll_screen.3.gz
+man/man3/select_mouse_cursor.3.gz
+man/man3/select_palette.3.gz
+man/man3/set_add_blender.3.gz
+man/man3/set_allegro_resource_path.3.gz
+man/man3/set_alpha_blender.3.gz
+man/man3/set_blender_mode.3.gz
+man/man3/set_blender_mode_ex.3.gz
+man/man3/set_burn_blender.3.gz
+man/man3/set_clip_rect.3.gz
+man/man3/set_clip_state.3.gz
+man/man3/set_close_button_callback.3.gz
+man/man3/set_color.3.gz
+man/man3/set_color_blender.3.gz
+man/man3/set_color_conversion.3.gz
+man/man3/set_color_depth.3.gz
+man/man3/set_config_data.3.gz
+man/man3/set_config_file.3.gz
+man/man3/set_config_float.3.gz
+man/man3/set_config_hex.3.gz
+man/man3/set_config_id.3.gz
+man/man3/set_config_int.3.gz
+man/man3/set_config_string.3.gz
+man/man3/set_dialog_color.3.gz
+man/man3/set_difference_blender.3.gz
+man/man3/set_display_switch_callback.3.gz
+man/man3/set_display_switch_mode.3.gz
+man/man3/set_dissolve_blender.3.gz
+man/man3/set_dodge_blender.3.gz
+man/man3/set_filename_encoding.3.gz
+man/man3/set_gfx_mode.3.gz
+man/man3/set_hardware_volume.3.gz
+man/man3/set_hue_blender.3.gz
+man/man3/set_invert_blender.3.gz
+man/man3/set_keyboard_rate.3.gz
+man/man3/set_leds.3.gz
+man/man3/set_luminance_blender.3.gz
+man/man3/set_mixer_quality.3.gz
+man/man3/set_mouse_cursor_bitmap.3.gz
+man/man3/set_mouse_range.3.gz
+man/man3/set_mouse_speed.3.gz
+man/man3/set_mouse_sprite.3.gz
+man/man3/set_mouse_sprite_focus.3.gz
+man/man3/set_multiply_blender.3.gz
+man/man3/set_palette.3.gz
+man/man3/set_palette_range.3.gz
+man/man3/set_projection_viewport.3.gz
+man/man3/set_saturation_blender.3.gz
+man/man3/set_screen_blender.3.gz
+man/man3/set_sound_input_source.3.gz
+man/man3/set_trans_blender.3.gz
+man/man3/set_ucodepage.3.gz
+man/man3/set_uformat.3.gz
+man/man3/set_volume.3.gz
+man/man3/set_volume_per_voice.3.gz
+man/man3/set_window_title.3.gz
+man/man3/set_write_alpha_blender.3.gz
+man/man3/set_zbuffer.3.gz
+man/man3/show_mouse.3.gz
+man/man3/show_os_cursor.3.gz
+man/man3/show_video_bitmap.3.gz
+man/man3/shutdown_dialog.3.gz
+man/man3/shutdown_menu.3.gz
+man/man3/simulate_keypress.3.gz
+man/man3/simulate_ukeypress.3.gz
+man/man3/solid_mode.3.gz
+man/man3/spline.3.gz
+man/man3/start_sound_input.3.gz
+man/man3/stop_audio_stream.3.gz
+man/man3/stop_midi.3.gz
+man/man3/stop_sample.3.gz
+man/man3/stop_sound_input.3.gz
+man/man3/stretch_blit.3.gz
+man/man3/stretch_sprite.3.gz
+man/man3/text_height.3.gz
+man/man3/text_length.3.gz
+man/man3/textout_centre_ex.3.gz
+man/man3/textout_ex.3.gz
+man/man3/textout_justify_ex.3.gz
+man/man3/textout_right_ex.3.gz
+man/man3/textprintf_centre_ex.3.gz
+man/man3/textprintf_ex.3.gz
+man/man3/textprintf_justify_ex.3.gz
+man/man3/textprintf_right_ex.3.gz
+man/man3/three_finger_flag.3.gz
+man/man3/transpose_font.3.gz
+man/man3/triangle.3.gz
+man/man3/triangle3d.3.gz
+man/man3/triangle3d_f.3.gz
+man/man3/uatof.3.gz
+man/man3/uconvert.3.gz
+man/man3/uconvert_ascii.3.gz
+man/man3/uconvert_size.3.gz
+man/man3/uconvert_toascii.3.gz
+man/man3/ucwidth.3.gz
+man/man3/ugetat.3.gz
+man/man3/ugetc.3.gz
+man/man3/ugetx.3.gz
+man/man3/ugetxc.3.gz
+man/man3/uinsert.3.gz
+man/man3/uisdigit.3.gz
+man/man3/uisok.3.gz
+man/man3/uisspace.3.gz
+man/man3/unload_datafile.3.gz
+man/man3/unload_datafile_object.3.gz
+man/man3/unscare_mouse.3.gz
+man/man3/unselect_palette.3.gz
+man/man3/uoffset.3.gz
+man/man3/update_dialog.3.gz
+man/man3/update_menu.3.gz
+man/man3/ureadkey.3.gz
+man/man3/uremove.3.gz
+man/man3/usetat.3.gz
+man/man3/usetc.3.gz
+man/man3/usprintf.3.gz
+man/man3/ustrcat.3.gz
+man/man3/ustrchr.3.gz
+man/man3/ustrcmp.3.gz
+man/man3/ustrcpy.3.gz
+man/man3/ustrdup.3.gz
+man/man3/ustrerror.3.gz
+man/man3/ustricmp.3.gz
+man/man3/ustrlen.3.gz
+man/man3/ustrlwr.3.gz
+man/man3/ustrncat.3.gz
+man/man3/ustrncmp.3.gz
+man/man3/ustrncpy.3.gz
+man/man3/ustrnicmp.3.gz
+man/man3/ustrpbrk.3.gz
+man/man3/ustrrchr.3.gz
+man/man3/ustrsize.3.gz
+man/man3/ustrsizez.3.gz
+man/man3/ustrstr.3.gz
+man/man3/ustrtod.3.gz
+man/man3/ustrtok.3.gz
+man/man3/ustrtok_r.3.gz
+man/man3/ustrtol.3.gz
+man/man3/ustrupr.3.gz
+man/man3/ustrzcat.3.gz
+man/man3/ustrzcpy.3.gz
+man/man3/ustrzncat.3.gz
+man/man3/ustrzncpy.3.gz
+man/man3/uszprintf.3.gz
+man/man3/utolower.3.gz
+man/man3/utoupper.3.gz
+man/man3/uvsprintf.3.gz
+man/man3/uvszprintf.3.gz
+man/man3/uwidth.3.gz
+man/man3/uwidth_max.3.gz
+man/man3/vector_length.3.gz
+man/man3/vector_length_f.3.gz
+man/man3/vline.3.gz
+man/man3/voice_check.3.gz
+man/man3/voice_get_frequency.3.gz
+man/man3/voice_get_pan.3.gz
+man/man3/voice_get_position.3.gz
+man/man3/voice_get_volume.3.gz
+man/man3/voice_ramp_volume.3.gz
+man/man3/voice_set_echo.3.gz
+man/man3/voice_set_frequency.3.gz
+man/man3/voice_set_pan.3.gz
+man/man3/voice_set_playmode.3.gz
+man/man3/voice_set_position.3.gz
+man/man3/voice_set_priority.3.gz
+man/man3/voice_set_tremolo.3.gz
+man/man3/voice_set_vibrato.3.gz
+man/man3/voice_set_volume.3.gz
+man/man3/voice_start.3.gz
+man/man3/voice_stop.3.gz
+man/man3/voice_stop_frequency_sweep.3.gz
+man/man3/voice_stop_pan_sweep.3.gz
+man/man3/voice_stop_volumeramp.3.gz
+man/man3/voice_sweep_frequency.3.gz
+man/man3/voice_sweep_pan.3.gz
+man/man3/vsync.3.gz
+man/man3/xor_mode.3.gz
+man/man3/xwin_set_window_name.3.gz
+%%DEMOS%%share/pixmaps/shooter.png
+%%DEMOS%%share/pixmaps/skater.png
 @dirrm lib/allegro/%%SHLIB_VER%%
 @dirrm lib/allegro
 @dirrm include/allegrogl/GLext
