Index: Makefile
===================================================================
--- Makefile	(revision 341319)
+++ Makefile	(working copy)
@@ -13,7 +13,7 @@
 MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Cross-platform library for games and multimedia programming
 
-LICENSE=	ZLIB BSD
+LICENSE=	ZLIB BSD3CLAUSE
 LICENSE_COMB=	dual
 
 LIB_DEPENDS+=	libjack.so:${PORTSDIR}/audio/jack \
@@ -23,9 +23,9 @@
 		libFLAC.so:${PORTSDIR}/audio/flac \
 		libphysfs.so:${PORTSDIR}/devel/physfs
 
-USES=		gmake cmake openal:soft pkgconfig
+USES=		gmake cmake openal:soft pkgconfig shebangfix
 USE_XORG=	x11 xpm xext xcursor xxf86vm xxf86dga
-USE_GL=		gl glu
+USE_GL=		glu
 USE_GNOME=	gtk20
 USE_FREETYPE=	yes
 USE_LDCONFIG=	yes
@@ -69,6 +69,29 @@
 PULSEAUDIO_CMAKE_ON=	-DWANT_PULSEAUDIO=on
 PULSEAUDIO_CMAKE_OFF=	-DWANT_PULSEAUDIO=off
 
+EXAMPLES_DEMOS=	ex_acodec ex_acodec_multi ex_audio_chain ex_audio_props \
+	ex_audio_simple ex_bitmap ex_bitmap_flip ex_bitmap_target ex_blend \
+	ex_blend2 ex_blend_bench ex_blend_test ex_blit ex_clip ex_color \
+	ex_config ex_convert ex_dir ex_disable_screensaver ex_display_events \
+	ex_display_options ex_draw ex_draw_bitmap ex_drawpixels ex_dualies \
+	ex_expose ex_file_slice ex_filter ex_font ex_font_justify ex_fs_resize \
+	ex_fs_window ex_get_path ex_gldepth ex_glext ex_haiku ex_icon \
+	ex_fs_resize ex_fs_window ex_get_path ex_gldepth ex_glext ex_haiku \
+	ex_icon ex_fs_resize ex_fs_window ex_get_path ex_gldepth ex_glext \
+	ex_haiku ex_icon ex_fs_resize ex_fs_window ex_get_path ex_gldepth \
+	ex_glext ex_haiku ex_icon ex_icon2 ex_joystick_events \
+	ex_joystick_hotplugging ex_kcm_direct ex_keyboard_events \
+	ex_keyboard_focus ex_lines ex_lockbitmap ex_logo ex_membmp ex_memfile \
+	ex_mixer_chain ex_mixer_pp ex_monitorinfo ex_mouse ex_mouse_cursor \
+	ex_mouse_events ex_mouse_focus ex_multisample ex_multiwin \
+	ex_native_filechooser ex_nodisplay ex_noframe ex_opengl \
+	ex_opengl_pixel_shader ex_path ex_path_test ex_physfs ex_pixelformat \
+	ex_premulalpha ex_premulalpha ex_prim ex_resample_test ex_resize \
+	ex_resize2 ex_rotate ex_saw ex_scale ex_stream_file ex_stream_seek \
+	ex_subbitmap ex_synth ex_threads ex_threads2 ex_timedwait ex_timer \
+	ex_transform ex_ttf ex_user_events ex_utf8 ex_vsync ex_warp_mouse \
+	ex_windows ex_winfull
+
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MDEMOS} && ${PORT_OPTIONS:MEXAMPLES}
@@ -78,10 +101,11 @@
 MASTER_SITES+=	SF/nemysisfreebsdp/${CATEGORIES}/${PORTNAME}/:icons
 DISTFILES+=	cosmic_protector.png:icons \
 		speed.png:icons
-DESKTOP_ENTRIES+="Cosmic Protector" "Protect the earth from a deadly meteor shower" "cosmic_protector" \
-		"cosmic_protector" "Game;ArcadeGame;" false
-DESKTOP_ENTRIES+="SPEED" "Simultaneous Projections Employing an Ensemble of Displays" "speed" \
-		"speed" "Game;ArcadeGame;" false
+SHEBANG_FILES+=	demos/speed/update.pl
+DESKTOP_ENTRIES+="Cosmic Protector" "Protect the earth from a deadly meteor shower" \
+		"cosmic_protector" "cosmic_protector" "Game;ArcadeGame;" false
+DESKTOP_ENTRIES+="SPEED" "Simultaneous Projections Employing an Ensemble of Displays" \
+		"speed" "speed" "Game;ArcadeGame;" false
 .else
 CMAKE_ARGS+=-DWANT_DEMO=off
 .endif
@@ -99,13 +123,27 @@
 	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${STAGEDIR}${DOCSDIR}
 	@(cd ${DOCSRCDIR2} && ${COPYTREE_SHARE} "src html" ${STAGEDIR}${DOCSDIR2})
 
-	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} . ${STAGEDIR}${EXAMPLESDIR})
+.if ${PORT_OPTIONS:MEXAMPLES}
+. for e in data ${EXAMPLES_DEMOS}
+	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} ${e} ${STAGEDIR}${EXAMPLESDIR})
+. endfor
+#	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} . ${STAGEDIR}${EXAMPLESDIR})
+	(cd ${STAGEDIR}${EXAMPLESDIR} ;${STRIP_CMD} ${EXAMPLES_DEMOS})
+	(cd ${STAGEDIR}${EXAMPLESDIR} ;${STRIP_CMD} *)
+.endif
 
 .if ${PORT_OPTIONS:MDEMOS} && ${PORT_OPTIONS:MEXAMPLES}
+#. for e in 
+#	@(cd ${WRKSRC} && ${COPYTREE_SHARE} ${e} ${STAGEDIR}${EXAMPLESDIR}/demos)
+#. endfor
 	@(cd ${WRKSRC} && ${COPYTREE_SHARE} demos ${STAGEDIR}${EXAMPLESDIR})
+
 	${INSTALL_SCRIPT} ${WRKDIR}/cosmic_protector ${STAGEDIR}${PREFIX}/bin/
 	${INSTALL_DATA} ${_DISTDIR}/cosmic_protector.png ${STAGEDIR}${PREFIX}/share/pixmaps/
 	@(cd ${STAGEDIR}${EXAMPLESDIR}/demos/cosmic_protector && ${CHMOD} 755 cosmic_protector)
+. for e in cosmic_protector speed
+	${STRIP_CMD} ${STAGEDIR}${EXAMPLESDIR}/demos/${e}/${e}
+. endfor
 	${INSTALL_SCRIPT} ${WRKDIR}/speed ${STAGEDIR}${PREFIX}/bin
 	${INSTALL_DATA} ${_DISTDIR}/speed.png ${STAGEDIR}${PREFIX}/share/pixmaps/
 	@(cd ${STAGEDIR}${EXAMPLESDIR}/demos/speed && ${CHMOD} 755 speed)
