Index: Makefile
===================================================================
--- Makefile	(revision 329200)
+++ Makefile	(working copy)
@@ -1,32 +1,30 @@
 # Created by: michaelnottebrock@gmx.net and lioux@FreeBSD.org
 # $FreeBSD$
 
-PORTNAME=	portaudio
+PORTNAME=	portaudio2
 DISTVERSION=	19_20111121
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	audio
 MASTER_SITES=	http://www.portaudio.com/archives/
 DISTNAME=	pa_stable_v${DISTVERSION}
 EXTRACT_SUFX=	.tgz
 
-MAINTAINER=	nemysis@gmx.ch
+MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Portable cross-platform Audio API
 
 LICENSE=	MIT
 
-LATEST_LINK=	${PORTNAME}19
-WRKSRC=		${WRKDIR}/${PORTNAME}
+WRKSRC=		${WRKDIR}/${PORTNAME:S/2//}
 
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS+=PKG_CONFIG_LIBDIR=${PREFIX}/libdata/pkgconfig \
 			--without-alsa \
-			--libdir=${PREFIX}/lib/${PORTNAME}2 \
-			--includedir=${PREFIX}/include/${PORTNAME}2
+			--libdir=${PREFIX}/lib/${PORTNAME} \
+			--includedir=${PREFIX}/include/${PORTNAME}
 
 USES=		gmake pkgconfig
-NO_STAGE=	yes
 USE_DOS2UNIX=	yes
-USE_LDCONFIG=	${PREFIX}/lib/${PORTNAME}2
+USE_LDCONFIG=	yes
 
 CPPFLAGS+=	-I${LOCALBASE}/include ${PTHREAD_CFLAGS}
 LDFLAGS+=	-L${LOCALBASE}/lib ${PTHREAD_LIBS}
@@ -34,16 +32,24 @@
 PORTDOCS=	*
 PORTEXAMPLES=	*
 
-DOCSDIR=	${PREFIX}/share/doc/${PORTNAME}2
-
 DOCSRCDIR1=	${WRKSRC}
 DOC_FILES1=	README.txt index.html
 
-OPTIONS_DEFINE=	JACK PATEST DOXYGEN
+OPTIONS_DEFINE=	DOCS DOXYGEN EXAMPLES JACK PATEST
 OPTIONS_DEFAULT=PATEST DOXYGEN
 
 PATEST_DESC=	PortAudio Test Programs
 DOXYGEN_DESC=	Install API documentation (requires DOCS)
+
+OPTIONS_SUB=	yes
+
+JACK_LIB_DEPENDS=	libjack.so:${PORTSDIR}/audio/jack
+JACK_CONFIGURE_ON=	--with-jack
+JACK_CONFIGURE_OFF=	--without-jack
+
+EXAMPLES_BIN=	pa_d* pa_f* paex_*
+PATEST_BIN=	pa_m* paqa_* patest*
+
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MDOXYGEN} && ${PORT_OPTIONS:MDOCS}
@@ -50,25 +56,12 @@
 BUILD_DEPENDS+=	doxygen:${PORTSDIR}/devel/doxygen
 .endif
 
-.if ${PORT_OPTIONS:MJACK}
-CONFIGURE_ARGS+=--with-jack
-LIB_DEPENDS+=	jack:${PORTSDIR}/audio/jack
-.else
-CONFIGURE_ARGS+=--without-jack
-.endif
-
-.if ${PORT_OPTIONS:MPATEST}
-PLIST_SUB+=	PATEST=""
-.else
-PLIST_SUB+=	PATEST="@comment "
-.endif
-
 post-patch:
 	@${REINPLACE_CMD} -e 's|#include <malloc.h>|#include <stdlib.h>|' ${WRKSRC}/ltmain.sh
 	@${REINPLACE_CMD} -e 's|machine/soundcard.h|sys/soundcard.h|' ${WRKSRC}/configure.in \
 		${WRKSRC}/src/hostapi/oss/pa_unix_oss.c ${WRKSRC}/src/SConscript
 	@${REINPLACE_CMD} -e 's|PACKAGE_NAME=|PACKAGE_NAME=portaudio2|' ${WRKSRC}/configure
-	@${REINPLACE_CMD} -e 's|$$(DESTDIR)$$(libdir)/pkgconfig|$$(prefix)/libdata/pkgconfig|g' \
+	@${REINPLACE_CMD} -e 's|$$(DESTDIR)$$(libdir)/pkgconfig|${STAGEDIR}$$(prefix)/libdata/pkgconfig|g' \
 		${WRKSRC}/Makefile.in
 
 .if ! ${PORT_OPTIONS:MEXAMPLES}
@@ -77,23 +70,34 @@
 .endif
 
 post-install:
-.if ${PORT_OPTIONS:MPATEST}
+.if ${PORT_OPTIONS:MEXAMPLES} || ${PORT_OPTIONS:MPATEST}
 	@(cd ${WRKSRC}/bin && for i in *; do ${MV} "$$i" "$${i}"-2; done)
-	${INSTALL_SCRIPT} ${WRKSRC}/bin/* ${PREFIX}/bin
 .endif
 
+.if ${PORT_OPTIONS:MEXAMPLES}
+.for f in ${EXAMPLES_BIN}
+	${INSTALL_SCRIPT} ${WRKSRC}/bin/${f} ${STAGEDIR}${PREFIX}/bin/
+.endfor
+.endif
+
+.if ${PORT_OPTIONS:MPATEST}
+.for f in ${PATEST_BIN}
+	${INSTALL_SCRIPT} ${WRKSRC}/bin/${f} ${STAGEDIR}${PREFIX}/bin/
+.endfor
+.endif
+
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${DOCSDIR}
+	@${MKDIR} ${STAGEDIR}${DOCSDIR}
+	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${STAGEDIR}${DOCSDIR}
 .endif
 
 .if ${PORT_OPTIONS:MDOXYGEN} && ${PORT_OPTIONS:MDOCS}
 	@cd ${WRKSRC} && doxygen
-	@(cd ${WRKSRC}/doc && ${COPYTREE_SHARE} html ${DOCSDIR})
+	@(cd ${WRKSRC}/doc && ${COPYTREE_SHARE} html ${STAGEDIR}${DOCSDIR})
 .endif
 
 .if ${PORT_OPTIONS:MEXAMPLES}
-	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} . ${EXAMPLESDIR})
+	@(cd ${WRKSRC}/examples && ${COPYTREE_SHARE} . ${STAGEDIR}${EXAMPLESDIR})
 .endif
 
 .include <bsd.port.mk>
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 329200)
+++ pkg-plist	(working copy)
@@ -40,6 +40,7 @@
 %%PATEST%%bin/patest_two_rates-2
 %%PATEST%%bin/patest_underflow-2
 %%PATEST%%bin/patest_wire-2
+%%JACK%%include/portaudio2/pa_jack.h
 include/portaudio2/portaudio.h
 lib/portaudio2/libportaudio.a
 lib/portaudio2/libportaudio.la
