===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/www/atutor/Makefile ./Makefile
--- /usr/ports/www/atutor/Makefile	2013-03-17 01:58:25.000000000 +0100
+++ ./Makefile	2013-05-28 14:15:40.000000000 +0200
@@ -4,7 +4,7 @@
 PORTNAME=	atutor
 PORTVERSION=	2.1.1
 CATEGORIES=	www
-MASTER_SITES=	SF/${PORTNAME}/ATutor%202
+MASTER_SITES=	SF/${PORTNAME}/ATutor%202/
 DISTNAME=	ATutor-${PORTVERSION}
 
 MAINTAINER=	wen@FreeBSD.org
diff -ruN --exclude=CVS /usr/ports/www/atutor/distinfo ./distinfo
--- /usr/ports/www/atutor/distinfo	2013-03-17 01:58:25.000000000 +0100
+++ ./distinfo	2013-05-28 14:17:49.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (ATutor-2.1.1.tar.gz) = fae60b293dc9db802dc7fe1685ffdb64825cff509a7bae46c9ceddb5ca0a5f75
-SIZE (ATutor-2.1.1.tar.gz) = 8100338
+SHA256 (ATutor-2.1.1.tar.gz) = 340b52210580e10956b320f11ab33e7095dd4de74c4f3ff2497d617ce4a95865
+SIZE (ATutor-2.1.1.tar.gz) = 8100482
===> Done
