===> Generating patch
===> Viewing diff with more
diff -ruN /usr/ports/multimedia/pymp/Makefile ./Makefile
--- /usr/ports/multimedia/pymp/Makefile	2013-06-02 18:56:38.000000000 +0200
+++ ./Makefile	2013-08-17 16:54:50.000000000 +0200
@@ -1,14 +1,14 @@
-# Created by:   nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/multimedia/pymp/Makefile 310871 2013-01-23 15:00:18Z makc $
 
 PORTNAME=	pymp
 PORTVERSION=	1.1
 PORTREVISION=	1
 CATEGORIES=	multimedia python
-MASTER_SITES=	http://jdolan.dyndns.org/files/
+MASTER_SITES=	SF/nemysisfreebsdp/
 DIST_SUBDIR=	python
 
-MAINTAINER=	nemysis@gmx.ch
+MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Lean, flexible frontend to MPlayer written in Python
 
 LICENSE=	GPLv2
@@ -16,20 +16,20 @@
 RUN_DEPENDS=	mplayer:${PORTSDIR}/multimedia/mplayer
 
 USE_BZIP2=	yes
+USES=		shebangfix
+SHEBANG_FILES=	${PORTNAME}
 USE_PYTHON=	yes
 USE_GNOME=	pygtk2
 
 PORTDOCS=	CHANGELOG README
 
-DESKTOP_ENTRIES="Pymp" "MPlayer frontend" \
-		"${PREFIX}/share/pixmaps/${PORTNAME}.png" \
-		"${PORTNAME}" "AudioVideo;Player;" "false"
+DESKTOP_ENTRIES="Pymp" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "AudioVideo;Player;" false
 
 .include <bsd.port.options.mk>
 
 post-patch:
-	@${REINPLACE_CMD} -e 's|/bin/bash|/bin/sh|' \
-	-e 's|PREFIX/lib/pymp|${DATADIR}|' \
+	@${REINPLACE_CMD} -e 's|PREFIX/lib/pymp|${DATADIR}|' \
 		${WRKSRC}/${PORTNAME}
 	${RM} -f ${WRKSRC}/build.py
 
@@ -40,7 +40,7 @@
 do-install:
 	${INSTALL_SCRIPT} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin/
 
-	${MKDIR} ${DATADIR}
+	@${MKDIR} ${DATADIR}
 .for d in *.py *.pyc *.pyo
 	${INSTALL_DATA} ${WRKSRC}/${d} ${DATADIR}
 .endfor
@@ -48,7 +48,7 @@
 	${INSTALL_DATA} ${WRKSRC}/${PORTNAME}.png ${PREFIX}/share/pixmaps/
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN /usr/ports/multimedia/pymp/pkg-descr ./pkg-descr
--- /usr/ports/multimedia/pymp/pkg-descr	2013-06-02 18:56:38.000000000 +0200
+++ ./pkg-descr	2013-08-17 16:18:38.000000000 +0200
@@ -3,4 +3,4 @@
 Pymp aims to blend in with your desktop environment, not stand out as many
 skinned players do. 
 
-WWW: http://jdolan.dyndns.org/trac/wiki/Pymp
+WWW: http://web.archive.org/web/20130114224609/http://jdolan.dyndns.org/trac/wiki/Pymp
===> Done
