Index: Makefile
===================================================================
--- Makefile	(revision 328992)
+++ Makefile	(working copy)
@@ -3,6 +3,7 @@
 
 PORTNAME=	libfm
 PORTVERSION=	1.1.2.2
+PORTREVISION=	1
 CATEGORIES=	x11
 MASTER_SITES=	SF/pcmanfm/PCManFM%20%2B%20Libfm%20%28tarball%20release%29/libfm%20%28required%20by%20PCManFM%29/
 
@@ -25,14 +26,11 @@
 CPPFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
 
-MAN1=		libfm-pref-apps.1
-
 OPTIONS_DEFINE=	NLS
 OPTIONS_SUB=	yes
 
 NLS_USES=	gettext
 
-NO_STAGE=	yes
 .include <bsd.port.options.mk>
 
 post-patch:
@@ -52,11 +50,10 @@
 	@(cd ${WRKDIR}/libfm-${PORTVERSION} && ${CHMOD} 755 install-sh)
 
 post-install:
-	@${MKDIR} ${PREFIX}/etc/xdg/libfm
-	${INSTALL_DATA} ${WRKSRC}/data/libfm.conf ${PREFIX}/etc/xdg/libfm/libfm.conf.sample
-	${INSTALL_DATA} ${WRKSRC}/data/pref-apps.conf ${PREFIX}/etc/xdg/libfm/pref-apps.conf.sample
-	@[ -f ${PREFIX}/etc/xdg/libfm/libfm.conf ] || \
-		${INSTALL_DATA} ${WRKSRC}/data/libfm.conf ${PREFIX}/etc/xdg/libfm/libfm.conf
-	@[ -f ${PREFIX}/etc/xdg/libfm/pref-apps.conf ] || \
-		${INSTALL_DATA} ${WRKSRC}/data/pref-apps.conf ${PREFIX}/etc/xdg/libfm/pref-apps.conf
+	@${MKDIR} ${STAGEDIR}${PREFIX}/etc/xdg/libfm
+	${INSTALL_DATA} ${WRKSRC}/data/libfm.conf \
+		${STAGEDIR}${PREFIX}/etc/xdg/libfm/libfm.conf.sample
+	${INSTALL_DATA} ${WRKSRC}/data/pref-apps.conf \
+		${STAGEDIR}${PREFIX}/etc/xdg/libfm/pref-apps.conf.sample
+
 .include <bsd.port.mk>
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 328992)
+++ pkg-plist	(working copy)
@@ -1,5 +1,4 @@
 bin/libfm-pref-apps
-include/libfm
 @unexec if cmp -s %D/etc/xdg/libfm/libfm.conf %D/etc/xdg/libfm/libfm.conf.sample; then rm -f %D/etc/xdg/libfm/libfm.conf; fi
 etc/xdg/libfm/libfm.conf.sample
 @exec if [ ! -f %D/etc/xdg/libfm/libfm.conf ] ; then cp -p %D/%F %B/libfm.conf; fi
@@ -6,6 +5,7 @@
 @unexec if cmp -s %D/etc/xdg/libfm/pref-apps.conf %D/etc/xdg/libfm/pref-apps.conf.sample; then rm -f %D/etc/xdg/libfm/pref-apps.conf; fi
 etc/xdg/libfm/pref-apps.conf.sample
 @exec if [ ! -f %D/etc/xdg/libfm/pref-apps.conf ] ; then cp -p %D/%F %B/pref-apps.conf; fi
+include/libfm
 include/libfm-1.0/fm-actions.h
 include/libfm-1.0/fm-app-chooser-combo-box.h
 include/libfm-1.0/fm-app-chooser-dlg.h
@@ -76,6 +76,7 @@
 libdata/pkgconfig/libfm-gtk.pc
 libdata/pkgconfig/libfm-gtk3.pc
 libdata/pkgconfig/libfm.pc
+man/man1/libfm-pref-apps.1.gz
 share/applications/libfm-pref-apps.desktop
 %%DATADIR%%/archivers.list
 %%DATADIR%%/ui/app-chooser.ui
@@ -143,8 +144,6 @@
 %%NLS%%share/locale/zh_CN/LC_MESSAGES/libfm.mo
 %%NLS%%share/locale/zh_TW/LC_MESSAGES/libfm.mo
 share/mime/packages/libfm.xml
-@dirrmtry etc/xdg/libfm
-@dirrmtry etc/xdg
 @dirrmtry share/mime/packages
 @dirrmtry share/mime
 %%NLS%%@dirrmtry share/locale/ur_PK/LC_MESSAGES
@@ -157,3 +156,5 @@
 @dirrm %%DATADIR%%
 @dirrmtry share/applications
 @dirrm include/libfm-1.0
+@dirrmtry etc/xdg/libfm
+@dirrmtry etc/xdg
