Index: Makefile
===================================================================
--- Makefile	(revision 329008)
+++ Makefile	(working copy)
@@ -3,6 +3,7 @@
 
 PORTNAME=	lxpanel
 PORTVERSION=	0.6.1
+PORTREVISION=	1
 CATEGORIES=	x11
 MASTER_SITES=	SF/lxde/LXPanel%20%28desktop%20panel%29/LXPanel%20${PORTVERSION}/
 
@@ -15,9 +16,14 @@
 LIB_DEPENDS=	libmenu-cache.so:${PORTSDIR}/x11/menu-cache
 RUN_DEPENDS=	${LOCALBASE}/share/desktop-directories/lxde-audio-video.directory:${PORTSDIR}/x11/lxmenu-data
 
-OPTIONS_DEFINE=	DOCS ALSA NLS
-OPTIONS_DEFAULT=NLS
+OPTIONS_DEFINE=	ALSA DOCS NLS
 
+OPTIONS_SUB=	yes
+
+NLS_USES=	gettext
+ALSA_RUN_DEPENDS=	${LINUXBASE}/lib/libasound.so.2:${PORTSDIR}/audio/linux-f10-alsa-lib
+ALSA_CONFIGURE_ENABLE=	alsa
+
 USE_XORG=	x11 xmu
 USE_GNOME=	gtk20 libwnck
 USES=		gmake pathfix pkgconfig
@@ -34,19 +40,6 @@
 
 .include <bsd.port.options.mk>
 
-.if ${PORT_OPTIONS:MALSA}
-RUN_DEPENDS+=	${LINUXBASE}/lib/libasound.so.2:${PORTSDIR}/audio/linux-f10-alsa-lib
-.else
-CONFIGURE_ARGS+=--disable-alsa
-.endif
-
-.if ${PORT_OPTIONS:MNLS}
-USES+=		gettext
-PLIST_SUB+=	NLS=""
-.else
-PLIST_SUB+=	NLS="@comment "
-.endif
-
 post-patch:
 	@${REINPLACE_CMD} -e 's,linux,sys,' \
 		${WRKSRC}/src/plugins/volume/volume-impl.c
