===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/comms/spandsp-devel/Makefile ./Makefile
--- /usr/ports/comms/spandsp-devel/Makefile	2013-05-24 15:27:25.000000000 +0200
+++ ./Makefile	2013-05-25 13:42:03.000000000 +0200
@@ -11,28 +11,30 @@
 MAINTAINER=	nemysis@gmx.ch
 COMMENT=	DSP library and software FAX machine
 
+LICENSE=	LGPL21
+
 LIB_DEPENDS=	tiff:${PORTSDIR}/graphics/tiff
 
 WRKSRC=		${WRKDIR}/spandsp-${DISTVERSION:C/pre.*//}
+
+USES=		pathfix
 USE_AUTOTOOLS=	libtool autoconf
 USE_GMAKE=	yes
-USE_GNOME=	gnomehack
-
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS=	CPPFLAGS="-I${LOCALBASE}/include" \
 		LDFLAGS="-L${LOCALBASE}/lib" \
 		-C
-
 USE_LDCONFIG=	yes
 
 CONFLICTS=	spandsp-[0-9]*
 
 OPTIONS_DEFINE=	DOXYGEN
 
+PORTDOCS=	*
+
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MDOXYGEN} && ${PORT_OPTIONS:MDOCS}
-PORTDOCS=	*
 CONFIGURE_ARGS+=--enable-doc
 BUILD_DEPENDS+=	doxygen:${PORTSDIR}/devel/doxygen \
 		docbook-xsl>=0:${PORTSDIR}/textproc/docbook-xsl
@@ -43,9 +45,9 @@
 
 post-install:
 .if ${PORT_OPTIONS:MDOXYGEN} && ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	(cd ${WRKSRC}/doc && \
-		${COPYTREE_SHARE} \* ${DOCSDIR} "-type d -or -mindepth 1")
+		${COPYTREE_SHARE} . ${DOCSDIR} "-type d -or -mindepth 1")
 .endif
 
 .include <bsd.port.mk>
===> Done
