===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/multimedia/dvdauthor/Makefile ./Makefile
--- /usr/ports/multimedia/dvdauthor/Makefile	2013-04-29 12:53:37.000000000 +0200
+++ ./Makefile	2013-05-27 12:20:46.000000000 +0200
@@ -20,12 +20,14 @@
 
 WRKSRC=		${WRKDIR}/${PORTNAME}
 
-USES=		pkgconfig iconv
+USES=		iconv pkgconfig
 USE_GNOME=	libxml2
 GNU_CONFIGURE=	yes
 USE_GMAKE=	yes
 CFLAGS+=	-I${LOCALBASE}/include -L${LOCALBASE}/lib
 
+PORTDOCS=	*
+
 DOCSRCDIR1=	${WRKSRC}
 DOC_FILES1=	AUTHORS ChangeLog README TODO
 
@@ -47,31 +49,27 @@
 
 .include <bsd.port.options.mk>
 
-.if ${PORT_OPTIONS:MNTSC}
-.if ${PORT_OPTIONS:MVIDEO-FORMAT}
+.if ${PORT_OPTIONS:MNTSC} && ${PORT_OPTIONS:MVIDEO-FORMAT}
 CONFIGURE_ENV+=	--enable-default-video-format=NTSC
 .endif
-.endif
 
-.if ${PORT_OPTIONS:MPAL}
-.if ${PORT_OPTIONS:MVIDEO-FORMAT}
+.if ${PORT_OPTIONS:MPAL} && ${PORT_OPTIONS:MVIDEO-FORMAT}
 CONFIGURE_ENV+=	--enable-default-video-format=PAL
 .endif
-.endif
 
 .if ${PORT_OPTIONS:MIMAGEMAGICK}
 LIB_DEPENDS+=	Magick:${PORTSDIR}/graphics/ImageMagick
 .endif
 
 pre-configure:
-.if !${PORT_OPTIONS:MIMAGEMAGICK}
+.if ! ${PORT_OPTIONS:MIMAGEMAGICK}
 	@${REINPLACE_CMD} 's|usemagick=1|usemagick=0|g' \
 		${WRKSRC}/configure
 .endif
 
 post-install:
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${DOCSDIR}
 	${INSTALL_DATA} ${DOC_FILES2:S|^|${DOCSRCDIR2}/|} ${DOCSDIR}
 .endif
diff -ruN --exclude=CVS /usr/ports/multimedia/dvdauthor/pkg-plist ./pkg-plist
--- /usr/ports/multimedia/dvdauthor/pkg-plist	2013-01-14 17:54:53.000000000 +0100
+++ ./pkg-plist	2013-03-25 20:50:45.000000000 +0100
@@ -4,13 +4,7 @@
 bin/mpeg2desc
 bin/spumux
 bin/spuunmux
-%%PORTDOCS%%%%DOCSDIR%%/AUTHORS
-%%PORTDOCS%%%%DOCSDIR%%/ChangeLog
-%%PORTDOCS%%%%DOCSDIR%%/README
-%%PORTDOCS%%%%DOCSDIR%%/README.FreeBSD
-%%PORTDOCS%%%%DOCSDIR%%/TODO
 %%DATADIR%%/common.xsd
 %%DATADIR%%/dvdauthor.xsd
 %%DATADIR%%/spumux.xsd
 @dirrm %%DATADIR%%
-%%PORTDOCS%%@dirrm %%DOCSDIR%%
===> Done
