Index: Makefile
===================================================================
--- Makefile	(revision 328620)
+++ Makefile	(working copy)
@@ -3,7 +3,7 @@
 
 PORTNAME=	dumb
 PORTVERSION=	0.9.3
-PORTREVISION=	3
+PORTREVISION=	4
 CATEGORIES=	audio
 MASTER_SITES=	SF
 
@@ -13,35 +13,55 @@
 USES=		gmake
 NO_STAGE=	yes
 
-OPTIONS_DEFINE=	ALLEGRO
+OPTIONS_RADIO=	ALLEGRO
+OPTIONS_RADIO_ALLEGRO=	ALLEGRO ALLEGRO_DEVEL
+
+OPTIONS_DEFAULT=	ALLEGRO
+
+OPTIONS_SUB=	yes
+
 ALLEGRO_DESC=	Enable allegro support
+ALLEGRO_DEVEL_DESC=	Enable allegro development version
 
+ALLEGRO_LIB_DEPENDS=		liballeg.so:${PORTSDIR}/devel/allegro
+ALLEGRO_DEVEL_LIB_DEPENDS=	liballegro.so:${PORTSDIR}/devel/allegro-devel
+
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MALLEGRO}
-LIB_DEPENDS+=	alleg:${PORTSDIR}/devel/allegro
-PLIST_SUB+=	ALLEGRO=""
-MAKE_ARGS=	_PORT_ALLEGRO_CFLAGS="`allegro-config --cflags`"
-MAKE_ARGS+=	LINK_ALLEGRO="-lm -L${LOCALBASE}/lib -lalleg"
+SET_ALLEGRO=	yes
 PKGNAMESUFFIX=	-allegro
 CONFLICTS=	dumb-[0-9]*
 .else
-PLIST_SUB+=	ALLEGRO="@comment "
-MAKE_ARGS=	_PORT_ALLEGRO_CFLAGS=
 CONFLICTS=	dumb-allegro-[0-9]*
 .endif
 
+.if ${PORT_OPTIONS:MALLEGRO_DEVEL}
+SET_ALLEGRO=	yes
+PKGNAMESUFFIX=	-allegro-devel
+CONFLICTS=	dumb-[0-9]*
+.else
+CONFLICTS=	dumb-allegro-devel-[0-9]*
+.endif
+
+.if defined(SET_ALLEGRO)
+MAKE_ARGS+=	_PORT_ALLEGRO_CFLAGS="`allegro-config --cflags`"
+.else
+MAKE_ARGS+=	_PORT_ALLEGRO_CFLAGS=
+.endif
+
 .if ${ARCH} == "amd64" || ${ARCH} == "ia64" || ${ARCH} == "sparc64"
 CFLAGS+=	-fPIC
 .endif
 
-pre-configure:
-	@${CP} ${FILESDIR}/config.txt ${WRKSRC}/make
-	@${REINPLACE_CMD} -e "s,@LOCALBASE@,${LOCALBASE}," ${WRKSRC}/make/config.txt
-.if ${PORT_OPTIONS:MALLEGRO}
-	@${REINPLACE_CMD} -e "s,@ALLEGRO@,," ${WRKSRC}/make/config.txt
+post-patch:
+	${CP} ${FILESDIR}/config.txt ${WRKSRC}/make
+	${REINPLACE_CMD} -e 's|@LOCALBASE@|${PREFIX}|' ${WRKSRC}/make/config.txt
+
+.if defined(SET_ALLEGRO)
+	${REINPLACE_CMD} -e 's|@ALLEGRO@ ||' ${WRKSRC}/make/config.txt
 .else
-	@${REINPLACE_CMD} -e "s,@ALLEGRO@,#," ${WRKSRC}/make/config.txt
+	${REINPLACE_CMD} -e 's|@ALLEGRO@ |#|' ${WRKSRC}/make/config.txt
 .endif
 
 .include <bsd.port.mk>
Index: pkg-plist
===================================================================
--- pkg-plist	(revision 328620)
+++ pkg-plist	(working copy)
@@ -1,7 +1,10 @@
 %%ALLEGRO%%bin/dumbplay
+%%ALLEGRO_DEVEL%%bin/dumbplay
 bin/dumbout
 bin/dumb2wav
 include/dumb.h
 %%ALLEGRO%%include/aldumb.h
+%%ALLEGRO_DEVEL%%include/aldumb.h
 lib/libdumb.a
 %%ALLEGRO%%lib/libaldmb.a
+%%ALLEGRO_DEVEL%%lib/libaldmb.a
