===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/audio/mp3info/Makefile ./Makefile
--- /usr/ports/audio/mp3info/Makefile	2013-01-14 17:54:18.000000000 +0100
+++ ./Makefile	2013-04-02 18:22:28.000000000 +0200
@@ -1,14 +1,10 @@
-# New ports collection makefile for:	mp3info
-# Date created:				19 June 1998
-# Whom:					Andrey Zakhvatov
-#
+# Created by: Andrey Zakhvatov
 # $FreeBSD: head/audio/mp3info/Makefile 310042 2013-01-07 12:33:59Z sylvio $
-#
 
 PORTNAME=	mp3info
 PORTVERSION=	0.8.5
 CATEGORIES=	audio
-MASTER_SITES=	${MASTER_SITE_SUNSITE}
+MASTER_SITES=	SUNSITE
 MASTER_SITE_SUBDIR=	apps/sound/mp3-utils/mp3info
 DISTNAME=	${PORTNAME}-${PORTVERSION}a
 EXTRACT_SUFX=	.tgz
@@ -32,14 +28,12 @@
 
 ALL_TARGET=	doc mp3info
 
-.include <bsd.port.pre.mk>
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MGTK2}
 PKGNAMESUFFIX=	-gtk
 USE_GNOME+=	gtk20
 ALL_TARGET+=	gmp3info
-MAKE_ARGS=	install-gmp3info
 PLIST_SUB+=	WITH_GTK=""
 .else
 PLIST_SUB+=	WITH_GTK="@comment "
@@ -47,12 +41,15 @@
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/mp3info ${PREFIX}/bin
-	${INSTALL_MAN} ${WRKSRC}/mp3info.1 ${PREFIX}/man/man1
-.if !defined(NOPORTDOCS)
+.if ${PORT_OPTIONS:MGTK2}
+	${INSTALL_PROGRAM} ${WRKSRC}/gmp3info ${PREFIX}/bin
+.endif
+
+	${INSTALL_MAN} ${WRKSRC}/${MAN1} ${MAN1PREFIX}/man/man1
+
+.if ${PORT_OPTIONS:MDOCS}
 	@${MKDIR} ${DOCSDIR}
-.for docs in ${PORTDOCS}
-	${INSTALL_DATA} ${WRKSRC}/${docs} ${DOCSDIR}
-.endfor
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
-.include <bsd.port.post.mk>
+.include <bsd.port.mk>
===> Done
