===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/audio/py-pyacoustid/Makefile ./Makefile
--- /usr/ports/audio/py-pyacoustid/Makefile	2013-02-15 20:10:11.000000000 +0100
+++ ./Makefile	2013-04-02 10:24:25.000000000 +0200
@@ -13,19 +13,19 @@
 LICENSE=	MIT
 
 LIB_DEPENDS=	chromaprint:${PORTSDIR}/audio/chromaprint
+BUILD_DEPENDS=	easy_install:${PORTSDIR}/devel/py-distribute
 
 USE_PYTHON=	yes
-USE_PYDISTUTILS=	easy_install
+USE_PYDISTUTILS=	yes
+PYDISTUTILS_NOEGGINFO=	yes
 PLIST_FILES=	%%PYTHON_SITELIBDIR%%/acoustid.py \
 		%%PYTHON_SITELIBDIR%%/chromaprint.py
 
-.include <bsd.port.pre.mk>
-
 do-build:
 	@(cd ${BUILD_WRKSRC}; ${SETENV} ${MAKE_ENV} ${PYTHON_CMD} \
 		${PYSETUP} build)
 
 do-install:
-	cd ${WRKSRC}/build/lib && ${COPYTREE_SHARE} \* ${PYTHONPREFIX_SITELIBDIR}/
+	@(cd ${WRKSRC}/build/lib && ${COPYTREE_SHARE} . ${PYTHONPREFIX_SITELIBDIR})
 
-.include <bsd.port.post.mk>
+.include <bsd.port.mk>
===> Done
