===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/anki/Makefile ./Makefile
--- /usr/ports/games/anki/Makefile	2013-06-02 18:54:54.000000000 +0200
+++ ./Makefile	2013-06-11 23:23:29.000000000 +0200
@@ -2,7 +2,7 @@
 # $FreeBSD: head/games/anki/Makefile 316434 2013-04-24 13:36:18Z ak $
 
 PORTNAME=	anki
-PORTVERSION=	2.0.8
+PORTVERSION=	2.0.9
 CATEGORIES=	games python
 MASTER_SITES=	http://ankisrs.net/download/mirror/
 EXTRACT_SUFX=	.tgz
@@ -13,9 +13,7 @@
 
 LICENSE=	AGPLv3 # (or later)
 
-BUILD_DEPENDS=	update-mime-database:${PORTSDIR}/misc/shared-mime-info
-RUN_DEPENDS:=	${BUILD_DEPENDS} \
-		${PYTHON_PKGNAMEPREFIX}pyaudio>0:${PORTSDIR}/audio/py-pyaudio \
+RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}pyaudio>0:${PORTSDIR}/audio/py-pyaudio \
 		${PYTHON_PKGNAMEPREFIX}sqlalchemy>0:${PORTSDIR}/databases/py-sqlalchemy \
 		${PYTHON_PKGNAMEPREFIX}sqlite3>0:${PORTSDIR}/databases/py-sqlite3 \
 		${PYTHON_PKGNAMEPREFIX}qt4-network>=4.4:${PORTSDIR}/net/py-qt4-network \
@@ -23,15 +21,17 @@
 		${PYTHON_PKGNAMEPREFIX}httplib2>0:${PORTSDIR}/www/py-httplib2 \
 		${PYTHON_PKGNAMEPREFIX}qt4-webkit>=4.4:${PORTSDIR}/www/py-qt4-webkit
 
+USES=		desktop-file-utils gettext:run shared-mime-info
 USE_PYTHON=	-2.7
 USE_QT4=	network_run webkit_run
-USES=		gettext:run
 
 MAN1=		${PORTNAME}.1
 
 post-patch:
 	@${REINPLACE_CMD} -e 's|/usr/share/|${PREFIX}/share/|' \
 		${WRKSRC}/runanki
+	@${REINPLACE_CMD} -e 's|^Categories=.*|Categories=Education;Languages;KDE;Qt;Game;|' \
+		${WRKSRC}/${PORTNAME}.desktop
 
 do-build:
 	@${PYTHON_CMD} -m compileall ${WRKSRC}
@@ -56,6 +56,5 @@
 	${INSTALL_DATA} ${WRKSRC}/${p} \
 		${PREFIX}/share/pixmaps
 .endfor
-	@-update-mime-database ${PREFIX}/share/mime
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/anki/distinfo ./distinfo
--- /usr/ports/games/anki/distinfo	2013-06-02 18:54:54.000000000 +0200
+++ ./distinfo	2013-06-11 22:24:36.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (python/anki-2.0.8.tgz) = dd670b02bf6b5c98dcfa9e1606ac00f6e1046f8debb5812e0f383ffe4778b868
-SIZE (python/anki-2.0.8.tgz) = 2924821
+SHA256 (python/anki-2.0.9.tgz) = bc6acc7c89f94df940057fcf5cc459233cad51388823429699c4f98d94160560
+SIZE (python/anki-2.0.9.tgz) = 2989053
diff -ruN --exclude=CVS /usr/ports/games/anki/pkg-descr ./pkg-descr
--- /usr/ports/games/anki/pkg-descr	2013-06-02 18:54:54.000000000 +0200
+++ ./pkg-descr	2013-02-09 17:55:40.000000000 +0100
@@ -11,3 +11,4 @@
 provided for Russian.
 
 WWW: http://ankisrs.net/
+WWW: http://ankisrs.net/anki2.html
diff -ruN --exclude=CVS /usr/ports/games/anki/pkg-plist ./pkg-plist
--- /usr/ports/games/anki/pkg-plist	2013-06-02 18:54:54.000000000 +0200
+++ ./pkg-plist	2013-06-11 23:09:29.000000000 +0200
@@ -55,6 +55,9 @@
 %%DATADIR%%/anki/importing/noteimp.py
 %%DATADIR%%/anki/importing/noteimp.pyc
 %%DATADIR%%/anki/importing/noteimp.pyo
+%%DATADIR%%/anki/importing/pauker.py
+%%DATADIR%%/anki/importing/pauker.pyc
+%%DATADIR%%/anki/importing/pauker.pyo
 %%DATADIR%%/anki/importing/supermemo_xml.py
 %%DATADIR%%/anki/importing/supermemo_xml.pyc
 %%DATADIR%%/anki/importing/supermemo_xml.pyo
@@ -673,8 +676,8 @@
 %%DATADIR%%/tests/support/invalid-ords.anki
 %%DATADIR%%/tests/support/media.apkg
 %%DATADIR%%/tests/support/mnemo.db
-%%DATADIR%%/tests/support/suspended12.anki
 %%DATADIR%%/tests/support/supermemo1.xml
+%%DATADIR%%/tests/support/suspended12.anki
 %%DATADIR%%/tests/support/text-2fields.txt
 %%DATADIR%%/tests/support/text-tags.txt
 %%DATADIR%%/tests/support/text-update.txt
@@ -727,13 +730,10 @@
 %%DATADIR%%/tools/build_ui.sh
 %%DATADIR%%/tools/tests.sh
 share/applications/anki.desktop
-share/mime/application/x-anki.xml
-share/mime/application/x-apkg.xml
 share/mime/packages/anki.xml
 share/pixmaps/anki.png
 share/pixmaps/anki.xpm
 @dirrmtry share/mime/packages
-@dirrmtry share/mime/application
 @dirrmtry share/mime
 @dirrmtry share/applications
 @dirrm %%DATADIR%%/tools
@@ -844,9 +844,8 @@
 @dirrm %%DATADIR%%/designer
 @dirrm %%DATADIR%%/aqt/forms
 @dirrm %%DATADIR%%/aqt
+@dirrm %%DATADIR%%/anki/~
 @dirrm %%DATADIR%%/anki/template
 @dirrm %%DATADIR%%/anki/importing
 @dirrm %%DATADIR%%/anki
 @dirrm %%DATADIR%%
-@exec %%LOCALBASE%%/bin/update-mime-database %D/share/mime > /dev/null || /usr/bin/true
-@unexec %%LOCALBASE%%/bin/update-mime-database %D/share/mime > /dev/null || /usr/bin/true
===> Done
