===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/asteroid/Makefile ./Makefile
--- /usr/ports/games/asteroid/Makefile	2013-04-29 12:53:31.000000000 +0200
+++ ./Makefile	2013-05-25 15:50:43.000000000 +0200
@@ -14,14 +14,14 @@
 GH_ACCOUNT=	chazomaticus
 GH_PROJECT=	asteroid
 GH_TAGNAME=	master
-GH_COMMIT=	6d23595
+GH_COMMIT=	ffeb0fd
 
+USES=		cmake desktop-file-utils
 USE_GL=		glut
 USE_SDL=	mixer sdl
-USE_GNOME=	desktopfileutils gtk20
+USE_GNOME=	gtk20
 USE_XORG=	xmu
 INSTALLS_ICONS=	yes
-USES=		cmake
 
 PORTDOCS=	AUTHORS NEWS README.md
 
@@ -29,9 +29,8 @@
 
 post-install:
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
-	@-update-desktop-database -q
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/asteroid/distinfo ./distinfo
--- /usr/ports/games/asteroid/distinfo	2013-02-19 23:46:57.000000000 +0100
+++ ./distinfo	2013-05-25 15:50:48.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (asteroid-1.2.1.tar.gz) = 83c22a15bce77b4d040cbdae3c19f8fe87197a3202a1c70415deb2744b5f1c0e
-SIZE (asteroid-1.2.1.tar.gz) = 307691
+SHA256 (asteroid-1.2.1.tar.gz) = e6cc3617428b686293b0972d8808e07e8c8f07e8b7ecfe1168f108a3987a4b04
+SIZE (asteroid-1.2.1.tar.gz) = 307676
diff -ruN --exclude=CVS /usr/ports/games/asteroid/pkg-descr ./pkg-descr
--- /usr/ports/games/asteroid/pkg-descr	2013-02-19 23:46:57.000000000 +0100
+++ ./pkg-descr	2013-05-25 15:35:20.000000000 +0200
@@ -18,4 +18,4 @@
 to figure out the different powerup types.
 
 
-WWW: http://chazomaticus.github.com/asteroid/
+WWW: http://chazomaticus.github.io/asteroid/
diff -ruN --exclude=CVS /usr/ports/games/asteroid/pkg-plist ./pkg-plist
--- /usr/ports/games/asteroid/pkg-plist	2013-04-29 12:53:31.000000000 +0200
+++ ./pkg-plist	2013-04-29 21:57:50.000000000 +0200
@@ -10,13 +10,7 @@
 %%DATADIR%%/sfx/spawn_powerup.wav
 %%DATADIR%%/sfx/spawn_shot.wav
 share/icons/hicolor/scalable/apps/asteroid.svg
-@dirrmtry share/icons/hicolor/scalable/apps
-@dirrmtry share/icons/hicolor/scalable
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrm %%DATADIR%%/sfx
 @dirrm %%DATADIR%%/gfx
 @dirrm %%DATADIR%%
 @dirrmtry share/applications
-@exec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
-@unexec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
===> Done
