===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/avoision/Makefile ./Makefile
--- /usr/ports/games/avoision/Makefile	2013-01-14 17:55:28.000000000 +0100
+++ ./Makefile	2013-03-16 00:52:50.000000000 +0100
@@ -1,18 +1,14 @@
-# New Ports collection makefile for:	Avoision
-# Date created:		2012-05-17
-# Whom:			nemysis@gmx.ch
-#
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/avoision/Makefile 303309 2012-08-29 03:02:15Z jgh $
-#
 
 PORTNAME=	avoision
 PORTVERSION=	1.1
-PORTREVISION=	2
+PORTREVISION=	3
 CATEGORIES=	games
 MASTER_SITES=	SF/avsn/
 
 MAINTAINER=	nemysis@gmx.ch
-COMMENT=	A simple collect-dots-avoid-blocks game
+COMMENT=	Simple collect-dots-avoid-blocks game
 
 LICENSE=	GPLv2 CCbyNCSA3
 LICENSE_COMB=	dual
@@ -35,10 +31,10 @@
 
 PLIST_FILES=	bin/${PORTNAME} \
 		share/applications/${PORTNAME}.desktop \
+		share/avoision/avoision.zip \
 		share/pixmaps/${PORTNAME}.png
 PLIST_DIRSTRY=	share/applications
 
-PORTDATA=	*
 PORTDOCS=	ChangeLog
 
 .include <bsd.port.options.mk>
@@ -46,7 +42,7 @@
 post-install:
 .if ${PORT_OPTIONS:MDOCS}
 	${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/ChangeLog ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
===> Done
