===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/barbie_seahorse_adventures/Makefile ./Makefile
--- /usr/ports/games/barbie_seahorse_adventures/Makefile	2013-03-13 12:33:23.000000000 +0100
+++ ./Makefile	2013-05-25 16:15:11.000000000 +0200
@@ -1,4 +1,4 @@
-# Created by:	nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/barbie_seahorse_adventures/Makefile 314003 2013-03-12 22:15:14Z beech $
 
 PORTNAME=	barbie_seahorse_adventures
@@ -31,7 +31,7 @@
 SUB_FILES=	${PORTNAME}
 
 DESKTOP_ENTRIES="Barbie Seahorse Adventures" "${COMMENT}" "${PORTNAME}" \
-		"${PORTNAME}" "Game;ArcadeGame;" ${FALSE}
+		"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
 
@@ -42,7 +42,7 @@
 
 do-install:
 	${INSTALL_SCRIPT} ${WRKDIR}/${PORTNAME} ${PREFIX}/bin
-	${MKDIR} ${DATADIR}
+	@${MKDIR} ${DATADIR}
 	${INSTALL_SCRIPT} ${WRKSRC}/*.py ${DATADIR}
 	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "data lib" ${DATADIR})
 
@@ -55,7 +55,7 @@
 	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/barbie_seahorse_adventures/pkg-plist ./pkg-plist
--- /usr/ports/games/barbie_seahorse_adventures/pkg-plist	2013-03-13 12:33:23.000000000 +0100
+++ ./pkg-plist	2013-03-23 13:16:20.000000000 +0100
@@ -421,18 +421,6 @@
 share/icons/hicolor/72x72/apps/barbie_seahorse_adventures.png
 share/icons/hicolor/96x96/apps/barbie_seahorse_adventures.png
 share/pixmaps/barbie_seahorse_adventures.png
-@dirrmtry share/icons/hicolor/96x96/apps
-@dirrmtry share/icons/hicolor/96x96
-@dirrmtry share/icons/hicolor/72x72/apps
-@dirrmtry share/icons/hicolor/72x72
-@dirrmtry share/icons/hicolor/64x64/apps
-@dirrmtry share/icons/hicolor/64x64
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrm %%DATADIR%%/lib/pgu/gui
 @dirrm %%DATADIR%%/lib/pgu
 @dirrm %%DATADIR%%/lib
===> Done
