===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/bubble-chains/Makefile ./Makefile
--- /usr/ports/games/bubble-chains/Makefile	2013-04-29 12:53:31.000000000 +0200
+++ ./Makefile	2013-05-25 16:30:29.000000000 +0200
@@ -1,4 +1,4 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/bubble-chains/Makefile 316755 2013-04-28 21:38:56Z beech $
 
 PORTNAME=	bubble-chains
@@ -29,8 +29,7 @@
 
 PORTDOCS=	README
 
-DESKTOP_ENTRIES="Bubble Chains" "${COMMENT}" \
-		"${PREFIX}/share/pixmaps/${PORTNAME}.png" \
+DESKTOP_ENTRIES="Bubble Chains" "${COMMENT}" "${PORTNAME}" \
 		"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
@@ -60,7 +59,7 @@
 		${PREFIX}/share/pixmaps/${PORTNAME}.ico
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
