===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/bugsquish/Makefile ./Makefile
--- /usr/ports/games/bugsquish/Makefile	2013-03-18 23:43:05.000000000 +0100
+++ ./Makefile	2013-05-27 23:29:47.000000000 +0200
@@ -28,9 +28,10 @@
 	@${REINPLACE_CMD} -e 's|CFLAGS=-Wall -O2|CFLAGS+=|' \
 	-e 's|SDL_CFLAGS :=|SDL_CFLAGS+=|' \
 	-e 's|SDL_LDFLAGS :=|SDL_LDFLAGS+=|' ${WRKSRC}/Makefile
+	@${FIND} ${WRKDIR} -name .xvpics | ${XARGS} ${RM} -rf
 
 do-install:
-	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin/
+	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin
 
 .  for d in embedded images music sounds
 	@(cd ${WRKSRC}/data && ${COPYTREE_SHARE} ${d} ${DATADIR})
@@ -39,7 +40,7 @@
 	${LN} -sf ${PREFIX}/share/bugsquish/embedded/images/icon.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/bugsquish/pkg-plist ./pkg-plist
--- /usr/ports/games/bugsquish/pkg-plist	2013-03-18 23:43:05.000000000 +0100
+++ ./pkg-plist	2013-03-26 19:57:27.000000000 +0100
@@ -1,6 +1,4 @@
 bin/bugsquish
-%%DATADIR%%/embedded/images/.xvpics/start-cont.png
-%%DATADIR%%/embedded/images/.xvpics/title.png
 %%DATADIR%%/embedded/images/arm1.png
 %%DATADIR%%/embedded/images/arm2.png
 %%DATADIR%%/embedded/images/arm3.png
@@ -114,7 +112,6 @@
 @dirrm %%DATADIR%%/sounds
 @dirrm %%DATADIR%%/music
 @dirrm %%DATADIR%%/images
-@dirrm %%DATADIR%%/embedded/images/.xvpics
 @dirrm %%DATADIR%%/embedded/images
 @dirrm %%DATADIR%%/embedded
 @dirrm %%DATADIR%%
===> Done
