===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/cretan-hubris/Makefile ./Makefile
--- /usr/ports/games/cretan-hubris/Makefile	2013-01-14 17:55:24.000000000 +0100
+++ ./Makefile	2013-05-25 16:44:30.000000000 +0200
@@ -1,9 +1,5 @@
-# New Ports collection makefile for:	Cretan Hubris
-# Date created:		2012-05-18
-# Whom:			nemysis@gmx.ch
-#
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/cretan-hubris/Makefile 303307 2012-08-29 02:14:30Z avilla $
-#
 
 PORTNAME=	cretan-hubris
 PORTVERSION=	0.2
@@ -27,17 +23,18 @@
 		physfs.1:${PORTSDIR}/devel/physfs
 RUN_DEPENDS=	zip:${PORTSDIR}/archivers/zip
 
-GNU_CONFIGURE=	yes
-ALL_TARGET=	${PORTNAME}
+USES=		pkgconfig
 USE_SDL=	sdl sound
 USE_GL=		glu
+GNU_CONFIGURE=	yes
+ALL_TARGET=	${PORTNAME}
 MAKE_JOBS_SAFE=	yes
 
 PLIST_FILES=	bin/${PORTNAME} \
 		share/applications/${PORTNAME}.desktop \
-		share/pixmaps/${PORTNAME}.png
+		share/pixmaps/${PORTNAME}.png \
+		%%DATADIR%%/${PORTNAME}.zip
+PLIST_DIRS=	%%DATADIR%%
 PLIST_DIRSTRY=	share/applications
 
-PORTDATA=	*
-
 .include <bsd.port.mk>
===> Done
