===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/cutemaze/Makefile ./Makefile
--- /usr/ports/games/cutemaze/Makefile	2013-01-14 17:55:38.000000000 +0100
+++ ./Makefile	2013-05-25 16:49:49.000000000 +0200
@@ -1,8 +1,9 @@
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/cutemaze/Makefile 303823 2012-09-07 20:08:28Z ak $
 
 PORTNAME=	cutemaze
 PORTVERSION=	1.1.0
-PORTREVISION=	3
+PORTREVISION=	4
 CATEGORIES=	games
 MASTER_SITES=	http://gottcode.org/${PORTNAME}/
 DISTNAME=	${PORTNAME}-${DISTVERSION}-src
@@ -15,6 +16,7 @@
 WRKSRC=		${WRKDIR}/${PORTNAME}-${PORTVERSION}
 
 USE_BZIP2=	yes
+Uses=		desktop-file-utils
 USE_QT4=	moc_build qmake_build rcc_build uic_build iconengines imageformats
 INSTALLS_ICONS=	yes
 MAKE_JOBS_SAFE=	yes
@@ -23,23 +25,20 @@
 
 .include <bsd.port.options.mk>
 
+post-extract:
+	@${RM} -f ${WRKSRC}/icons/oxygen/LGPL
+
 do-configure:
 	@cd ${WRKSRC} && ${SETENV} ${CONFIGURE_ENV} \
-		${QMAKE} ${QMAKEFLAGS} PREFIX=${PREFIX} ${WRKSRC}/${PORTNAME}.pro
+		${QMAKE} ${QMAKEFLAGS} PREFIX=${PREFIX}
 
 post-install:
-# Icons
-	@(cd ${WRKSRC}/icons && ${COPYTREE_SHARE} oxygen ${PREFIX}/share/icons/)
-
-# Pixmaps
-	${INSTALL_DATA} ${WRKSRC}/icons/${PORTNAME}.png ${PREFIX}/share/pixmaps/
+	@(cd ${WRKSRC}/icons && ${COPYTREE_SHARE} oxygen ${PREFIX}/share/icons)
+	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
-# Documentation
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
-.  for f in ${PORTDOCS}
-	${INSTALL_DATA} ${WRKSRC}/${f} ${DOCSDIR}
-.  endfor
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/cutemaze/pkg-plist ./pkg-plist
--- /usr/ports/games/cutemaze/pkg-plist	2013-01-14 17:55:38.000000000 +0100
+++ ./pkg-plist	2013-05-21 18:10:12.000000000 +0200
@@ -19,7 +19,6 @@
 share/icons/oxygen/22x22/media-playback-pause.png
 share/icons/oxygen/22x22/zoom-in.png
 share/icons/oxygen/22x22/zoom-out.png
-share/icons/oxygen/LGPL
 share/icons/oxygen/scalable/application-exit.svgz
 share/icons/oxygen/scalable/document-new.svgz
 share/icons/oxygen/scalable/games-config-options.svgz
@@ -34,8 +33,4 @@
 @dirrmtry share/icons/oxygen/22x22
 @dirrmtry share/icons/oxygen/16x16
 @dirrmtry share/icons/oxygen
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrmtry share/applications
===> Done
