===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/defendguin/Makefile ./Makefile
--- /usr/ports/games/defendguin/Makefile	2013-03-15 15:37:39.000000000 +0100
+++ ./Makefile	2013-05-27 23:33:45.000000000 +0200
@@ -26,11 +26,9 @@
 
 MAN6=		${PORTNAME}.6
 
-DESKTOP_ENTRIES="Defendguin" "${COMMENT}"  \
-		"share/pixmaps/${PORTNAME}" \
+DESKTOP_ENTRIES="Defendguin" "${COMMENT}" "${PORTNAME}"  \
 		"${PORTNAME}" "Game;ArcadeGame;" false
 
-
 .include <bsd.port.options.mk>
 
 do-build:
@@ -39,13 +37,13 @@
 		`${SDL_CONFIG} --libs` -lSDL_mixer
 
 do-install:
-	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin/
+	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin
 	@(cd ${WRKSRC}/../data && ${COPYTREE_SHARE} "images music sounds" ${DATADIR})
-	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps/
-	${INSTALL_MAN} ${MAN6:S|^|${WRKSRC}/|} ${MAN6PREFIX}/man/man6/
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
+	${INSTALL_MAN} ${MAN6:S|^|${WRKSRC}/|} ${MAN6PREFIX}/man/man6
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/../docs/|} ${DOCSDIR}
 .endif
 
===> Done
