===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/empire/Makefile ./Makefile
--- /usr/ports/games/empire/Makefile	2013-01-14 17:55:27.000000000 +0100
+++ ./Makefile	2013-05-25 17:03:40.000000000 +0200
@@ -1,9 +1,5 @@
-# New ports collection makefile for:	empire
-# Date created:				13 May 1997
-# Whom:					Andrey Zakhvatov
-#
+# Created by: Andrey Zakhvatov
 # $FreeBSD: head/games/empire/Makefile 305367 2012-10-06 14:17:56Z jhale $
-#
 
 PORTNAME=	empire
 PORTVERSION=	1.9
@@ -39,7 +35,7 @@
 	${INSTALL_DATA} ${WRKSRC}/empire-logo.png ${PREFIX}/share/pixmaps/empire.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
