===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/epiar/Makefile ./Makefile
--- /usr/ports/games/epiar/Makefile	2013-04-26 16:36:08.000000000 +0200
+++ ./Makefile	2013-05-22 00:27:17.000000000 +0200
@@ -3,7 +3,7 @@
 
 PORTNAME=	epiar
 PORTVERSION=	0.5.1
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	games
 MASTER_SITES=	http://epiar.net/files/epiar/releases/0.5/
 
@@ -21,19 +21,18 @@
 		xml2:${PORTSDIR}/textproc/libxml2 \
 		physfs:${PORTSDIR}/devel/physfs
 
-USES=		pkgconfig
 USE_BZIP2=	yes
 USE_SDL=	image mixer sdl
 USE_GL=		glu
 GNU_CONFIGURE=	yes
 USE_AUTOTOOLS=	aclocal
 USE_GMAKE=	yes
+USES=		pkgconfig
 MAKE_JOBS_SAFE=	yes
 
 PORTDOCS=	AUTHORS ChangeLog README
 
 pre-configure:
-	@${REINPLACE_CMD} -e 's|9.22|2.4|' ${WRKSRC}/configure.in
 	@(cd ${WRKSRC} && ${SETENV} ${CONFIGURE_ENV} ./autogen.sh)
 
 .include <bsd.port.options.mk>
diff -ruN --exclude=CVS /usr/ports/games/epiar/files/patch-configure.in ./files/patch-configure.in
--- /usr/ports/games/epiar/files/patch-configure.in	1970-01-01 01:00:00.000000000 +0100
+++ ./files/patch-configure.in	2013-05-10 23:46:03.000000000 +0200
@@ -0,0 +1,32 @@
+--- configure.in.orig	2012-05-31 04:16:51.000000000 +0200
++++ configure.in	2013-05-10 23:45:57.000000000 +0200
+@@ -30,29 +30,6 @@
+ 	PKG_CONFIG=`which pkg-config`
+ fi
+ 
+-dnl Check for Freetype
+-case "$target" in
+-	*-*-linux* | *-*-cygwin* | *-*-mingw32* | *-*-freebsd*)
+-	PKG_CHECK_MODULES([freetype2], [freetype2 >= 9.22])
+-	CFLAGS="$CFLAGS $freetype2_CFLAGS"
+-	LIBS="$LIBS $freetype2_LIBS"
+-esac
+-case "$target" in *-apple-darwin*)
+-	CFLAGS="$CFLAGS -I/usr/X11/include -I/usr/X11/include/freetype2"
+-	LIBS="$LIBS -L/usr/X11/lib"
+-esac
+-
+-dnl Some additional flags for Freetype
+-dnl case "$target" in *-*-linux*)
+-dnl	CFLAGS="$CFLAGS -I/usr/include/freetype2"
+-dnl esac
+-dnl case "$target" in *-*-freebsd*)
+-dnl	CFLAGS="$CFLAGS -I/usr/local/include/freetype2"
+-dnl esac
+-dnl case "$target" in *-apple-darwin*)
+-dnl	CFLAGS="$CFLAGS -I/opt/local/include/ -I/opt/local/include/freetype"
+-dnl esac
+-
+ dnl Check for PhysicsFS (physfs)
+ LIBS="$LIBS -lphysfs"
+ CFLAGS="$CFLAGS -DUSE_PHYSICSFS"
===> Done
