===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/gemdropx/Makefile ./Makefile
--- /usr/ports/games/gemdropx/Makefile	2013-01-14 17:55:25.000000000 +0100
+++ ./Makefile	2013-03-26 21:58:11.000000000 +0100
@@ -1,16 +1,16 @@
-# New ports collection makefile for:	gemdropx
-# Date created:				9 March 1998
-# Whom:					Andrey Zakhvatov
-#
+# Created by: Andrey Zakhvatov
 # $FreeBSD: head/games/gemdropx/Makefile 304062 2012-09-10 19:52:36Z swills $
-#
 
 PORTNAME=	gemdropx
 PORTVERSION=	0.9
-PORTREVISION=	9
+PORTREVISION=	10
 CATEGORIES=	games
 MASTER_SITES=	ftp://ftp.tuxpaint.org/unix/x/gemdropx/src/ \
-		ftp://ftp.billsgames.com/unix/x/gemdropx/src/
+		ftp://ftp.billsgames.com/unix/x/gemdropx/src/ \
+		SF/nemysisfreebsdp/:icons
+DISTFILES=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX} \
+		${PORTNAME}.png:icons
+EXTRACT_ONLY=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX}
 
 MAINTAINER=	nemysis@gmx.ch
 COMMENT=	Interesting one-player puzzle game for X (SDL)
@@ -22,40 +22,31 @@
 
 USE_GMAKE=	yes
 
-PLIST_FILES=	bin/${PORTNAME} \
-		share/pixmaps/${PORTNAME}.xmp
-
-PORTDATA=	*
 PORTDOCS=	AUTHORS.txt CHANGES.txt ICON.txt README.txt TODO.txt
 
+DESKTOP_ENTRIES="Gem Drop X" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Game;ArcadeGame;" false
+
 .include <bsd.port.options.mk>
 
 post-patch:
-# Fix SDL, CFLAGS
 	@${REINPLACE_CMD} \
 	-e 's|sdl-config|$(SDL_CONFIG)|' \
 	-e 's|CC=gcc|CC?=g++|' \
 	-e 's|CXX=gcc|CXX?=g++|' \
 	-e 's|CFLAGS=-Wall $(SDL_CFLAGS)|CFLAGS+=$(SDL_CFLAGS)|' \
 		${WRKSRC}/Makefile
+	@${FIND} ${WRKDIR} -name .xvpics | ${XARGS} ${RM} -rf
 
 do-install:
-# Executable
 	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin
-
-# Data
-	${MKDIR} ${DATADIR}
+	@${MKDIR} ${DATADIR}
 	@(cd ${WRKSRC}/data && ${COPYTREE_SHARE} "images sounds" ${DATADIR})
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
-# Pixmaps
-	${INSTALL_DATA} ${WRKSRC}/data/images/gemdropx-icon.xpm ${PREFIX}/share/pixmaps/${PORTNAME}.xmp
-
-# Documentation
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
-.  for f in ${PORTDOCS}
-	${INSTALL_DATA} ${WRKSRC}/${f} ${DOCSDIR}
-.  endfor
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/gemdropx/distinfo ./distinfo
--- /usr/ports/games/gemdropx/distinfo	2013-01-14 17:55:25.000000000 +0100
+++ ./distinfo	2013-03-26 21:58:22.000000000 +0100
@@ -1,2 +1,4 @@
 SHA256 (gemdropx-0.9.tar.gz) = e50495d292a1d456c28044efbf07c16d8865f8d95e1caba86f4c5b2e3fb1d28f
 SIZE (gemdropx-0.9.tar.gz) = 903036
+SHA256 (gemdropx.png) = 9a7edea28ecbe442b5842c4c6f98d9f9f59079f9b1323faf4ef8a6936413d011
+SIZE (gemdropx.png) = 2228
diff -ruN --exclude=CVS /usr/ports/games/gemdropx/pkg-plist ./pkg-plist
--- /usr/ports/games/gemdropx/pkg-plist	1970-01-01 01:00:00.000000000 +0100
+++ ./pkg-plist	2013-03-26 19:55:38.000000000 +0100
@@ -0,0 +1,69 @@
+bin/gemdropx
+%%DATADIR%%/images/README
+%%DATADIR%%/images/action-mask.xbm
+%%DATADIR%%/images/action.xbm
+%%DATADIR%%/images/author.bmp
+%%DATADIR%%/images/block.bmp
+%%DATADIR%%/images/bluegem.bmp
+%%DATADIR%%/images/bomb.bmp
+%%DATADIR%%/images/brick.bmp
+%%DATADIR%%/images/congratulations.bmp
+%%DATADIR%%/images/credits.bmp
+%%DATADIR%%/images/diamond.bmp
+%%DATADIR%%/images/disc.bmp
+%%DATADIR%%/images/dot.bmp
+%%DATADIR%%/images/explode0.bmp
+%%DATADIR%%/images/explode1.bmp
+%%DATADIR%%/images/explode2.bmp
+%%DATADIR%%/images/explode3.bmp
+%%DATADIR%%/images/fuji.bmp
+%%DATADIR%%/images/gemdropx-icon.xpm
+%%DATADIR%%/images/happy.bmp
+%%DATADIR%%/images/left-mask.xbm
+%%DATADIR%%/images/left.xbm
+%%DATADIR%%/images/level.bmp
+%%DATADIR%%/images/lines.bmp
+%%DATADIR%%/images/need.bmp
+%%DATADIR%%/images/nothing-mask.xbm
+%%DATADIR%%/images/nothing.xbm
+%%DATADIR%%/images/numbers.bmp
+%%DATADIR%%/images/patience.bmp
+%%DATADIR%%/images/paused.bmp
+%%DATADIR%%/images/right-mask.xbm
+%%DATADIR%%/images/right.xbm
+%%DATADIR%%/images/sad.bmp
+%%DATADIR%%/images/score.bmp
+%%DATADIR%%/images/stopwatch.bmp
+%%DATADIR%%/images/title.bmp
+%%DATADIR%%/images/triangle.bmp
+%%DATADIR%%/images/uhoh.bmp
+%%DATADIR%%/images/warning.bmp
+%%DATADIR%%/images/warning1.bmp
+%%DATADIR%%/images/warning2.bmp
+%%DATADIR%%/images/warning3.bmp
+%%DATADIR%%/images/warning4.bmp
+%%DATADIR%%/images/wildcard.bmp
+%%DATADIR%%/images/win1.bmp
+%%DATADIR%%/images/win2.bmp
+%%DATADIR%%/images/yellowgem.bmp
+%%DATADIR%%/sounds/22drops.mod
+%%DATADIR%%/sounds/2force.xm
+%%DATADIR%%/sounds/README
+%%DATADIR%%/sounds/applause.wav
+%%DATADIR%%/sounds/begin.wav
+%%DATADIR%%/sounds/bomb.wav
+%%DATADIR%%/sounds/buzzer.wav
+%%DATADIR%%/sounds/citron.mod
+%%DATADIR%%/sounds/clock.wav
+%%DATADIR%%/sounds/gameover.wav
+%%DATADIR%%/sounds/got.wav
+%%DATADIR%%/sounds/grab.wav
+%%DATADIR%%/sounds/hr_suds.xm
+%%DATADIR%%/sounds/levelselect.wav
+%%DATADIR%%/sounds/throw.wav
+%%DATADIR%%/sounds/wildcard.wav
+%%DATADIR%%/sounds/win.wav
+share/pixmaps/gemdropx.png
+@dirrm %%DATADIR%%/sounds
+@dirrm %%DATADIR%%/images
+@dirrm %%DATADIR%%
===> Done
