===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/glightoff/Makefile ./Makefile
--- /usr/ports/games/glightoff/Makefile	2013-04-26 16:36:09.000000000 +0200
+++ ./Makefile	2013-05-25 17:35:08.000000000 +0200
@@ -1,4 +1,4 @@
-# Created by:			nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/glightoff/Makefile 316464 2013-04-24 18:10:30Z ak $
 
 PORTNAME=	glightoff
@@ -12,10 +12,11 @@
 
 LICENSE=	GPLv2
 
-GNU_CONFIGURE=	yes
-USE_GMAKE=	yes
+USES=		desktop-file-utils
 USE_GNOME=	glib20 gtk20
 USE_XORG=	x11
+GNU_CONFIGURE=	yes
+USE_GMAKE=	yes
 
 CFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
@@ -34,13 +35,13 @@
 .endif
 
 post-configure:
-.if !${PORT_OPTIONS:MNLS}
+.if ! ${PORT_OPTIONS:MNLS}
 	@${REINPLACE_CMD} 's|src po|src|' ${WRKSRC}/Makefile
 .endif
 
 post-install:
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
