===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/help_hannahs_horse/Makefile ./Makefile
--- /usr/ports/games/help_hannahs_horse/Makefile	2013-02-10 14:22:24.000000000 +0100
+++ ./Makefile	2013-03-26 22:15:05.000000000 +0100
@@ -1,4 +1,4 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/help_hannahs_horse/Makefile 311995 2013-02-10 07:48:40Z miwi $
 
 PORTNAME=	help_hannahs_horse
@@ -30,8 +30,8 @@
 INSTALLS_ICONS=	yes
 ICON_SIZES=	32x32 48x48 64x64 72x72 96x96
 
-DESKTOP_ENTRIES="Help Hannah's Horse" "${COMMENT}" "${PORTNAME}.png" \
-		"${PORTNAME}" "Game;ArcadeGame;" ${FALSE}
+DESKTOP_ENTRIES="Help Hannah's Horse" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
 
@@ -63,7 +63,7 @@
 	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/help_hannahs_horse/pkg-plist ./pkg-plist
--- /usr/ports/games/help_hannahs_horse/pkg-plist	2013-02-10 14:22:24.000000000 +0100
+++ ./pkg-plist	2013-03-26 22:15:33.000000000 +0100
@@ -154,18 +154,6 @@
 share/icons/hicolor/72x72/apps/help_hannahs_horse.png
 share/icons/hicolor/96x96/apps/help_hannahs_horse.png
 share/pixmaps/help_hannahs_horse.png
-@dirrmtry share/icons/hicolor/96x96/apps
-@dirrmtry share/icons/hicolor/96x96
-@dirrmtry share/icons/hicolor/72x72/apps
-@dirrmtry share/icons/hicolor/72x72
-@dirrmtry share/icons/hicolor/64x64/apps
-@dirrmtry share/icons/hicolor/64x64
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrm %%DATADIR%%/stop/default
 @dirrm %%DATADIR%%/stop
 @dirrm %%DATADIR%%/sounds
===> Done
