===> Generating patch
===> Viewing diff with more
diff -ruN /usr/ports/games/hitori/Makefile ./Makefile
--- /usr/ports/games/hitori/Makefile	2013-06-02 18:54:53.000000000 +0200
+++ ./Makefile	2013-08-09 21:59:45.000000000 +0200
@@ -2,7 +2,7 @@
 # $FreeBSD: head/games/hitori/Makefile 316464 2013-04-24 18:10:30Z ak $
 
 PORTNAME=	hitori
-PORTVERSION=	0.4.0
+PORTVERSION=	0.4.1
 CATEGORIES=	games
 MASTER_SITES=	GNOME
 DIST_SUBDIR=	gnome2
@@ -21,37 +21,33 @@
 LIB_DEPENDS=	cairo:${PORTSDIR}/graphics/cairo
 
 USE_XZ=		yes
-GNU_CONFIGURE=	yes
-USE_GMAKE=	yes
+USES=		gmake desktop-file-utils
 USE_GNOME=	glib20 gtk30
+GNU_CONFIGURE=	yes
 MAKE_JOBS_SAFE=	yes
 INSTALLS_ICONS=	yes
 
 PORTDOCS=	AUTHORS ChangeLog MAINTAINERS NEWS README
 
-OPTIONS_DEFINE=	NLS
+OPTIONS_DEFINE=	DOCS NLS
+OPTIONS_SUB=	yes
 
-.include <bsd.port.options.mk>
+NLS_CONFIGURE_ENABLE=	nls
+NLS_USES=	gettext
 
-.if ${PORT_OPTIONS:MNLS}
-USES+=		gettext
-PLIST_SUB+=	NLS=""
-.else
-CONFIGURE_ARGS+=	--disable-nls
-PLIST_SUB+=	NLS="@comment "
-.endif
+.include <bsd.port.options.mk>
 
 post-patch:
 .if ${PORT_OPTIONS:MNLS}
-	@${REINPLACE_CMD} -e 's|itlocaledir = $$(prefix)/$$(DATADIRNAME)/locale|itlocaledir = $$(prefix)/share/locale|' \
+	@${REINPLACE_CMD} -e 's|$$(prefix)/$$(DATADIRNAME)/locale|$$(prefix)/share/locale|' \
 		${WRKSRC}/po/Makefile.in.in
 .endif
 
 post-install:
-	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps/
+	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN /usr/ports/games/hitori/distinfo ./distinfo
--- /usr/ports/games/hitori/distinfo	2013-06-02 18:54:53.000000000 +0200
+++ ./distinfo	2013-08-09 21:59:52.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (gnome2/hitori-0.4.0.tar.xz) = 5d9e1c0f5d33c960167a126242bf57ff4865a6aaf6602da4a25aae56dc491030
-SIZE (gnome2/hitori-0.4.0.tar.xz) = 448052
+SHA256 (gnome2/hitori-0.4.1.tar.xz) = 7a34ea697086cabd48ef740370f69995c9487def78ad7c9f280279eea2f282af
+SIZE (gnome2/hitori-0.4.1.tar.xz) = 456752
diff -ruN /usr/ports/games/hitori/pkg-descr ./pkg-descr
--- /usr/ports/games/hitori/pkg-descr	2013-06-02 18:54:53.000000000 +0200
+++ ./pkg-descr	2013-08-09 21:24:42.000000000 +0200
@@ -2,4 +2,4 @@
 puzzle game, which is similar in theme to more popular puzzles such as Sudoku
 created by Nikoli.
 
-WWW: https://live.gnome.org/Hitori
+WWW: https://wiki.gnome.org/Apps/Hitori
diff -ruN /usr/ports/games/hitori/pkg-plist ./pkg-plist
--- /usr/ports/games/hitori/pkg-plist	2013-06-02 18:54:53.000000000 +0200
+++ ./pkg-plist	2013-08-10 00:44:48.000000000 +0200
@@ -49,6 +49,13 @@
 share/help/fr/hitori/license.page
 share/help/fr/hitori/playing.page
 share/help/fr/hitori/strategy.page
+share/help/hu/hitori/customization.page
+share/help/hu/hitori/figures/hitori_main_window.png
+share/help/hu/hitori/game.page
+share/help/hu/hitori/index.page
+share/help/hu/hitori/license.page
+share/help/hu/hitori/playing.page
+share/help/hu/hitori/strategy.page
 share/help/ro/hitori/customization.page
 share/help/ro/hitori/figures/hitori_main_window.png
 share/help/ro/hitori/game.page
@@ -90,6 +97,7 @@
 %%NLS%%share/locale/es/LC_MESSAGES/hitori.mo
 %%NLS%%share/locale/fr/LC_MESSAGES/hitori.mo
 %%NLS%%share/locale/gl/LC_MESSAGES/hitori.mo
+%%NLS%%share/locale/he/LC_MESSAGES/hitori.mo
 %%NLS%%share/locale/hu/LC_MESSAGES/hitori.mo
 %%NLS%%share/locale/id/LC_MESSAGES/hitori.mo
 %%NLS%%share/locale/lv/LC_MESSAGES/hitori.mo
@@ -108,49 +116,42 @@
 share/pixmaps/hitori.png
 %%NLS%%@dirrmtry share/locale/sr@latin/LC_MESSAGES
 %%NLS%%@dirrmtry share/locale/sr@latin
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor/22x22/apps
-@dirrmtry share/icons/hicolor/22x22
-@dirrmtry share/icons/hicolor/16x16/apps
-@dirrmtry share/icons/hicolor/16x16
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrm %%DATADIR%%
-@dirrmtry share/help/zh_CN/hitori/figures
-@dirrmtry share/help/zh_CN/hitori
+@dirrm share/help/zh_CN/hitori/figures
+@dirrm share/help/zh_CN/hitori
 @dirrmtry share/help/zh_CN
-@dirrmtry share/help/sv/hitori/figures
-@dirrmtry share/help/sv/hitori
+@dirrm share/help/sv/hitori/figures
+@dirrm share/help/sv/hitori
 @dirrmtry share/help/sv
-@dirrmtry share/help/sl/hitori/figures
-@dirrmtry share/help/sl/hitori
+@dirrm share/help/sl/hitori/figures
+@dirrm share/help/sl/hitori
 @dirrmtry share/help/sl
-@dirrmtry share/help/ro/hitori/figures
-@dirrmtry share/help/ro/hitori
+@dirrm share/help/ro/hitori/figures
+@dirrm share/help/ro/hitori
 @dirrmtry share/help/ro
-@dirrmtry share/help/fr/hitori/figures
-@dirrmtry share/help/fr/hitori
+@dirrm share/help/hu/hitori/figures
+@dirrm share/help/hu/hitori
+@dirrmtry share/help/hu
+@dirrm share/help/fr/hitori/figures
+@dirrm share/help/fr/hitori
 @dirrmtry share/help/fr
-@dirrmtry share/help/es/hitori/figures
-@dirrmtry share/help/es/hitori
+@dirrm share/help/es/hitori/figures
+@dirrm share/help/es/hitori
 @dirrmtry share/help/es
-@dirrmtry share/help/en_GB/hitori/figures
-@dirrmtry share/help/en_GB/hitori
+@dirrm share/help/en_GB/hitori/figures
+@dirrm share/help/en_GB/hitori
 @dirrmtry share/help/en_GB
-@dirrmtry share/help/de/hitori/figures
-@dirrmtry share/help/de/hitori
+@dirrm share/help/de/hitori/figures
+@dirrm share/help/de/hitori
 @dirrmtry share/help/de
-@dirrmtry share/help/da/hitori/figures
-@dirrmtry share/help/da/hitori
+@dirrm share/help/da/hitori/figures
+@dirrm share/help/da/hitori
 @dirrmtry share/help/da
-@dirrmtry share/help/cs/hitori/figures
-@dirrmtry share/help/cs/hitori
+@dirrm share/help/cs/hitori/figures
+@dirrm share/help/cs/hitori
 @dirrmtry share/help/cs
-@dirrmtry share/help/C/hitori/figures
-@dirrmtry share/help/C/hitori
+@dirrm share/help/C/hitori/figures
+@dirrm share/help/C/hitori
 @dirrmtry share/help/C
 @dirrmtry share/help
 @dirrmtry share/applications
===> Done
