===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/icebreaker/Makefile ./Makefile
--- /usr/ports/games/icebreaker/Makefile	2013-01-17 13:26:12.000000000 +0100
+++ ./Makefile	2013-03-27 01:06:01.000000000 +0100
@@ -3,30 +3,30 @@
 
 PORTNAME=	icebreaker
 PORTVERSION=	1.9.8
+PORTREVISION=	1
 CATEGORIES=	games
-MASTER_SITES=	http://mattdm.org/icebreaker/1.9.x/
-EXTRACT_SUFX=	.tgz
+MASTER_SITES=	http://mattdm.org/icebreaker/1.9.x/ \
+		SF/nemysisfreebsdp/ \
+		SF/nemysisfreebsdp/:icons
+DISTFILES=	${PORTNAME}-${DISTVERSION}.tgz \
+		${PORTNAME}.png:icons
+EXTRACT_ONLY=	${PORTNAME}-${DISTVERSION}.tgz
 
 MAINTAINER=	nemysis@gmx.ch
 COMMENT=	Game similar to Jezzball or Barrack
 
 LICENSE=	GPLv2
 
-BUILD_DEPENDS=	pnmtopng:${PORTSDIR}/graphics/netpbm
-
 USE_SDL=	mixer sdl
 USE_GMAKE=	yes
 
-PLIST_FILES=	bin/${PORTNAME} \
-		share/pixmaps/${PORTNAME}.png \
-		share/applications/${PORTNAME}.desktop
-PLIST_DIRSTRY=	share/applications
-
-PORTDATA=	*
 PORTDOCS=	ChangeLog README README.SDL README.ifyoumakechanges README.themes TODO
 
 MAN6=		${PORTNAME}.6
 
+DESKTOP_ENTRIES="IceBreaker" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Game;ArcadeGame;" false
+
 .include <bsd.port.options.mk>
 
 post-patch:
@@ -38,27 +38,18 @@
 	-e 's|highscoredir=/var/local/lib/games|highscoredir=/var/games|' \
 	-e 's|CFLAGS=|CFLAGS+=|' \
 	-e 's|OPTIMIZE=-O3|OPTIMIZE=|' \
+	-e 's|*.ico *.desktop *.man.in|*.ico *.man.in|' \
 	-e 's|$$(CC) $$(CFLAGS) $$^ -o icebreaker $$(SDL_LIB)|$$(CC) $$(LDFLAGS) $$(CFLAGS) $$^ -o icebreaker $$(SDL_LIB)|' \
 	-e 's|install -s -g games -m 2755 icebreaker $(bindir)|install -g games -m 2755 icebreaker $(bindir)|' \
 	-e 's|install -s -m 755 icebreaker $(bindir)|install -m 755 icebreaker $(bindir)|' \
 	-e 's|chown -f games:games|chown -f root:games|' \
 		${WRKSRC}/Makefile
-	@${REINPLACE_CMD} \
-	-e 's|%{Summary}|Game similar to Jezzball or Barrack|' \
-	-e 's|/usr/share/icebreaker/icebreaker_48.bmp|icebreaker|' \
-		${WRKSRC}/icebreaker.desktop
-
-post-build:
-	@(cd ${WRKSRC} && ${LOCALBASE}/bin/winicontoppm -bestqual icebreaker_32.ico | \
-		${LOCALBASE}/bin/pnmtopng > icebreaker.png)
 
 post-install:
-	${INSTALL_DATA} ${WRKSRC}/${PORTNAME}.png ${PREFIX}/share/pixmaps
-	${MKDIR} ${PREFIX}/share/applications
-	${INSTALL_DATA} ${WRKSRC}/${PORTNAME}.desktop ${PREFIX}/share/applications
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/icebreaker/distinfo ./distinfo
--- /usr/ports/games/icebreaker/distinfo	2013-01-17 13:26:12.000000000 +0100
+++ ./distinfo	2013-03-27 00:10:07.000000000 +0100
@@ -1,2 +1,4 @@
 SHA256 (icebreaker-1.9.8.tgz) = 2e9b1e898c50562c503cb109172c422544d2e5de09e642bb8eb3401a9180adc4
 SIZE (icebreaker-1.9.8.tgz) = 177355
+SHA256 (icebreaker.png) = bee29260a7e3fcd858b4286246d82df8747b73f2e5e31bc59b732b4df84efcd5
+SIZE (icebreaker.png) = 838
diff -ruN --exclude=CVS /usr/ports/games/icebreaker/pkg-plist ./pkg-plist
--- /usr/ports/games/icebreaker/pkg-plist	1970-01-01 01:00:00.000000000 +0100
+++ ./pkg-plist	2013-03-27 00:59:40.000000000 +0100
@@ -0,0 +1,40 @@
+bin/icebreaker
+%%DATADIR%%/atom.bmp
+%%DATADIR%%/bgeveryone.wav
+%%DATADIR%%/bgvvint.wav
+%%DATADIR%%/billg.bmp
+%%DATADIR%%/billg.ibt
+%%DATADIR%%/blocks.ibt
+%%DATADIR%%/bwop.wav
+%%DATADIR%%/cheese.ibt
+%%DATADIR%%/cow.bmp
+%%DATADIR%%/cows.ibt
+%%DATADIR%%/crash.wav
+%%DATADIR%%/drum.wav
+%%DATADIR%%/ewww.wav
+%%DATADIR%%/icebreaker_48.bmp
+%%DATADIR%%/jz-like.ibt
+%%DATADIR%%/kitty.bmp
+%%DATADIR%%/kitty.ibt
+%%DATADIR%%/linux.ibt
+%%DATADIR%%/moo.wav
+%%DATADIR%%/mouse-r.bmp
+%%DATADIR%%/mouse.bmp
+%%DATADIR%%/nbwoanp.wav
+%%DATADIR%%/ouch.wav
+%%DATADIR%%/penguin.bmp
+%%DATADIR%%/penguinicon_32.bmp
+%%DATADIR%%/phase.wav
+%%DATADIR%%/pop.wav
+%%DATADIR%%/scrunch.wav
+%%DATADIR%%/splash.wav
+%%DATADIR%%/splat.wav
+%%DATADIR%%/squeak.wav
+%%DATADIR%%/star.bmp
+%%DATADIR%%/stars.ibt
+%%DATADIR%%/tap.wav
+%%DATADIR%%/turtle.bmp
+%%DATADIR%%/turtles.ibt
+%%DATADIR%%/woodblock.bmp
+share/pixmaps/icebreaker.png
+@dirrm %%DATADIR%%
===> Done
