===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/impossible_mission_puzzle/Makefile ./Makefile
--- /usr/ports/games/impossible_mission_puzzle/Makefile	2013-01-24 11:57:58.000000000 +0100
+++ ./Makefile	2013-03-27 01:33:41.000000000 +0100
@@ -1,3 +1,4 @@
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/impossible_mission_puzzle/Makefile 310871 2013-01-23 15:00:18Z makc $
 
 PORTNAME=	impossible_mission_puzzle
@@ -27,7 +28,7 @@
 PORTDOCS=	README
 
 DESKTOP_ENTRIES="Impossible Mission puzzle" "${COMMENT}" "${PORTNAME}" \
-		"${PORTNAME}" "Game;ArcadeGame;" ${FALSE}
+		"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
 
@@ -40,7 +41,7 @@
 	${INSTALL_DATA} ${DISTDIR}/${DIST_SUBDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
