===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/krank/Makefile ./Makefile
--- /usr/ports/games/krank/Makefile	2013-01-24 11:57:59.000000000 +0100
+++ ./Makefile	2013-05-25 20:00:17.000000000 +0200
@@ -1,3 +1,4 @@
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/krank/Makefile 310871 2013-01-23 15:00:18Z makc $
 
 PORTNAME=	krank
@@ -41,10 +42,10 @@
 .endfor
 	${INSTALL_DATA} ${WRKSRC}/krank.iss ${DATADIR}
 	${INSTALL_DATA} ${WRKSRC}/logo.xcf ${DATADIR}
-	${INSTALL_DATA} ${WRKSRC}/html/gfx/logo.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
+	${LN} -sf ${DATADIR}/html/gfx/anchor.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
