===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/kye/Makefile ./Makefile
--- /usr/ports/games/kye/Makefile	2013-03-15 15:37:40.000000000 +0100
+++ ./Makefile	2013-05-27 23:42:31.000000000 +0200
@@ -36,7 +36,7 @@
 .include <bsd.port.options.mk>
 
 post-extract-script:
-	${MKDIR} ${WRKDIR}/${PORTNAME}
+	@${MKDIR} ${WRKDIR}/${PORTNAME}
 .for l in kye-data_moria_levels.20121101.tar.gz kye-data_nuke_levels.20121101.tar.gz kye-data_extra_levels.20120818.tar.gz
 	${TAR} xfjp ${DISTDIR}/${DIST_SUBDIR}/${l} -C ${WRKDIR}/${PORTNAME}
 .endfor
@@ -44,10 +44,10 @@
 post-install:
 	${INSTALL_DATA} ${WRKSRC}/kye-icon.png ${PREFIX}/share/pixmaps/Kye.png
 	${INSTALL_DATA} ${WRKSRC}/kye-edit-icon.png ${PREFIX}/share/pixmaps/Kye-edit.png
-	@cd ${WRKDIR}/${PORTNAME} && ${COPYTREE_SHARE} \* ${DATADIR}
+	@cd ${WRKDIR}/${PORTNAME} && ${COPYTREE_SHARE} . ${DATADIR}
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
