===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/lmarbles/Makefile ./Makefile
--- /usr/ports/games/lmarbles/Makefile	2013-03-08 22:52:54.000000000 +0100
+++ ./Makefile	2013-05-25 21:40:23.000000000 +0200
@@ -35,7 +35,6 @@
 
 post-patch:
 	@${REINPLACE_CMD} -e 's|(datadir)/icons|(datadir)/pixmaps|g' \
-	-e 's|(datadir)/pixmaps/lmarbles32.gif|(datadir)/pixmaps/lmarbles32.png|g' \
 	-e 's|(datadir)/pixmaps/lmarbles48.gif|(datadir)/pixmaps/lmarbles48.png|g' \
 		${WRKSRC}/Makefile.in
 	@${REINPLACE_CMD} -e 's|-m 666||g' \
@@ -43,15 +42,17 @@
 		${WRKSRC}/src/Makefile.in
 
 post-install:
-	${INSTALL_DATA} ${WRKDIR}/${PORTNAME}*.png ${PREFIX}/share/pixmaps/
+	${INSTALL_DATA} ${WRKDIR}/${PORTNAME}*.png ${PREFIX}/share/pixmaps
 	@if [ ! -f ${ETCDIR}/lmarbles.prfs ] ; then \
 		${INSTALL_DATA} ${ETCDIR}/lmarbles.prfs.sample ${ETCDIR}/lmarbles.prfs ; \
+		${CHMOD} 664 ${ETCDIR}/lmarbles.prfs ; \
+		${CHOWN} root:games ${ETCDIR}/lmarbles.prfs ; \
 	fi
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${DOC_FILES1:S|^|${DOCSRCDIR1}/|} ${DOCSDIR}
-	${MKDIR} ${DOCSDIR2}
+	@${MKDIR} ${DOCSDIR2}
 	${INSTALL_DATA} ${DOC_FILES2:S|^|${DOCSRCDIR2}/|} ${DOCSDIR2}
 .endif
 
===> Done
