===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/madbomber/Makefile ./Makefile
--- /usr/ports/games/madbomber/Makefile	2013-03-15 15:37:39.000000000 +0100
+++ ./Makefile	2013-05-25 21:17:22.000000000 +0200
@@ -20,8 +20,7 @@
 
 PORTDOCS=	AUTHORS.txt CHANGES.txt README.txt TODO.txt
 
-DESKTOP_ENTRIES="Mad Bomber" "${COMMENT}" \
-		"${PREFIX}/share/pixmaps/${PORTNAME}" \
+DESKTOP_ENTRIES="Mad Bomber" "${COMMENT}" "${PORTNAME}" \
 		"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
@@ -31,17 +30,17 @@
 		${PORTNAME}.c ${LDFLAGS} -lSDL_mixer -lSDL_image
 
 do-install:
-	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin/
+	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin
 
-	${MKDIR} ${DATADIR}
+	@${MKDIR} ${DATADIR}
 .for d in images music sounds
 	@(cd ${WRKSRC}/data && ${COPYTREE_SHARE} ${d} ${DATADIR})
 .endfor
 
-	${INSTALL_DATA} ${WRKSRC}/data/images/icon.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
+	${LN} -sf ${DATADIR}/images/icon.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
