===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/magicmaze/Makefile ./Makefile
--- /usr/ports/games/magicmaze/Makefile	2013-01-24 11:57:58.000000000 +0100
+++ ./Makefile	2013-03-27 14:34:12.000000000 +0100
@@ -1,9 +1,9 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/magicmaze/Makefile 310871 2013-01-23 15:00:18Z makc $
 
 PORTNAME=	magicmaze
 PORTVERSION=	1.4.3.6
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	games ruby
 MASTER_SITES=	RUBYFORGE
 DISTNAME=	${PORTNAME}-${DISTVERSION:S/./_/g}
@@ -13,7 +13,7 @@
 
 LICENSE=	GPLv2
 
-RUN_DEPENDS=	ruby18-sdl>=1.3:${PORTSDIR}/devel/ruby-sdl
+RUN_DEPENDS=	ruby18-sdl>=0:${PORTSDIR}/devel/ruby-sdl
 
 WRKSRC=		${WRKDIR}/${PORTNAME}-${DISTVERSION:S/./_/g}
 
@@ -21,10 +21,6 @@
 USE_RUBY=	yes
 NO_BUILD=	yes
 
-PLIST_FILES=	bin/${PORTNAME} \
-		share/pixmaps/${PORTNAME}.png
-
-PORTDATA=	*
 PORTDOCS=	*
 
 SUB_FILES=	${PORTNAME}
@@ -37,13 +33,12 @@
 
 do-install:
 	${INSTALL_SCRIPT} ${WRKDIR}/${PORTNAME} ${PREFIX}/bin
-	${MKDIR} ${DATADIR}
-	${INSTALL_SCRIPT} ${WRKSRC}/*.rb ${DATADIR}
 	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "data magicmaze" ${DATADIR})
-	${INSTALL_DATA} ${WRKSRC}/data/gfx/icon.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
+	${INSTALL_SCRIPT} ${WRKSRC}/*.rb ${DATADIR}
+	${LN} -sf ${DATADIR}/data/gfx/icon.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${WRKSRC}/README.TXT ${WRKSRC}/doc/BUGS.TXT ${WRKSRC}/doc/TODO.TXT ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/magicmaze/pkg-plist ./pkg-plist
--- /usr/ports/games/magicmaze/pkg-plist	1970-01-01 01:00:00.000000000 +0100
+++ ./pkg-plist	2013-03-27 13:36:08.000000000 +0100
@@ -0,0 +1,59 @@
+bin/magicmaze
+%%DATADIR%%/data/gfx/background.pcx
+%%DATADIR%%/data/gfx/end.pcx
+%%DATADIR%%/data/gfx/fraktmod.ttf
+%%DATADIR%%/data/gfx/fraktmod.txt
+%%DATADIR%%/data/gfx/icon.png
+%%DATADIR%%/data/gfx/sprites.dat
+%%DATADIR%%/data/gfx/sprites.pcx
+%%DATADIR%%/data/gfx/title.pcx
+%%DATADIR%%/data/maps/mm_map.001
+%%DATADIR%%/data/maps/mm_map.002
+%%DATADIR%%/data/maps/mm_map.003
+%%DATADIR%%/data/maps/mm_map.004
+%%DATADIR%%/data/maps/mm_map.005
+%%DATADIR%%/data/maps/mm_map.006
+%%DATADIR%%/data/maps/mm_map.007
+%%DATADIR%%/data/maps/mm_map.008
+%%DATADIR%%/data/maps/mm_map.009
+%%DATADIR%%/data/maps/mm_map.010
+%%DATADIR%%/data/sound/sound1.wav
+%%DATADIR%%/data/sound/sound2.wav
+%%DATADIR%%/data/sound/sound3.wav
+%%DATADIR%%/data/sound/sound4.wav
+%%DATADIR%%/magicmaze/entity.rb
+%%DATADIR%%/magicmaze/filemap.rb
+%%DATADIR%%/magicmaze/game.rb
+%%DATADIR%%/magicmaze/gameloop.rb
+%%DATADIR%%/magicmaze/graphics.rb
+%%DATADIR%%/magicmaze/images.rb
+%%DATADIR%%/magicmaze/input.rb
+%%DATADIR%%/magicmaze/inventory.rb
+%%DATADIR%%/magicmaze/magicmaze.rb
+%%DATADIR%%/magicmaze/map.rb
+%%DATADIR%%/magicmaze/mapeditor.rb
+%%DATADIR%%/magicmaze/movement.rb
+%%DATADIR%%/magicmaze/player.rb
+%%DATADIR%%/magicmaze/sound.rb
+%%DATADIR%%/magicmaze/spellbook.rb
+%%DATADIR%%/magicmaze/spelltile.rb
+%%DATADIR%%/magicmaze/test_filemap.rb
+%%DATADIR%%/magicmaze/test_game.rb
+%%DATADIR%%/magicmaze/test_graphics.rb
+%%DATADIR%%/magicmaze/test_images.rb
+%%DATADIR%%/magicmaze/test_magicmaze.rb
+%%DATADIR%%/magicmaze/test_map.rb
+%%DATADIR%%/magicmaze/test_movement.rb
+%%DATADIR%%/magicmaze/test_player.rb
+%%DATADIR%%/magicmaze/test_sound.rb
+%%DATADIR%%/magicmaze/test_tile.rb
+%%DATADIR%%/magicmaze/tile.rb
+%%DATADIR%%/mm.rb
+%%DATADIR%%/testmm.rb
+share/pixmaps/magicmaze.png
+@dirrm %%DATADIR%%/magicmaze
+@dirrm %%DATADIR%%/data/sound
+@dirrm %%DATADIR%%/data/maps
+@dirrm %%DATADIR%%/data/gfx
+@dirrm %%DATADIR%%/data
+@dirrm %%DATADIR%%
===> Done
