===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/mopesnake/Makefile ./Makefile
--- /usr/ports/games/mopesnake/Makefile	2013-03-15 15:37:40.000000000 +0100
+++ ./Makefile	2013-03-27 16:42:32.000000000 +0100
@@ -23,8 +23,7 @@
 
 SUB_FILES=	${PORTNAME}
 
-DESKTOP_ENTRIES="Master Of Pain (Eating) - Snake" "${COMMENT}" \
-		"${PREFIX}/share/pixmaps/${PORTNAME}" \
+DESKTOP_ENTRIES="Master Of Pain (Eating) - Snake" "${COMMENT}" "${PORTNAME}" \
 		"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
@@ -35,15 +34,14 @@
 	@${FIND} ${WRKSRC} -name "*.bak" -delete
 
 do-install:
-	${INSTALL_SCRIPT} ${WRKDIR}/${PORTNAME} ${PREFIX}/bin/
-	${MKDIR} ${DATADIR}
-	${INSTALL_SCRIPT} ${WRKSRC}/${PORTNAME}.py ${DATADIR}
+	${INSTALL_SCRIPT} ${WRKDIR}/${PORTNAME} ${PREFIX}/bin
 	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "images mopelib music sounds" ${DATADIR})
-	${INSTALL_DATA} ${WRKSRC}/doc/screenshot1.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
+	${INSTALL_SCRIPT} ${WRKSRC}/${PORTNAME}.py ${DATADIR}
 	${INSTALL_DATA} ${WRKSRC}/version ${DATADIR}
+	${INSTALL_DATA} ${WRKSRC}/doc/screenshot1.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/doc/|} ${DOCSDIR}
 .endif
 
===> Done
