===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/netwalk/Makefile ./Makefile
--- /usr/ports/games/netwalk/Makefile	2013-04-29 12:53:30.000000000 +0200
+++ ./Makefile	2013-05-25 22:57:45.000000000 +0200
@@ -1,3 +1,4 @@
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/netwalk/Makefile 316654 2013-04-27 08:50:46Z mva $
 
 PORTNAME=	netwalk
@@ -55,7 +56,7 @@
 	${INSTALL_DATA} ${_DISTDIR}/screenshot-0.4.8.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
