===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/nuclearchess/Makefile ./Makefile
--- /usr/ports/games/nuclearchess/Makefile	2013-02-24 17:34:00.000000000 +0100
+++ ./Makefile	2013-05-21 15:04:08.000000000 +0200
@@ -18,7 +18,6 @@
 LICENSE=	GPLv2
 
 USE_SDL=	sdl image
-USE_GNOME=	desktopfileutils
 INSTALLS_ICONS=	yes
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS=	--localstatedir=${DATADIR}
@@ -26,19 +25,17 @@
 
 PORTDOCS=	AUTHORS ChangeLog
 
-DESKTOP_ENTRIES="NuclearChess" "${COMMENT}"  \
-		"${PREFIX}/share/pixmaps/${PORTNAME}.png" \
+DESKTOP_ENTRIES="NuclearChess" "${COMMENT}" "${PORTNAME}" \
 		"${PORTNAME}" "Game;BoardGame;" false
 
 .include <bsd.port.options.mk>
 
 post-install:
-	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps/
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
-	@-update-desktop-database
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/nuclearchess/pkg-plist ./pkg-plist
--- /usr/ports/games/nuclearchess/pkg-plist	2013-02-24 17:34:00.000000000 +0100
+++ ./pkg-plist	2013-03-27 18:04:38.000000000 +0100
@@ -36,5 +36,3 @@
 share/pixmaps/nuclearchess.png
 @dirrm %%DATADIR%%/gfx
 @dirrm %%DATADIR%%
-@exec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
-@unexec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
===> Done
