===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/palomino/Makefile ./Makefile
--- /usr/ports/games/palomino/Makefile	2013-04-29 12:53:30.000000000 +0200
+++ ./Makefile	2013-05-26 00:38:09.000000000 +0200
@@ -15,6 +15,8 @@
 MAINTAINER=	nemysis@gmx.ch
 COMMENT=	Action flight simulation computer program using OSG
 
+LICENSE=	GPLv2
+
 LIB_DEPENDS=	osg:${PORTSDIR}/graphics/osg
 
 USE_BZIP2=	yes
@@ -40,8 +42,10 @@
 PLIB_DESC=	Enable sound support through plib
 GUI_DESC=	Enable FLTK GUI
 
+PORTDOCS=	*
+
 DESKTOP_ENTRIES=	"Palomino" "${COMMENT}" "${PORTNAME}" \
-			"${PORTNAME}" "Game;ArcadeGame;" ${FALSE}
+			"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
 
@@ -71,30 +75,35 @@
 	@${REINPLACE_CMD} -e '/FIND_PACKAGE(/ s|)| REQUIRED)|' \
 		${WRKSRC}/CMakeModules/libraries.cmake
 	@${REINPLACE_CMD} -e '/COMPILE_WITH_OSSIM/ d' ${WRKSRC}/CMakeLists.txt
-.if empty(PORT_OPTIONS:MJOYSTICK)
+
+.if ! ${PORT_OPTIONS:MJOYSTICK}
 	@${REINPLACE_CMD} -e '/COMPILE_WITH_LINUX_JOYSTICK/ d' ${WRKSRC}/CMakeLists.txt
 .endif
-.if empty(PORT_OPTIONS:MSDL)
+
+.if ! ${PORT_OPTIONS:MSDL}
 	@${REINPLACE_CMD} -e '/COMPILE_WITH_SDL/ d' ${WRKSRC}/CMakeLists.txt
 .endif
-.if empty(PORT_OPTIONS:MPLIB)
+
+.if ! ${PORT_OPTIONS:MPLIB}
 	@${REINPLACE_CMD} -e '/COMPILE_WITH_PLIB/ d' ${WRKSRC}/CMakeLists.txt
 .endif
-.if empty(PORT_OPTIONS:MGUI)
+
+.if ! ${PORT_OPTIONS:MGUI}
 	@${REINPLACE_CMD} -e '/COMPILE_WITH_FLTK/ d' ${WRKSRC}/CMakeLists.txt
 .endif
+
 	@${REINPLACE_CMD} -e 's|string sDataDir|&="${DATADIR}/data/"|'\
-	       	${WRKSRC}/src/base/conf.cc
+		${WRKSRC}/src/base/conf.cc
 	@${REINPLACE_CMD} -e '/LUABIND_SCRIPTS_DIR/ s|"scripts"|"${DATADIR}/scripts"|' \
-	       	${WRKSRC}/src/lua_bind/defs.hh
+		${WRKSRC}/src/lua_bind/defs.hh
 	@${REINPLACE_CMD} -e 's|doc/html/|${DOCSDIR}/|' ${WRKSRC}/src/gui/gui.cc
 
 post-install:
-	${MKDIR} ${DATADIR}
-	cd ${WRKSRC}/ && ${COPYTREE_SHARE} "data scripts" ${DATADIR}/
+	@${MKDIR} ${DATADIR}
+	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "data scripts" ${DATADIR})
 # docs are used from within the game, so this is unconditional
-	${MKDIR} ${DOCSDIR}
-	cd ${WRKSRC}/doc/html && ${COPYTREE_SHARE} \* ${DOCSDIR}/
+	@${MKDIR} ${DOCSDIR}
+	@(cd ${WRKSRC}/doc/html && ${COPYTREE_SHARE} . ${DOCSDIR})
 
 	${LN} -sf ${DOCSDIR}/images/palominoLogo.png \
 		${PREFIX}/share/pixmaps/${PORTNAME}.png
diff -ruN --exclude=CVS /usr/ports/games/palomino/pkg-plist ./pkg-plist
--- /usr/ports/games/palomino/pkg-plist	2013-01-14 17:55:40.000000000 +0100
+++ ./pkg-plist	2013-05-21 19:25:26.000000000 +0200
@@ -1,52 +1,5 @@
 bin/palomino
 share/pixmaps/palomino.png
-%%DOCSDIR%%/coding.html
-%%DOCSDIR%%/compression.html
-%%DOCSDIR%%/crash.html
-%%DOCSDIR%%/docs.css
-%%DOCSDIR%%/images/arrowBlue.gif
-%%DOCSDIR%%/images/arrowGreen.gif
-%%DOCSDIR%%/images/arrowRed.gif
-%%DOCSDIR%%/images/arrowWhite.gif
-%%DOCSDIR%%/images/arrowYellow.gif
-%%DOCSDIR%%/images/favicon.png
-%%DOCSDIR%%/images/palominoLogo.png
-%%DOCSDIR%%/images/palomino_pegasus_50x40.png
-%%DOCSDIR%%/images/sprite1.png
-%%DOCSDIR%%/images/sprite2.png
-%%DOCSDIR%%/images/sprite3.png
-%%DOCSDIR%%/index.html
-%%DOCSDIR%%/issues.html
-%%DOCSDIR%%/lua_scripts.html
-%%DOCSDIR%%/manual.css
-%%DOCSDIR%%/manual.html
-%%DOCSDIR%%/math.html
-%%DOCSDIR%%/module_base.html
-%%DOCSDIR%%/module_base_configuration.html
-%%DOCSDIR%%/module_collision.html
-%%DOCSDIR%%/module_control.html
-%%DOCSDIR%%/module_fx.html
-%%DOCSDIR%%/module_game.html
-%%DOCSDIR%%/module_gfx.html
-%%DOCSDIR%%/module_graph.html
-%%DOCSDIR%%/module_gui.html
-%%DOCSDIR%%/module_hud.html
-%%DOCSDIR%%/module_init.html
-%%DOCSDIR%%/module_input.html
-%%DOCSDIR%%/module_lua.html
-%%DOCSDIR%%/module_math.html
-%%DOCSDIR%%/module_object.html
-%%DOCSDIR%%/module_physics.html
-%%DOCSDIR%%/module_program.html
-%%DOCSDIR%%/module_shader.html
-%%DOCSDIR%%/module_shadows.html
-%%DOCSDIR%%/module_sky.html
-%%DOCSDIR%%/module_sound.html
-%%DOCSDIR%%/module_view.html
-%%DOCSDIR%%/module_world.html
-%%DOCSDIR%%/rules.html
-%%DOCSDIR%%/scene.html
-%%DOCSDIR%%/terrain.html
 %%DATADIR%%/data/fonts/README.txt
 %%DATADIR%%/data/fonts/arial.ttf
 %%DATADIR%%/data/fonts/masque.ttf
@@ -516,5 +469,3 @@
 @dirrm %%DATADIR%%/data/fonts
 @dirrm %%DATADIR%%/data
 @dirrm %%DATADIR%%
-@dirrm %%DOCSDIR%%/images
-@dirrm %%DOCSDIR%%
===> Done
