===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/patapizza-tetris/Makefile ./Makefile
--- /usr/ports/games/patapizza-tetris/Makefile	2013-01-14 17:55:35.000000000 +0100
+++ ./Makefile	2013-03-27 21:05:02.000000000 +0100
@@ -1,9 +1,9 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/patapizza-tetris/Makefile 304720 2012-09-22 20:01:11Z bsam $
 
 PORTNAME=	patapizza-tetris
 PORTVERSION=	1.0
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	games
 MASTER_SITES=	https://github.com/patapizza/tetris/downloads/
 DISTNAME=	${PORTNAME:S/patapizza-//}-${DISTVERSION}
@@ -17,14 +17,13 @@
 USE_GMAKE=	yes
 USE_SDL=	sdl image gfx ttf
 
-PLIST_FILES=	bin/${PORTNAME} \
-		share/pixmaps/${PORTNAME}.png
-
-PORTDATA=	*
 PORTDOCS=	README.md
 
 SUB_FILES=	${PORTNAME}.sh
 
+DESKTOP_ENTRIES="Patapizza Tetris" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Game;ArcadeGame;" false
+
 .include <bsd.port.options.mk>
 
 post-patch:
@@ -36,22 +35,13 @@
 		-lm `${SDL_CONFIG} --cflags --libs` -lSDL -lSDL_image -lSDL_gfx -lSDL_ttf -lm tetris.c
 
 do-install:
-# Scripts
 	${INSTALL_SCRIPT} ${WRKDIR}/${PORTNAME}.sh ${PREFIX}/bin/${PORTNAME}
-
-# Executable
-	${MKDIR} ${DATADIR}
-	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${DATADIR}
-
-# Data
 	@(cd ${WRKSRC} && ${COPYTREE_SHARE} images ${DATADIR})
+	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${DATADIR}
+	${LN} -sf ${DATADIR}/images/logo.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
-# Pixmaps
-	${INSTALL_DATA} ${WRKSRC}/images/logo.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
-
-# Documentation
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/patapizza-tetris/files/patapizza-tetris.sh.in ./files/patapizza-tetris.sh.in
--- /usr/ports/games/patapizza-tetris/files/patapizza-tetris.sh.in	2013-01-14 17:55:35.000000000 +0100
+++ ./files/patapizza-tetris.sh.in	2013-03-27 20:11:36.000000000 +0100
@@ -1,7 +1,7 @@
 #!/bin/sh
 #
 # $FreeBSD: head/games/patapizza-tetris/files/patapizza-tetris.sh.in 304694 2012-09-22 16:22:48Z bsam $
+#
 
-echo "patapizza Tetris: Starting up..."
 cd "%%DATADIR%%"
-./patapizza-tetris
+exec ./patapizza-tetris
diff -ruN --exclude=CVS /usr/ports/games/patapizza-tetris/pkg-plist ./pkg-plist
--- /usr/ports/games/patapizza-tetris/pkg-plist	1970-01-01 01:00:00.000000000 +0100
+++ ./pkg-plist	2013-03-27 20:16:59.000000000 +0100
@@ -0,0 +1,49 @@
+bin/patapizza-tetris
+%%DATADIR%%/images/0.jpg
+%%DATADIR%%/images/1.jpg
+%%DATADIR%%/images/2.jpg
+%%DATADIR%%/images/3.jpg
+%%DATADIR%%/images/4.jpg
+%%DATADIR%%/images/5.jpg
+%%DATADIR%%/images/6.jpg
+%%DATADIR%%/images/7.jpg
+%%DATADIR%%/images/8.jpg
+%%DATADIR%%/images/9.jpg
+%%DATADIR%%/images/A.jpg
+%%DATADIR%%/images/B.jpg
+%%DATADIR%%/images/C.jpg
+%%DATADIR%%/images/D.jpg
+%%DATADIR%%/images/E.jpg
+%%DATADIR%%/images/F.jpg
+%%DATADIR%%/images/G.jpg
+%%DATADIR%%/images/H.jpg
+%%DATADIR%%/images/I.jpg
+%%DATADIR%%/images/J.jpg
+%%DATADIR%%/images/K.jpg
+%%DATADIR%%/images/L.jpg
+%%DATADIR%%/images/M.jpg
+%%DATADIR%%/images/N.jpg
+%%DATADIR%%/images/O.jpg
+%%DATADIR%%/images/P.jpg
+%%DATADIR%%/images/Q.jpg
+%%DATADIR%%/images/R.jpg
+%%DATADIR%%/images/S.jpg
+%%DATADIR%%/images/T.jpg
+%%DATADIR%%/images/U.jpg
+%%DATADIR%%/images/V.jpg
+%%DATADIR%%/images/W.jpg
+%%DATADIR%%/images/X.jpg
+%%DATADIR%%/images/Y.jpg
+%%DATADIR%%/images/Z.jpg
+%%DATADIR%%/images/g.jpg
+%%DATADIR%%/images/i.jpg
+%%DATADIR%%/images/l.jpg
+%%DATADIR%%/images/logo.png
+%%DATADIR%%/images/o.jpg
+%%DATADIR%%/images/s.jpg
+%%DATADIR%%/images/t.jpg
+%%DATADIR%%/images/z.jpg
+%%DATADIR%%/patapizza-tetris
+share/pixmaps/patapizza-tetris.png
+@dirrm %%DATADIR%%/images
+@dirrm %%DATADIR%%
===> Done
