===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/sdl_scavenger/Makefile ./Makefile
--- /usr/ports/games/sdl_scavenger/Makefile	2013-01-24 11:57:59.000000000 +0100
+++ ./Makefile	2013-03-28 02:48:15.000000000 +0100
@@ -1,8 +1,9 @@
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/sdl_scavenger/Makefile 310871 2013-01-23 15:00:18Z makc $
 
 PORTNAME=	sdl_scavenger
 PORTVERSION=	145.1.2010.11.5
-PORTREVISION=	2
+PORTREVISION=	3
 CATEGORIES=	games
 MASTER_SITES=	SF/sdlscavenger/
 DISTNAME=	sdlscav-145.1_2010_11_5
@@ -10,7 +11,7 @@
 MAINTAINER=	nemysis@gmx.ch
 COMMENT=	Lode Runner like game
 
-LICENSE=	GPLv2
+LICENSE=	GPLv1
 
 WRKSRC=		${WRKDIR}/sdlscav-145
 
@@ -20,29 +21,24 @@
 MAKE_JOBS_SAFE=	yes
 ALL_TARGET=
 
-PLIST_FILES=	bin/sdlscavenger \
-		share/pixmaps/sdlscavenger.png
-
-PORTDATA=	*
-PORTDOCS=	CREDITS DOC NEW_FEATURES README STRATEGY
+PORTDOCS=	CREDITS DOC NEW_FEATURES README STRATEGY changelog
 
 DESKTOP_ENTRIES="SDL Scavenger" "${COMMENT}" "sdlscavenger" \
-		"sdlscavenger" "Game;ArcadeGame;" ${FALSE}
+		"sdlscavenger" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
 
 post-patch:
-		@${REINPLACE_CMD} -e 's|data:/usr/local/games/scavenger|data:${DATADIR}/data|' \
+		@${REINPLACE_CMD} -e 's|/usr/local/games/scavenger|${DATADIR}/data|' \
 			${WRKSRC}/names.h
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/sdlscav ${PREFIX}/bin/sdlscavenger
-	${MKDIR} ${DATADIR}
-	@(cd ${WRKSRC} && ${COPYTREE_SHARE} data ${DATADIR})
-	${INSTALL_DATA} ${WRKSRC}/data/scav.png ${PREFIX}/share/pixmaps/sdlscavenger.png
+	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "data scav" ${DATADIR})
+	${LN} -sf ${DATADIR}/data/scav.png ${PREFIX}/share/pixmaps/sdlscavenger.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/sdl_scavenger/pkg-plist ./pkg-plist
--- /usr/ports/games/sdl_scavenger/pkg-plist	1970-01-01 01:00:00.000000000 +0100
+++ ./pkg-plist	2013-03-28 02:37:56.000000000 +0100
@@ -0,0 +1,44 @@
+bin/sdlscavenger
+share/pixmaps/sdlscavenger.png
+%%DATADIR%%/data/badguy.lbm
+%%DATADIR%%/data/brownblue.lbm
+%%DATADIR%%/data/death.raw
+%%DATADIR%%/data/death.wav
+%%DATADIR%%/data/devil.xpm
+%%DATADIR%%/data/dig.raw
+%%DATADIR%%/data/dig.wav
+%%DATADIR%%/data/fall.raw
+%%DATADIR%%/data/fall.wav
+%%DATADIR%%/data/highrc
+%%DATADIR%%/data/highstreet.lbm
+%%DATADIR%%/data/leprechaun.lbm
+%%DATADIR%%/data/levels.scl
+%%DATADIR%%/data/masters.scl
+%%DATADIR%%/data/microman.lbm
+%%DATADIR%%/data/microman.xpm
+%%DATADIR%%/data/micromanrc
+%%DATADIR%%/data/pop.raw
+%%DATADIR%%/data/pop.wav
+%%DATADIR%%/data/redbrick.lbm
+%%DATADIR%%/data/reddevil.lbm
+%%DATADIR%%/data/reddevilrc
+%%DATADIR%%/data/regularguy.lbm
+%%DATADIR%%/data/scav.bmp
+%%DATADIR%%/data/scav.png
+%%DATADIR%%/data/scav.xpm
+%%DATADIR%%/data/scavrc
+%%DATADIR%%/data/shamrc
+%%DATADIR%%/data/shamrock.lbm
+%%DATADIR%%/data/shamrock.xpm
+%%DATADIR%%/data/spiral.lbm
+%%DATADIR%%/data/spiralthing.lbm
+%%DATADIR%%/data/victory.raw
+%%DATADIR%%/data/victory.wav
+%%DATADIR%%/scav/highrc
+%%DATADIR%%/scav/micromanrc
+%%DATADIR%%/scav/reddevilrc
+%%DATADIR%%/scav/scavrc
+%%DATADIR%%/scav/shamrc
+@dirrm %%DATADIR%%/scav
+@dirrm %%DATADIR%%/data
+@dirrm %%DATADIR%%
===> Done
