===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/sdlroids/Makefile ./Makefile
--- /usr/ports/games/sdlroids/Makefile	2013-01-24 11:57:59.000000000 +0100
+++ ./Makefile	2013-05-26 13:08:32.000000000 +0200
@@ -18,10 +18,10 @@
 LICENSE=	GPLv2
 
 USE_BZIP2=	yes
+USE_SDL=	mixer sdl
 GNU_CONFIGURE=	yes
 USE_GMAKE=	yes
 USE_CSTD=	gnu89
-USE_SDL=	mixer sdl
 
 MAN6=		sdlroids.6
 
@@ -33,10 +33,10 @@
 .include <bsd.port.options.mk>
 
 post-install:
-	${INSTALL_DATA} ${DISTDIR}/${DIST_SUBDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/sdlroids/pkg-plist ./pkg-plist
--- /usr/ports/games/sdlroids/pkg-plist	2013-01-14 17:55:32.000000000 +0100
+++ ./pkg-plist	2013-05-26 13:10:20.000000000 +0200
@@ -1,35 +1,35 @@
 bin/sdlroids
 share/pixmaps/sdlroids.png
-share/sdlroids/bmp/blank.bmp
-share/sdlroids/bmp/bomb.bmp
-share/sdlroids/bmp/icon.bmp
-share/sdlroids/bmp/level.bmp
-share/sdlroids/bmp/life.bmp
-share/sdlroids/bmp/num0.bmp
-share/sdlroids/bmp/num1.bmp
-share/sdlroids/bmp/num2.bmp
-share/sdlroids/bmp/num3.bmp
-share/sdlroids/bmp/num4.bmp
-share/sdlroids/bmp/num5.bmp
-share/sdlroids/bmp/num6.bmp
-share/sdlroids/bmp/num7.bmp
-share/sdlroids/bmp/num8.bmp
-share/sdlroids/bmp/num9.bmp
-share/sdlroids/bmp/plus.bmp
-share/sdlroids/bmp/score.bmp
-share/sdlroids/bmp/shield.bmp
-share/sdlroids/sounds/bonusgot.wav
-share/sdlroids/sounds/bshot.wav
-share/sdlroids/sounds/bwound.wav
-share/sdlroids/sounds/explode.wav
-share/sdlroids/sounds/explode2.wav
-share/sdlroids/sounds/newbad.wav
-share/sdlroids/sounds/newbonus.wav
-share/sdlroids/sounds/phit.wav
-share/sdlroids/sounds/pshot.wav
-share/sdlroids/sounds/swarmsplit.wav
-share/sdlroids/sounds/thrust.wav
-share/sdlroids/sounds/title.wav
-@dirrm share/sdlroids/sounds
-@dirrm share/sdlroids/bmp
-@dirrm share/sdlroids
+%%DATADIR%%/bmp/blank.bmp
+%%DATADIR%%/bmp/bomb.bmp
+%%DATADIR%%/bmp/icon.bmp
+%%DATADIR%%/bmp/level.bmp
+%%DATADIR%%/bmp/life.bmp
+%%DATADIR%%/bmp/num0.bmp
+%%DATADIR%%/bmp/num1.bmp
+%%DATADIR%%/bmp/num2.bmp
+%%DATADIR%%/bmp/num3.bmp
+%%DATADIR%%/bmp/num4.bmp
+%%DATADIR%%/bmp/num5.bmp
+%%DATADIR%%/bmp/num6.bmp
+%%DATADIR%%/bmp/num7.bmp
+%%DATADIR%%/bmp/num8.bmp
+%%DATADIR%%/bmp/num9.bmp
+%%DATADIR%%/bmp/plus.bmp
+%%DATADIR%%/bmp/score.bmp
+%%DATADIR%%/bmp/shield.bmp
+%%DATADIR%%/sounds/bonusgot.wav
+%%DATADIR%%/sounds/bshot.wav
+%%DATADIR%%/sounds/bwound.wav
+%%DATADIR%%/sounds/explode.wav
+%%DATADIR%%/sounds/explode2.wav
+%%DATADIR%%/sounds/newbad.wav
+%%DATADIR%%/sounds/newbonus.wav
+%%DATADIR%%/sounds/phit.wav
+%%DATADIR%%/sounds/pshot.wav
+%%DATADIR%%/sounds/swarmsplit.wav
+%%DATADIR%%/sounds/thrust.wav
+%%DATADIR%%/sounds/title.wav
+@dirrm %%DATADIR%%/sounds
+@dirrm %%DATADIR%%/bmp
+@dirrm %%DATADIR%%
===> Done
