===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/simsu/Makefile ./Makefile
--- /usr/ports/games/simsu/Makefile	2013-02-24 17:33:59.000000000 +0100
+++ ./Makefile	2013-03-28 11:43:50.000000000 +0100
@@ -1,4 +1,4 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/simsu/Makefile 312847 2013-02-23 19:13:43Z rm $
 
 PORTNAME=	simsu
@@ -27,10 +27,10 @@
 		${QMAKE} ${QMAKEFLAGS} PREFIX=${PREFIX}
 
 post-install:
-	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps/
+	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/simsu/pkg-plist ./pkg-plist
--- /usr/ports/games/simsu/pkg-plist	2013-02-20 21:01:01.000000000 +0100
+++ ./pkg-plist	2013-03-28 11:47:04.000000000 +0100
@@ -25,26 +25,4 @@
 %%DATADIR%%/translations/simsu_uk.qm
 @dirrm %%DATADIR%%/translations
 @dirrm %%DATADIR%%
-@dirrmtry share/icons/hicolor/scalable/apps
-@dirrmtry share/icons/hicolor/scalable
-@dirrmtry share/icons/hicolor/64x64/apps
-@dirrmtry share/icons/hicolor/64x64
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor/256x256/apps
-@dirrmtry share/icons/hicolor/256x256
-@dirrmtry share/icons/hicolor/24x24/apps
-@dirrmtry share/icons/hicolor/24x24
-@dirrmtry share/icons/hicolor/22x22/apps
-@dirrmtry share/icons/hicolor/22x22
-@dirrmtry share/icons/hicolor/16x16/apps
-@dirrmtry share/icons/hicolor/16x16
-@dirrmtry share/icons/hicolor/128x128/apps
-@dirrmtry share/icons/hicolor/128x128
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrmtry share/applications
-@exec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
-@unexec %%LOCALBASE%%/bin/update-desktop-database -q || /usr/bin/true
===> Done
