===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/smashbattle/Makefile ./Makefile
--- /usr/ports/games/smashbattle/Makefile	2013-01-14 17:55:33.000000000 +0100
+++ ./Makefile	2013-05-26 13:37:23.000000000 +0200
@@ -1,9 +1,9 @@
-# Created by:		nemysis@gmx.ch
-#
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/smashbattle/Makefile 305698 2012-10-11 05:33:04Z jgh $
 
 PORTNAME=	smashbattle
 PORTVERSION=	110224
+PORTREVISION=	1
 CATEGORIES=	games
 MASTER_SITES=	SF/${PORTNAME}/${PORTNAME}/beta-${PORTVERSION}
 DISTNAME=	${PORTNAME}-${DISTVERSION}-src
@@ -15,23 +15,16 @@
 
 USE_ZIP=	yes
 EXTRACT_AFTER_ARGS=	-d ${WRKDIR}/${DISTNAME}
+USES=		desktop-file-utils
+USE_SDL=	sdl image mixer
 USE_GMAKE=	yes
 USE_DOS2UNIX=	*.cpp *.h *.ini Makefile
 ALL_TARGET=	battle
-USE_SDL=	sdl image mixer
 MAKE_JOBS_SAFE=	yes
 
 pre-extract:
 	@${MKDIR} ${WRKDIR}/${DISTNAME}
 
-PLIST_FILES=	bin/${PORTNAME} \
-		share/applications/${PORTNAME}.desktop \
-		share/applications/${PORTNAME}fs.desktop \
-		share/pixmaps/${PORTNAME}.png
-PLIST_DIRSTRY=	share/applications
-
-PORTDATA=	*
-
 SUB_FILES=	${PORTNAME}
 
 .include <bsd.port.options.mk>
@@ -39,16 +32,14 @@
 post-patch:
 	@${REINPLACE_CMD} -e 's|/usr/share/games/smashbattle/gfx/SB.png|${PORTNAME}|' \
 		-e 's|/usr/local/bin/smashbattle|${PORTNAME}|' \
-		${WRKSRC}/linux/${PORTNAME}.desktop ${WRKSRC}/linux/${PORTNAME}fs.desktop
+		${WRKSRC}/linux/*.desktop
 
 do-install:
 	${INSTALL_SCRIPT} ${WRKDIR}/${PORTNAME} ${PREFIX}/bin
-	${MKDIR} ${DATADIR}
-	${INSTALL_PROGRAM} ${WRKSRC}/battle ${DATADIR}/${PORTNAME}
 	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "gfx music sfx stage" ${DATADIR})
-	${INSTALL_DATA} ${WRKSRC}/gfx/SB.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
-	${MKDIR} ${PREFIX}/share/applications
-	${INSTALL_DATA} ${WRKSRC}/linux/${PORTNAME}.desktop ${PREFIX}/share/applications/
-	${INSTALL_DATA} ${WRKSRC}/linux/${PORTNAME}fs.desktop ${PREFIX}/share/applications/
+	${INSTALL_PROGRAM} ${WRKSRC}/battle ${DATADIR}/${PORTNAME}
+	${LN} -sf ${DATADIR}/gfx/SB.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
+	@${MKDIR} ${PREFIX}/share/applications
+	${INSTALL_DATA} ${WRKSRC}/linux/*.desktop ${PREFIX}/share/applications
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/smashbattle/pkg-plist ./pkg-plist
--- /usr/ports/games/smashbattle/pkg-plist	1970-01-01 01:00:00.000000000 +0100
+++ ./pkg-plist	2013-05-21 18:32:27.000000000 +0200
@@ -0,0 +1,99 @@
+bin/smashbattle
+share/applications/smashbattle.desktop
+share/applications/smashbattlefs.desktop
+share/pixmaps/smashbattle.png
+%%DATADIR%%/gfx/SB.bmp
+%%DATADIR%%/gfx/SB.ico
+%%DATADIR%%/gfx/SB.png
+%%DATADIR%%/gfx/anton.bmp
+%%DATADIR%%/gfx/arjan.bmp
+%%DATADIR%%/gfx/berry.bmp
+%%DATADIR%%/gfx/bert.bmp
+%%DATADIR%%/gfx/bg_black.bmp
+%%DATADIR%%/gfx/bg_blue.bmp
+%%DATADIR%%/gfx/bg_grey.bmp
+%%DATADIR%%/gfx/bg_orange.bmp
+%%DATADIR%%/gfx/bg_white.bmp
+%%DATADIR%%/gfx/bomb.bmp
+%%DATADIR%%/gfx/cannon.bmp
+%%DATADIR%%/gfx/chick.bmp
+%%DATADIR%%/gfx/common.bmp
+%%DATADIR%%/gfx/cups.bmp
+%%DATADIR%%/gfx/donja.bmp
+%%DATADIR%%/gfx/eva.bmp
+%%DATADIR%%/gfx/font_l.bmp
+%%DATADIR%%/gfx/font_m.bmp
+%%DATADIR%%/gfx/font_s.bmp
+%%DATADIR%%/gfx/gatling.bmp
+%%DATADIR%%/gfx/gijs.bmp
+%%DATADIR%%/gfx/jay.bmp
+%%DATADIR%%/gfx/jeremy.bmp
+%%DATADIR%%/gfx/jeroen.bmp
+%%DATADIR%%/gfx/kim.bmp
+%%DATADIR%%/gfx/marcel.bmp
+%%DATADIR%%/gfx/okke.bmp
+%%DATADIR%%/gfx/peter.bmp
+%%DATADIR%%/gfx/player1hp.bmp
+%%DATADIR%%/gfx/player2hp.bmp
+%%DATADIR%%/gfx/pmarkers.bmp
+%%DATADIR%%/gfx/powerups.bmp
+%%DATADIR%%/gfx/props.bmp
+%%DATADIR%%/gfx/rob.bmp
+%%DATADIR%%/gfx/rutger.bmp
+%%DATADIR%%/gfx/ruud.bmp
+%%DATADIR%%/gfx/shield.bmp
+%%DATADIR%%/gfx/steven.bmp
+%%DATADIR%%/gfx/tedje.bmp
+%%DATADIR%%/gfx/tiles.bmp
+%%DATADIR%%/gfx/tobias.bmp
+%%DATADIR%%/gfx/ton.bmp
+%%DATADIR%%/gfx/weapons.bmp
+%%DATADIR%%/music/a_hero_rises.ogg
+%%DATADIR%%/music/graveyard.ogg
+%%DATADIR%%/music/hydro_dreams.ogg
+%%DATADIR%%/music/space_farer.ogg
+%%DATADIR%%/music/victory.ogg
+%%DATADIR%%/sfx/airstrike.wav
+%%DATADIR%%/sfx/bounce.wav
+%%DATADIR%%/sfx/countdown.wav
+%%DATADIR%%/sfx/explode.wav
+%%DATADIR%%/sfx/go.wav
+%%DATADIR%%/sfx/hit.wav
+%%DATADIR%%/sfx/item.wav
+%%DATADIR%%/sfx/jump.wav
+%%DATADIR%%/sfx/laser.wav
+%%DATADIR%%/sfx/pause.wav
+%%DATADIR%%/sfx/select.wav
+%%DATADIR%%/sfx/select_character.wav
+%%DATADIR%%/sfx/shield.wav
+%%DATADIR%%/sfx/shoot.wav
+%%DATADIR%%/sfx/youlose.wav
+%%DATADIR%%/smashbattle
+%%DATADIR%%/stage/blastbowl.lvl
+%%DATADIR%%/stage/boulderdash.lvl
+%%DATADIR%%/stage/chick_invasion.lvl
+%%DATADIR%%/stage/chick_invasion_ii.lvl
+%%DATADIR%%/stage/chicken_run.lvl
+%%DATADIR%%/stage/commongrounds.lvl
+%%DATADIR%%/stage/ducknhunt.lvl
+%%DATADIR%%/stage/easy_does_it.lvl
+%%DATADIR%%/stage/lamoustache.lvl
+%%DATADIR%%/stage/piepit.lvl
+%%DATADIR%%/stage/pitfall.lvl
+%%DATADIR%%/stage/pitofdeath.lvl
+%%DATADIR%%/stage/platformalley.lvl
+%%DATADIR%%/stage/pogostick.lvl
+%%DATADIR%%/stage/rabbithole.lvl
+%%DATADIR%%/stage/slipnslide.lvl
+%%DATADIR%%/stage/snowfight.lvl
+%%DATADIR%%/stage/stayhigh.lvl
+%%DATADIR%%/stage/thefunnel.lvl
+%%DATADIR%%/stage/titlescreen.lvl
+%%DATADIR%%/stage/trainingdojo.lvl
+%%DATADIR%%/stage/tryout.lvl
+@dirrm %%DATADIR%%/stage
+@dirrm %%DATADIR%%/sfx
+@dirrm %%DATADIR%%/music
+@dirrm %%DATADIR%%/gfx
+@dirrm %%DATADIR%%
+@dirrmtry share/applications
===> Done
