===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/starfighter/Makefile ./Makefile
--- /usr/ports/games/starfighter/Makefile	2013-04-29 12:53:31.000000000 +0200
+++ ./Makefile	2013-05-26 14:04:47.000000000 +0200
@@ -3,7 +3,7 @@
 
 PORTNAME=	starfighter
 PORTVERSION=	1.2
-PORTREVISION=	1
+PORTREVISION=	2
 CATEGORIES=	games
 MASTER_SITES=	SF/pr-${PORTNAME}/ \
 		SF/nemysisfreebsdp/:icons
@@ -24,10 +24,17 @@
 USE_GMAKE=	yes
 MAKEFILE=	makefile
 
+PLIST_FILES=	bin/${PORTNAME} \
+		share/pixmaps/${PORTNAME}.png \
+		%%DATADIR%%/${PORTNAME}.pak
+PLIST_DIRS=	%%DATADIR%%
+
+PORTDOCS=	*
+
 INSTALLS_ICONS=	yes
 
 DESKTOP_ENTRIES="Project Starfighter" "${COMMENT}" "${PORTNAME}" \
-		"${PORTNAME}" "Game;ArcadeGame;" ${FALSE}
+		"${PORTNAME}" "Game;ArcadeGame;" false
 
 .include <bsd.port.options.mk>
 
@@ -40,11 +47,15 @@
 	-e 's|/share/games/parallelrealities/|/share/starfighter/|' \
 		${WRKSRC}/makefile
 
-.if !${PORT_OPTIONS:MDOCS}
+.if ${PORT_OPTIONS:MDOCS}
+	@${REINPLACE_CMD} -e 's|docs/index.html|docs/index.html ChangeLog|' ${WRKSRC}/makefile
+.endif
+
+.if ! ${PORT_OPTIONS:MDOCS}
 	@${REINPLACE_CMD} -i '' -e '/$$(DESTDIR)$$(DOCDIR)/d' ${WRKSRC}/makefile
 .endif
 
 post-install:
-	${INSTALL_DATA} ${DISTDIR}/${DIST_SUBDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/starfighter/pkg-plist ./pkg-plist
--- /usr/ports/games/starfighter/pkg-plist	2013-01-14 17:55:25.000000000 +0100
+++ ./pkg-plist	1970-01-01 01:00:00.000000000 +0100
@@ -1,6 +0,0 @@
-bin/starfighter
-%%PORTDOCS%%%%DOCSDIR%%/index.html
-share/pixmaps/starfighter.png
-%%DATADIR%%/starfighter.pak
-@dirrm %%DATADIR%%
-%%PORTDOCS%%@dirrm %%DOCSDIR%%
===> Done
