===> Generating patch
===> Viewing diff with more
diff -ruN /usr/ports/games/tanglet/Makefile ./Makefile
--- /usr/ports/games/tanglet/Makefile	2013-06-02 18:54:48.000000000 +0200
+++ ./Makefile	2013-08-09 21:03:24.000000000 +0200
@@ -1,13 +1,8 @@
-# New Ports collection makefile for:	tanglet
-# Date created:		2012-04-21
-# Whom:			nemysis@gmx.ch
-#
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/tanglet/Makefile 304401 2012-09-17 14:25:55Z jhale $
-#
 
 PORTNAME=	tanglet
-PORTVERSION=	1.2.1
-PORTREVISION=	2
+PORTVERSION=	1.2.2
 CATEGORIES=	games
 MASTER_SITES=	http://gottcode.org/${PORTNAME}/
 EXTRACT_SUFX=	-src.tar.bz2
@@ -18,25 +13,28 @@
 LICENSE=	GPLv3
 
 USE_BZIP2=	yes
-USE_QT4=	moc_build qmake_build rcc_build gui
+USES=		desktop-file-utils
+USE_QT4=	moc_build qmake_build rcc_build gui iconengines imageformats
 INSTALLS_ICONS=	yes
 MAKE_JOBS_SAFE=	yes
 
-PORTDOCS=	CREDITS ChangeLog
+PORTDOCS=	CREDITS ChangeLog NEWS
 
 .include <bsd.port.options.mk>
 
+post-patch:
+	@${REINPLACE_CMD} -i '' -e '/PKGCONFIG += zlib/d' \
+		${WRKSRC}/tanglet.pro
+
 do-configure:
 	@cd ${WRKSRC} && ${SETENV} ${CONFIGURE_ENV} \
 		${QMAKE} ${QMAKEFLAGS} PREFIX=${PREFIX}
 
 post-install:
-# Pixmaps
-	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps/
+	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
-# Documentation
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN /usr/ports/games/tanglet/distinfo ./distinfo
--- /usr/ports/games/tanglet/distinfo	2013-06-02 18:54:48.000000000 +0200
+++ ./distinfo	2013-08-09 20:58:20.000000000 +0200
@@ -1,2 +1,2 @@
-SHA256 (tanglet-1.2.1-src.tar.bz2) = 94f4c5c76020ec1fe502ff076fbe5471e497d4b7bcabd1e9059e1022d1ce80c8
-SIZE (tanglet-1.2.1-src.tar.bz2) = 7814483
+SHA256 (tanglet-1.2.2-src.tar.bz2) = c90d5c36b7e6cca54cdc001c7eecfb333795add16e6ae01f100026c61f4a84da
+SIZE (tanglet-1.2.2-src.tar.bz2) = 8102968
diff -ruN /usr/ports/games/tanglet/pkg-plist ./pkg-plist
--- /usr/ports/games/tanglet/pkg-plist	2013-06-02 18:54:48.000000000 +0200
+++ ./pkg-plist	2013-08-09 21:02:04.000000000 +0200
@@ -10,68 +10,61 @@
 share/icons/hicolor/64x64/apps/tanglet.png
 share/icons/hicolor/scalable/apps/tanglet.svg
 share/pixmaps/tanglet.png
-share/pixmaps/tanglet_32.xpm
-share/tanglet/data/cs/dice
-share/tanglet/data/cs/dictionary
-share/tanglet/data/cs/name
-share/tanglet/data/cs/words
-share/tanglet/data/de/README
-share/tanglet/data/de/dice
-share/tanglet/data/de/dictionary
-share/tanglet/data/de/name
-share/tanglet/data/de/words
-share/tanglet/data/en/README
-share/tanglet/data/en/dice
-share/tanglet/data/en/dictionary
-share/tanglet/data/en/name
-share/tanglet/data/en/scowl-word-lists-used
-share/tanglet/data/en/words
-share/tanglet/data/fr/dice
-share/tanglet/data/fr/dictionary
-share/tanglet/data/fr/name
-share/tanglet/data/fr/words
-share/tanglet/data/he/README
-share/tanglet/data/he/dice
-share/tanglet/data/he/dictionary
-share/tanglet/data/he/name
-share/tanglet/data/he/words
-share/tanglet/data/nl/dice
-share/tanglet/data/nl/dictionary
-share/tanglet/data/nl/name
-share/tanglet/data/nl/words
-share/tanglet/translations/tanglet_cs.qm
-share/tanglet/translations/tanglet_de.qm
-share/tanglet/translations/tanglet_en.qm
-share/tanglet/translations/tanglet_fr.qm
-share/tanglet/translations/tanglet_he.qm
-share/tanglet/translations/tanglet_nl.qm
-@dirrm share/tanglet/translations
-@dirrm share/tanglet/data/nl
-@dirrm share/tanglet/data/he
-@dirrm share/tanglet/data/fr
-@dirrm share/tanglet/data/en
-@dirrm share/tanglet/data/de
-@dirrm share/tanglet/data/cs
-@dirrm share/tanglet/data
-@dirrm share/tanglet
-@dirrmtry share/icons/hicolor/scalable/apps
-@dirrmtry share/icons/hicolor/scalable
-@dirrmtry share/icons/hicolor/64x64/apps
-@dirrmtry share/icons/hicolor/64x64
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor/256x256/apps
-@dirrmtry share/icons/hicolor/256x256
-@dirrmtry share/icons/hicolor/24x24/apps
-@dirrmtry share/icons/hicolor/24x24
-@dirrmtry share/icons/hicolor/22x22/apps
-@dirrmtry share/icons/hicolor/22x22
-@dirrmtry share/icons/hicolor/16x16/apps
-@dirrmtry share/icons/hicolor/16x16
-@dirrmtry share/icons/hicolor/128x128/apps
-@dirrmtry share/icons/hicolor/128x128
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
+share/pixmaps/tanglet.xpm
+%%DATADIR%%/data/cs/dice
+%%DATADIR%%/data/cs/dictionary
+%%DATADIR%%/data/cs/name
+%%DATADIR%%/data/cs/words
+%%DATADIR%%/data/de/README
+%%DATADIR%%/data/de/dice
+%%DATADIR%%/data/de/dictionary
+%%DATADIR%%/data/de/name
+%%DATADIR%%/data/de/words
+%%DATADIR%%/data/en/README
+%%DATADIR%%/data/en/dice
+%%DATADIR%%/data/en/dictionary
+%%DATADIR%%/data/en/name
+%%DATADIR%%/data/en/scowl-word-lists-used
+%%DATADIR%%/data/en/words
+%%DATADIR%%/data/fr/dice
+%%DATADIR%%/data/fr/dictionary
+%%DATADIR%%/data/fr/name
+%%DATADIR%%/data/fr/words
+%%DATADIR%%/data/he/README
+%%DATADIR%%/data/he/dice
+%%DATADIR%%/data/he/dictionary
+%%DATADIR%%/data/he/name
+%%DATADIR%%/data/he/words
+%%DATADIR%%/data/hu/README
+%%DATADIR%%/data/hu/dice
+%%DATADIR%%/data/hu/dictionary
+%%DATADIR%%/data/hu/name
+%%DATADIR%%/data/hu/words
+%%DATADIR%%/data/nl/dice
+%%DATADIR%%/data/nl/dictionary
+%%DATADIR%%/data/nl/name
+%%DATADIR%%/data/nl/words
+%%DATADIR%%/translations/tanglet_cs.qm
+%%DATADIR%%/translations/tanglet_de.qm
+%%DATADIR%%/translations/tanglet_en.qm
+%%DATADIR%%/translations/tanglet_es.qm
+%%DATADIR%%/translations/tanglet_es_CL.qm
+%%DATADIR%%/translations/tanglet_fr.qm
+%%DATADIR%%/translations/tanglet_he.qm
+%%DATADIR%%/translations/tanglet_hu.qm
+%%DATADIR%%/translations/tanglet_it.qm
+%%DATADIR%%/translations/tanglet_nl.qm
+%%DATADIR%%/translations/tanglet_ro.qm
+%%DATADIR%%/translations/tanglet_tr.qm
+%%DATADIR%%/translations/tanglet_uk.qm
+@dirrm %%DATADIR%%/translations
+@dirrm %%DATADIR%%/data/nl
+@dirrm %%DATADIR%%/data/hu
+@dirrm %%DATADIR%%/data/he
+@dirrm %%DATADIR%%/data/fr
+@dirrm %%DATADIR%%/data/en
+@dirrm %%DATADIR%%/data/de
+@dirrm %%DATADIR%%/data/cs
+@dirrm %%DATADIR%%/data
+@dirrm %%DATADIR%%
 @dirrmtry share/applications
===> Done
