===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/tanglet/Makefile ./Makefile
--- /usr/ports/games/tanglet/Makefile	2013-01-14 17:55:30.000000000 +0100
+++ ./Makefile	2013-03-28 16:09:47.000000000 +0100
@@ -1,9 +1,5 @@
-# New Ports collection makefile for:	tanglet
-# Date created:		2012-04-21
-# Whom:			nemysis@gmx.ch
-#
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/tanglet/Makefile 304401 2012-09-17 14:25:55Z jhale $
-#
 
 PORTNAME=	tanglet
 PORTVERSION=	1.2.1
@@ -18,7 +14,7 @@
 LICENSE=	GPLv3
 
 USE_BZIP2=	yes
-USE_QT4=	moc_build qmake_build rcc_build gui
+USE_QT4=	moc_build qmake_build rcc_build gui iconengines imageformats
 INSTALLS_ICONS=	yes
 MAKE_JOBS_SAFE=	yes
 
@@ -31,12 +27,10 @@
 		${QMAKE} ${QMAKEFLAGS} PREFIX=${PREFIX}
 
 post-install:
-# Pixmaps
-	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps/
+	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
-# Documentation
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/tanglet/pkg-plist ./pkg-plist
--- /usr/ports/games/tanglet/pkg-plist	2013-01-14 17:55:30.000000000 +0100
+++ ./pkg-plist	2013-03-28 14:51:55.000000000 +0100
@@ -11,67 +11,47 @@
 share/icons/hicolor/scalable/apps/tanglet.svg
 share/pixmaps/tanglet.png
 share/pixmaps/tanglet_32.xpm
-share/tanglet/data/cs/dice
-share/tanglet/data/cs/dictionary
-share/tanglet/data/cs/name
-share/tanglet/data/cs/words
-share/tanglet/data/de/README
-share/tanglet/data/de/dice
-share/tanglet/data/de/dictionary
-share/tanglet/data/de/name
-share/tanglet/data/de/words
-share/tanglet/data/en/README
-share/tanglet/data/en/dice
-share/tanglet/data/en/dictionary
-share/tanglet/data/en/name
-share/tanglet/data/en/scowl-word-lists-used
-share/tanglet/data/en/words
-share/tanglet/data/fr/dice
-share/tanglet/data/fr/dictionary
-share/tanglet/data/fr/name
-share/tanglet/data/fr/words
-share/tanglet/data/he/README
-share/tanglet/data/he/dice
-share/tanglet/data/he/dictionary
-share/tanglet/data/he/name
-share/tanglet/data/he/words
-share/tanglet/data/nl/dice
-share/tanglet/data/nl/dictionary
-share/tanglet/data/nl/name
-share/tanglet/data/nl/words
-share/tanglet/translations/tanglet_cs.qm
-share/tanglet/translations/tanglet_de.qm
-share/tanglet/translations/tanglet_en.qm
-share/tanglet/translations/tanglet_fr.qm
-share/tanglet/translations/tanglet_he.qm
-share/tanglet/translations/tanglet_nl.qm
-@dirrm share/tanglet/translations
-@dirrm share/tanglet/data/nl
-@dirrm share/tanglet/data/he
-@dirrm share/tanglet/data/fr
-@dirrm share/tanglet/data/en
-@dirrm share/tanglet/data/de
-@dirrm share/tanglet/data/cs
-@dirrm share/tanglet/data
-@dirrm share/tanglet
-@dirrmtry share/icons/hicolor/scalable/apps
-@dirrmtry share/icons/hicolor/scalable
-@dirrmtry share/icons/hicolor/64x64/apps
-@dirrmtry share/icons/hicolor/64x64
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor/256x256/apps
-@dirrmtry share/icons/hicolor/256x256
-@dirrmtry share/icons/hicolor/24x24/apps
-@dirrmtry share/icons/hicolor/24x24
-@dirrmtry share/icons/hicolor/22x22/apps
-@dirrmtry share/icons/hicolor/22x22
-@dirrmtry share/icons/hicolor/16x16/apps
-@dirrmtry share/icons/hicolor/16x16
-@dirrmtry share/icons/hicolor/128x128/apps
-@dirrmtry share/icons/hicolor/128x128
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
+%%DATADIR%%/data/cs/dice
+%%DATADIR%%/data/cs/dictionary
+%%DATADIR%%/data/cs/name
+%%DATADIR%%/data/cs/words
+%%DATADIR%%/data/de/README
+%%DATADIR%%/data/de/dice
+%%DATADIR%%/data/de/dictionary
+%%DATADIR%%/data/de/name
+%%DATADIR%%/data/de/words
+%%DATADIR%%/data/en/README
+%%DATADIR%%/data/en/dice
+%%DATADIR%%/data/en/dictionary
+%%DATADIR%%/data/en/name
+%%DATADIR%%/data/en/scowl-word-lists-used
+%%DATADIR%%/data/en/words
+%%DATADIR%%/data/fr/dice
+%%DATADIR%%/data/fr/dictionary
+%%DATADIR%%/data/fr/name
+%%DATADIR%%/data/fr/words
+%%DATADIR%%/data/he/README
+%%DATADIR%%/data/he/dice
+%%DATADIR%%/data/he/dictionary
+%%DATADIR%%/data/he/name
+%%DATADIR%%/data/he/words
+%%DATADIR%%/data/nl/dice
+%%DATADIR%%/data/nl/dictionary
+%%DATADIR%%/data/nl/name
+%%DATADIR%%/data/nl/words
+%%DATADIR%%/translations/tanglet_cs.qm
+%%DATADIR%%/translations/tanglet_de.qm
+%%DATADIR%%/translations/tanglet_en.qm
+%%DATADIR%%/translations/tanglet_fr.qm
+%%DATADIR%%/translations/tanglet_he.qm
+%%DATADIR%%/translations/tanglet_nl.qm
+@dirrm %%DATADIR%%/translations
+@dirrm %%DATADIR%%/data/nl
+@dirrm %%DATADIR%%/data/he
+@dirrm %%DATADIR%%/data/fr
+@dirrm %%DATADIR%%/data/en
+@dirrm %%DATADIR%%/data/de
+@dirrm %%DATADIR%%/data/cs
+@dirrm %%DATADIR%%/data
+@dirrm %%DATADIR%%
 @dirrmtry share/applications
===> Done
