===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/tong/Makefile ./Makefile
--- /usr/ports/games/tong/Makefile	2013-01-24 11:57:57.000000000 +0100
+++ ./Makefile	2013-05-26 17:32:07.000000000 +0200
@@ -1,4 +1,4 @@
-# Created by:   nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/tong/Makefile 310871 2013-01-23 15:00:18Z makc $
 
 PORTNAME=	tong
@@ -16,7 +16,7 @@
 USE_GMAKE=	yes
 MAKE_JOBS_SAFE=	yes
 
-PORTDOCS=	CHANGELOG README making-of.txt notes.txt
+PORTDOCS=	CHANGELOG CREDITS README making-of.txt notes.txt
 
 WRKSRC=		${WRKDIR}/${PORTNAME}
 
@@ -27,18 +27,17 @@
 
 post-patch:
 	@${REINPLACE_CMD} -e 's|GAME_DATA_DIR := .|GAME_DATA_DIR := ${DATADIR}|' \
-	-e 's|CC = g++ -O3 |CC = ${CXX} ${CXXFLAGS} |' \
+		-e 's|CC = g++ -O3 |CC = ${CXX} ${CXXFLAGS} |' \
 		${WRKSRC}/Makefile
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${PREFIX}/bin
-	@cd ${WRKSRC} && ${COPYTREE_SHARE} "*.png" ${DATADIR}
-	${MKDIR} ${DATADIR}/media
-	@cd ${WRKSRC}/media && ${COPYTREE_SHARE} "*.png *.xcf *.ogg *.ico" ${DATADIR}/media
+	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "*.png" ${DATADIR})
+	@(cd ${WRKSRC}/media && ${COPYTREE_SHARE} "*.png *.xcf *.ogg *.ico" ${DATADIR}/media)
 	${LN} -sf ${DATADIR}/media/icon.png ${PREFIX}/share/pixmaps/${PORTNAME}.png
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
===> Done
