===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/ttt/Makefile ./Makefile
--- /usr/ports/games/ttt/Makefile	2013-01-14 17:55:38.000000000 +0100
+++ ./Makefile	2013-03-28 17:47:37.000000000 +0100
@@ -1,16 +1,16 @@
-# New ports collection makefile for:	ttt
-# Date created:				11 July 2005
-# Whom:					Alejandro Pulver <alejandro@varnet.biz>
-#
+# Created by: Alejandro Pulver <alejandro@varnet.biz>
 # $FreeBSD: head/games/ttt/Makefile 304004 2012-09-09 22:08:43Z swills $
-#
 
 PORTNAME=	ttt
 PORTVERSION=	2011.04
-PORTREVISION=	2
+PORTREVISION=	3
 CATEGORIES=	games
 MASTER_SITES=	ftp://ftp.tuxpaint.org/unix/x/ttt/ \
-		ftp://ftp.billsgames.com/unix/x/ttt/
+		ftp://ftp.billsgames.com/unix/x/ttt/ \
+		SF/nemysisfreebsdp/:icons
+DISTFILES=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX} \
+		${PORTNAME}.png:icons
+EXTRACT_ONLY=	${PORTNAME}-${DISTVERSION}${EXTRACT_SUFX}
 
 MAINTAINER=	nemysis@gmx.ch
 COMMENT=	Simple one or two player Tic Tac Toe game
@@ -25,34 +25,24 @@
 CPPFLAGS+=	-I${LOCALBASE}/include
 LDFLAGS+=	-L${LOCALBASE}/lib
 
-PLIST_FILES=	bin/${PORTNAME} \
-		share/pixmaps/${PORTNAME}.bmp
-
-PORTDATA=	*
-PORTDOCS=	README
+PORTDOCS=	 README
 
 SUB_FILES=	${PORTNAME}
 
+DESKTOP_ENTRIES="TTT" "${COMMENT}" "${PORTNAME}" \
+		"${PORTNAME}" "Game;ArcadeGame;" false
+
 .include <bsd.port.options.mk>
 
 do-install:
-# Scripts
 	${INSTALL_SCRIPT} ${WRKDIR}/${PORTNAME} ${PREFIX}/bin
-
-# Executable
-	${MKDIR} ${DATADIR}
-	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${DATADIR}
-
-# Data
 	@(cd ${WRKSRC} && ${COPYTREE_SHARE} "images sounds" ${DATADIR})
+	${INSTALL_PROGRAM} ${WRKSRC}/${PORTNAME} ${DATADIR}
+	${INSTALL_DATA} ${_DISTDIR}/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
-# Pixmaps
-	${INSTALL_DATA} ${WRKSRC}/images/program_logo.bmp ${PREFIX}/share/pixmaps/${PORTNAME}.bmp
-
-# Documentation
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
-	${INSTALL_DATA} ${WRKSRC}/README ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
+	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
 .include <bsd.port.mk>
diff -ruN --exclude=CVS /usr/ports/games/ttt/distinfo ./distinfo
--- /usr/ports/games/ttt/distinfo	2013-01-14 17:55:38.000000000 +0100
+++ ./distinfo	2013-03-28 17:26:57.000000000 +0100
@@ -1,2 +1,4 @@
 SHA256 (ttt-2011.04.tar.gz) = 747e3ed66e37aae9ceb32599b209d36451e910899cdd36dbf44763e90213b81e
 SIZE (ttt-2011.04.tar.gz) = 140666
+SHA256 (ttt.png) = 32dd4cf8a6d03ddbc2fc90709f4567c542c240020af629722cb6f2c1841c12bd
+SIZE (ttt.png) = 2361
diff -ruN --exclude=CVS /usr/ports/games/ttt/files/ttt.in ./files/ttt.in
--- /usr/ports/games/ttt/files/ttt.in	2013-01-14 17:55:38.000000000 +0100
+++ ./files/ttt.in	2013-03-28 17:08:57.000000000 +0100
@@ -1,7 +1,7 @@
 #!/bin/sh
 #
 # $FreeBSD: head/games/ttt/files/ttt.in 304004 2012-09-09 22:08:43Z swills $
+#
 
-echo "Tic Tac Toe: Starting up..."
 cd "%%DATADIR%%"
-./ttt
+exec ./ttt
diff -ruN --exclude=CVS /usr/ports/games/ttt/pkg-plist ./pkg-plist
--- /usr/ports/games/ttt/pkg-plist	1970-01-01 01:00:00.000000000 +0100
+++ ./pkg-plist	2013-03-28 17:42:25.000000000 +0100
@@ -0,0 +1,24 @@
+bin/ttt
+share/pixmaps/ttt.png
+%%DATADIR%%/images/game_board.bmp
+%%DATADIR%%/images/human_vs_computer.bmp
+%%DATADIR%%/images/human_vs_human.bmp
+%%DATADIR%%/images/mark_o.bmp
+%%DATADIR%%/images/mark_x.bmp
+%%DATADIR%%/images/program_logo.bmp
+%%DATADIR%%/images/status_draw.bmp
+%%DATADIR%%/images/status_win_o.bmp
+%%DATADIR%%/images/status_win_x.bmp
+%%DATADIR%%/images/turn_o.bmp
+%%DATADIR%%/images/turn_x.bmp
+%%DATADIR%%/sounds/badspot.wav
+%%DATADIR%%/sounds/draw.wav
+%%DATADIR%%/sounds/intro.wav
+%%DATADIR%%/sounds/o.wav
+%%DATADIR%%/sounds/start.wav
+%%DATADIR%%/sounds/win.wav
+%%DATADIR%%/sounds/x.wav
+%%DATADIR%%/ttt
+@dirrm %%DATADIR%%/sounds
+@dirrm %%DATADIR%%/images
+@dirrm %%DATADIR%%
===> Done
