Index: Makefile
===================================================================
--- Makefile	(revision 330670)
+++ Makefile	(working copy)
@@ -2,11 +2,11 @@
 # $FreeBSD$
 
 PORTNAME=	recoll
-PORTVERSION=	1.18.2
+PORTVERSION=	1.19.7
 CATEGORIES=	deskutils
 MASTER_SITES=	http://www.lesbonscomptes.com/${PORTNAME}/
 
-MAINTAINER=	nemysis@gmx.ch
+MAINTAINER=	nemysis@FreeBSD.org
 COMMENT=	Personal full text search package, based on Xapian
 
 LICENSE=	GPLv2
@@ -13,7 +13,7 @@
 
 BUILD_DEPENDS=	xapian-core>=0:${PORTSDIR}/databases/xapian-core
 
-USES=		iconv gmake
+USES=		gmake iconv
 GNU_CONFIGURE=	yes
 
 CPPFLAGS+=	-I${LOCALBASE}/include
@@ -26,10 +26,20 @@
 X11MON_DESC=	X11 session monitoring support
 OPTIONS_DEFAULT=QT4 ASPELL PYTHON X11MON
 
+OPTIONS_SUB=	yes
+
+QT4_CONFIGURE_ON=		qtgui
+
+ASPELL_LIB_DEPENDS=		libaspell.so:${PORTSDIR}/textproc/aspell
+ASPELL_CONFIGURE_ENABLE=	aspell
+
+PYTHON_CONFIGURE_ENABLE=	python-module
+
+X11MON_CONFIGURE_ENABLE=	x11mon
+
 # Wrong versioning upstream
 PORTSCOUT=	skipv:1.19
 
-NO_STAGE=	yes
 .include <bsd.port.options.mk>
 
 .if ${PORT_OPTIONS:MQT4}
@@ -37,29 +47,15 @@
 		corelib gui xml webkit
 QT_NONSTANDARD=	yes
 CONFIGURE_ENV+=	QTDIR="${QT_PREFIX}" QMAKE="${QMAKE}"
-CONFIGURE_ARGS+=--enable-qtgui
 INSTALLS_ICONS=	yes
 PLIST_SUB+=	QT=""
 MAN1+=		recollq.1 recoll.1
-.else
-CONFIGURE_ARGS+=--disable-qtgui
-PLIST_SUB+=	QT="@comment "
 .endif
 
-.if ${PORT_OPTIONS:MASPELL}
-LIB_DEPENDS+=	aspell:${PORTSDIR}/textproc/aspell
-.else
-CONFIGURE_ARGS+=--without-aspell
-.endif
-
 .if ${PORT_OPTIONS:MPYTHON}
 USE_PYTHON=	yes
 INSTALLS_EGGINFO=	yes
 PYDISTUTILS_EGGINFO=	Recoll-1.0-py${PYTHON_VER}.egg-info
-PLIST_SUB+=	PYTHON=""
-.else
-CONFIGURE_ARGS+=--disable-python-module
-PLIST_SUB+=	PYTHON="@comment "
 .endif
 
 .if ${PORT_OPTIONS:MX11MON}
@@ -66,8 +62,6 @@
 USE_XORG+=	ice sm x11
 USE_FAM=	yes
 CONFIGURE_ARGS+=--with-fam
-.else
-CONFIGURE_ARGS+=--disable-x11mon
 .endif
 
 post-patch:
@@ -85,7 +79,7 @@
 post-configure:
 .if ${PORT_OPTIONS:MQT4}
 	@cd ${WRKSRC}/qtgui && ${SETENV} ${CONFIGURE_ENV} \
-	${QMAKE} ${QMAKEFLAGS} PREFIX=${PREFIX}
+		${QMAKE} ${QMAKEFLAGS} PREFIX=${PREFIX}
 .endif
 
 .include <bsd.port.mk>
Index: distinfo
===================================================================
--- distinfo	(revision 330670)
+++ distinfo	(working copy)
@@ -1,2 +1,2 @@
-SHA256 (recoll-1.18.2.tar.gz) = 4729785354bd0d465c92e7d2a6fced688d8c08f1342a9bef2f040e7b10b1f510
-SIZE (recoll-1.18.2.tar.gz) = 1969421
+SHA256 (recoll-1.19.7.tar.gz) = 6d647f4c17282994cefb37e068cfb571cf9de14a22087701c59ff8c78a4e0b9d
+SIZE (recoll-1.19.7.tar.gz) = 2005198
